<html><head>
<link rel="stylesheet" media="all" type="text/css" href="http://rna.bgsu.edu/nrlist/css/nrlists.css">
<script src="http://rna.bgsu.edu/nrlist/js/sorttable.js"></script>
<title>Non-redundant list of PDB structures of all resolutions as of 2011-06-11</title>
</head>
<body>
<h3>Non-redundant list of 1009 PDB structures of all resolutions and containing at least one RNA nucleotide, as of 2011-06-11</h3>
They are sorted by the length of the longest unique chain, longest to shortest.<br>
<a href="Nonredundant_All_Resolution_2011-06-11_list.pdb">Click here for the list file to use with FR3D</a> (Save in FR3D/PDBFiles.)<br><a href="index.html">Click here for the complete list</a><br>
Click here for the non-redundant list of high-resolution x-ray structures up to: <a href="Nonredundant_1A.html">1A</a> <a href="Nonredundant_1,5A.html">1.5A</a> <a href="Nonredundant_2A.html">2A</a> <a href="Nonredundant_2,5A.html">2.5A</a> <a href="Nonredundant_3A.html">3A</a> <a href="Nonredundant_3,5A.html">3.5A</a> <a href="Nonredundant_4A.html">4A</a> <a href="Nonredundant_20A.html">20A</a> <a href="Nonredundant_All_Resolution.html">All Resolutions</a> 
<table border=1 class="sortable"><tr><th class="narrow">PDBID</th><th class="narrow">FR3D analysis</th><th class="narrow">Chain(s)</th><th class="narrow">NTs</th><th class="narrow">Pairs</th><th class="narrow">Res</th><th class="narrow">Date</th><th>Organism</th><th>Description provided by author</th><th>Author</th><th>Also represents</th></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZ9" title="Link to PDB">3IZ9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZ9/index.html" title="Link to FR3D analysis. 0.3793 pairs per nucleotide">3IZ9</a></td><td align="center">ABC</td><td align="center">3472</td><td align="center">1317</td><td align="center">5.5</td><td sorttable_customkey="20100112000000">12-01-10</td><td>TRITICUM AESTIVUM,TRITICUM AESTIVUM,TRITICUM AESTIVUM</td><td>MODEL OF THE LARGE SUBUNIT RNA BASED ON A 5.5 A CRYO-EM MAP OF TRITICUM AESTIVUM TRANSLATING 80S RIBOSOME</td><td>J.-P.ARMACHE,A.JARASCH,A.M.ANGER,E.VILLA,T.BECKER,S.BHUSHAN, F.JOSSINET,M.HABECK,G.DINDAR,S.FRANCKENBERG,V.MARQUEZ,T.MIELKE, M.THOMM,O.BERNINGHAUSEN,B.BEATRIX,J.SOEDING,E.WESTHOF,D.N.WILSON, R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3O58" title="Link to PDB">3O58</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O58/index.html" title="Link to FR3D analysis. 0.3122 pairs per nucleotide">3O58</a></td><td align="center">123</td><td align="center">3485</td><td align="center">1088</td><td align="center">4.0</td><td sorttable_customkey="20110312000000">12-15-10</td><td>SACCHAROMYCES CEREVISIAE S288C,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>YEAST 80S RIBOSOME. THIS ENTRY CONSISTS OF THE 60S SUBUNIT OF THE FIRST 80S IN THE ASYMMETRIC UNIT.</td><td>A.BEN-SHEM,L.JENNER,G.YUSUPOVA,M.YUSUPOV</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O5H/index.html" title="Resolution 4.0A, 3485 nucleotides, 1086 basepairs, 0.3116 pairs per nucleotide">3O5H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O58/3O58_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZF" title="Link to PDB">3IZF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZF/index.html" title="Link to FR3D analysis. 0.3583 pairs per nucleotide">3IZF</a></td><td align="center">ABC</td><td align="center">3436</td><td align="center">1231</td><td align="center">10.0</td><td sorttable_customkey="20100112000000">12-01-10</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>MODEL OF THE LARGE SUBUNIT RNA BASED ON A 6.1 A CRYO-EM MAP OF SACCHAROMYCES CEREVISIAE TRANSLATING 80S RIBOSOME (WITH ES27L-IN CONFORMATION)</td><td>J.-P.ARMACHE,A.JARASCH,A.M.ANGER,E.VILLA,T.BECKER,S.BHUSHAN, F.JOSSINET,M.HABECK,G.DINDAR,S.FRANCKENBERG,V.MARQUEZ,T.MIELKE, M.THOMM,O.BERNINGHAUSEN,B.BEATRIX,J.SOEDING,E.WESTHOF,D.N.WILSON, R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3JYX" title="Link to PDB">3JYX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JYX/index.html" title="Link to FR3D analysis. 0.1988 pairs per nucleotide">3JYX</a></td><td align="center">345</td><td align="center">3440</td><td align="center">684</td><td align="center">8.9</td><td sorttable_customkey="20101012000000">12-22-09</td><td>THERMOMYCES LANUGINOSUS,THERMOMYCES LANUGINOSUS,THERMOMYCES LANUGINOSUS</td><td>STRUCTURE OF THE 60S RRNA FOR EUKARYOTIC RIBOSOME BASED ON CRYO-EM MAP OF THERMOMYCES LANUGINOSUS RIBOSOME AT 8.9A RESOLUTION</td><td>D.J.TAYLOR,B.DEVKOTA,A.D.HUANG,M.TOPF,E.NARAYANAN,A.SALI,S.C.HARVEY, J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PYO" title="Link to PDB">3PYO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYO/index.html" title="Link to FR3D analysis. 0.4474 pairs per nucleotide">3PYO</a></td><td align="center">AB</td><td align="center">2879</td><td align="center">1288</td><td align="center">3.5</td><td sorttable_customkey="20110902000000">2-09-11</td><td>THERMUS THERMOPHILUS,THERMUS THERMOPHILUS</td><td>CRYSTAL STRUCTURE OF A COMPLEX CONTAINING DOMAIN 3 FROM THE PSIV IGR IRES RNA BOUND TO THE 70S RIBOSOME. THIS FILE CONTAINS THE 50S SUBUNIT OF THE FIRST 70S RIBOSOME.</td><td>J.ZHU,A.KOROSTELEV,D.COSTANTINO,H.F.NOLLER,J.S.KIEFT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYR/index.html" title="Resolution 3.5A, 2879 nucleotides, 1287 basepairs, 0.4470 pairs per nucleotide">3PYR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I8I/index.html" title="Resolution 3.1A, 3034 nucleotides, 1349 basepairs, 0.4446 pairs per nucleotide">3I8I</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYV/index.html" title="Resolution 3.4A, 2879 nucleotides, 1272 basepairs, 0.4418 pairs per nucleotide">3PYV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYT/index.html" title="Resolution 3.4A, 2878 nucleotides, 1270 basepairs, 0.4413 pairs per nucleotide">3PYT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDN/index.html" title="Resolution 3.5A, 2926 nucleotides, 1291 basepairs, 0.4412 pairs per nucleotide">2WDN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDL/index.html" title="Resolution 3.5A, 2926 nucleotides, 1286 basepairs, 0.4395 pairs per nucleotide">2WDL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WH2/index.html" title="Resolution 3.5A, 3005 nucleotides, 1318 basepairs, 0.4386 pairs per nucleotide">2WH2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WH4/index.html" title="Resolution 3.5A, 3005 nucleotides, 1316 basepairs, 0.4379 pairs per nucleotide">2WH4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MS1/index.html" title="Resolution 3.6A, 2998 nucleotides, 1311 basepairs, 0.4373 pairs per nucleotide">3MS1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MRZ/index.html" title="Resolution 3.6A, 2998 nucleotides, 1310 basepairs, 0.4370 pairs per nucleotide">3MRZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F1H/index.html" title="Resolution 3.0A, 2998 nucleotides, 1310 basepairs, 0.4370 pairs per nucleotide">3F1H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F1F/index.html" title="Resolution 3.0A, 2998 nucleotides, 1307 basepairs, 0.4360 pairs per nucleotide">3F1F</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDJ/index.html" title="Resolution 3.3A, 2926 nucleotides, 1275 basepairs, 0.4357 pairs per nucleotide">2WDJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I9E/index.html" title="Resolution 3.5A, 3006 nucleotides, 1308 basepairs, 0.4351 pairs per nucleotide">3I9E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDI/index.html" title="Resolution 3.3A, 2926 nucleotides, 1273 basepairs, 0.4351 pairs per nucleotide">2WDI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHJ/index.html" title="Resolution 3.0A, 2844 nucleotides, 1233 basepairs, 0.4335 pairs per nucleotide">3OHJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHZ/index.html" title="Resolution 3.0A, 2844 nucleotides, 1232 basepairs, 0.4332 pairs per nucleotide">3OHZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIT/index.html" title="Resolution 3.3A, 2967 nucleotides, 1284 basepairs, 0.4328 pairs per nucleotide">3KIT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I8F/index.html" title="Resolution 3.1A, 3031 nucleotides, 1311 basepairs, 0.4325 pairs per nucleotide">3I8F</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI1/index.html" title="Resolution 3.0A, 2844 nucleotides, 1229 basepairs, 0.4321 pairs per nucleotide">3OI1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIR/index.html" title="Resolution 3.3A, 2967 nucleotides, 1282 basepairs, 0.4321 pairs per nucleotide">3KIR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHK/index.html" title="Resolution 3.0A, 2844 nucleotides, 1226 basepairs, 0.4311 pairs per nucleotide">3OHK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OH5/index.html" title="Resolution 3.0A, 2844 nucleotides, 1224 basepairs, 0.4304 pairs per nucleotide">3OH5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OH7/index.html" title="Resolution 3.0A, 2844 nucleotides, 1224 basepairs, 0.4304 pairs per nucleotide">3OH7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNM/index.html" title="Resolution 3.5A, 2908 nucleotides, 1251 basepairs, 0.4302 pairs per nucleotide">3KNM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HUX/index.html" title="Resolution 3.1A, 2886 nucleotides, 1240 basepairs, 0.4297 pairs per nucleotide">3HUX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIY/index.html" title="Resolution 3.6A, 2967 nucleotides, 1272 basepairs, 0.4287 pairs per nucleotide">3KIY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIW/index.html" title="Resolution 3.6A, 2967 nucleotides, 1271 basepairs, 0.4284 pairs per nucleotide">3KIW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XG2/index.html" title="Resolution 3.2A, 2926 nucleotides, 1251 basepairs, 0.4275 pairs per nucleotide">2XG2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X9S/index.html" title="Resolution 3.1A, 3020 nucleotides, 1291 basepairs, 0.4275 pairs per nucleotide">2X9S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XG0/index.html" title="Resolution 3.2A, 2926 nucleotides, 1250 basepairs, 0.4272 pairs per nucleotide">2XG0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X9U/index.html" title="Resolution 3.1A, 3020 nucleotides, 1290 basepairs, 0.4272 pairs per nucleotide">2X9U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D5B/index.html" title="Resolution 3.2A, 2998 nucleotides, 1279 basepairs, 0.4266 pairs per nucleotide">3D5B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D5D/index.html" title="Resolution 3.2A, 2998 nucleotides, 1277 basepairs, 0.4260 pairs per nucleotide">3D5D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J01/index.html" title="Resolution 2.8A, 2891 nucleotides, 1228 basepairs, 0.4248 pairs per nucleotide">2J01</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J03/index.html" title="Resolution 2.8A, 2891 nucleotides, 1226 basepairs, 0.4241 pairs per nucleotide">2J03</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I9C/index.html" title="Resolution 3.5A, 3005 nucleotides, 1267 basepairs, 0.4216 pairs per nucleotide">3I9C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FIN/index.html" title="Resolution 6.4A, 2971 nucleotides, 1252 basepairs, 0.4214 pairs per nucleotide">3FIN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNO/index.html" title="Resolution 3.5A, 2894 nucleotides, 1209 basepairs, 0.4178 pairs per nucleotide">3KNO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0X/index.html" title="Resolution 3.1A, 3020 nucleotides, 1261 basepairs, 0.4175 pairs per nucleotide">2Y0X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HUZ/index.html" title="Resolution 3.1A, 2896 nucleotides, 1207 basepairs, 0.4168 pairs per nucleotide">3HUZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0V/index.html" title="Resolution 3.1A, 3020 nucleotides, 1257 basepairs, 0.4162 pairs per nucleotide">2Y0V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRJ/index.html" title="Resolution 3.6A, 3020 nucleotides, 1256 basepairs, 0.4159 pairs per nucleotide">2WRJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRL/index.html" title="Resolution 3.6A, 3020 nucleotides, 1255 basepairs, 0.4156 pairs per nucleotide">2WRL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI5/index.html" title="Resolution 3.1A, 2844 nucleotides, 1180 basepairs, 0.4149 pairs per nucleotide">3OI5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI3/index.html" title="Resolution 3.1A, 2844 nucleotides, 1179 basepairs, 0.4146 pairs per nucleotide">3OI3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XUX/index.html" title="Resolution 7.6A, 3020 nucleotides, 1251 basepairs, 0.4142 pairs per nucleotide">2XUX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y11/index.html" title="Resolution 3.1A, 3020 nucleotides, 1248 basepairs, 0.4132 pairs per nucleotide">2Y11</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y19/index.html" title="Resolution 3.1A, 3020 nucleotides, 1246 basepairs, 0.4126 pairs per nucleotide">2Y19</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XTG/index.html" title="Resolution 7.8A, 3020 nucleotides, 1239 basepairs, 0.4103 pairs per nucleotide">2XTG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XQE/index.html" title="Resolution 3.1A, 3020 nucleotides, 1229 basepairs, 0.4070 pairs per nucleotide">2XQE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNI/index.html" title="Resolution 3.0A, 2918 nucleotides, 1186 basepairs, 0.4064 pairs per nucleotide">3KNI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0Z/index.html" title="Resolution 3.1A, 3020 nucleotides, 1224 basepairs, 0.4053 pairs per nucleotide">2Y0Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y15/index.html" title="Resolution 3.1A, 3020 nucleotides, 1221 basepairs, 0.4043 pairs per nucleotide">2Y15</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y17/index.html" title="Resolution 3.1A, 3020 nucleotides, 1217 basepairs, 0.4030 pairs per nucleotide">2Y17</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y13/index.html" title="Resolution 3.1A, 3020 nucleotides, 1189 basepairs, 0.3937 pairs per nucleotide">2Y13</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V47/index.html" title="Resolution 3.8A, 2886 nucleotides, 1104 basepairs, 0.3825 pairs per nucleotide">2V47</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V49/index.html" title="Resolution 3.8A, 2886 nucleotides, 1101 basepairs, 0.3815 pairs per nucleotide">2V49</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VSP/index.html" title="Resolution 3.8A, 3009 nucleotides, 1136 basepairs, 0.3775 pairs per nucleotide">1VSP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNK/index.html" title="Resolution 3.0A, 2918 nucleotides, 1085 basepairs, 0.3718 pairs per nucleotide">3KNK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VSA/index.html" title="Resolution 3.7A, 3009 nucleotides, 1096 basepairs, 0.3642 pairs per nucleotide">1VSA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRR/index.html" title="Resolution 3.6A, 3020 nucleotides, 1051 basepairs, 0.3480 pairs per nucleotide">2WRR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRO/index.html" title="Resolution 3.6A, 3020 nucleotides, 1046 basepairs, 0.3464 pairs per nucleotide">2WRO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGU/index.html" title="Resolution 4.5A, 3012 nucleotides, 920 basepairs, 0.3054 pairs per nucleotide">2HGU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGJ/index.html" title="Resolution 5.0A, 3012 nucleotides, 906 basepairs, 0.3008 pairs per nucleotide">2HGJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGQ/index.html" title="Resolution 5.5A, 3012 nucleotides, 896 basepairs, 0.2975 pairs per nucleotide">2HGQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B9P/index.html" title="Resolution 6.5A, 2937 nucleotides, 816 basepairs, 0.2778 pairs per nucleotide">2B9P</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B66/index.html" title="Resolution 5.9A, 2937 nucleotides, 811 basepairs, 0.2761 pairs per nucleotide">2B66</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B9N/index.html" title="Resolution 6.8A, 2937 nucleotides, 806 basepairs, 0.2744 pairs per nucleotide">2B9N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YL3/index.html" title="Resolution 5.5A, 2995 nucleotides, 777 basepairs, 0.2594 pairs per nucleotide">1YL3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYO/3PYO_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QBG" title="Link to PDB">2QBG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBG/index.html" title="Link to FR3D analysis. 0.4588 pairs per nucleotide">2QBG</a></td><td align="center">AB</td><td align="center">2958</td><td align="center">1357</td><td align="center">3.3</td><td sorttable_customkey="20090109000000">9-25-07</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF THE BACTERIAL RIBOSOME FROM ESCHERICHIA COLI IN COMPLEX WITH RIBOSOME RECYCLING FACTOR (RRF). THIS FILE CONTAINS THE 50S SUBUNIT OF THE SECOND 70S RIBOSOME, WITH RRF BOUND. THE ENTIRE CRYSTAL STRUCTURE CONTAINS TWO 70S RIBOSOMES AND IS DESCRIBED IN REMARK 400.</td><td>M.A.BOROVINSKAYA,R.D.PAI,W.ZHANG,B.-S.SCHUWIRTH,J.M.HOLTON, G.HIROKAWA,H.KAJI,A.KAJI,J.H.D.CATE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBE/index.html" title="Resolution 3.3A, 2958 nucleotides, 1355 basepairs, 0.4581 pairs per nucleotide">2QBE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFC/index.html" title="Resolution 3.2A, 2972 nucleotides, 1359 basepairs, 0.4573 pairs per nucleotide">3OFC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ORB/index.html" title="Resolution 3.3A, 2972 nucleotides, 1359 basepairs, 0.4573 pairs per nucleotide">3ORB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1N/index.html" title="Resolution 3.2A, 2972 nucleotides, 1356 basepairs, 0.4563 pairs per nucleotide">3I1N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OAT/index.html" title="Resolution 3.2A, 2972 nucleotides, 1355 basepairs, 0.4559 pairs per nucleotide">3OAT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QAO/index.html" title="Resolution 3.2A, 2958 nucleotides, 1348 basepairs, 0.4557 pairs per nucleotide">2QAO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AWB/index.html" title="Resolution 3.5A, 2958 nucleotides, 1346 basepairs, 0.4550 pairs per nucleotide">2AWB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J28/index.html" title="Resolution 8.0A, 3032 nucleotides, 1379 basepairs, 0.4548 pairs per nucleotide">2J28</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QAM/index.html" title="Resolution 3.2A, 2958 nucleotides, 1343 basepairs, 0.4540 pairs per nucleotide">2QAM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFR/index.html" title="Resolution 3.1A, 2972 nucleotides, 1348 basepairs, 0.4536 pairs per nucleotide">3OFR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AW4/index.html" title="Resolution 3.5A, 2958 nucleotides, 1341 basepairs, 0.4533 pairs per nucleotide">2AW4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VS8/index.html" title="Resolution 3.5A, 2958 nucleotides, 1338 basepairs, 0.4523 pairs per nucleotide">1VS8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I2V/index.html" title="Resolution 3.2A, 2958 nucleotides, 1338 basepairs, 0.4523 pairs per nucleotide">2I2V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBC/index.html" title="Resolution 3.5A, 2958 nucleotides, 1338 basepairs, 0.4523 pairs per nucleotide">2QBC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I2T/index.html" title="Resolution 3.2A, 2958 nucleotides, 1336 basepairs, 0.4517 pairs per nucleotide">2I2T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBA/index.html" title="Resolution 3.5A, 2958 nucleotides, 1335 basepairs, 0.4513 pairs per nucleotide">2QBA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VS6/index.html" title="Resolution 3.5A, 2958 nucleotides, 1335 basepairs, 0.4513 pairs per nucleotide">1VS6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBK/index.html" title="Resolution 4.0A, 2958 nucleotides, 1334 basepairs, 0.4510 pairs per nucleotide">2QBK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFZ/index.html" title="Resolution 3.3A, 2972 nucleotides, 1340 basepairs, 0.4509 pairs per nucleotide">3OFZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DF2/index.html" title="Resolution 3.5A, 2958 nucleotides, 1330 basepairs, 0.4496 pairs per nucleotide">3DF2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DF4/index.html" title="Resolution 3.5A, 2958 nucleotides, 1329 basepairs, 0.4493 pairs per nucleotide">3DF4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z4N/index.html" title="Resolution 4.5A, 2958 nucleotides, 1328 basepairs, 0.4490 pairs per nucleotide">2Z4N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOZ/index.html" title="Resolution 3.5A, 2958 nucleotides, 1326 basepairs, 0.4483 pairs per nucleotide">2QOZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QP1/index.html" title="Resolution 3.5A, 2958 nucleotides, 1325 basepairs, 0.4479 pairs per nucleotide">2QP1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBI/index.html" title="Resolution 4.0A, 2958 nucleotides, 1324 basepairs, 0.4476 pairs per nucleotide">2QBI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z4L/index.html" title="Resolution 4.5A, 2958 nucleotides, 1324 basepairs, 0.4476 pairs per nucleotide">2Z4L</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R8S/index.html" title="Resolution 3.0A, 3016 nucleotides, 1347 basepairs, 0.4466 pairs per nucleotide">3R8S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I20/index.html" title="Resolution 3.7A, 2972 nucleotides, 1323 basepairs, 0.4452 pairs per nucleotide">3I20</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1R/index.html" title="Resolution 3.8A, 2972 nucleotides, 1322 basepairs, 0.4448 pairs per nucleotide">3I1R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOX/index.html" title="Resolution 3.9A, 2958 nucleotides, 1308 basepairs, 0.4422 pairs per nucleotide">2QOX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOV/index.html" title="Resolution 3.9A, 2958 nucleotides, 1307 basepairs, 0.4419 pairs per nucleotide">2QOV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1T/index.html" title="Resolution 3.8A, 2958 nucleotides, 1281 basepairs, 0.4331 pairs per nucleotide">3I1T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R8T/index.html" title="Resolution 3.0A, 3014 nucleotides, 1305 basepairs, 0.4330 pairs per nucleotide">3R8T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1P/index.html" title="Resolution 3.2A, 2958 nucleotides, 1263 basepairs, 0.4270 pairs per nucleotide">3I1P</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FIK/index.html" title="Resolution 6.7A, 3020 nucleotides, 1270 basepairs, 0.4205 pairs per nucleotide">3FIK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I22/index.html" title="Resolution 3.7A, 2958 nucleotides, 1241 basepairs, 0.4195 pairs per nucleotide">3I22</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WWQ/index.html" title="Resolution 5.8A, 3105 nucleotides, 1299 basepairs, 0.4184 pairs per nucleotide">2WWQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFD/index.html" title="Resolution 3.2A, 2958 nucleotides, 1230 basepairs, 0.4158 pairs per nucleotide">3OFD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OAS/index.html" title="Resolution 3.2A, 2958 nucleotides, 1230 basepairs, 0.4158 pairs per nucleotide">3OAS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3J01/index.html" title="Resolution 7.1A, 3024 nucleotides, 1256 basepairs, 0.4153 pairs per nucleotide">3J01</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OG0/index.html" title="Resolution 3.3A, 2958 nucleotides, 1220 basepairs, 0.4124 pairs per nucleotide">3OG0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KCR/index.html" title="Resolution 9.6A, 3020 nucleotides, 1212 basepairs, 0.4013 pairs per nucleotide">3KCR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VT2/index.html" title="Resolution 3.3A, 2958 nucleotides, 1187 basepairs, 0.4013 pairs per nucleotide">1VT2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBX/index.html" title="Resolution 10.0A, 2958 nucleotides, 1185 basepairs, 0.4006 pairs per nucleotide">3BBX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZU/index.html" title="Resolution 10.0A, 3000 nucleotides, 1177 basepairs, 0.3923 pairs per nucleotide">3IZU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFQ/index.html" title="Resolution 3.1A, 2958 nucleotides, 1138 basepairs, 0.3847 pairs per nucleotide">3OFQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZT/index.html" title="Resolution 10.0A, 3000 nucleotides, 1138 basepairs, 0.3793 pairs per nucleotide">3IZT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VHM/index.html" title="Resolution 3.7A, 2958 nucleotides, 980 basepairs, 0.3313 pairs per nucleotide">2VHM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VHN/index.html" title="Resolution 3.7A, 2958 nucleotides, 967 basepairs, 0.3269 pairs per nucleotide">2VHN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GYA/index.html" title="Resolution 15.0A, 2848 nucleotides, 791 basepairs, 0.2777 pairs per nucleotide">2GYA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GYC/index.html" title="Resolution 15.0A, 2848 nucleotides, 659 basepairs, 0.2314 pairs per nucleotide">2GYC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E1D/index.html" title="Resolution 9.0A, 3020 nucleotides, 586 basepairs, 0.1940 pairs per nucleotide">3E1D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E1B/index.html" title="Resolution 9.0A, 3020 nucleotides, 555 basepairs, 0.1838 pairs per nucleotide">3E1B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBG/2QBG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1C2W" title="Link to PDB">1C2W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C2W/index.html" title="Link to FR3D analysis. 0.2926 pairs per nucleotide">1C2W</a></td><td align="center">B</td><td align="center">2891</td><td align="center">846</td><td align="center">7.5</td><td sorttable_customkey="20001004000000">4-10-00</td><td>ESCHERICHIA COLI</td><td>S RRNA STRUCTURE FITTED TO A CRYO-ELECTRON MICROSCOPIC MAP AT 7.5 ANGSTROMS RESOLUTION</td><td>R.BRIMACOMBE,F.MUELLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PNU" title="Link to PDB">1PNU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNU/index.html" title="Link to FR3D analysis. 0.3187 pairs per nucleotide">1PNU</a></td><td align="center">09</td><td align="center">2943</td><td align="center">938</td><td align="center">8.7</td><td sorttable_customkey="20040307000000">7-15-03</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF A STREPTOMYCIN DEPENDENT RIBOSOME FROM ESCHERICHIA COLI, 50S SUBUNIT OF 70S RIBOSOME. THIS FILE, 1PNU, CONTAINS ONLY MOLECULES OF THE 50S RIBOSOMAL SUBUNIT. THE 30S SUBUNIT, MRNA, P-SITE TRNA, AND A-SITE TRNA ARE IN THE PDB FILE 1PNS.</td><td>A.VILA-SANJURJO,W.K.RIDGEWAY,V.SEYMANER,W.ZHANG,S.SANTOSO, K.YU,J.H.D.CATE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNY/index.html" title="Resolution 9.5A, 2943 nucleotides, 938 basepairs, 0.3187 pairs per nucleotide">1PNY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOY/index.html" title="Resolution 11.5A, 2943 nucleotides, 588 basepairs, 0.1998 pairs per nucleotide">1VOY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOU/index.html" title="Resolution 11.5A, 2943 nucleotides, 586 basepairs, 0.1991 pairs per nucleotide">1VOU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOR/index.html" title="Resolution 11.5A, 2943 nucleotides, 584 basepairs, 0.1984 pairs per nucleotide">1VOR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VP0/index.html" title="Resolution 11.5A, 2943 nucleotides, 583 basepairs, 0.1981 pairs per nucleotide">1VP0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOW/index.html" title="Resolution 11.5A, 2943 nucleotides, 583 basepairs, 0.1981 pairs per nucleotide">1VOW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNU/1PNU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZJR" title="Link to PDB">2ZJR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZJR/index.html" title="Link to FR3D analysis. 0.4202 pairs per nucleotide">2ZJR</a></td><td align="center">XY</td><td align="center">2808</td><td align="center">1180</td><td align="center">2.9</td><td sorttable_customkey="20090506000000">6-17-08</td><td>DEINOCOCCUS RADIODURANS,DEINOCOCCUS RADIODURANS</td><td>REFINED NATIVE STRUCTURE OF THE LARGE RIBOSOMAL SUBUNIT (50S) FROM DEINOCOCCUS RADIODURANS</td><td>J.M.HARMS,D.N.WILSON,F.SCHLUENZEN,S.R.CONNELL,T.STACHELHAUS, Z.ZABOROWSKA,C.M.T.SPAHN,P.FUCINI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CF5/index.html" title="Resolution 3.3A, 2808 nucleotides, 1160 basepairs, 0.4131 pairs per nucleotide">3CF5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZJQ/index.html" title="Resolution 3.3A, 2808 nucleotides, 1127 basepairs, 0.4014 pairs per nucleotide">2ZJQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZJP/index.html" title="Resolution 3.7A, 2808 nucleotides, 1125 basepairs, 0.4006 pairs per nucleotide">2ZJP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DLL/index.html" title="Resolution 3.5A, 2808 nucleotides, 1101 basepairs, 0.3921 pairs per nucleotide">3DLL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AAR/index.html" title="Resolution 3.5A, 2766 nucleotides, 1014 basepairs, 0.3666 pairs per nucleotide">2AAR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OGM/index.html" title="Resolution 3.5A, 2766 nucleotides, 1004 basepairs, 0.3630 pairs per nucleotide">2OGM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PIO/index.html" title="Resolution 3.2A, 2777 nucleotides, 964 basepairs, 0.3471 pairs per nucleotide">3PIO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D3O/index.html" title="Resolution 3.4A, 2802 nucleotides, 953 basepairs, 0.3401 pairs per nucleotide">2D3O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JQ4/index.html" title="Resolution 3.5A, 2883 nucleotides, 980 basepairs, 0.3399 pairs per nucleotide">3JQ4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JZY/index.html" title="Resolution 3.5A, 2774 nucleotides, 923 basepairs, 0.3327 pairs per nucleotide">1JZY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OGO/index.html" title="Resolution 3.7A, 2765 nucleotides, 920 basepairs, 0.3327 pairs per nucleotide">2OGO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K01/index.html" title="Resolution 3.5A, 2774 nucleotides, 922 basepairs, 0.3324 pairs per nucleotide">1K01</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y69/index.html" title="Resolution 3.3A, 2884 nucleotides, 958 basepairs, 0.3322 pairs per nucleotide">1Y69</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PIP/index.html" title="Resolution 3.5A, 2764 nucleotides, 917 basepairs, 0.3318 pairs per nucleotide">3PIP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JZX/index.html" title="Resolution 3.1A, 2774 nucleotides, 920 basepairs, 0.3317 pairs per nucleotide">1JZX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JZZ/index.html" title="Resolution 3.8A, 2774 nucleotides, 913 basepairs, 0.3291 pairs per nucleotide">1JZZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J5A/index.html" title="Resolution 3.5A, 2774 nucleotides, 912 basepairs, 0.3288 pairs per nucleotide">1J5A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O44/index.html" title="Resolution 3.3A, 2766 nucleotides, 891 basepairs, 0.3221 pairs per nucleotide">2O44</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OGN/index.html" title="Resolution 3.6A, 2766 nucleotides, 889 basepairs, 0.3214 pairs per nucleotide">2OGN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OND/index.html" title="Resolution 3.4A, 2765 nucleotides, 875 basepairs, 0.3165 pairs per nucleotide">1OND</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NWX/index.html" title="Resolution 3.5A, 2884 nucleotides, 910 basepairs, 0.3155 pairs per nucleotide">1NWX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NWY/index.html" title="Resolution 3.3A, 2884 nucleotides, 906 basepairs, 0.3141 pairs per nucleotide">1NWY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NKW/index.html" title="Resolution 3.1A, 2884 nucleotides, 902 basepairs, 0.3128 pairs per nucleotide">1NKW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NJO/index.html" title="Resolution 3.7A, 2769 nucleotides, 865 basepairs, 0.3124 pairs per nucleotide">1NJO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NJP/index.html" title="Resolution 3.5A, 2790 nucleotides, 871 basepairs, 0.3122 pairs per nucleotide">1NJP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FWO/index.html" title="Resolution 3.7A, 2883 nucleotides, 899 basepairs, 0.3118 pairs per nucleotide">3FWO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NJM/index.html" title="Resolution 3.6A, 2790 nucleotides, 868 basepairs, 0.3111 pairs per nucleotide">1NJM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P9X/index.html" title="Resolution 3.4A, 2738 nucleotides, 850 basepairs, 0.3104 pairs per nucleotide">1P9X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NJN/index.html" title="Resolution 3.7A, 2766 nucleotides, 856 basepairs, 0.3095 pairs per nucleotide">1NJN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O43/index.html" title="Resolution 3.6A, 2766 nucleotides, 837 basepairs, 0.3026 pairs per nucleotide">2O43</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SM1/index.html" title="Resolution 3.4A, 2884 nucleotides, 862 basepairs, 0.2989 pairs per nucleotide">1SM1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XBP/index.html" title="Resolution 3.5A, 2884 nucleotides, 844 basepairs, 0.2926 pairs per nucleotide">1XBP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z58/index.html" title="Resolution 3.8A, 2766 nucleotides, 754 basepairs, 0.2726 pairs per nucleotide">1Z58</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O45/index.html" title="Resolution 3.6A, 2766 nucleotides, 744 basepairs, 0.2690 pairs per nucleotide">2O45</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZJR/2ZJR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S1I" title="Link to PDB">1S1I</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S1I/index.html" title="Link to FR3D analysis. 0.4513 pairs per nucleotide">1S1I</a></td><td align="center">34</td><td align="center">2916</td><td align="center">1316</td><td align="center">11.7</td><td sorttable_customkey="20060105000000">5-25-04</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>STRUCTURE OF THE RIBOSOMAL 80S-EEF2-SORDARIN COMPLEX FROM YEAST OBTAINED BY DOCKING ATOMIC MODELS FOR RNA AND PROTEIN COMPONENTS INTO A 11.7 A CRYO-EM MAP. THIS FILE, 1S1I, CONTAINS 60S SUBUNIT. THE 40S RIBOSOMAL SUBUNIT IS IN FILE 1S1H.</td><td>C.M.SPAHN,M.G.GOMEZ-LORENZO,R.A.GRASSUCCI,R.JORGENSEN, G.R.ANDERSEN,R.BECKMANN,P.A.PENCZEK,J.P.G.BALLESTA,J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S72" title="Link to PDB">1S72</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S72/index.html" title="Link to FR3D analysis. 0.4793 pairs per nucleotide">1S72</a></td><td align="center">09</td><td align="center">2871</td><td align="center">1376</td><td align="center">2.4</td><td sorttable_customkey="20050306000000">6-15-04</td><td>HALOARCULA MARISMORTUI,HALOARCULA MARISMORTUI</td><td>REFINED CRYSTAL STRUCTURE OF THE HALOARCULA MARISMORTUI LARGE RIBOSOMAL SUBUNIT AT 2.4 ANGSTROM RESOLUTION</td><td>D.J.KLEIN,T.M.SCHMEING,P.B.MOORE,T.A.STEITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ6/index.html" title="Resolution 2.7A, 2875 nucleotides, 1373 basepairs, 0.4776 pairs per nucleotide">1VQ6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ4/index.html" title="Resolution 2.7A, 2875 nucleotides, 1372 basepairs, 0.4772 pairs per nucleotide">1VQ4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CC2/index.html" title="Resolution 2.4A, 2871 nucleotides, 1370 basepairs, 0.4772 pairs per nucleotide">3CC2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CPW/index.html" title="Resolution 2.7A, 2879 nucleotides, 1373 basepairs, 0.4769 pairs per nucleotide">3CPW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQO/index.html" title="Resolution 2.2A, 2873 nucleotides, 1370 basepairs, 0.4769 pairs per nucleotide">1VQO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ8/index.html" title="Resolution 2.2A, 2873 nucleotides, 1370 basepairs, 0.4769 pairs per nucleotide">1VQ8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K73/index.html" title="Resolution 3.0A, 2876 nucleotides, 1371 basepairs, 0.4767 pairs per nucleotide">1K73</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQP/index.html" title="Resolution 2.2A, 2874 nucleotides, 1370 basepairs, 0.4767 pairs per nucleotide">1VQP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M90/index.html" title="Resolution 2.8A, 2879 nucleotides, 1372 basepairs, 0.4766 pairs per nucleotide">1M90</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQK/index.html" title="Resolution 2.3A, 2873 nucleotides, 1369 basepairs, 0.4765 pairs per nucleotide">1VQK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JJ2/index.html" title="Resolution 2.4A, 2876 nucleotides, 1370 basepairs, 0.4764 pairs per nucleotide">1JJ2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ9/index.html" title="Resolution 2.4A, 2874 nucleotides, 1368 basepairs, 0.4760 pairs per nucleotide">1VQ9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CC4/index.html" title="Resolution 2.7A, 2871 nucleotides, 1366 basepairs, 0.4758 pairs per nucleotide">3CC4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCU/index.html" title="Resolution 2.8A, 2871 nucleotides, 1366 basepairs, 0.4758 pairs per nucleotide">3CCU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQM/index.html" title="Resolution 2.3A, 2873 nucleotides, 1366 basepairs, 0.4755 pairs per nucleotide">1VQM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQL/index.html" title="Resolution 2.3A, 2873 nucleotides, 1366 basepairs, 0.4755 pairs per nucleotide">1VQL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G6E/index.html" title="Resolution 2.7A, 2871 nucleotides, 1365 basepairs, 0.4754 pairs per nucleotide">3G6E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCM/index.html" title="Resolution 2.5A, 2871 nucleotides, 1364 basepairs, 0.4751 pairs per nucleotide">3CCM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YHQ/index.html" title="Resolution 2.4A, 2871 nucleotides, 1364 basepairs, 0.4751 pairs per nucleotide">1YHQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NJI/index.html" title="Resolution 3.0A, 2876 nucleotides, 1366 basepairs, 0.4750 pairs per nucleotide">1NJI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QVF/index.html" title="Resolution 3.1A, 2888 nucleotides, 1371 basepairs, 0.4747 pairs per nucleotide">1QVF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YI2/index.html" title="Resolution 2.6A, 2871 nucleotides, 1362 basepairs, 0.4744 pairs per nucleotide">1YI2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ7/index.html" title="Resolution 2.5A, 2876 nucleotides, 1364 basepairs, 0.4743 pairs per nucleotide">1VQ7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KQS/index.html" title="Resolution 3.1A, 2881 nucleotides, 1366 basepairs, 0.4741 pairs per nucleotide">1KQS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQN/index.html" title="Resolution 2.4A, 2876 nucleotides, 1363 basepairs, 0.4739 pairs per nucleotide">1VQN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CXC/index.html" title="Resolution 3.0A, 2879 nucleotides, 1363 basepairs, 0.4734 pairs per nucleotide">3CXC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCQ/index.html" title="Resolution 2.9A, 2871 nucleotides, 1359 basepairs, 0.4734 pairs per nucleotide">3CCQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CC7/index.html" title="Resolution 2.7A, 2871 nucleotides, 1359 basepairs, 0.4734 pairs per nucleotide">3CC7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCV/index.html" title="Resolution 2.9A, 2871 nucleotides, 1359 basepairs, 0.4734 pairs per nucleotide">3CCV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KC8/index.html" title="Resolution 3.0A, 2876 nucleotides, 1361 basepairs, 0.4732 pairs per nucleotide">1KC8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VQ5/index.html" title="Resolution 2.6A, 2874 nucleotides, 1360 basepairs, 0.4732 pairs per nucleotide">1VQ5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCL/index.html" title="Resolution 2.9A, 2871 nucleotides, 1358 basepairs, 0.4730 pairs per nucleotide">3CCL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q86/index.html" title="Resolution 3.0A, 2882 nucleotides, 1363 basepairs, 0.4729 pairs per nucleotide">1Q86</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K9M/index.html" title="Resolution 3.0A, 2876 nucleotides, 1360 basepairs, 0.4729 pairs per nucleotide">1K9M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YIJ/index.html" title="Resolution 2.6A, 2871 nucleotides, 1357 basepairs, 0.4727 pairs per nucleotide">1YIJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CMA/index.html" title="Resolution 2.8A, 2874 nucleotides, 1357 basepairs, 0.4722 pairs per nucleotide">3CMA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FFK/index.html" title="Resolution 2.4A, 2828 nucleotides, 1335 basepairs, 0.4721 pairs per nucleotide">1FFK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YIT/index.html" title="Resolution 2.8A, 2871 nucleotides, 1355 basepairs, 0.4720 pairs per nucleotide">1YIT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YJW/index.html" title="Resolution 2.9A, 2871 nucleotides, 1354 basepairs, 0.4716 pairs per nucleotide">1YJW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q82/index.html" title="Resolution 3.0A, 2878 nucleotides, 1357 basepairs, 0.4715 pairs per nucleotide">1Q82</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N8R/index.html" title="Resolution 3.0A, 2876 nucleotides, 1356 basepairs, 0.4715 pairs per nucleotide">1N8R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCE/index.html" title="Resolution 2.8A, 2871 nucleotides, 1353 basepairs, 0.4713 pairs per nucleotide">3CCE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G71/index.html" title="Resolution 2.9A, 2871 nucleotides, 1352 basepairs, 0.4709 pairs per nucleotide">3G71</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YJN/index.html" title="Resolution 3.0A, 2871 nucleotides, 1352 basepairs, 0.4709 pairs per nucleotide">1YJN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M1K/index.html" title="Resolution 3.2A, 2876 nucleotides, 1354 basepairs, 0.4708 pairs per nucleotide">1M1K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OTL/index.html" title="Resolution 2.7A, 2871 nucleotides, 1351 basepairs, 0.4706 pairs per nucleotide">2OTL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q81/index.html" title="Resolution 3.0A, 2878 nucleotides, 1354 basepairs, 0.4705 pairs per nucleotide">1Q81</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QVG/index.html" title="Resolution 2.9A, 2885 nucleotides, 1357 basepairs, 0.4704 pairs per nucleotide">1QVG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K8A/index.html" title="Resolution 3.0A, 2876 nucleotides, 1352 basepairs, 0.4701 pairs per nucleotide">1K8A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OTJ/index.html" title="Resolution 2.9A, 2871 nucleotides, 1349 basepairs, 0.4699 pairs per nucleotide">2OTJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q7Y/index.html" title="Resolution 3.2A, 2878 nucleotides, 1352 basepairs, 0.4698 pairs per nucleotide">1Q7Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CD6/index.html" title="Resolution 2.8A, 2873 nucleotides, 1349 basepairs, 0.4695 pairs per nucleotide">3CD6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KD1/index.html" title="Resolution 3.0A, 2876 nucleotides, 1350 basepairs, 0.4694 pairs per nucleotide">1KD1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YJ9/index.html" title="Resolution 2.9A, 2871 nucleotides, 1347 basepairs, 0.4692 pairs per nucleotide">1YJ9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I56/index.html" title="Resolution 2.9A, 2871 nucleotides, 1342 basepairs, 0.4674 pairs per nucleotide">3I56</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I55/index.html" title="Resolution 3.1A, 2875 nucleotides, 1343 basepairs, 0.4671 pairs per nucleotide">3I55</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QEX/index.html" title="Resolution 2.9A, 2871 nucleotides, 1341 basepairs, 0.4671 pairs per nucleotide">2QEX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCS/index.html" title="Resolution 3.0A, 2871 nucleotides, 1338 basepairs, 0.4660 pairs per nucleotide">3CCS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CME/index.html" title="Resolution 3.0A, 2873 nucleotides, 1337 basepairs, 0.4654 pairs per nucleotide">3CME</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCR/index.html" title="Resolution 3.0A, 2871 nucleotides, 1324 basepairs, 0.4612 pairs per nucleotide">3CCR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1W2B/index.html" title="Resolution 3.5A, 2876 nucleotides, 1314 basepairs, 0.4569 pairs per nucleotide">1W2B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CCJ/index.html" title="Resolution 2.7A, 2871 nucleotides, 1299 basepairs, 0.4525 pairs per nucleotide">3CCJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QA4/index.html" title="Resolution 3.0A, 2870 nucleotides, 1291 basepairs, 0.4498 pairs per nucleotide">2QA4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G4S/index.html" title="Resolution 3.2A, 2871 nucleotides, 1272 basepairs, 0.4431 pairs per nucleotide">3G4S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FG0/index.html" title="Resolution 3.0A, 498 nucleotides, 203 basepairs, 0.4076 pairs per nucleotide">1FG0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FFZ/index.html" title="Resolution 3.2A, 498 nucleotides, 193 basepairs, 0.3876 pairs per nucleotide">1FFZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S72/1S72_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BBO" title="Link to PDB">3BBO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBO/index.html" title="Link to FR3D analysis. 0.2791 pairs per nucleotide">3BBO</a></td><td align="center">ABC</td><td align="center">2952</td><td align="center">824</td><td align="center">9.4</td><td sorttable_customkey="20100305000000">5-27-08</td><td>SPINACEA OLERACEA,SPINACEA OLERACEA,SPINACEA OLERACEA</td><td>HOMOLOGY MODEL FOR THE SPINACH CHLOROPLAST 50S SUBUNIT FITTED TO 9.4A CRYO-EM MAP OF THE 70S CHLORORIBOSOME</td><td>M.R.SHARMA,D.N.WILSON,P.P.DATTA,C.BARAT,F.SCHLUENZEN, P.FUCINI,R.K.AGRAWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZE" title="Link to PDB">3IZE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZE/index.html" title="Link to FR3D analysis. 0.2769 pairs per nucleotide">3IZE</a></td><td align="center">ABC</td><td align="center">1582</td><td align="center">438</td><td align="center">10.0</td><td sorttable_customkey="20100112000000">12-01-10</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>MODEL OF THE SMALL SUBUNIT RNA BASED ON A 6.1 A CRYO-EM MAP OF SACCHAROMYCES CEREVISIAE TRANSLATING 80S RIBOSOME</td><td>J.-P.ARMACHE,A.JARASCH,A.M.ANGER,E.VILLA,T.BECKER,S.BHUSHAN, F.JOSSINET,M.HABECK,G.DINDAR,S.FRANCKENBERG,V.MARQUEZ,T.MIELKE, M.THOMM,O.BERNINGHAUSEN,B.BEATRIX,J.SOEDING,E.WESTHOF,D.N.WILSON, R.BECKMANN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O30/index.html" title="Resolution 4.0A, 1789 nucleotides, 482 basepairs, 0.2694 pairs per nucleotide">3O30</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O2Z/index.html" title="Resolution 4.0A, 1789 nucleotides, 475 basepairs, 0.2655 pairs per nucleotide">3O2Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZE/3IZE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3JYV" title="Link to PDB">3JYV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JYV/index.html" title="Link to FR3D analysis. 0.2780 pairs per nucleotide">3JYV</a></td><td align="center">7A</td><td align="center">1820</td><td align="center">506</td><td align="center">8.9</td><td sorttable_customkey="20101012000000">12-22-09</td><td>THERMOMYCES LANUGINOSUS,THERMOMYCES LANUGINOSUS</td><td>STRUCTURE OF THE 40S RRNA AND PROTEINS AND P/E TRNA FOR EUKARYOTIC RIBOSOME BASED ON CRYO-EM MAP OF THERMOMYCES LANUGINOSUS RIBOSOME AT 8.9A RESOLUTION</td><td>D.J.TAYLOR,B.DEVKOTA,A.D.HUANG,M.TOPF,E.NARAYANAN,A.SALI,S.C.HARVEY, J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XZM" title="Link to PDB">2XZM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XZM/index.html" title="Link to FR3D analysis. 0.3181 pairs per nucleotide">2XZM</a></td><td align="center">A</td><td align="center">1745</td><td align="center">555</td><td align="center">3.9</td><td sorttable_customkey="20120701000000">1-19-11</td><td>TETRAHYMENA THERMOPHILA</td><td>CRYSTAL STRUCTURE OF THE EUKARYOTIC 40S RIBOSOMAL SUBUNIT IN COMPLEX WITH INITIATION FACTOR 1. THIS FILE CONTAINS THE 40S SUBUNIT AND INITIATION FACTOR FOR MOLECULE 1</td><td>J.RABL,M.LEIBUNDGUT,S.F.ATAIDE,A.HAAG,N.BAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XZN/index.html" title="Resolution 3.9A, 1745 nucleotides, 553 basepairs, 0.3169 pairs per nucleotide">2XZN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XZM/2XZM_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AW7" title="Link to PDB">2AW7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AW7/index.html" title="Link to FR3D analysis. 0.4484 pairs per nucleotide">2AW7</a></td><td align="center">A</td><td align="center">1530</td><td align="center">686</td><td align="center">3.5</td><td sorttable_customkey="20050811000000">11-08-05</td><td>ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF THE BACTERIAL RIBOSOME FROM ESCHERICHIA COLI AT 3.5 A RESOLUTION. THIS FILE CONTAINS THE 30S SUBUNIT OF THE SECOND 70S RIBOSOME. THE ENTIRE CRYSTAL STRUCTURE CONTAINS TWO 70S RIBOSOMES AND IS DESCRIBED IN REMARK 400.</td><td>B.S.SCHUWIRTH,M.A.BOROVINSKAYA,C.W.HAU,W.ZHANG,A.VILA- SANJURJO,J.M.HOLTON,J.H.D.CATE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VS7/index.html" title="Resolution 3.5A, 1530 nucleotides, 686 basepairs, 0.4484 pairs per nucleotide">1VS7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QAN/index.html" title="Resolution 3.2A, 1530 nucleotides, 684 basepairs, 0.4471 pairs per nucleotide">2QAN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBF/index.html" title="Resolution 3.3A, 1530 nucleotides, 681 basepairs, 0.4451 pairs per nucleotide">2QBF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AVY/index.html" title="Resolution 3.5A, 1530 nucleotides, 679 basepairs, 0.4438 pairs per nucleotide">2AVY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QAL/index.html" title="Resolution 3.2A, 1530 nucleotides, 678 basepairs, 0.4431 pairs per nucleotide">2QAL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBD/index.html" title="Resolution 3.3A, 1530 nucleotides, 677 basepairs, 0.4425 pairs per nucleotide">2QBD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DF3/index.html" title="Resolution 3.5A, 1530 nucleotides, 675 basepairs, 0.4412 pairs per nucleotide">3DF3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1M/index.html" title="Resolution 3.2A, 1533 nucleotides, 674 basepairs, 0.4397 pairs per nucleotide">3I1M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBB/index.html" title="Resolution 3.5A, 1530 nucleotides, 671 basepairs, 0.4386 pairs per nucleotide">2QBB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DF1/index.html" title="Resolution 3.5A, 1530 nucleotides, 671 basepairs, 0.4386 pairs per nucleotide">3DF1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z4M/index.html" title="Resolution 4.5A, 1530 nucleotides, 671 basepairs, 0.4386 pairs per nucleotide">2Z4M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VS5/index.html" title="Resolution 3.5A, 1530 nucleotides, 670 basepairs, 0.4379 pairs per nucleotide">1VS5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QB9/index.html" title="Resolution 3.5A, 1530 nucleotides, 669 basepairs, 0.4373 pairs per nucleotide">2QB9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QP0/index.html" title="Resolution 3.5A, 1530 nucleotides, 667 basepairs, 0.4359 pairs per nucleotide">2QP0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBH/index.html" title="Resolution 4.0A, 1530 nucleotides, 667 basepairs, 0.4359 pairs per nucleotide">2QBH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFX/index.html" title="Resolution 3.3A, 1533 nucleotides, 668 basepairs, 0.4357 pairs per nucleotide">3OFX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z4K/index.html" title="Resolution 4.5A, 1530 nucleotides, 666 basepairs, 0.4353 pairs per nucleotide">2Z4K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBJ/index.html" title="Resolution 4.0A, 1530 nucleotides, 665 basepairs, 0.4346 pairs per nucleotide">2QBJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OAQ/index.html" title="Resolution 3.2A, 1533 nucleotides, 666 basepairs, 0.4344 pairs per nucleotide">3OAQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOY/index.html" title="Resolution 3.5A, 1530 nucleotides, 662 basepairs, 0.4327 pairs per nucleotide">2QOY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OR9/index.html" title="Resolution 3.3A, 1533 nucleotides, 663 basepairs, 0.4325 pairs per nucleotide">3OR9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFA/index.html" title="Resolution 3.2A, 1533 nucleotides, 661 basepairs, 0.4312 pairs per nucleotide">3OFA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOW/index.html" title="Resolution 3.9A, 1530 nucleotides, 656 basepairs, 0.4288 pairs per nucleotide">2QOW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1Q/index.html" title="Resolution 3.8A, 1556 nucleotides, 666 basepairs, 0.4280 pairs per nucleotide">3I1Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I2P/index.html" title="Resolution 3.2A, 1553 nucleotides, 662 basepairs, 0.4263 pairs per nucleotide">2I2P</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1Z/index.html" title="Resolution 3.7A, 1573 nucleotides, 670 basepairs, 0.4259 pairs per nucleotide">3I1Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I2U/index.html" title="Resolution 3.2A, 1553 nucleotides, 660 basepairs, 0.4250 pairs per nucleotide">2I2U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1O/index.html" title="Resolution 3.2A, 1530 nucleotides, 647 basepairs, 0.4229 pairs per nucleotide">3I1O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QOU/index.html" title="Resolution 3.9A, 1530 nucleotides, 646 basepairs, 0.4222 pairs per nucleotide">2QOU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I21/index.html" title="Resolution 3.7A, 1570 nucleotides, 662 basepairs, 0.4217 pairs per nucleotide">3I21</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I1S/index.html" title="Resolution 3.8A, 1553 nucleotides, 654 basepairs, 0.4211 pairs per nucleotide">3I1S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFO/index.html" title="Resolution 3.1A, 1533 nucleotides, 644 basepairs, 0.4201 pairs per nucleotide">3OFO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFB/index.html" title="Resolution 3.2A, 1530 nucleotides, 640 basepairs, 0.4183 pairs per nucleotide">3OFB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OAR/index.html" title="Resolution 3.2A, 1530 nucleotides, 634 basepairs, 0.4144 pairs per nucleotide">3OAR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ORA/index.html" title="Resolution 3.3A, 1530 nucleotides, 633 basepairs, 0.4137 pairs per nucleotide">3ORA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R8O/index.html" title="Resolution 3.0A, 1631 nucleotides, 674 basepairs, 0.4132 pairs per nucleotide">3R8O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFY/index.html" title="Resolution 3.3A, 1530 nucleotides, 625 basepairs, 0.4085 pairs per nucleotide">3OFY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R8N/index.html" title="Resolution 3.0A, 1629 nucleotides, 663 basepairs, 0.4070 pairs per nucleotide">3R8N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WWL/index.html" title="Resolution 5.8A, 1617 nucleotides, 651 basepairs, 0.4026 pairs per nucleotide">2WWL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3J00/index.html" title="Resolution 7.1A, 1630 nucleotides, 655 basepairs, 0.4018 pairs per nucleotide">3J00</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FIH/index.html" title="Resolution 6.7A, 1769 nucleotides, 708 basepairs, 0.4002 pairs per nucleotide">3FIH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OFP/index.html" title="Resolution 3.1A, 1530 nucleotides, 611 basepairs, 0.3993 pairs per nucleotide">3OFP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KC4/index.html" title="Resolution 9.6A, 1530 nucleotides, 596 basepairs, 0.3895 pairs per nucleotide">3KC4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZW/index.html" title="Resolution 10.0A, 1687 nucleotides, 651 basepairs, 0.3859 pairs per nucleotide">3IZW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZV/index.html" title="Resolution 10.0A, 1687 nucleotides, 620 basepairs, 0.3675 pairs per nucleotide">3IZV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GY9/index.html" title="Resolution 15.0A, 1716 nucleotides, 574 basepairs, 0.3345 pairs per nucleotide">2GY9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GYB/index.html" title="Resolution 15.0A, 1716 nucleotides, 517 basepairs, 0.3013 pairs per nucleotide">2GYB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VHO/index.html" title="Resolution 3.7A, 1530 nucleotides, 416 basepairs, 0.2719 pairs per nucleotide">2VHO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VHP/index.html" title="Resolution 3.7A, 1530 nucleotides, 409 basepairs, 0.2673 pairs per nucleotide">2VHP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E1C/index.html" title="Resolution 9.0A, 1776 nucleotides, 390 basepairs, 0.2196 pairs per nucleotide">3E1C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E1A/index.html" title="Resolution 9.0A, 1776 nucleotides, 385 basepairs, 0.2168 pairs per nucleotide">3E1A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AW7/2AW7_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PNX" title="Link to PDB">1PNX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNX/index.html" title="Link to FR3D analysis. 0.4312 pairs per nucleotide">1PNX</a></td><td align="center">A</td><td align="center">1533</td><td align="center">661</td><td align="center">9.5</td><td sorttable_customkey="20040307000000">7-15-03</td><td>ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF THE WILD TYPE RIBOSOME FROM E. COLI, 30S SUBUNIT OF 70S RIBOSOME. THIS FILE, 1PNX, CONTAINS ONLY MOLECULES OF THE 30S RIBOSOMAL SUBUNIT. THE 50S SUBUNIT IS IN THE PDB FILE 1PNY.</td><td>A.VILA-SANJURJO,W.K.RIDGEWAY,V.SEYMANER,W.ZHANG,S.SANTOSO, K.YU,J.H.D.CATE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNS/index.html" title="Resolution 8.7A, 1683 nucleotides, 719 basepairs, 0.4272 pairs per nucleotide">1PNS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOS/index.html" title="Resolution 11.5A, 1526 nucleotides, 390 basepairs, 0.2556 pairs per nucleotide">1VOS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOX/index.html" title="Resolution 11.5A, 1526 nucleotides, 388 basepairs, 0.2543 pairs per nucleotide">1VOX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOQ/index.html" title="Resolution 11.5A, 1526 nucleotides, 388 basepairs, 0.2543 pairs per nucleotide">1VOQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOV/index.html" title="Resolution 11.5A, 1526 nucleotides, 386 basepairs, 0.2529 pairs per nucleotide">1VOV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOZ/index.html" title="Resolution 11.5A, 1526 nucleotides, 386 basepairs, 0.2529 pairs per nucleotide">1VOZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PNX/1PNX_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FJG" title="Link to PDB">1FJG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FJG/index.html" title="Link to FR3D analysis. 0.4560 pairs per nucleotide">1FJG</a></td><td align="center">AX</td><td align="center">1513</td><td align="center">690</td><td align="center">3.0</td><td sorttable_customkey="20020109000000">9-25-00</td><td>THERMUS THERMOPHILUS,THERMUS THERMOPHILUS</td><td>STRUCTURE OF THE THERMUS THERMOPHILUS 30S RIBOSOMAL SUBUNIT IN COMPLEX WITH THE ANTIBIOTICS STREPTOMYCIN, SPECTINOMYCIN, AND PAROMOMYCIN</td><td>A.P.CARTER,W.M.CLEMONS JR.,D.E.BRODERSEN,B.T.WIMBERLY, R.J.MORGAN-WARREN,V.RAMAKRISHNAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J5E/index.html" title="Resolution 3.0A, 1513 nucleotides, 690 basepairs, 0.4560 pairs per nucleotide">1J5E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UUA/index.html" title="Resolution 2.9A, 1528 nucleotides, 691 basepairs, 0.4522 pairs per nucleotide">2UUA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IBL/index.html" title="Resolution 3.1A, 1532 nucleotides, 688 basepairs, 0.4491 pairs per nucleotide">1IBL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N32/index.html" title="Resolution 3.0A, 1531 nucleotides, 687 basepairs, 0.4487 pairs per nucleotide">1N32</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HR0/index.html" title="Resolution 3.2A, 1513 nucleotides, 678 basepairs, 0.4481 pairs per nucleotide">1HR0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UUC/index.html" title="Resolution 3.1A, 1527 nucleotides, 684 basepairs, 0.4479 pairs per nucleotide">2UUC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UU9/index.html" title="Resolution 3.1A, 1528 nucleotides, 684 basepairs, 0.4476 pairs per nucleotide">2UU9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VQE/index.html" title="Resolution 2.5A, 1522 nucleotides, 681 basepairs, 0.4474 pairs per nucleotide">2VQE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UUB/index.html" title="Resolution 2.9A, 1526 nucleotides, 681 basepairs, 0.4463 pairs per nucleotide">2UUB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HNX/index.html" title="Resolution 3.4A, 1513 nucleotides, 671 basepairs, 0.4435 pairs per nucleotide">1HNX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HNW/index.html" title="Resolution 3.4A, 1513 nucleotides, 670 basepairs, 0.4428 pairs per nucleotide">1HNW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VQF/index.html" title="Resolution 2.9A, 1523 nucleotides, 674 basepairs, 0.4425 pairs per nucleotide">2VQF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UXC/index.html" title="Resolution 2.9A, 1525 nucleotides, 671 basepairs, 0.4400 pairs per nucleotide">2UXC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IBK/index.html" title="Resolution 3.3A, 1512 nucleotides, 665 basepairs, 0.4398 pairs per nucleotide">1IBK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HNZ/index.html" title="Resolution 3.3A, 1513 nucleotides, 663 basepairs, 0.4382 pairs per nucleotide">1HNZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IBM/index.html" title="Resolution 3.3A, 1528 nucleotides, 667 basepairs, 0.4365 pairs per nucleotide">1IBM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XNR/index.html" title="Resolution 3.1A, 1521 nucleotides, 662 basepairs, 0.4352 pairs per nucleotide">1XNR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XMQ/index.html" title="Resolution 3.0A, 1521 nucleotides, 662 basepairs, 0.4352 pairs per nucleotide">1XMQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XNQ/index.html" title="Resolution 3.0A, 1521 nucleotides, 660 basepairs, 0.4339 pairs per nucleotide">1XNQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E5L/index.html" title="Resolution 3.3A, 1527 nucleotides, 659 basepairs, 0.4316 pairs per nucleotide">2E5L</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZM6/index.html" title="Resolution 3.3A, 1506 nucleotides, 649 basepairs, 0.4309 pairs per nucleotide">2ZM6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYN/index.html" title="Resolution 3.5A, 1541 nucleotides, 664 basepairs, 0.4309 pairs per nucleotide">3PYN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYQ/index.html" title="Resolution 3.5A, 1541 nucleotides, 663 basepairs, 0.4302 pairs per nucleotide">3PYQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HHH/index.html" title="Resolution 3.4A, 1505 nucleotides, 643 basepairs, 0.4272 pairs per nucleotide">2HHH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F1E/index.html" title="Resolution 3.0A, 1668 nucleotides, 709 basepairs, 0.4251 pairs per nucleotide">3F1E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D5A/index.html" title="Resolution 3.2A, 1670 nucleotides, 709 basepairs, 0.4246 pairs per nucleotide">3D5A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UXD/index.html" title="Resolution 3.2A, 1496 nucleotides, 635 basepairs, 0.4245 pairs per nucleotide">2UXD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XMO/index.html" title="Resolution 3.2A, 1519 nucleotides, 644 basepairs, 0.4240 pairs per nucleotide">1XMO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F1G/index.html" title="Resolution 3.0A, 1668 nucleotides, 707 basepairs, 0.4239 pairs per nucleotide">3F1G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D5C/index.html" title="Resolution 3.2A, 1670 nucleotides, 707 basepairs, 0.4234 pairs per nucleotide">3D5C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N33/index.html" title="Resolution 3.4A, 1527 nucleotides, 641 basepairs, 0.4198 pairs per nucleotide">1N33</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MR8/index.html" title="Resolution 3.6A, 1588 nucleotides, 664 basepairs, 0.4181 pairs per nucleotide">3MR8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I8H/index.html" title="Resolution 3.1A, 1770 nucleotides, 739 basepairs, 0.4175 pairs per nucleotide">3I8H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MS0/index.html" title="Resolution 3.6A, 1590 nucleotides, 663 basepairs, 0.4170 pairs per nucleotide">3MS0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y10/index.html" title="Resolution 3.1A, 1746 nucleotides, 728 basepairs, 0.4170 pairs per nucleotide">2Y10</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UXB/index.html" title="Resolution 3.1A, 1524 nucleotides, 634 basepairs, 0.4160 pairs per nucleotide">2UXB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I96/index.html" title="Resolution 4.2A, 1514 nucleotides, 629 basepairs, 0.4155 pairs per nucleotide">1I96</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I8G/index.html" title="Resolution 3.1A, 1771 nucleotides, 734 basepairs, 0.4145 pairs per nucleotide">3I8G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y18/index.html" title="Resolution 3.1A, 1746 nucleotides, 723 basepairs, 0.4141 pairs per nucleotide">2Y18</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0W/index.html" title="Resolution 3.1A, 1746 nucleotides, 718 basepairs, 0.4112 pairs per nucleotide">2Y0W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0U/index.html" title="Resolution 3.1A, 1746 nucleotides, 714 basepairs, 0.4089 pairs per nucleotide">2Y0U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HUY/index.html" title="Resolution 3.1A, 1586 nucleotides, 648 basepairs, 0.4086 pairs per nucleotide">3HUY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y0Y/index.html" title="Resolution 3.1A, 1740 nucleotides, 710 basepairs, 0.4080 pairs per nucleotide">2Y0Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRK/index.html" title="Resolution 3.6A, 1667 nucleotides, 680 basepairs, 0.4079 pairs per nucleotide">2WRK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XSY/index.html" title="Resolution 7.8A, 1592 nucleotides, 649 basepairs, 0.4077 pairs per nucleotide">2XSY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNN/index.html" title="Resolution 3.5A, 1729 nucleotides, 704 basepairs, 0.4072 pairs per nucleotide">3KNN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XQD/index.html" title="Resolution 3.1A, 1737 nucleotides, 707 basepairs, 0.4070 pairs per nucleotide">2XQD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I94/index.html" title="Resolution 3.2A, 1514 nucleotides, 616 basepairs, 0.4069 pairs per nucleotide">1I94</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I9D/index.html" title="Resolution 3.5A, 1692 nucleotides, 688 basepairs, 0.4066 pairs per nucleotide">3I9D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNL/index.html" title="Resolution 3.5A, 1730 nucleotides, 703 basepairs, 0.4064 pairs per nucleotide">3KNL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XUY/index.html" title="Resolution 7.6A, 1592 nucleotides, 646 basepairs, 0.4058 pairs per nucleotide">2XUY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I9B/index.html" title="Resolution 3.5A, 1694 nucleotides, 685 basepairs, 0.4044 pairs per nucleotide">3I9B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRI/index.html" title="Resolution 3.6A, 1667 nucleotides, 674 basepairs, 0.4043 pairs per nucleotide">2WRI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y14/index.html" title="Resolution 3.1A, 1740 nucleotides, 703 basepairs, 0.4040 pairs per nucleotide">2Y14</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J02/index.html" title="Resolution 2.8A, 1686 nucleotides, 677 basepairs, 0.4015 pairs per nucleotide">2J02</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J00/index.html" title="Resolution 2.8A, 1686 nucleotides, 676 basepairs, 0.4009 pairs per nucleotide">2J00</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y16/index.html" title="Resolution 3.1A, 1740 nucleotides, 697 basepairs, 0.4006 pairs per nucleotide">2Y16</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIQ/index.html" title="Resolution 3.3A, 1666 nucleotides, 666 basepairs, 0.3998 pairs per nucleotide">3KIQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OGE/index.html" title="Resolution 3.0A, 1504 nucleotides, 601 basepairs, 0.3996 pairs per nucleotide">3OGE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIS/index.html" title="Resolution 3.3A, 1666 nucleotides, 665 basepairs, 0.3992 pairs per nucleotide">3KIS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FIC/index.html" title="Resolution 6.4A, 1724 nucleotides, 687 basepairs, 0.3985 pairs per nucleotide">3FIC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDM/index.html" title="Resolution 3.5A, 1741 nucleotides, 693 basepairs, 0.3980 pairs per nucleotide">2WDM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDH/index.html" title="Resolution 3.3A, 1741 nucleotides, 693 basepairs, 0.3980 pairs per nucleotide">2WDH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OGY/index.html" title="Resolution 3.0A, 1504 nucleotides, 598 basepairs, 0.3976 pairs per nucleotide">3OGY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDG/index.html" title="Resolution 3.3A, 1741 nucleotides, 692 basepairs, 0.3975 pairs per nucleotide">2WDG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WDK/index.html" title="Resolution 3.5A, 1741 nucleotides, 692 basepairs, 0.3975 pairs per nucleotide">2WDK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X9T/index.html" title="Resolution 3.1A, 1662 nucleotides, 658 basepairs, 0.3959 pairs per nucleotide">2X9T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHC/index.html" title="Resolution 3.0A, 1504 nucleotides, 595 basepairs, 0.3956 pairs per nucleotide">3OHC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHD/index.html" title="Resolution 3.0A, 1504 nucleotides, 593 basepairs, 0.3943 pairs per nucleotide">3OHD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI0/index.html" title="Resolution 3.0A, 1504 nucleotides, 593 basepairs, 0.3943 pairs per nucleotide">3OI0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYU/index.html" title="Resolution 3.4A, 1540 nucleotides, 607 basepairs, 0.3942 pairs per nucleotide">3PYU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X9R/index.html" title="Resolution 3.1A, 1662 nucleotides, 655 basepairs, 0.3941 pairs per nucleotide">2X9R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OHY/index.html" title="Resolution 3.0A, 1504 nucleotides, 591 basepairs, 0.3930 pairs per nucleotide">3OHY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PYS/index.html" title="Resolution 3.4A, 1540 nucleotides, 605 basepairs, 0.3929 pairs per nucleotide">3PYS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I95/index.html" title="Resolution 4.5A, 1514 nucleotides, 592 basepairs, 0.3910 pairs per nucleotide">1I95</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HUW/index.html" title="Resolution 3.1A, 1577 nucleotides, 614 basepairs, 0.3893 pairs per nucleotide">3HUW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F4V/index.html" title="Resolution 3.8A, 1511 nucleotides, 588 basepairs, 0.3891 pairs per nucleotide">2F4V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIX/index.html" title="Resolution 3.6A, 1669 nucleotides, 649 basepairs, 0.3889 pairs per nucleotide">3KIX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KIU/index.html" title="Resolution 3.6A, 1669 nucleotides, 648 basepairs, 0.3883 pairs per nucleotide">3KIU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WH3/index.html" title="Resolution 3.5A, 1664 nucleotides, 645 basepairs, 0.3876 pairs per nucleotide">2WH3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y12/index.html" title="Resolution 3.1A, 1740 nucleotides, 672 basepairs, 0.3862 pairs per nucleotide">2Y12</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WH1/index.html" title="Resolution 3.5A, 1664 nucleotides, 641 basepairs, 0.3852 pairs per nucleotide">2WH1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI2/index.html" title="Resolution 3.1A, 1504 nucleotides, 573 basepairs, 0.3810 pairs per nucleotide">3OI2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OI4/index.html" title="Resolution 3.1A, 1504 nucleotides, 570 basepairs, 0.3790 pairs per nucleotide">3OI4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OTO/index.html" title="Resolution 3.7A, 1513 nucleotides, 571 basepairs, 0.3774 pairs per nucleotide">3OTO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XFZ/index.html" title="Resolution 3.2A, 1676 nucleotides, 615 basepairs, 0.3669 pairs per nucleotide">2XFZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QNH/index.html" title="Resolution 3.8A, 1674 nucleotides, 614 basepairs, 0.3668 pairs per nucleotide">2QNH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNH/index.html" title="Resolution 3.0A, 1732 nucleotides, 626 basepairs, 0.3614 pairs per nucleotide">3KNH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNJ/index.html" title="Resolution 3.1A, 1732 nucleotides, 620 basepairs, 0.3580 pairs per nucleotide">3KNJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V46/index.html" title="Resolution 3.8A, 1608 nucleotides, 574 basepairs, 0.3570 pairs per nucleotide">2V46</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B9O/index.html" title="Resolution 6.5A, 1670 nucleotides, 595 basepairs, 0.3563 pairs per nucleotide">2B9O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B64/index.html" title="Resolution 5.9A, 1670 nucleotides, 595 basepairs, 0.3563 pairs per nucleotide">2B64</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B9M/index.html" title="Resolution 6.8A, 1670 nucleotides, 589 basepairs, 0.3527 pairs per nucleotide">2B9M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V48/index.html" title="Resolution 3.8A, 1608 nucleotides, 566 basepairs, 0.3520 pairs per nucleotide">2V48</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XG1/index.html" title="Resolution 3.2A, 1669 nucleotides, 577 basepairs, 0.3457 pairs per nucleotide">2XG1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OW8/index.html" title="Resolution 3.7A, 1638 nucleotides, 565 basepairs, 0.3449 pairs per nucleotide">2OW8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGI/index.html" title="Resolution 5.0A, 1667 nucleotides, 574 basepairs, 0.3443 pairs per nucleotide">2HGI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N34/index.html" title="Resolution 3.8A, 1517 nucleotides, 520 basepairs, 0.3428 pairs per nucleotide">1N34</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRQ/index.html" title="Resolution 3.6A, 1738 nucleotides, 589 basepairs, 0.3389 pairs per nucleotide">2WRQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGR/index.html" title="Resolution 4.5A, 1694 nucleotides, 571 basepairs, 0.3371 pairs per nucleotide">2HGR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WRN/index.html" title="Resolution 3.6A, 1738 nucleotides, 580 basepairs, 0.3337 pairs per nucleotide">2WRN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N36/index.html" title="Resolution 3.6A, 1513 nucleotides, 502 basepairs, 0.3318 pairs per nucleotide">1N36</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGP/index.html" title="Resolution 5.5A, 1793 nucleotides, 585 basepairs, 0.3263 pairs per nucleotide">2HGP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YL4/index.html" title="Resolution 5.5A, 1733 nucleotides, 552 basepairs, 0.3185 pairs per nucleotide">1YL4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I97/index.html" title="Resolution 4.5A, 1514 nucleotides, 436 basepairs, 0.2880 pairs per nucleotide">1I97</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FKA/index.html" title="Resolution 3.3A, 1174 nucleotides, 300 basepairs, 0.2555 pairs per nucleotide">1FKA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FJG/1FJG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZ7" title="Link to PDB">3IZ7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZ7/index.html" title="Link to FR3D analysis. 0.3008 pairs per nucleotide">3IZ7</a></td><td align="center">ABC</td><td align="center">1586</td><td align="center">477</td><td align="center">5.5</td><td sorttable_customkey="20100112000000">12-01-10</td><td>TRITICUM AESTIVUM,TRITICUM AESTIVUM,TRITICUM AESTIVUM</td><td>MODEL OF THE SMALL SUBUNIT RNA BASED ON A 5.5 A CRYO-EM MAP OF TRITICUM AESTIVUM TRANSLATING 80S RIBOSOME</td><td>J.-P.ARMACHE,A.JARASCH,A.M.ANGER,E.VILLA,T.BECKER,S.BHUSHAN, F.JOSSINET,M.HABECK,G.DINDAR,S.FRANCKENBERG,V.MARQUEZ,T.MIELKE, M.THOMM,O.BERNINGHAUSEN,B.BEATRIX,J.SOEDING,E.WESTHOF,D.N.WILSON, R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BBN" title="Link to PDB">3BBN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBN/index.html" title="Link to FR3D analysis. 0.3681 pairs per nucleotide">3BBN</a></td><td align="center">A</td><td align="center">1478</td><td align="center">544</td><td align="center">9.4</td><td sorttable_customkey="20090105000000">5-13-08</td><td>SPINACEA OLERACEA</td><td>HOMOLOGY MODEL FOR THE SPINACH CHLOROPLAST 30S SUBUNIT FITTED TO 9.4A CRYO-EM MAP OF THE 70S CHLORORIBOSOME.</td><td>M.R.SHARMA,D.N.WILSON,P.P.DATTA,C.BARAT,F.SCHLUENZEN, P.FUCINI,R.K.AGRAWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S1H" title="Link to PDB">1S1H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S1H/index.html" title="Link to FR3D analysis. 0.4270 pairs per nucleotide">1S1H</a></td><td align="center">A</td><td align="center">1466</td><td align="center">626</td><td align="center">11.7</td><td sorttable_customkey="20060105000000">5-25-04</td><td>SACCHAROMYCES CEREVISIAE</td><td>STRUCTURE OF THE RIBOSOMAL 80S-EEF2-SORDARIN COMPLEX FROM YEAST OBTAINED BY DOCKING ATOMIC MODELS FOR RNA AND PROTEIN COMPONENTS INTO A 11.7 A CRYO-EM MAP. THIS FILE, 1S1H, CONTAINS 40S SUBUNIT. THE 60S RIBOSOMAL SUBUNIT IS IN FILE 1S1I.</td><td>C.M.SPAHN,M.G.GOMEZ-LORENZO,R.A.GRASSUCCI,R.JORGENSEN, G.R.ANDERSEN,R.BECKMANN,P.A.PENCZEK,J.P.G.BALLESTA,J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1H1K" title="Link to PDB">1H1K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H1K/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1H1K</a></td><td align="center">IJKLMN</td><td align="center">1906</td><td align="center">953</td><td align="center">10.0</td><td sorttable_customkey="20040209000000">9-26-02</td><td>BLUETONGUE VIRUS,BLUETONGUE VIRUS,BLUETONGUE VIRUS,BLUETONGUE VIRUS,BLUETONGUE VIRUS,BLUETONGUE VIRUS</td><td>THE BLUETONGUE VIRUS (BTV) CORE BINDS DSRNA</td><td>J.M.DIPROSE,J.M.GRIMES,G.C.SUTTON,J.N.BURROUGHS,A.MEYER, S.MAAN,P.P.C.MERTENS,D.I.STUART</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3G78" title="Link to PDB">3G78</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G78/index.html" title="Link to FR3D analysis. 0.4548 pairs per nucleotide">3G78</a></td><td align="center">AZ</td><td align="center">398</td><td align="center">181</td><td align="center">2.8</td><td sorttable_customkey="20110402000000">2-16-10</td><td>OCEANOBACILLUS IHEYENSIS,synthetic</td><td>INSIGHT INTO GROUP II INTRON CATALYSIS FROM REVISED CRYSTAL STRUCTURE</td><td>J.WANG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IGI/index.html" title="Resolution 3.1A, 395 nucleotides, 179 basepairs, 0.4532 pairs per nucleotide">3IGI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BWP/index.html" title="Resolution 3.1A, 349 nucleotides, 154 basepairs, 0.4413 pairs per nucleotide">3BWP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EOH/index.html" title="Resolution 3.1A, 381 nucleotides, 164 basepairs, 0.4304 pairs per nucleotide">3EOH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EOG/index.html" title="Resolution 3.4A, 388 nucleotides, 160 basepairs, 0.4124 pairs per nucleotide">3EOG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G78/3G78_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZ4" title="Link to PDB">3IZ4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZ4/index.html" title="Link to FR3D analysis. 0.2785 pairs per nucleotide">3IZ4</a></td><td align="center">A</td><td align="center">377</td><td align="center">105</td><td align="center">13.6</td><td sorttable_customkey="20110810000000">10-20-10</td><td>ESCHERICHIA COLI</td><td>MODIFIED E. COLI TMRNA IN THE RESUME STATE WITH THE TRNA-LIKE DOMAIN IN THE RIBOSOMAL P SITE INTERACTING WITH THE SMPB</td><td>Y.HASHEM,J.FU,J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IYR" title="Link to PDB">3IYR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IYR/index.html" title="Link to FR3D analysis. 0.1547 pairs per nucleotide">3IYR</a></td><td align="center">A</td><td align="center">349</td><td align="center">54</td><td align="center">13.0</td><td sorttable_customkey="20110810000000">10-20-10</td><td>THERMUS THERMOPHILUS</td><td>TMRNA-SMPB: A JOURNEY TO THE CENTER OF THE BACTERIAL RIBOSOME</td><td>F.WEIS,P.BRON,E.GIUDICE,J.P.ROLLAND,D.THOMAS,B.FELDEN,R.GILLET</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IYQ" title="Link to PDB">3IYQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IYQ/index.html" title="Link to FR3D analysis. 0.1662 pairs per nucleotide">3IYQ</a></td><td align="center">A</td><td align="center">349</td><td align="center">58</td><td align="center">13.0</td><td sorttable_customkey="20110810000000">10-20-10</td><td>THERMUS THERMOPHILUS</td><td>TMRNA-SMPB: A JOURNEY TO THE CENTER OF THE BACTERIAL RIBOSOME</td><td>F.WEIS,P.BRON,E.GIUDICE,J.P.ROLLAND,D.THOMAS,B.FELDEN,R.GILLET</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q1Q" title="Link to PDB">3Q1Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q1Q/index.html" title="Link to FR3D analysis. 0.4043 pairs per nucleotide">3Q1Q</a></td><td align="center">BC</td><td align="center">423</td><td align="center">171</td><td align="center">3.8</td><td sorttable_customkey="20110903000000">3-09-11</td><td>synthetic,synthetic</td><td>STRUCTURE OF A BACTERIAL RIBONUCLEASE P HOLOENZYME IN COMPLEX WITH TRNA</td><td>N.J.REITER,A.OSTERMAN,A.TORRES-LARIOS,K.K.SWINGER,T.PAN,A.MONDRAGON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q1R/index.html" title="Resolution 4.2A, 423 nucleotides, 169 basepairs, 0.3995 pairs per nucleotide">3Q1R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A2E/index.html" title="Resolution 3.9A, 266 nucleotides, 100 basepairs, 0.3759 pairs per nucleotide">2A2E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q1Q/3Q1Q_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A64" title="Link to PDB">2A64</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A64/index.html" title="Link to FR3D analysis. 0.3926 pairs per nucleotide">2A64</a></td><td align="center">A</td><td align="center">298</td><td align="center">117</td><td align="center">3.3</td><td sorttable_customkey="20060809000000">9-20-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF BACTERIAL RIBONUCLEASE P RNA</td><td>A.V.KAZANTSEV,A.A.KRIVENKO,D.J.HARRINGTON,S.R.HOLBROOK, P.D.ADAMS,N.R.PACE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2J37" title="Link to PDB">2J37</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J37/index.html" title="Link to FR3D analysis. 0.4020 pairs per nucleotide">2J37</a></td><td align="center">AZ</td><td align="center">408</td><td align="center">164</td><td align="center">8.0</td><td sorttable_customkey="20060811000000">11-08-06</td><td>HALOARCULA MARISMORTUI,TRITICUM SP.</td><td>MODEL OF MAMMALIAN SRP BOUND TO 80S RNCS</td><td>M.HALIC,M.BLAU,T.BECKER,T.MIELKE,M.R.POOL,K.WILD,I.SINNING, R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1X8W" title="Link to PDB">1X8W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1X8W/index.html" title="Link to FR3D analysis. 0.3926 pairs per nucleotide">1X8W</a></td><td align="center">A</td><td align="center">242</td><td align="center">95</td><td align="center">3.8</td><td sorttable_customkey="20040911000000">11-09-04</td><td>synthetic</td><td>STRUCTURE OF THE TETRAHYMENA RIBOZYME: BASE TRIPLE SANDWICH AND METAL ION AT THE ACTIVE SITE</td><td>F.GUO,A.R.GOODING,T.R.CECH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GRZ/index.html" title="Resolution 5.0A, 247 nucleotides, 90 basepairs, 0.3644 pairs per nucleotide">1GRZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1X8W/1X8W_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RKJ" title="Link to PDB">2RKJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RKJ/index.html" title="Link to FR3D analysis. 0.3285 pairs per nucleotide">2RKJ</a></td><td align="center">CDGH</td><td align="center">968</td><td align="center">318</td><td align="center">4.5</td><td sorttable_customkey="20080801000000">1-08-08</td><td>NEUROSPORA CRASSA,STAPHYLOCOCCUS PHAGE TWORT</td><td>COCRYSTAL STRUCTURE OF A TYROSYL-TRNA SYNTHETASE SPLICING FACTOR WITH A GROUP I INTRON RNA</td><td>P.J.PAUKSTELIS,J.-H.CHEN,E.CHASE,A.M.LAMBOWITZ,B.L.GOLDEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Y0Q" title="Link to PDB">1Y0Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y0Q/index.html" title="Link to FR3D analysis. 0.3820 pairs per nucleotide">1Y0Q</a></td><td align="center">AB</td><td align="center">233</td><td align="center">89</td><td align="center">3.6</td><td sorttable_customkey="20050912000000">12-21-04</td><td>STAPHYLOCOCCUS PHAGE TWORT,synthetic</td><td>CRYSTAL STRUCTURE OF AN ACTIVE GROUP I RIBOZYME-PRODUCT COMPLEX</td><td>B.L.GOLDEN,H.KIM,E.CHASE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DHS" title="Link to PDB">3DHS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DHS/index.html" title="Link to FR3D analysis. 0.3721 pairs per nucleotide">3DHS</a></td><td align="center">A</td><td align="center">215</td><td align="center">80</td><td align="center">3.6</td><td sorttable_customkey="20110301000000">1-27-09</td><td>synthetic</td><td>MAPPING METAL-BINDING SITES IN THE CATALYTIC DOMAIN OF BACTERIAL RNASE P RNA</td><td>N.R.PACE,A.V.KAZANTSEV,A.A.KRIVENKO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U6B" title="Link to PDB">1U6B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U6B/index.html" title="Link to FR3D analysis. 0.4932 pairs per nucleotide">1U6B</a></td><td align="center">BCD</td><td align="center">219</td><td align="center">108</td><td align="center">3.1</td><td sorttable_customkey="20041008000000">8-10-04</td><td>synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A SELF-SPLICING GROUP I INTRON WITH BOTH EXONS</td><td>P.L.ADAMS,M.R.STAHLEY,A.B.KOSEK,J.WANG,S.A.STROBEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO2/index.html" title="Resolution 3.3A, 220 nucleotides, 102 basepairs, 0.4636 pairs per nucleotide">3BO2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO3/index.html" title="Resolution 3.4A, 220 nucleotides, 97 basepairs, 0.4409 pairs per nucleotide">3BO3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO4/index.html" title="Resolution 3.3A, 218 nucleotides, 96 basepairs, 0.4404 pairs per nucleotide">3BO4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IIN/index.html" title="Resolution 4.2A, 216 nucleotides, 95 basepairs, 0.4398 pairs per nucleotide">3IIN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZZN/index.html" title="Resolution 3.4A, 219 nucleotides, 89 basepairs, 0.4064 pairs per nucleotide">1ZZN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U6B/1U6B_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NOQ" title="Link to PDB">2NOQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NOQ/index.html" title="Link to FR3D analysis. 0.3628 pairs per nucleotide">2NOQ</a></td><td align="center">ABCDE</td><td align="center">317</td><td align="center">115</td><td align="center">7.3</td><td sorttable_customkey="20070911000000">11-21-06</td><td>synthetic,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>STRUCTURE OF RIBOSOME-BOUND CRICKET PARALYSIS VIRUS IRES RNA</td><td>M.SCHULER,S.R.CONNELL,A.LESCOUTE,J.GIESEBRECHT,M.DABROWSKI, B.SCHROEER,T.MIELKE,P.A.PENCZEK,E.WESTHOF,C.M.T.SPAHN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DIR" title="Link to PDB">3DIR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIR/index.html" title="Link to FR3D analysis. 0.4709 pairs per nucleotide">3DIR</a></td><td align="center">A</td><td align="center">172</td><td align="center">81</td><td align="center">2.9</td><td sorttable_customkey="20090409000000">9-16-08</td><td>synthetic</td><td>CRYSTALLIZATION OF THE THERMOTOGA MARITIMA LYSINE RIBOSWITCH BOUND TO N6-1-IMINOETHYL-L-LYSINE</td><td>A.A.SERGANOV</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIL/index.html" title="Resolution 1.9A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DIL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DJ2/index.html" title="Resolution 2.5A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DJ2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIQ/index.html" title="Resolution 2.7A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DIQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIX/index.html" title="Resolution 2.9A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DIX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIG/index.html" title="Resolution 2.8A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DIG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DJ0/index.html" title="Resolution 2.5A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3DJ0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIS/index.html" title="Resolution 3.1A, 173 nucleotides, 80 basepairs, 0.4624 pairs per nucleotide">3DIS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIM/index.html" title="Resolution 2.9A, 173 nucleotides, 80 basepairs, 0.4624 pairs per nucleotide">3DIM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIY/index.html" title="Resolution 2.7A, 172 nucleotides, 79 basepairs, 0.4593 pairs per nucleotide">3DIY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIZ/index.html" title="Resolution 2.9A, 173 nucleotides, 79 basepairs, 0.4566 pairs per nucleotide">3DIZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIO/index.html" title="Resolution 2.4A, 173 nucleotides, 79 basepairs, 0.4566 pairs per nucleotide">3DIO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DIR/3DIR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P49" title="Link to PDB">3P49</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P49/index.html" title="Link to FR3D analysis. 0.2840 pairs per nucleotide">3P49</a></td><td align="center">A</td><td align="center">169</td><td align="center">48</td><td align="center">3.5</td><td sorttable_customkey="20110604000000">4-06-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A GLYCINE RIBOSWITCH FROM FUSOBACTERIUM NUCLEATUM</td><td>E.B.BUTLER,J.WANG,Y.XIONG,S.STROBEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3D0U" title="Link to PDB">3D0U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D0U/index.html" title="Link to FR3D analysis. 0.4720 pairs per nucleotide">3D0U</a></td><td align="center">A</td><td align="center">161</td><td align="center">76</td><td align="center">2.8</td><td sorttable_customkey="20080107000000">7-01-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF LYSINE RIBOSWITCH BOUND TO LYSINE</td><td>A.D.GARST,A.HEROUX,R.P.RAMBO,R.T.BATEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D0X/index.html" title="Resolution 3.0A, 161 nucleotides, 72 basepairs, 0.4472 pairs per nucleotide">3D0X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D0U/3D0U_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QBZ" title="Link to PDB">2QBZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBZ/index.html" title="Link to FR3D analysis. 0.4771 pairs per nucleotide">2QBZ</a></td><td align="center">X</td><td align="center">153</td><td align="center">73</td><td align="center">2.6</td><td sorttable_customkey="20080609000000">9-18-07</td><td>synthetic</td><td>STRUCTURE OF THE M-BOX RIBOSWITCH APTAMER DOMAIN</td><td>C.E.DANN III,W.C.WINKLER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PDR/index.html" title="Resolution 1.9A, 322 nucleotides, 148 basepairs, 0.4596 pairs per nucleotide">3PDR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QBZ/2QBZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GID" title="Link to PDB">1GID</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GID/index.html" title="Link to FR3D analysis. 0.4842 pairs per nucleotide">1GID</a></td><td align="center">AB</td><td align="center">316</td><td align="center">153</td><td align="center">2.5</td><td sorttable_customkey="19980712000000">12-31-96</td><td>SYNTHETIC CONSTRUCT</td><td>CRYSTAL STRUCTURE OF A GROUP I RIBOZYME DOMAIN: PRINCIPLES OF RNA PACKING</td><td>J.H.CATE,A.R.GOODING,E.PODELL,K.ZHOU,B.L.GOLDEN,C.E.KUNDROT, T.R.CECH,J.A.DOUDNA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HR2/index.html" title="Resolution 2.2A, 315 nucleotides, 149 basepairs, 0.4730 pairs per nucleotide">1HR2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R8S/index.html" title="Resolution 1.9A, 159 nucleotides, 73 basepairs, 0.4591 pairs per nucleotide">2R8S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L8V/index.html" title="Resolution 2.8A, 314 nucleotides, 136 basepairs, 0.4331 pairs per nucleotide">1L8V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GID/1GID_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U9S" title="Link to PDB">1U9S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U9S/index.html" title="Link to FR3D analysis. 0.4710 pairs per nucleotide">1U9S</a></td><td align="center">A</td><td align="center">155</td><td align="center">73</td><td align="center">2.9</td><td sorttable_customkey="20060210000000">10-26-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE SPECIFICITY DOMAIN OF RIBONUCLEASE P OF THE A-TYPE</td><td>A.S.KRASILNIKOV,Y.XIAO,T.PAN,A.MONDRAGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NBS" title="Link to PDB">1NBS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NBS/index.html" title="Link to FR3D analysis. 0.4222 pairs per nucleotide">1NBS</a></td><td align="center">AB</td><td align="center">270</td><td align="center">114</td><td align="center">3.1</td><td sorttable_customkey="20040602000000">2-18-03</td><td>BACILLUS SUBTILIS</td><td>CRYSTAL STRUCTURE OF THE SPECIFICITY DOMAIN OF RIBONUCLEASE P RNA</td><td>A.S.KRASILNIKOV,X.YANG,T.PAN,A.MONDRAGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NZ4" title="Link to PDB">2NZ4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NZ4/index.html" title="Link to FR3D analysis. 0.4007 pairs per nucleotide">2NZ4</a></td><td align="center">GHRS</td><td align="center">594</td><td align="center">238</td><td align="center">2.5</td><td sorttable_customkey="20080401000000">1-16-07</td><td>synthetic,synthetic</td><td>STRUCTURAL INVESTIGATION OF THE GLMS RIBOZYME BOUND TO ITS CATALYTIC COFACTOR</td><td>J.C.COCHRANE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G96/index.html" title="Resolution 3.0A, 595 nucleotides, 233 basepairs, 0.3916 pairs per nucleotide">3G96</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3L3C/index.html" title="Resolution 2.9A, 595 nucleotides, 232 basepairs, 0.3899 pairs per nucleotide">3L3C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G8T/index.html" title="Resolution 3.0A, 151 nucleotides, 57 basepairs, 0.3775 pairs per nucleotide">3G8T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G9C/index.html" title="Resolution 2.9A, 601 nucleotides, 223 basepairs, 0.3710 pairs per nucleotide">3G9C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G8S/index.html" title="Resolution 3.1A, 594 nucleotides, 212 basepairs, 0.3569 pairs per nucleotide">3G8S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NZ4/2NZ4_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZD" title="Link to PDB">3IZD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZD/index.html" title="Link to FR3D analysis. 0.4275 pairs per nucleotide">3IZD</a></td><td align="center">A</td><td align="center">138</td><td align="center">59</td><td align="center">10.0</td><td sorttable_customkey="20100112000000">12-01-10</td><td>SACCHAROMYCES CEREVISIAE</td><td>MODEL OF THE LARGE SUBUNIT RNA EXPANSION SEGMENT ES27L-OUT BASED ON A 6.1 A CRYO-EM MAP OF SACCHAROMYCES CEREVISIAE TRANSLATING 80S RIBOSOME. 3IZD IS A SMALL PART (AN EXPANSION SEGMENT) WHICH IS IN AN ALTERNATIVE CONFORMATION TO WHAT IS IN ALREADY 3IZF.</td><td>J.-P.ARMACHE,A.JARASCH,A.M.ANGER,E.VILLA,T.BECKER,S.BHUSHAN, F.JOSSINET,M.HABECK,G.DINDAR,S.FRANCKENBERG,V.MARQUEZ,T.MIELKE, M.THOMM,O.BERNINGHAUSEN,B.BEATRIX,J.SOEDING,E.WESTHOF,D.N.WILSON, R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LNG" title="Link to PDB">1LNG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LNG/index.html" title="Link to FR3D analysis. 0.4536 pairs per nucleotide">1LNG</a></td><td align="center">B</td><td align="center">97</td><td align="center">44</td><td align="center">2.3</td><td sorttable_customkey="20020706000000">6-07-02</td><td>METHANOCALDOCOCCUS JANNASCHII</td><td>CRYSTAL STRUCTURE OF THE SRP19-7S.S SRP RNA COMPLEX OF M. JANNASCHII</td><td>T.HAINZL,S.HUANG,A.E.SAUER-ERIKSSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V3C/index.html" title="Resolution 2.5A, 192 nucleotides, 86 basepairs, 0.4479 pairs per nucleotide">2V3C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NDB/index.html" title="Resolution 3.0A, 136 nucleotides, 59 basepairs, 0.4338 pairs per nucleotide">3NDB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z43/index.html" title="Resolution 2.6A, 101 nucleotides, 39 basepairs, 0.3861 pairs per nucleotide">1Z43</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LNG/1LNG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IVK" title="Link to PDB">3IVK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IVK/index.html" title="Link to FR3D analysis. 0.4453 pairs per nucleotide">3IVK</a></td><td align="center">C</td><td align="center">256</td><td align="center">114</td><td align="center">3.1</td><td sorttable_customkey="20100203000000">3-02-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE CATALYTIC CORE OF AN RNA POLYMERASE RIBOZYME COMPLEXED WITH AN ANTIGEN BINDING ANTIBODY FRAGMENT</td><td>Y.KOLDOBSKAYA,E.M.DUGUID,D.M.SHECHNER,S.KOIDE, A.A.KOSSIAKOFF,D.P.BARTEL,J.A.PICCIRILLI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HHN/index.html" title="Resolution 3.0A, 272 nucleotides, 113 basepairs, 0.4154 pairs per nucleotide">3HHN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IVK/3IVK_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IL9" title="Link to PDB">2IL9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IL9/index.html" title="Link to FR3D analysis. 0.1835 pairs per nucleotide">2IL9</a></td><td align="center">AM</td><td align="center">267</td><td align="center">49</td><td align="center">3.1</td><td sorttable_customkey="20080102000000">2-13-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PLAUTIA STALI INTESTINE VIRUS INTERGENIC REGION INTERNAL RIBOSOME ENTRY SITE RIBOSOMAL BINDING DOMAIN RNA AT 3.1 ANGSTROMS</td><td>J.S.PFINGSTEN,D.A.COSTANTINO,J.S.KIEFT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MFQ" title="Link to PDB">1MFQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MFQ/index.html" title="Link to FR3D analysis. 0.4173 pairs per nucleotide">1MFQ</a></td><td align="center">A</td><td align="center">127</td><td align="center">53</td><td align="center">3.1</td><td sorttable_customkey="20030809000000">9-20-02</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE ANALYSIS OF A TERNARY S-DOMAIN COMPLEX OF HUMAN SIGNAL RECOGNITION PARTICLE</td><td>A.KUGLSTATTER,C.OUBRIDGE,K.NAGAI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GO5/index.html" title="Resolution 7.4A, 217 nucleotides, 88 basepairs, 0.4055 pairs per nucleotide">2GO5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KTV/index.html" title="Resolution 3.8A, 215 nucleotides, 80 basepairs, 0.3721 pairs per nucleotide">3KTV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RY1/index.html" title="Resolution 12.0A, 301 nucleotides, 112 basepairs, 0.3721 pairs per nucleotide">1RY1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L9A/index.html" title="Resolution 2.9A, 125 nucleotides, 45 basepairs, 0.3600 pairs per nucleotide">1L9A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MFQ/1MFQ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HO7" title="Link to PDB">2HO7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HO7/index.html" title="Link to FR3D analysis. 0.4539 pairs per nucleotide">2HO7</a></td><td align="center">AB</td><td align="center">141</td><td align="center">64</td><td align="center">2.9</td><td sorttable_customkey="20080209000000">9-26-06</td><td>synthetic,synthetic</td><td>PRE-CLEAVAGE STATE OF THE THERMOANAEROBACTER TENGCONGENSIS GLMS RIBOZYME BOUND TO GLUCOSE-6-PHOSPHATE</td><td>D.J.KLEIN,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z75/index.html" title="Resolution 1.7A, 140 nucleotides, 62 basepairs, 0.4429 pairs per nucleotide">2Z75</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H0X/index.html" title="Resolution 2.3A, 140 nucleotides, 61 basepairs, 0.4357 pairs per nucleotide">2H0X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GCS/index.html" title="Resolution 2.1A, 140 nucleotides, 61 basepairs, 0.4357 pairs per nucleotide">2GCS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B4B/index.html" title="Resolution 2.7A, 141 nucleotides, 61 basepairs, 0.4326 pairs per nucleotide">3B4B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GCV/index.html" title="Resolution 2.1A, 140 nucleotides, 60 basepairs, 0.4286 pairs per nucleotide">2GCV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z74/index.html" title="Resolution 2.2A, 140 nucleotides, 60 basepairs, 0.4286 pairs per nucleotide">2Z74</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H0W/index.html" title="Resolution 2.4A, 140 nucleotides, 60 basepairs, 0.4286 pairs per nucleotide">2H0W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H0S/index.html" title="Resolution 2.4A, 140 nucleotides, 59 basepairs, 0.4214 pairs per nucleotide">2H0S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B4A/index.html" title="Resolution 2.7A, 141 nucleotides, 59 basepairs, 0.4184 pairs per nucleotide">3B4A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HO6/index.html" title="Resolution 2.8A, 140 nucleotides, 58 basepairs, 0.4143 pairs per nucleotide">2HO6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H0Z/index.html" title="Resolution 2.7A, 141 nucleotides, 58 basepairs, 0.4113 pairs per nucleotide">2H0Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B4C/index.html" title="Resolution 3.0A, 139 nucleotides, 55 basepairs, 0.3957 pairs per nucleotide">3B4C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HO7/2HO7_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1C2X" title="Link to PDB">1C2X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C2X/index.html" title="Link to FR3D analysis. 0.2917 pairs per nucleotide">1C2X</a></td><td align="center">C</td><td align="center">120</td><td align="center">35</td><td align="center">7.5</td><td sorttable_customkey="20001004000000">4-10-00</td><td>ESCHERICHIA COLI</td><td>S RRNA STRUCTURE FITTED TO A CRYO-ELECTRON MICROSCOPIC MAP AT 7.5 ANGSTROMS RESOLUTION</td><td>R.BRIMACOMBE,F.MUELLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NPB" title="Link to PDB">3NPB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NPB/index.html" title="Link to FR3D analysis. 0.3866 pairs per nucleotide">3NPB</a></td><td align="center">A</td><td align="center">119</td><td align="center">46</td><td align="center">3.0</td><td sorttable_customkey="20100311000000">11-03-10</td><td>BACILLUS SUBTILIS</td><td>TL5 RNA STRUCTURE</td><td>C.LU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZZ" title="Link to PDB">3IZZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZZ/index.html" title="Link to FR3D analysis. 0.3620 pairs per nucleotide">3IZZ</a></td><td align="center">ABDE</td><td align="center">337</td><td align="center">122</td><td align="center">10.8</td><td sorttable_customkey="20130603000000">3-30-11</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>MODELS FOR RIBOSOME COMPONENTS THAT ARE NEAREST NEIGHBORS TO THE BOVINE MITOCHONDRIAL INITIATION FACTOR2 BOUND TO THE E. COLI RIBOSOME</td><td>A.S.YASSIN,E.HAQUE,P.P.DATTA,K.ELMORE,N.K.BANAVALI,L.L.SPREMULLI, R.K.AGRAWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FOQ" title="Link to PDB">1FOQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FOQ/index.html" title="Link to FR3D analysis. 0.3670 pairs per nucleotide">1FOQ</a></td><td align="center">A</td><td align="center">545</td><td align="center">200</td><td align="center">20.0</td><td sorttable_customkey="20011012000000">12-22-00</td><td>BACILLUS PHAGE PHI29</td><td>PENTAMERIC MODEL OF THE BACTERIOPHAGE PHI29 PROHEAD RNA</td><td>A.A.SIMPSON,Y.TAO,P.G.LEIMAN,M.O.BADASSO,Y.HE,P.J.JARDINE, N.H.OLSON,M.C.MORAIS,S.GRIMES,D.L.ANDERSON,T.S.BAKER, M.G.ROSSMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3F4G" title="Link to PDB">3F4G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F4G/index.html" title="Link to FR3D analysis. 0.4679 pairs per nucleotide">3F4G</a></td><td align="center">XY</td><td align="center">109</td><td align="center">51</td><td align="center">3.0</td><td sorttable_customkey="20110301000000">1-27-09</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE FMN RIBOSWITCH BOUND TO RIBOFLAVIN.</td><td>A.A.SERGANOV,L.HUANG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F4E/index.html" title="Resolution 3.0A, 109 nucleotides, 50 basepairs, 0.4587 pairs per nucleotide">3F4E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F2Q/index.html" title="Resolution 3.0A, 107 nucleotides, 49 basepairs, 0.4579 pairs per nucleotide">3F2Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F2Y/index.html" title="Resolution 3.2A, 107 nucleotides, 49 basepairs, 0.4579 pairs per nucleotide">3F2Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F4H/index.html" title="Resolution 3.0A, 109 nucleotides, 49 basepairs, 0.4495 pairs per nucleotide">3F4H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F2W/index.html" title="Resolution 3.5A, 107 nucleotides, 48 basepairs, 0.4486 pairs per nucleotide">3F2W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F30/index.html" title="Resolution 3.1A, 107 nucleotides, 47 basepairs, 0.4393 pairs per nucleotide">3F30</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F2X/index.html" title="Resolution 3.1A, 107 nucleotides, 47 basepairs, 0.4393 pairs per nucleotide">3F2X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F2T/index.html" title="Resolution 3.0A, 106 nucleotides, 46 basepairs, 0.4340 pairs per nucleotide">3F2T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F4G/3F4G_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRL" title="Link to PDB">2KRL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRL/index.html" title="Link to FR3D analysis. 0.2157 pairs per nucleotide">2KRL</a></td><td align="center">A</td><td align="center">102</td><td align="center">22</td><td align="center">SOLUTION NMR; SOLUTION SCATTERING</td><td sorttable_customkey="20110902000000">2-09-11</td><td>TURNIP CRINKLE VIRUS</td><td>THE ENSEMBLE OF THE SOLUTION GLOBAL STRUCTURES OF THE 102-NT RIBOSOME BINDING STRUCTURE ELEMENT OF THE TURNIP CRINKLE VIRUS 3' UTR RNA</td><td>X.ZUO,J.WANG,P.YU,D.EYLER,H.XU,M.STARICH,D.TIEDE,A.SIMON,W.KASPRZAK, C.SCHWIETERS,B.SHAPIRO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XXA" title="Link to PDB">2XXA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XXA/index.html" title="Link to FR3D analysis. 0.3922 pairs per nucleotide">2XXA</a></td><td align="center">G</td><td align="center">204</td><td align="center">80</td><td align="center">3.9</td><td sorttable_customkey="20110203000000">3-02-11</td><td>DEINOCOCCUS RADIODURANS</td><td>THE CRYSTAL STRUCTURE OF THE SIGNAL RECOGNITION PARTICLE (SRP) IN COMPLEX WITH ITS RECEPTOR(SR)</td><td>S.F.ATAIDE,N.SCHMITZ,K.SHEN,A.KE,S.SHAN,J.A.DOUDNA,N.BAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S9S" title="Link to PDB">1S9S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S9S/index.html" title="Link to FR3D analysis. 0.3733 pairs per nucleotide">1S9S</a></td><td align="center">A</td><td align="center">2020</td><td align="center">754</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040604000000">4-06-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF MLV PSI SITE</td><td>V.D'SOUZA,A.DEY,D.HABIB,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YSH" title="Link to PDB">1YSH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YSH/index.html" title="Link to FR3D analysis. 0.4012 pairs per nucleotide">1YSH</a></td><td align="center">ABF</td><td align="center">162</td><td align="center">65</td><td align="center">9.5</td><td sorttable_customkey="20050507000000">7-05-05</td><td>synthetic,synthetic,synthetic</td><td>LOCALIZATION AND DYNAMIC BEHAVIOR OF RIBOSOMAL PROTEIN L30E</td><td>M.HALIC,T.BECKER,J.FRANK,C.M.SPAHN,R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GTT" title="Link to PDB">2GTT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GTT/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2GTT</a></td><td align="center">W</td><td align="center">198</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="20070709000000">9-19-06</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE RABIES VIRUS NUCLEOPROTEIN-RNA COMPLEX</td><td>A.A.V.ALBERTINI,A.K.WERNIMONT,T.MUZIOL,R.B.G.RAVELLI, W.WEISSENHORN,R.W.H.RUIGROK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KTW" title="Link to PDB">3KTW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KTW/index.html" title="Link to FR3D analysis. 0.3646 pairs per nucleotide">3KTW</a></td><td align="center">C</td><td align="center">96</td><td align="center">35</td><td align="center">3.2</td><td sorttable_customkey="20110402000000">2-16-10</td><td>SULFOLOBUS SOLFATARICUS</td><td>CRYSTAL STRUCTURE OF THE SRP19/S-DOMAIN SRP RNA COMPLEX OF SULFOLOBUS SOLFATARICUS</td><td>K.WILD,G.BANGE,G.BOZKURT,I.SINNING</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PXV" title="Link to PDB">2PXV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXV/index.html" title="Link to FR3D analysis. 0.4490 pairs per nucleotide">2PXV</a></td><td align="center">B</td><td align="center">49</td><td align="center">22</td><td align="center">2.0</td><td sorttable_customkey="20070708000000">8-07-07</td><td>synthetic</td><td>VARIANT 6 OF RIBONUCLEOPROTEIN CORE OF THE E. COLI SIGNAL RECOGNITION PARTICLE</td><td>A.Y.KEEL,R.P.RAMBO,R.T.BATEY,J.S.KIEFT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXK/index.html" title="Resolution 2.0A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXD/index.html" title="Resolution 2.0A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXQ/index.html" title="Resolution 2.5A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXB/index.html" title="Resolution 2.0A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXT/index.html" title="Resolution 2.5A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXP/index.html" title="Resolution 2.5A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXE/index.html" title="Resolution 2.0A, 49 nucleotides, 22 basepairs, 0.4490 pairs per nucleotide">2PXE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DUL/index.html" title="Resolution 1.8A, 48 nucleotides, 21 basepairs, 0.4375 pairs per nucleotide">1DUL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LQX/index.html" title="Resolution 1.9A, 48 nucleotides, 21 basepairs, 0.4375 pairs per nucleotide">3LQX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HQ1/index.html" title="Resolution 1.5A, 48 nucleotides, 21 basepairs, 0.4375 pairs per nucleotide">1HQ1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXU/index.html" title="Resolution 2.5A, 49 nucleotides, 21 basepairs, 0.4286 pairs per nucleotide">2PXU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXF/index.html" title="Resolution 2.0A, 49 nucleotides, 21 basepairs, 0.4286 pairs per nucleotide">2PXF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXL/index.html" title="Resolution 2.5A, 47 nucleotides, 20 basepairs, 0.4255 pairs per nucleotide">2PXL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XKV/index.html" title="Resolution 13.5A, 96 nucleotides, 9 basepairs, 0.0938 pairs per nucleotide">2XKV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PXV/2PXV_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OM7" title="Link to PDB">2OM7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OM7/index.html" title="Link to FR3D analysis. 0.3319 pairs per nucleotide">2OM7</a></td><td align="center">ABCDFGHIJM</td><td align="center">455</td><td align="center">151</td><td align="center">7.3</td><td sorttable_customkey="20090301000000">1-15-08</td><td>THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS</td><td>STRUCTURAL BASIS FOR INTERACTION OF THE RIBOSOME WITH THE SWITCH REGIONS OF GTP-BOUND ELONGATION FACTORS</td><td>S.R.CONNELL,D.N.WILSON,M.ROST,M.SCHUELER,J.GIESEBRECHT, M.DABROWSKI,T.MIELKE,P.FUCINI,C.M.T.SPAHN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GIS" title="Link to PDB">2GIS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIS/index.html" title="Link to FR3D analysis. 0.4681 pairs per nucleotide">2GIS</a></td><td align="center">A</td><td align="center">94</td><td align="center">44</td><td align="center">2.9</td><td sorttable_customkey="20060407000000">7-04-06</td><td>synthetic</td><td>STRUCTURE OF THE S-ADENOSYLMETHIONINE RIBOSWITCH MRNA REGULATORY ELEMENT</td><td>R.K.MONTANGE,R.T.BATEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GX5/index.html" title="Resolution 2.4A, 94 nucleotides, 43 basepairs, 0.4574 pairs per nucleotide">3GX5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GX6/index.html" title="Resolution 2.8A, 94 nucleotides, 42 basepairs, 0.4468 pairs per nucleotide">3GX6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IQR/index.html" title="Resolution 2.5A, 94 nucleotides, 42 basepairs, 0.4468 pairs per nucleotide">3IQR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GX2/index.html" title="Resolution 2.9A, 94 nucleotides, 41 basepairs, 0.4362 pairs per nucleotide">3GX2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IQN/index.html" title="Resolution 2.7A, 94 nucleotides, 41 basepairs, 0.4362 pairs per nucleotide">3IQN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GX3/index.html" title="Resolution 2.7A, 94 nucleotides, 40 basepairs, 0.4255 pairs per nucleotide">3GX3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IQP/index.html" title="Resolution 2.9A, 94 nucleotides, 40 basepairs, 0.4255 pairs per nucleotide">3IQP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GX7/index.html" title="Resolution 3.0A, 94 nucleotides, 40 basepairs, 0.4255 pairs per nucleotide">3GX7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIS/2GIS_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U6P" title="Link to PDB">1U6P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U6P/index.html" title="Link to FR3D analysis. 0.3635 pairs per nucleotide">1U6P</a></td><td align="center">B</td><td align="center">2000</td><td align="center">727</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040211000000">11-02-04</td><td>MOLONEY MURINE LEUKEMIA VIRUS</td><td>NMR STRUCTURE OF THE MLV ENCAPSIDATION SIGNAL BOUND TO THE NUCLEOCAPSID PROTEIN</td><td>V.D'SOUZA,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IWN" title="Link to PDB">3IWN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IWN/index.html" title="Link to FR3D analysis. 0.3763 pairs per nucleotide">3IWN</a></td><td align="center">A</td><td align="center">93</td><td align="center">35</td><td align="center">3.2</td><td sorttable_customkey="20091011000000">11-10-09</td><td>synthetic</td><td>CO-CRYSTAL STRUCTURE OF A BACTERIAL C-DI-GMP RIBOSWITCH</td><td>N.KULSHINA,N.J.BAIRD,A.R.FERRE-D'AMARE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ADD" title="Link to PDB">3ADD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADD/index.html" title="Link to FR3D analysis. 0.4148 pairs per nucleotide">3ADD</a></td><td align="center">C</td><td align="center">176</td><td align="center">73</td><td align="center">2.4</td><td sorttable_customkey="20120407000000">7-28-10</td><td>METHANOPYRUS KANDLERI</td><td>CRYSTAL STRUCTURE OF O-PHOSPHOSERYL-TRNA KINASE COMPLEXED WITH SELENOCYSTEINE TRNA AND AMPPNP (CRYSTAL TYPE 3)</td><td>Y.ITOH,S.CHIBA,S.SEKINE,S.YOKOYAMA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADB/index.html" title="Resolution 2.8A, 184 nucleotides, 72 basepairs, 0.3913 pairs per nucleotide">3ADB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADD/3ADD_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CUN" title="Link to PDB">3CUN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CUN/index.html" title="Link to FR3D analysis. 0.3736 pairs per nucleotide">3CUN</a></td><td align="center">C</td><td align="center">91</td><td align="center">34</td><td align="center">3.0</td><td sorttable_customkey="20091206000000">6-24-08</td><td>HOMO SAPIENS</td><td>AMINOACYL-TRNA SYNTHETASE RIBOZYME</td><td>H.XIAO,H.MURAKAMI,H.SUGA,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CUL/index.html" title="Resolution 2.8A, 182 nucleotides, 67 basepairs, 0.3681 pairs per nucleotide">3CUL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CUN/3CUN_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M5O" title="Link to PDB">1M5O</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5O/index.html" title="Link to FR3D analysis. 0.4196 pairs per nucleotide">1M5O</a></td><td align="center">DE</td><td align="center">224</td><td align="center">94</td><td align="center">2.2</td><td sorttable_customkey="20021210000000">10-12-02</td><td>synthetic,synthetic</td><td>TRANSITION STATE STABILIZATION BY A CATALYTIC RNA</td><td>P.B.RUPERT,A.P.MASSEY,S.T.SIGURDSSON,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5V/index.html" title="Resolution 2.4A, 224 nucleotides, 92 basepairs, 0.4107 pairs per nucleotide">1M5V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5K/index.html" title="Resolution 2.4A, 220 nucleotides, 89 basepairs, 0.4045 pairs per nucleotide">1M5K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5P/index.html" title="Resolution 2.6A, 112 nucleotides, 44 basepairs, 0.3929 pairs per nucleotide">1M5P</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5O/1M5O_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ADC" title="Link to PDB">3ADC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADC/index.html" title="Link to FR3D analysis. 0.3889 pairs per nucleotide">3ADC</a></td><td align="center">C</td><td align="center">180</td><td align="center">70</td><td align="center">2.9</td><td sorttable_customkey="20120407000000">7-28-10</td><td>METHANOPYRUS KANDLERI</td><td>CRYSTAL STRUCTURE OF O-PHOSPHOSERYL-TRNA KINASE COMPLEXED WITH SELENOCYSTEINE TRNA AND AMPPNP (CRYSTAL TYPE 2)</td><td>Y.ITOH,S.CHIBA,S.SEKINE,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MUR" title="Link to PDB">3MUR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MUR/index.html" title="Link to FR3D analysis. 0.4176 pairs per nucleotide">3MUR</a></td><td align="center">R</td><td align="center">91</td><td align="center">38</td><td align="center">3.0</td><td sorttable_customkey="20120108000000">8-25-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE C92U MUTANT C-DI-GMP RIBOSWITH BOUND TO C-DI- GMP</td><td>S.A.STROBEL,K.D.SMITH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MXH/index.html" title="Resolution 2.3A, 91 nucleotides, 37 basepairs, 0.4066 pairs per nucleotide">3MXH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MUV/index.html" title="Resolution 3.2A, 90 nucleotides, 35 basepairs, 0.3889 pairs per nucleotide">3MUV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MUM/index.html" title="Resolution 2.9A, 91 nucleotides, 35 basepairs, 0.3846 pairs per nucleotide">3MUM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IRW/index.html" title="Resolution 2.7A, 90 nucleotides, 34 basepairs, 0.3778 pairs per nucleotide">3IRW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MUT/index.html" title="Resolution 3.0A, 91 nucleotides, 33 basepairs, 0.3626 pairs per nucleotide">3MUT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MUR/3MUR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WZ2" title="Link to PDB">1WZ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WZ2/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1WZ2</a></td><td align="center">D</td><td align="center">176</td><td align="center">66</td><td align="center">3.2</td><td sorttable_customkey="20050609000000">9-06-05</td><td>synthetic</td><td>THE CRYSTAL STRUCTURE OF LEUCYL-TRNA SYNTHETASE AND TRNA(LEUCINE) COMPLEX</td><td>R.FUKUNAGA,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DUH" title="Link to PDB">1DUH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DUH/index.html" title="Link to FR3D analysis. 0.4318 pairs per nucleotide">1DUH</a></td><td align="center">A</td><td align="center">88</td><td align="center">38</td><td align="center">2.7</td><td sorttable_customkey="20000805000000">5-08-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE CONSERVED DOMAIN IV OF E. COLI 4.5S RNA</td><td>L.JOVINE,T.HAINZL,C.OUBRIDGE,W.G.SCOTT,J.LI,T.K.SIXMA, A.WONACOTT,T.SKARZYNSKI,K.NAGAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OWZ" title="Link to PDB">3OWZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OWZ/index.html" title="Link to FR3D analysis. 0.4702 pairs per nucleotide">3OWZ</a></td><td align="center">AB</td><td align="center">168</td><td align="center">79</td><td align="center">3.0</td><td sorttable_customkey="20120512000000">12-29-10</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF GLYCINE RIBOSWITCH, SOAKED IN IRIDIUM</td><td>L.HUANG,A.SERGANOV,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OX0/index.html" title="Resolution 3.0A, 173 nucleotides, 81 basepairs, 0.4682 pairs per nucleotide">3OX0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OXJ/index.html" title="Resolution 3.2A, 171 nucleotides, 79 basepairs, 0.4620 pairs per nucleotide">3OXJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OXM/index.html" title="Resolution 3.0A, 173 nucleotides, 79 basepairs, 0.4566 pairs per nucleotide">3OXM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OXE/index.html" title="Resolution 2.9A, 173 nucleotides, 79 basepairs, 0.4566 pairs per nucleotide">3OXE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OWW/index.html" title="Resolution 2.8A, 172 nucleotides, 78 basepairs, 0.4535 pairs per nucleotide">3OWW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OXD/index.html" title="Resolution 3.0A, 172 nucleotides, 78 basepairs, 0.4535 pairs per nucleotide">3OXD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OWI/index.html" title="Resolution 2.9A, 173 nucleotides, 78 basepairs, 0.4509 pairs per nucleotide">3OWI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OXB/index.html" title="Resolution 3.0A, 172 nucleotides, 75 basepairs, 0.4360 pairs per nucleotide">3OXB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OWZ/3OWZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3RG5" title="Link to PDB">3RG5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3RG5/index.html" title="Link to FR3D analysis. 0.4128 pairs per nucleotide">3RG5</a></td><td align="center">A</td><td align="center">172</td><td align="center">71</td><td align="center">2.0</td><td sorttable_customkey="20110806000000">6-08-11</td><td>MUS MUSCULUS</td><td>CRYSTAL STRUCTURE OF MOUSE TRNA(SEC)</td><td>M.C.WAHL,O.M.GANICHKIN,E.A.ANEDCHENKO</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3A3A/index.html" title="Resolution 3.1A, 86 nucleotides, 34 basepairs, 0.3953 pairs per nucleotide">3A3A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HL2/index.html" title="Resolution 2.8A, 82 nucleotides, 31 basepairs, 0.3780 pairs per nucleotide">3HL2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3RG5/3RG5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HOJ" title="Link to PDB">2HOJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOJ/index.html" title="Link to FR3D analysis. 0.4459 pairs per nucleotide">2HOJ</a></td><td align="center">A</td><td align="center">74</td><td align="center">33</td><td align="center">2.5</td><td sorttable_customkey="20070709000000">9-19-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN E. COLI THI-BOX RIBOSWITCH BOUND TO THIAMINE PYROPHOSPHATE, MANGANESE IONS</td><td>T.E.EDWARDS,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOO/index.html" title="Resolution 3.0A, 74 nucleotides, 33 basepairs, 0.4459 pairs per nucleotide">2HOO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOL/index.html" title="Resolution 2.9A, 75 nucleotides, 33 basepairs, 0.4400 pairs per nucleotide">2HOL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOK/index.html" title="Resolution 3.2A, 75 nucleotides, 33 basepairs, 0.4400 pairs per nucleotide">2HOK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K0J/index.html" title="Resolution 3.1A, 166 nucleotides, 70 basepairs, 0.4217 pairs per nucleotide">3K0J</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOM/index.html" title="Resolution 2.9A, 76 nucleotides, 31 basepairs, 0.4079 pairs per nucleotide">2HOM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOP/index.html" title="Resolution 3.3A, 69 nucleotides, 23 basepairs, 0.3333 pairs per nucleotide">2HOP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HOJ/2HOJ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZZM" title="Link to PDB">2ZZM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZZM/index.html" title="Link to FR3D analysis. 0.3214 pairs per nucleotide">2ZZM</a></td><td align="center">B</td><td align="center">84</td><td align="center">27</td><td align="center">2.6</td><td sorttable_customkey="20100309000000">9-15-09</td><td>synthetic</td><td>THE COMPLEX STRUCTURE OF ATRM5 AND TRNALEU</td><td>S.GOTO-ITO,T.ITO,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AM1" title="Link to PDB">3AM1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AM1/index.html" title="Link to FR3D analysis. 0.4198 pairs per nucleotide">3AM1</a></td><td align="center">B</td><td align="center">81</td><td align="center">34</td><td align="center">2.4</td><td sorttable_customkey="20120512000000">12-29-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF O-PHOSPHOSERYL-TRNA KINASE COMPLEXED WITH ANTICODON-STEM/LOOP TRUNCATED TRNA(SEC)</td><td>Y.ARAISO,R.ISHITANI,O.NUREKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GDI" title="Link to PDB">2GDI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GDI/index.html" title="Link to FR3D analysis. 0.4744 pairs per nucleotide">2GDI</a></td><td align="center">XY</td><td align="center">156</td><td align="center">74</td><td align="center">2.0</td><td sorttable_customkey="20060407000000">7-04-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THIAMINE PYROPHOSPHATE-SPECIFIC RIBOSWITCH IN COMPLEX WITH THIAMINE PYROPHOSPHATE</td><td>A.SERGANOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BTE" title="Link to PDB">2BTE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BTE/index.html" title="Link to FR3D analysis. 0.4423 pairs per nucleotide">2BTE</a></td><td align="center">BE</td><td align="center">156</td><td align="center">69</td><td align="center">2.9</td><td sorttable_customkey="20060309000000">9-15-05</td><td>THERMUS THERMOPHILUS</td><td>THERMUS THERMOPHILUS LEUCYL-TRNA SYNTHETASE COMPLEXED WITH WITH A TRNALEU TRANSCRIPT IN THE POST-EDITING CONFORMATION AND A POST-TRANSFER EDITING SUBSTRATE ANALOGUE</td><td>S.CUSACK,M.TUKALO,A.YAREMCHUK,R.FUKUNAGA,S.YOKOYAMA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V0G/index.html" title="Resolution 3.5A, 150 nucleotides, 61 basepairs, 0.4067 pairs per nucleotide">2V0G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BYT/index.html" title="Resolution 3.3A, 156 nucleotides, 63 basepairs, 0.4038 pairs per nucleotide">2BYT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BTE/2BTE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1P5P" title="Link to PDB">1P5P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P5P/index.html" title="Link to FR3D analysis. 0.3636 pairs per nucleotide">1P5P</a></td><td align="center">A</td><td align="center">77</td><td align="center">28</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030411000000">11-04-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HCV IRES DOMAIN II (MINIMIZED AVERAGE STRUCTURE)</td><td>P.J.LUKAVSKY,I.KIM,G.A.OTTO,J.D.PUGLISI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P5O/index.html" title="SOLUTION NMR, 924 nucleotides, 333 basepairs, 0.3604 pairs per nucleotide">1P5O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P5P/1P5P_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3D2V" title="Link to PDB">3D2V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D2V/index.html" title="Link to FR3D analysis. 0.4610 pairs per nucleotide">3D2V</a></td><td align="center">A</td><td align="center">154</td><td align="center">71</td><td align="center">2.0</td><td sorttable_customkey="20080107000000">7-01-08</td><td>ARABIDOPSIS THALIANA</td><td>STRUCTURE OF THE EUKARYOTIC TPP-SPECIFIC RIBOSWITCH BOUND TO THE ANTIBACTERIAL COMPOUND PYRITHIAMINE PYROPHOSPHATE</td><td>S.THORE,C.FRICK,N.BAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D2X/index.html" title="Resolution 2.5A, 154 nucleotides, 70 basepairs, 0.4545 pairs per nucleotide">3D2X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D2G/index.html" title="Resolution 2.2A, 154 nucleotides, 70 basepairs, 0.4545 pairs per nucleotide">3D2G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CKY/index.html" title="Resolution 2.9A, 154 nucleotides, 68 basepairs, 0.4416 pairs per nucleotide">2CKY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D2V/3D2V_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J2B" title="Link to PDB">1J2B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J2B/index.html" title="Link to FR3D analysis. 0.2703 pairs per nucleotide">1J2B</a></td><td align="center">CD</td><td align="center">148</td><td align="center">40</td><td align="center">3.3</td><td sorttable_customkey="20050305000000">5-27-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ARCHAEOSINE TRNA-GUANINE TRANSGLYCOSYLASE COMPLEXED WITH LAMBDA-FORM TRNA(VAL)</td><td>R.ISHITANI,O.NUREKI,N.NAMEKI,N.OKADA,S.NISHIMURA,S.YOKOYAMA, RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1H3E" title="Link to PDB">1H3E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H3E/index.html" title="Link to FR3D analysis. 0.4026 pairs per nucleotide">1H3E</a></td><td align="center">B</td><td align="center">77</td><td align="center">31</td><td align="center">2.9</td><td sorttable_customkey="20040310000000">10-27-02</td><td>THERMUS THERMOPHILUS</td><td>TYROSYL-TRNA SYNTHETASE FROM THERMUS THERMOPHILUS COMPLEXED WITH WILD-TYPE TRNATYR(GUA) AND WITH ATP AND TYROSINOL</td><td>S.CUSACK,A.YAREMCHUK,I.KRIKLIVYI,M.TUKALO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3A2K" title="Link to PDB">3A2K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3A2K/index.html" title="Link to FR3D analysis. 0.3377 pairs per nucleotide">3A2K</a></td><td align="center">C</td><td align="center">154</td><td align="center">52</td><td align="center">3.6</td><td sorttable_customkey="20100810000000">10-20-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF TILS COMPLEXED WITH TRNA</td><td>K.NAKANISHI,L.BONNEFOND,R.ISHITANI,O.NUREKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I9V" title="Link to PDB">1I9V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I9V/index.html" title="Link to FR3D analysis. 0.3973 pairs per nucleotide">1I9V</a></td><td align="center">A</td><td align="center">73</td><td align="center">29</td><td align="center">2.6</td><td sorttable_customkey="20010406000000">6-04-01</td><td>SACCHAROMYCES CEREVISIAE</td><td>CRYSTAL STRUCTURE ANALYSIS OF A TRNA-NEOMYCIN COMPLEX</td><td>N.E.MIKKELSEN,K.JOHANSSON,A.VIRTANEN,L.A.KIRSEBOM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZO3/index.html" title="Resolution 13.8A, 152 nucleotides, 60 basepairs, 0.3947 pairs per nucleotide">1ZO3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZO1/index.html" title="Resolution 13.8A, 76 nucleotides, 30 basepairs, 0.3947 pairs per nucleotide">1ZO1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FCW/index.html" title="Resolution 17.0A, 310 nucleotides, 118 basepairs, 0.3806 pairs per nucleotide">1FCW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EHZ/index.html" title="Resolution 1.9A, 62 nucleotides, 23 basepairs, 0.3710 pairs per nucleotide">1EHZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/6TNA/index.html" title="Resolution 2.7A, 62 nucleotides, 23 basepairs, 0.3710 pairs per nucleotide">6TNA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TRA/index.html" title="Resolution 3.0A, 62 nucleotides, 23 basepairs, 0.3710 pairs per nucleotide">1TRA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/4TRA/index.html" title="Resolution 3.0A, 62 nucleotides, 23 basepairs, 0.3710 pairs per nucleotide">4TRA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MJ1/index.html" title="Resolution 13.0A, 124 nucleotides, 45 basepairs, 0.3629 pairs per nucleotide">1MJ1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TN2/index.html" title="Resolution 3.0A, 62 nucleotides, 22 basepairs, 0.3548 pairs per nucleotide">1TN2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EVV/index.html" title="Resolution 2.0A, 62 nucleotides, 22 basepairs, 0.3548 pairs per nucleotide">1EVV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OB5/index.html" title="Resolution 3.1A, 186 nucleotides, 66 basepairs, 0.3548 pairs per nucleotide">1OB5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SZ1/index.html" title="Resolution 6.2A, 124 nucleotides, 44 basepairs, 0.3548 pairs per nucleotide">1SZ1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ML5/index.html" title="Resolution 14.0A, 68 nucleotides, 24 basepairs, 0.3529 pairs per nucleotide">1ML5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OB2/index.html" title="Resolution 3.4A, 63 nucleotides, 22 basepairs, 0.3492 pairs per nucleotide">1OB2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TTT/index.html" title="Resolution 2.7A, 186 nucleotides, 64 basepairs, 0.3441 pairs per nucleotide">1TTT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/4TNA/index.html" title="Resolution 2.5A, 62 nucleotides, 21 basepairs, 0.3387 pairs per nucleotide">4TNA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I9V/1I9V_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EIY" title="Link to PDB">1EIY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EIY/index.html" title="Link to FR3D analysis. 0.3553 pairs per nucleotide">1EIY</a></td><td align="center">C</td><td align="center">76</td><td align="center">27</td><td align="center">3.3</td><td sorttable_customkey="20000603000000">3-06-00</td><td>THERMUS THERMOPHILUS HB8</td><td>THE CRYSTAL STRUCTURE OF PHENYLALANYL-TRNA SYNTHETASE FROM THERMUS THERMOPHILUS COMPLEXED WITH COGNATE TRNAPHE</td><td>Y.GOLDGUR,L.MOSYAK,L.RESHETNIKOVA,V.ANKILOVA,M.SAFRO</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IY5/index.html" title="Resolution 3.1A, 76 nucleotides, 23 basepairs, 0.3026 pairs per nucleotide">2IY5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EIY/1EIY_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K4C" title="Link to PDB">2K4C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K4C/index.html" title="Link to FR3D analysis. 0.3947 pairs per nucleotide">2K4C</a></td><td align="center">A</td><td align="center">76</td><td align="center">30</td><td align="center">SOLUTION NMR; SOLUTION SCATTERING</td><td sorttable_customkey="20080912000000">12-09-08</td><td>ESCHERICHIA COLI</td><td>TRNAPHE-BASED HOMOLOGY MODEL FOR TRNAVAL REFINED AGAINST BASE N-H RDCS IN TWO MEDIA AND SAXS DATA</td><td>A.GRISHAEV,J.YING,M.D.CANNY,A.PARDI,A.BAX</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IZY" title="Link to PDB">3IZY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IZY/index.html" title="Link to FR3D analysis. 0.0526 pairs per nucleotide">3IZY</a></td><td align="center">N</td><td align="center">76</td><td align="center">4</td><td align="center">10.8</td><td sorttable_customkey="20121103000000">3-23-11</td><td>BOS TAURUS</td><td>MAMMALIAN MITOCHONDRIAL TRANSLATION INITIATION FACTOR 2</td><td>A.S.YASSIN,E HAQUE,P.P DATTA,K ELMORE,N.K.BANAVALI,L.L.SPREMULLI, R.K.AGRAWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NR0" title="Link to PDB">2NR0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NR0/index.html" title="Link to FR3D analysis. 0.3640 pairs per nucleotide">2NR0</a></td><td align="center">E</td><td align="center">272</td><td align="center">99</td><td align="center">3.9</td><td sorttable_customkey="20080305000000">5-15-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PSEUDOUDIRINDE SYNTHASE TRUA IN COMPLEX WITH LEUCYL TRNA</td><td>S.HUR,R.M.STROUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DEG" title="Link to PDB">3DEG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DEG/index.html" title="Link to FR3D analysis. 0.3792 pairs per nucleotide">3DEG</a></td><td align="center">JK</td><td align="center">298</td><td align="center">113</td><td align="center">10.0</td><td sorttable_customkey="20090708000000">8-19-08</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>COMPLEX OF ELONGATING ESCHERICHIA COLI 70S RIBOSOME AND EF4(LEPA)-GMPPNP</td><td>S.R.CONNELL,M.TOPF,Y.QIN,D.N.WILSON,T.MIELKE,P.FUCINI, K.H.NIERHAUS,C.M.T.SPAHN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CW5/index.html" title="Resolution 3.1A, 72 nucleotides, 25 basepairs, 0.3472 pairs per nucleotide">3CW5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CW6/index.html" title="Resolution 3.3A, 72 nucleotides, 24 basepairs, 0.3333 pairs per nucleotide">3CW6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FMT/index.html" title="Resolution 2.8A, 144 nucleotides, 44 basepairs, 0.3056 pairs per nucleotide">2FMT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DEG/3DEG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2CV1" title="Link to PDB">2CV1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CV1/index.html" title="Link to FR3D analysis. 0.4333 pairs per nucleotide">2CV1</a></td><td align="center">D</td><td align="center">150</td><td align="center">65</td><td align="center">2.4</td><td sorttable_customkey="20060509000000">9-05-06</td><td>synthetic</td><td>GLUTAMYL-TRNA SYNTHETASE FROM THERMUS THERMOPHILUS IN COMPLEX WITH TRNA(GLU), ATP, AND AN ANALOG OF L-GLUTAMATE: A QUATERNARY COMPLEX</td><td>S.SEKINE,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DXI/index.html" title="Resolution 2.2A, 150 nucleotides, 64 basepairs, 0.4267 pairs per nucleotide">2DXI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G59/index.html" title="Resolution 2.4A, 150 nucleotides, 63 basepairs, 0.4200 pairs per nucleotide">1G59</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N77/index.html" title="Resolution 2.4A, 150 nucleotides, 63 basepairs, 0.4200 pairs per nucleotide">1N77</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N78/index.html" title="Resolution 2.1A, 150 nucleotides, 63 basepairs, 0.4200 pairs per nucleotide">1N78</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CV2/index.html" title="Resolution 2.7A, 150 nucleotides, 62 basepairs, 0.4133 pairs per nucleotide">2CV2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CV0/index.html" title="Resolution 2.4A, 150 nucleotides, 61 basepairs, 0.4067 pairs per nucleotide">2CV0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CV1/2CV1_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZUE" title="Link to PDB">2ZUE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZUE/index.html" title="Link to FR3D analysis. 0.3867 pairs per nucleotide">2ZUE</a></td><td align="center">B</td><td align="center">75</td><td align="center">29</td><td align="center">2.0</td><td sorttable_customkey="20100608000000">8-18-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PYROCOCCUS HORIKOSHII ARGINYL-TRNA SYNTHETASE COMPLEXED WITH TRNA(ARG) AND AN ATP ANALOG (ANP)</td><td>M.KONNO,T.SUMIDA,E.UCHIKAWA,Y.MORI,T.YANAGISAWA,S.SEKINE, S.YOKOYAMA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZUF/index.html" title="Resolution 2.3A, 75 nucleotides, 29 basepairs, 0.3867 pairs per nucleotide">2ZUF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZUE/2ZUE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2CSX" title="Link to PDB">2CSX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CSX/index.html" title="Link to FR3D analysis. 0.3733 pairs per nucleotide">2CSX</a></td><td align="center">C</td><td align="center">75</td><td align="center">28</td><td align="center">2.7</td><td sorttable_customkey="20060809000000">9-20-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AQUIFEX AEOLICUS METHIONYL-TRNA SYNTHETASE COMPLEXED WITH TRNA(MET)</td><td>K.NAKANISHI,Y.OGISO,T.NAKAMA,S.FUKAI,O.NUREKI,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CT8/index.html" title="Resolution 2.7A, 148 nucleotides, 53 basepairs, 0.3581 pairs per nucleotide">2CT8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CSX/2CSX_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IHX" title="Link to PDB">2IHX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IHX/index.html" title="Link to FR3D analysis. 0.3580 pairs per nucleotide">2IHX</a></td><td align="center">B</td><td align="center">1500</td><td align="center">537</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080401000000">1-16-07</td><td>ROUS SARCOMA VIRUS</td><td>SOLUTION STRUCTURE OF THE ROUS SARCOMA VIRUS NUCLEOCAPSID PROTEIN:UPSI RNA PACKAGING SIGNAL COMPLEX</td><td>J.ZHOU,M.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AKE" title="Link to PDB">2AKE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AKE/index.html" title="Link to FR3D analysis. 0.2917 pairs per nucleotide">2AKE</a></td><td align="center">B</td><td align="center">72</td><td align="center">21</td><td align="center">3.1</td><td sorttable_customkey="20061107000000">7-11-06</td><td>BOS TAURUS</td><td>STRUCTURE OF HUMAN TRYPTOPHANYL-TRNA SYNTHETASE IN COMPLEX WITH TRNA(TRP)</td><td>N.SHEN,L.GUO,B.YANG,Y.JIN,J.DING</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR2/index.html" title="Resolution 3.0A, 75 nucleotides, 20 basepairs, 0.2667 pairs per nucleotide">2DR2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AKE/2AKE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QU2" title="Link to PDB">1QU2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QU2/index.html" title="Link to FR3D analysis. 0.4133 pairs per nucleotide">1QU2</a></td><td align="center">T</td><td align="center">75</td><td align="center">31</td><td align="center">2.2</td><td sorttable_customkey="20010708000000">8-31-99</td><td>STAPHYLOCOCCUS AUREUS</td><td>INSIGHTS INTO EDITING FROM AN ILE-TRNA SYNTHETASE STRUCTURE WITH TRNA(ILE) AND MUPIROCIN</td><td>L.F.SILVIAN,J.WANG,T.A.STEITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FFY/index.html" title="Resolution 2.2A, 75 nucleotides, 31 basepairs, 0.4133 pairs per nucleotide">1FFY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QU3/index.html" title="Resolution 2.9A, 75 nucleotides, 30 basepairs, 0.4000 pairs per nucleotide">1QU3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QU2/1QU2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GAX" title="Link to PDB">1GAX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GAX/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1GAX</a></td><td align="center">D</td><td align="center">150</td><td align="center">60</td><td align="center">2.9</td><td sorttable_customkey="20000612000000">12-06-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THERMUS THERMOPHILUS VALYL-TRNA SYNTHETASE COMPLEXED WITH TRNA(VAL) AND VALYL-ADENYLATE ANALOGUE</td><td>S.FUKAI,O.NUREKI,S.SEKINE,A.SHIMADA,J.TAO,D.G.VASSYLYEV, S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IVS/index.html" title="Resolution 2.9A, 75 nucleotides, 30 basepairs, 0.4000 pairs per nucleotide">1IVS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GAX/1GAX_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1O0C" title="Link to PDB">1O0C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O0C/index.html" title="Link to FR3D analysis. 0.4189 pairs per nucleotide">1O0C</a></td><td align="center">B</td><td align="center">74</td><td align="center">31</td><td align="center">2.7</td><td sorttable_customkey="20040304000000">4-15-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF L-GLUTAMATE AND AMPCPP BOUND TO GLUTAMINE AMINOACYL TRNA SYNTHETASE</td><td>T.L.BULLOCK,J.J.PERONA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QRS/index.html" title="Resolution 2.6A, 74 nucleotides, 30 basepairs, 0.4054 pairs per nucleotide">1QRS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QRT/index.html" title="Resolution 2.7A, 74 nucleotides, 30 basepairs, 0.4054 pairs per nucleotide">1QRT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QTQ/index.html" title="Resolution 2.2A, 74 nucleotides, 30 basepairs, 0.4054 pairs per nucleotide">1QTQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GTR/index.html" title="Resolution 2.5A, 74 nucleotides, 30 basepairs, 0.4054 pairs per nucleotide">1GTR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GTS/index.html" title="Resolution 2.8A, 74 nucleotides, 29 basepairs, 0.3919 pairs per nucleotide">1GTS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QRU/index.html" title="Resolution 3.0A, 74 nucleotides, 29 basepairs, 0.3919 pairs per nucleotide">1QRU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O0B/index.html" title="Resolution 2.7A, 74 nucleotides, 29 basepairs, 0.3919 pairs per nucleotide">1O0B</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EUY/index.html" title="Resolution 2.6A, 73 nucleotides, 28 basepairs, 0.3836 pairs per nucleotide">1EUY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EXD/index.html" title="Resolution 2.7A, 73 nucleotides, 27 basepairs, 0.3699 pairs per nucleotide">1EXD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RE8/index.html" title="Resolution 2.6A, 74 nucleotides, 27 basepairs, 0.3649 pairs per nucleotide">2RE8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZJW/index.html" title="Resolution 2.5A, 74 nucleotides, 25 basepairs, 0.3378 pairs per nucleotide">1ZJW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RD2/index.html" title="Resolution 2.6A, 74 nucleotides, 25 basepairs, 0.3378 pairs per nucleotide">2RD2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EUQ/index.html" title="Resolution 3.1A, 72 nucleotides, 24 basepairs, 0.3333 pairs per nucleotide">1EUQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O0C/1O0C_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q3Z" title="Link to PDB">3Q3Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q3Z/index.html" title="Link to FR3D analysis. 0.4324 pairs per nucleotide">3Q3Z</a></td><td align="center">AV</td><td align="center">148</td><td align="center">64</td><td align="center">2.5</td><td sorttable_customkey="20111105000000">5-11-11</td><td>CLOSTRIDIUM ACETOBUTYLICUM</td><td>STRUCTURE OF A C-DI-GMP-II RIBOSWITCH FROM C. ACETOBUTYLICUM BOUND TO C-DI-GMP</td><td>K.D.SMITH,C.A.SHANAHAN,E.L.MOORE,A.C.SIMON,S.A.STROBEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1VBY" title="Link to PDB">1VBY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VBY/index.html" title="Link to FR3D analysis. 0.4110 pairs per nucleotide">1VBY</a></td><td align="center">B</td><td align="center">73</td><td align="center">30</td><td align="center">2.9</td><td sorttable_customkey="20050605000000">5-18-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HEPATITIS DELTA VIRUS GEMONIC RIBOZYME PRECURSOR, WITH C75U MUTAION, AND MN2+ BOUND</td><td>A.KE,K.ZHOU,F.DING,J.H.D.CATE,J.A.DOUDNA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VC7/index.html" title="Resolution 2.5A, 74 nucleotides, 30 basepairs, 0.4054 pairs per nucleotide">1VC7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VC5/index.html" title="Resolution 3.4A, 70 nucleotides, 28 basepairs, 0.4000 pairs per nucleotide">1VC5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SJ4/index.html" title="Resolution 2.7A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">1SJ4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SJ3/index.html" title="Resolution 2.2A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">1SJ3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VBX/index.html" title="Resolution 2.7A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">1VBX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VC0/index.html" title="Resolution 2.5A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">1VC0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OIH/index.html" title="Resolution 2.4A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">2OIH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VBZ/index.html" title="Resolution 2.8A, 73 nucleotides, 29 basepairs, 0.3973 pairs per nucleotide">1VBZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SJF/index.html" title="Resolution 2.8A, 74 nucleotides, 29 basepairs, 0.3919 pairs per nucleotide">1SJF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VC6/index.html" title="Resolution 2.8A, 72 nucleotides, 28 basepairs, 0.3889 pairs per nucleotide">1VC6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DRZ/index.html" title="Resolution 2.3A, 72 nucleotides, 28 basepairs, 0.3889 pairs per nucleotide">1DRZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CX0/index.html" title="Resolution 2.3A, 72 nucleotides, 28 basepairs, 0.3889 pairs per nucleotide">1CX0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OJ3/index.html" title="Resolution 2.9A, 73 nucleotides, 28 basepairs, 0.3836 pairs per nucleotide">2OJ3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VBY/1VBY_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J1U" title="Link to PDB">1J1U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J1U/index.html" title="Link to FR3D analysis. 0.4189 pairs per nucleotide">1J1U</a></td><td align="center">B</td><td align="center">74</td><td align="center">31</td><td align="center">1.9</td><td sorttable_customkey="20030306000000">6-03-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ARCHAEAL TYROSYL-TRNA SYNTHETASE COMPLEXED WITH TRNA(TYR) AND L-TYROSINE</td><td>T.KOBAYASHI,O.NUREKI,R.ISHITANI,M.TUKALO,S.CUSACK, K.SAKAMOTO,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZM5" title="Link to PDB">2ZM5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZM5/index.html" title="Link to FR3D analysis. 0.4324 pairs per nucleotide">2ZM5</a></td><td align="center">C</td><td align="center">74</td><td align="center">32</td><td align="center">2.5</td><td sorttable_customkey="20100204000000">4-14-09</td><td>ESCHERICHIA COLI K12</td><td>CRYSTAL STRUCTURE OF TRNA MODIFICATION ENZYME MIAA IN THE COMPLEX WITH TRNA(PHE)</td><td>J.SAKAI,M.YAO,S.CHIMNARONK,I.TANAKA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZXU/index.html" title="Resolution 2.8A, 74 nucleotides, 32 basepairs, 0.4324 pairs per nucleotide">2ZXU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FOZ/index.html" title="Resolution 2.5A, 143 nucleotides, 58 basepairs, 0.4056 pairs per nucleotide">3FOZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZM5/2ZM5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DER" title="Link to PDB">2DER</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DER/index.html" title="Link to FR3D analysis. 0.3793 pairs per nucleotide">2DER</a></td><td align="center">D</td><td align="center">145</td><td align="center">55</td><td align="center">3.1</td><td sorttable_customkey="20070308000000">8-15-06</td><td>synthetic</td><td>COCRYSTAL STRUCTURE OF AN RNA SULFURATION ENZYME MNMA AND TRNA-GLU IN THE INITIAL TRNA BINDING STATE</td><td>T.NUMATA,Y.IKEUCHI,S.FUKAI,T.SUZUKI,O.NUREKI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DET/index.html" title="Resolution 3.4A, 70 nucleotides, 24 basepairs, 0.3429 pairs per nucleotide">2DET</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DEU/index.html" title="Resolution 3.4A, 148 nucleotides, 46 basepairs, 0.3108 pairs per nucleotide">2DEU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DER/2DER_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AL0" title="Link to PDB">3AL0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AL0/index.html" title="Link to FR3D analysis. 0.4054 pairs per nucleotide">3AL0</a></td><td align="center">E</td><td align="center">74</td><td align="center">30</td><td align="center">3.4</td><td sorttable_customkey="20120509000000">9-29-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE GLUTAMINE TRANSAMIDOSOME FROM THERMOTOGA MARITIMA IN THE GLUTAMYLATION STATE.</td><td>T.ITO,S.YOKOYAMA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AKZ/index.html" title="Resolution 2.9A, 296 nucleotides, 112 basepairs, 0.3784 pairs per nucleotide">3AKZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AL0/3AL0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U0B" title="Link to PDB">1U0B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U0B/index.html" title="Link to FR3D analysis. 0.4054 pairs per nucleotide">1U0B</a></td><td align="center">A</td><td align="center">74</td><td align="center">30</td><td align="center">2.3</td><td sorttable_customkey="20051111000000">11-23-04</td><td>ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF CYSTEINYL-TRNA SYNTHETASE BINARY COMPLEX WITH TRNACYS</td><td>S.HAUENSTEIN,C.M.ZHANG,Y.M.HOU,J.J.PERONA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D6F" title="Link to PDB">2D6F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D6F/index.html" title="Link to FR3D analysis. 0.3425 pairs per nucleotide">2D6F</a></td><td align="center">F</td><td align="center">146</td><td align="center">50</td><td align="center">3.1</td><td sorttable_customkey="20061107000000">7-11-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GLU-TRNA(GLN) AMIDOTRANSFERASE IN THE COMPLEX WITH TRNA(GLN)</td><td>O.NUREKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3L0U" title="Link to PDB">3L0U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3L0U/index.html" title="Link to FR3D analysis. 0.4110 pairs per nucleotide">3L0U</a></td><td align="center">A</td><td align="center">73</td><td align="center">30</td><td align="center">3.0</td><td sorttable_customkey="20110403000000">3-16-10</td><td>synthetic</td><td>THE CRYSTAL STRUCTURE OF UNMODIFIED TRNAPHE FROM ESCHERICHIA COLI</td><td>R.T.BYRNE,A.L.KONEVEGA,M.V.RODNINA,A.A.ANTSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AZX" title="Link to PDB">2AZX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AZX/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">2AZX</a></td><td align="center">C</td><td align="center">144</td><td align="center">60</td><td align="center">2.8</td><td sorttable_customkey="20060108000000">8-01-06</td><td>synthetic</td><td>CHARGED AND UNCHARGED TRNAS ADOPT DISTINCT CONFORMATIONS WHEN COMPLEXED WITH HUMAN TRYPTOPHANYL-TRNA SYNTHETASE</td><td>X.L.YANG,F.J.OTERO,K.L.EWALT,J.LIU,M.A.SWAIRJO,C.KOHRER, U.L.RAJBHANDARY,R.J.SKENE,D.E.MCREE,P.SCHIMMEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZNI" title="Link to PDB">2ZNI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZNI/index.html" title="Link to FR3D analysis. 0.3611 pairs per nucleotide">2ZNI</a></td><td align="center">C</td><td align="center">144</td><td align="center">52</td><td align="center">3.1</td><td sorttable_customkey="20100612000000">12-30-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PYRROLYSYL-TRNA SYNTHETASE-TRNA(PYL) COMPLEX FROM DESULFITOBACTERIUM HAFNIENSE</td><td>K.NOZAWA,Y.ARAISO,D.SOLL,R.ISHITANI,O.NUREKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Y26" title="Link to PDB">1Y26</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y26/index.html" title="Link to FR3D analysis. 0.4930 pairs per nucleotide">1Y26</a></td><td align="center">X</td><td align="center">71</td><td align="center">35</td><td align="center">2.1</td><td sorttable_customkey="20060412000000">12-28-04</td><td>VIBRIO VULNIFICUS</td><td>A-RIBOSWITCH-ADENINE COMPLEX</td><td>A.SERGANOV,Y.R.YUAN,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DU3" title="Link to PDB">2DU3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DU3/index.html" title="Link to FR3D analysis. 0.3662 pairs per nucleotide">2DU3</a></td><td align="center">D</td><td align="center">71</td><td align="center">26</td><td align="center">2.6</td><td sorttable_customkey="20080103000000">3-13-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ARCHAEOGLOBUS FULGIDUS O-PHOSPHOSERYL- TRNA SYNTHETASE COMPLEXED WITH TRNACYS AND O-PHOSPHOSERINE</td><td>R.FUKUNAGA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DU4/index.html" title="Resolution 2.8A, 71 nucleotides, 24 basepairs, 0.3380 pairs per nucleotide">2DU4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DU5/index.html" title="Resolution 3.2A, 71 nucleotides, 18 basepairs, 0.2535 pairs per nucleotide">2DU5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DU3/2DU3_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KFU" title="Link to PDB">3KFU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KFU/index.html" title="Link to FR3D analysis. 0.3957 pairs per nucleotide">3KFU</a></td><td align="center">K</td><td align="center">278</td><td align="center">110</td><td align="center">3.0</td><td sorttable_customkey="20120108000000">8-25-10</td><td>THERMUS THERMOPHILUS</td><td>CRYSTAL STRUCTURE OF THE TRANSAMIDOSOME</td><td>M.BLAISE,M.BAILLY,M.FRECHIN,S.THIRUP,H.D.BECKER,D.KERN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DU6" title="Link to PDB">2DU6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DU6/index.html" title="Link to FR3D analysis. 0.2535 pairs per nucleotide">2DU6</a></td><td align="center">D</td><td align="center">71</td><td align="center">18</td><td align="center">3.3</td><td sorttable_customkey="20080103000000">3-13-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ARCHAEOGLOBUS FULGIDUS O-PHOSPHOSERYL- TRNA SYNTHETASE E418N/E420N MUTANT COMPLEXED WITH TRNAAMBER AND O-PHOSPHOSERINE ("AMBER COMPLEX")</td><td>R.FUKUNAGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L3J" title="Link to PDB">2L3J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L3J/index.html" title="Link to FR3D analysis. 0.4359 pairs per nucleotide">2L3J</a></td><td align="center">B</td><td align="center">1349</td><td align="center">588</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120310000000">10-27-10</td><td>RATTUS NORVEGICUS</td><td>THE SOLUTION STRUCTURE OF THE ADAR2 DSRBM-RNA COMPLEX REVEALS A SEQUENCE-SPECIFIC READ OUT OF THE MINOR GROOVE</td><td>R.STEFL,F.C.OBERSTRASS,F.H.-T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZZN" title="Link to PDB">2ZZN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZZN/index.html" title="Link to FR3D analysis. 0.4085 pairs per nucleotide">2ZZN</a></td><td align="center">C</td><td align="center">142</td><td align="center">58</td><td align="center">3.0</td><td sorttable_customkey="20100309000000">9-15-09</td><td>synthetic</td><td>THE COMPLEX STRUCTURE OF ATRM5 AND TRNACYS</td><td>S.GOTO-ITO,T.ITO,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OIU" title="Link to PDB">2OIU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OIU/index.html" title="Link to FR3D analysis. 0.4366 pairs per nucleotide">2OIU</a></td><td align="center">PQ</td><td align="center">142</td><td align="center">62</td><td align="center">2.6</td><td sorttable_customkey="20080103000000">3-13-07</td><td>synthetic</td><td>L1 RIBOZYME LIGASE CIRCULAR ADDUCT</td><td>M.P ROBERTSON,W.G.SCOTT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3LA5" title="Link to PDB">3LA5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LA5/index.html" title="Link to FR3D analysis. 0.4930 pairs per nucleotide">3LA5</a></td><td align="center">A</td><td align="center">71</td><td align="center">35</td><td align="center">1.7</td><td sorttable_customkey="20120201000000">1-26-10</td><td>BACILLUS SUBTILIS</td><td>X-RAY CRYSTAL STRUCTURE OF MC6 RNA RIBOSWITCH BOUND TO AZACYTOSINE</td><td>M.S.DUNSTAN,D.LEYS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KXK" title="Link to PDB">1KXK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KXK/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1KXK</a></td><td align="center">A</td><td align="center">70</td><td align="center">28</td><td align="center">3.0</td><td sorttable_customkey="20031003000000">3-22-02</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A RNA MOLECULE CONTAINING DOMAIN 5 AND 6 OF THE YEAST AI5G GROUP II SELF-SPLICING INTRON</td><td>L.ZHANG,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IVN" title="Link to PDB">3IVN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IVN/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">3IVN</a></td><td align="center">A</td><td align="center">138</td><td align="center">60</td><td align="center">2.8</td><td sorttable_customkey="20110701000000">1-19-10</td><td>BACILLUS SUBTILIS</td><td>STRUCTURE OF THE U65C MUTANT A-RIBOSWITCH APTAMER FROM THE BACILLUS SUBTILIS PBUE OPERON</td><td>V.DELFOSSE,P.DAGENAIS,D.CHAUSSE,G.DI TOMASSO,P.LEGAULT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3EPH" title="Link to PDB">3EPH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EPH/index.html" title="Link to FR3D analysis. 0.3478 pairs per nucleotide">3EPH</a></td><td align="center">E</td><td align="center">138</td><td align="center">48</td><td align="center">3.0</td><td sorttable_customkey="20080411000000">11-04-08</td><td>synthetic</td><td>CRYSTALLOGRAPHIC SNAPSHOTS OF EUKARYOTIC DIMETHYLALLYLTRANSFERASE ACTING ON TRNA: INSIGHT INTO TRNA RECOGNITION AND REACTION MECHANISM</td><td>R.H.HUANG,C.ZHOU</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EPJ/index.html" title="Resolution 3.1A, 138 nucleotides, 48 basepairs, 0.3478 pairs per nucleotide">3EPJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EPK/index.html" title="Resolution 3.2A, 138 nucleotides, 47 basepairs, 0.3406 pairs per nucleotide">3EPK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EPL/index.html" title="Resolution 3.6A, 137 nucleotides, 43 basepairs, 0.3139 pairs per nucleotide">3EPL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EPH/3EPH_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QF6" title="Link to PDB">1QF6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QF6/index.html" title="Link to FR3D analysis. 0.4203 pairs per nucleotide">1QF6</a></td><td align="center">B</td><td align="center">69</td><td align="center">29</td><td align="center">2.9</td><td sorttable_customkey="19990605000000">5-06-99</td><td>ESCHERICHIA COLI</td><td>STRUCTURE OF E. COLI THREONYL-TRNA SYNTHETASE COMPLEXED WITH ITS COGNATE TRNA</td><td>R.SANKARANARAYANAN,A.C.DOCK-BREGEON,B.REES,D.MORAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MOJ" title="Link to PDB">3MOJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MOJ/index.html" title="Link to FR3D analysis. 0.3768 pairs per nucleotide">3MOJ</a></td><td align="center">A</td><td align="center">69</td><td align="center">26</td><td align="center">2.9</td><td sorttable_customkey="20100408000000">8-04-10</td><td>ESCHERICHIA COLI</td><td>STRUCTURE OF THE RNA BINDING DOMAIN OF THE BACILLUS SUBTILIS YXIN PROTEIN COMPLEXED WITH A FRAGMENT OF 23S RIBOSOMAL RNA</td><td>J.W.HARDIN,Y.HU,D.B.MCKAY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GIX" title="Link to PDB">1GIX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GIX/index.html" title="Link to FR3D analysis. 0.3586 pairs per nucleotide">1GIX</a></td><td align="center">1BC</td><td align="center">198</td><td align="center">71</td><td align="center">5.5</td><td sorttable_customkey="20010405000000">5-04-01</td><td>THERMUS THERMOPHILUS,synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE RIBOSOME AT 5.5 A RESOLUTION. THIS FILE, 1GIX, CONTAINS THE 30S RIBOSOME SUBUNIT, THREE TRNA, AND MRNA MOLECULES. 50S RIBOSOME SUBUNIT IS IN THE FILE 1GIY</td><td>M.M.YUSUPOV,G.Z.YUSUPOVA,A.BAUCOM,K.LIEBERMAN,T.N.EARNEST, J.H.D.CATE,H.F.NOLLER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JGO/index.html" title="Resolution 5.6A, 198 nucleotides, 71 basepairs, 0.3586 pairs per nucleotide">1JGO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JGP/index.html" title="Resolution 7.0A, 198 nucleotides, 71 basepairs, 0.3586 pairs per nucleotide">1JGP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JGQ/index.html" title="Resolution 5.0A, 195 nucleotides, 69 basepairs, 0.3538 pairs per nucleotide">1JGQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GIX/1GIX_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EFW" title="Link to PDB">1EFW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EFW/index.html" title="Link to FR3D analysis. 0.3968 pairs per nucleotide">1EFW</a></td><td align="center">C</td><td align="center">126</td><td align="center">50</td><td align="center">3.0</td><td sorttable_customkey="20010706000000">6-19-00</td><td>ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF ASPARTYL-TRNA SYNTHETASE FROM THERMUS THERMOPHILUS COMPLEXED TO TRNAASP FROM ESCHERICHIA COLI</td><td>C.BRIAND,A.POTERSZMAN,S.EILER,G.WEBSTER,J.-C.THIERRY,D.MORAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C0A/index.html" title="Resolution 2.4A, 68 nucleotides, 25 basepairs, 0.3676 pairs per nucleotide">1C0A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EFW/1EFW_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Y9B" title="Link to PDB">2Y9B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9B/index.html" title="Link to FR3D analysis. 0.3431 pairs per nucleotide">2Y9B</a></td><td align="center">V</td><td align="center">204</td><td align="center">70</td><td align="center">3.6</td><td sorttable_customkey="20111105000000">5-11-11</td><td>HOMO SAPIENS</td><td>STRUCTURE OF THE SPLICEOSOMAL U4 SNRNP CORE DOMAIN</td><td>A.K.W.LEUNG,K.NAGAI,J.LI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9D/index.html" title="Resolution 3.6A, 68 nucleotides, 23 basepairs, 0.3382 pairs per nucleotide">2Y9D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9A/index.html" title="Resolution 3.6A, 204 nucleotides, 69 basepairs, 0.3382 pairs per nucleotide">2Y9A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9C/index.html" title="Resolution 3.6A, 204 nucleotides, 66 basepairs, 0.3235 pairs per nucleotide">2Y9C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9B/2Y9B_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QUW" title="Link to PDB">2QUW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QUW/index.html" title="Link to FR3D analysis. 0.3731 pairs per nucleotide">2QUW</a></td><td align="center">AB</td><td align="center">67</td><td align="center">25</td><td align="center">2.2</td><td sorttable_customkey="20090307000000">7-15-08</td><td>synthetic,synthetic</td><td>HAMMERHEAD RIBOZYME G12A MUTANT AFTER CLEAVAGE</td><td>Y.I.CHI,W.G.SCOTT,S.H.KIM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QUS/index.html" title="Resolution 2.4A, 136 nucleotides, 48 basepairs, 0.3529 pairs per nucleotide">2QUS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QUW/2QUW_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HAY" title="Link to PDB">3HAY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HAY/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">3HAY</a></td><td align="center">EF</td><td align="center">81</td><td align="center">27</td><td align="center">5.0</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A SUBSTRATE-BOUND FULL H/ACA RNP FROM PYROCOCCUS FURIOSUS</td><td>K.YE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IL2" title="Link to PDB">1IL2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IL2/index.html" title="Link to FR3D analysis. 0.4341 pairs per nucleotide">1IL2</a></td><td align="center">C</td><td align="center">129</td><td align="center">56</td><td align="center">2.6</td><td sorttable_customkey="20030409000000">9-28-01</td><td>SACCHAROMYCES CEREVISIAE</td><td>CRYSTAL STRUCTURE OF THE E. COLI ASPARTYL-TRNA SYNTHETASE:YEAST TRNAASP:ASPARTYL-ADENYLATE COMPLEX</td><td>L.MOULINIER,S.EILER,G.ERIANI,J.GANGLOFF,J.C.THIERRY, K.GABRIEL,W.H.MCCLAIN,D.MORAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ASZ/index.html" title="Resolution 3.0A, 134 nucleotides, 54 basepairs, 0.4030 pairs per nucleotide">1ASZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ASY/index.html" title="Resolution 2.9A, 134 nucleotides, 54 basepairs, 0.4030 pairs per nucleotide">1ASY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IL2/1IL2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FO4" title="Link to PDB">3FO4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FO4/index.html" title="Link to FR3D analysis. 0.5079 pairs per nucleotide">3FO4</a></td><td align="center">A</td><td align="center">63</td><td align="center">32</td><td align="center">1.9</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GUANINE RIBOSWITCH C74U MUTANT BOUND TO 6-CHLOROGUANINE</td><td>S.D.GILBERT,R.T.BATEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XNZ/index.html" title="Resolution 1.6A, 65 nucleotides, 33 basepairs, 0.5077 pairs per nucleotide">2XNZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GOG/index.html" title="Resolution 2.1A, 65 nucleotides, 33 basepairs, 0.5077 pairs per nucleotide">3GOG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FO6/index.html" title="Resolution 1.9A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">3FO6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EET/index.html" title="Resolution 1.9A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">2EET</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GAO/index.html" title="Resolution 1.9A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">3GAO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G9C/index.html" title="Resolution 1.7A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">2G9C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U8D/index.html" title="Resolution 1.9A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">1U8D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EEV/index.html" title="Resolution 1.9A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">2EEV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GER/index.html" title="Resolution 1.7A, 67 nucleotides, 34 basepairs, 0.5075 pairs per nucleotide">3GER</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EES/index.html" title="Resolution 1.8A, 67 nucleotides, 33 basepairs, 0.4925 pairs per nucleotide">2EES</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EEW/index.html" title="Resolution 2.2A, 67 nucleotides, 33 basepairs, 0.4925 pairs per nucleotide">2EEW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y27/index.html" title="Resolution 2.4A, 67 nucleotides, 33 basepairs, 0.4925 pairs per nucleotide">1Y27</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DS7/index.html" title="Resolution 1.9A, 134 nucleotides, 66 basepairs, 0.4925 pairs per nucleotide">3DS7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GES/index.html" title="Resolution 2.1A, 67 nucleotides, 33 basepairs, 0.4925 pairs per nucleotide">3GES</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EEU/index.html" title="Resolution 1.9A, 67 nucleotides, 33 basepairs, 0.4925 pairs per nucleotide">2EEU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G4M/index.html" title="Resolution 2.4A, 67 nucleotides, 32 basepairs, 0.4776 pairs per nucleotide">3G4M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GOT/index.html" title="Resolution 1.9A, 67 nucleotides, 32 basepairs, 0.4776 pairs per nucleotide">3GOT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B57/index.html" title="Resolution 2.1A, 65 nucleotides, 31 basepairs, 0.4769 pairs per nucleotide">2B57</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FO4/3FO4_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BBV" title="Link to PDB">3BBV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBV/index.html" title="Link to FR3D analysis. 0.3134 pairs per nucleotide">3BBV</a></td><td align="center">z</td><td align="center">67</td><td align="center">21</td><td align="center">10.0</td><td sorttable_customkey="20090910000000">10-21-08</td><td>THERMUS THERMOPHILUS</td><td>THE TRNA(PHE) FITTED INTO THE LOW RESOLUTION CRYO-EM MAP OF THE 50S.NC-TRNA.HSP15 COMPLEX</td><td>L.JIANG,J.P.ABRAHAMS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1B23" title="Link to PDB">1B23</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1B23/index.html" title="Link to FR3D analysis. 0.3485 pairs per nucleotide">1B23</a></td><td align="center">R</td><td align="center">66</td><td align="center">23</td><td align="center">2.6</td><td sorttable_customkey="19980712000000">12-07-98</td><td>ESCHERICHIA COLI</td><td>E. COLI CYSTEINYL-TRNA AND T. AQUATICUS ELONGATION FACTOR EF-TU:GTP TERNARY COMPLEX</td><td>P.NISSEN,M.KJELDGAARD,S.THIRUP,J.NYBORG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L1F" title="Link to PDB">2L1F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L1F/index.html" title="Link to FR3D analysis. 0.4080 pairs per nucleotide">2L1F</a></td><td align="center">AB</td><td align="center">2620</td><td align="center">1069</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120310000000">10-27-10</td><td>MOLONEY MURINE LEUKEMIA VIRUS,MOLONEY MURINE LEUKEMIA VIRUS</td><td>STRUCTURE OF A CONSERVED RETROVIRAL RNA PACKAGING ELEMENT BY NMR SPECTROSCOPY AND CRYO-ELECTRON TOMOGRAPHY</td><td>M.F.SUMMERS,R.N.IROBALIEVA,B.TOLBERT,A.SMALLS-MANTY,K.IYALLA, K.LOELIGER,V.D'SOUZA,H.KHANT,M.SCHMID,E.GARCIA,A.TELESNITSKY,W.CHIU, Y.MIYAZAKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3R4F" title="Link to PDB">3R4F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R4F/index.html" title="Link to FR3D analysis. 0.3485 pairs per nucleotide">3R4F</a></td><td align="center">A</td><td align="center">66</td><td align="center">23</td><td align="center">3.5</td><td sorttable_customkey="20120804000000">4-20-11</td><td>synthetic</td><td>PROHEAD RNA</td><td>F.DING,C.LU,W.ZHANO,K.R.RAJASHANKAR,D.L.ANDERSON,P.J.JARDINE, S.GRIMES,A.KE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TRA" title="Link to PDB">2TRA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TRA/index.html" title="Link to FR3D analysis. 0.4308 pairs per nucleotide">2TRA</a></td><td align="center">A</td><td align="center">65</td><td align="center">28</td><td align="center">3.0</td><td sorttable_customkey="19870611000000">11-06-87</td><td>SACCHAROMYCES CEREVISIAE</td><td>RESTRAINED REFINEMENT OF TWO CRYSTALLINE FORMS OF YEAST ASPARTIC ACID AND PHENYLALANINE TRANSFER RNA CRYSTALS</td><td>E.WESTHOF,P.DUMAS,D.MORAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3TRA/index.html" title="Resolution 3.0A, 64 nucleotides, 26 basepairs, 0.4062 pairs per nucleotide">3TRA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TRA/2TRA_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1H4S" title="Link to PDB">1H4S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H4S/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1H4S</a></td><td align="center">T</td><td align="center">65</td><td align="center">26</td><td align="center">2.9</td><td sorttable_customkey="20020606000000">6-18-01</td><td>THERMUS THERMOPHILUS</td><td>PROLYL-TRNA SYNTHETASE FROM THERMUS THERMOPHILUS COMPLEXED WITH TRNAPRO(CGG) AND A PROLYL-ADENYLATE ANALOGUE</td><td>A.YAREMCHUK,M.TUKALO,S.CUSACK</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H4Q/index.html" title="Resolution 3.0A, 65 nucleotides, 26 basepairs, 0.4000 pairs per nucleotide">1H4Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H4S/1H4S_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NQP" title="Link to PDB">2NQP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NQP/index.html" title="Link to FR3D analysis. 0.3231 pairs per nucleotide">2NQP</a></td><td align="center">F</td><td align="center">65</td><td align="center">21</td><td align="center">3.5</td><td sorttable_customkey="20080305000000">5-15-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PSEUDOUDIRINDE SYNTHASE TRUA IN COMPLEX WITH LEUCYL TRNA</td><td>S.HUR,R.M.STROUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=486D" title="Link to PDB">486D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/486D/index.html" title="Link to FR3D analysis. 0.4051 pairs per nucleotide">486D</a></td><td align="center">CD</td><td align="center">195</td><td align="center">79</td><td align="center">7.5</td><td sorttable_customkey="19990410000000">10-04-99</td><td>THERMUS THERMOPHILUS,ESCHERICHIA COLI,SACCHAROMYCES CEREVISIAE,THERMUS THERMOPHILUS,THERMUS THERMOPHILUS,synthetic,synthetic</td><td>X-RAY CRYSTAL STRUCTURES OF 70S RIBOSOME FUNCTIONAL COMPLEXES</td><td>J.H.CATE,M.M.YUSUPOV,G.ZH.YUSUPOVA,T.N.EARNEST,H.F.NOLLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NKB" title="Link to PDB">3NKB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NKB/index.html" title="Link to FR3D analysis. 0.4429 pairs per nucleotide">3NKB</a></td><td align="center">AB</td><td align="center">70</td><td align="center">31</td><td align="center">1.9</td><td sorttable_customkey="20100109000000">9-01-10</td><td>HOMO SAPIENS</td><td>A 1.9A CRYSTAL STRUCTURE OF THE HDV RIBOZYME PRECLEAVAGE SUGGESTS BOTH LEWIS ACID AND GENERAL ACID MECHANISMS CONTRIBUTE TO PHOSPHODIESTER CLEAVAGE</td><td>J.-H.CHEN,R.YAJIMA,D.M.CHADALAVADA,E.CHASE,P.C.BEVILACQUA,B.L.GOLDEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3EGZ" title="Link to PDB">3EGZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EGZ/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">3EGZ</a></td><td align="center">B</td><td align="center">64</td><td align="center">20</td><td align="center">2.2</td><td sorttable_customkey="20100410000000">10-28-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN IN VITRO EVOLVED TETRACYCLINE APTAMER AND ARTIFICIAL RIBOSWITCH</td><td>H.XIAO,T.E.EDWARDS,A.R.FERRE-D'AMARE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YFG" title="Link to PDB">1YFG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YFG/index.html" title="Link to FR3D analysis. 0.3594 pairs per nucleotide">1YFG</a></td><td align="center">A</td><td align="center">64</td><td align="center">23</td><td align="center">3.0</td><td sorttable_customkey="19970309000000">9-03-97</td><td>SACCHAROMYCES CEREVISIAE</td><td>YEAST INITIATOR TRNA</td><td>R.BASAVAPPA,P.B.SIGLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2WWA" title="Link to PDB">2WWA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WWA/index.html" title="Link to FR3D analysis. 0.3214 pairs per nucleotide">2WWA</a></td><td align="center">DEFG</td><td align="center">140</td><td align="center">45</td><td align="center">8.9</td><td sorttable_customkey="20090812000000">12-08-09</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>CRYO-EM STRUCTURES OF IDLE YEAST SSH1 COMPLEX BOUND TO THE YEAST 80S RIBOSOME</td><td>T.BECKER,E.MANDON,S.BHUSHAN,A.JARASCH,J.P.ARMACHE,S.FUNES, F.JOSSINET,J.GUMBART,T.MIELKE,O.BERNINGHAUSEN,K.SCHULTEN, E.WESTHOF,R.GILMORE,R.BECKMANN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WW9/index.html" title="Resolution 8.6A, 140 nucleotides, 44 basepairs, 0.3143 pairs per nucleotide">2WW9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WWA/2WWA_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F7V" title="Link to PDB">1F7V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7V/index.html" title="Link to FR3D analysis. 0.3667 pairs per nucleotide">1F7V</a></td><td align="center">B</td><td align="center">60</td><td align="center">22</td><td align="center">2.9</td><td sorttable_customkey="20030306000000">6-27-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF YEAST ARGINYL-TRNA SYNTHETASE COMPLEXED WITH THE TRNAARG</td><td>B.DELAGOUTTE,D.MORAS,J.CAVARELLI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7U/index.html" title="Resolution 2.2A, 63 nucleotides, 22 basepairs, 0.3492 pairs per nucleotide">1F7U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7V/1F7V_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2WWB" title="Link to PDB">2WWB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WWB/index.html" title="Link to FR3D analysis. 0.3456 pairs per nucleotide">2WWB</a></td><td align="center">DEFG</td><td align="center">136</td><td align="center">47</td><td align="center">6.5</td><td sorttable_customkey="20090812000000">12-08-09</td><td>TRITICUM AESTIVUM,TRITICUM AESTIVUM,TRITICUM AESTIVUM,TRITICUM AESTIVUM</td><td>CRYO-EM STRUCTURE OF THE MAMMALIAN SEC61 COMPLEX BOUND TO THE ACTIVELY TRANSLATING WHEAT GERM 80S RIBOSOME</td><td>T.BECKER,E.MANDON,S.BHUSHAN,A.JARASCH,J.P.ARMACHE,S.FUNES, F.JOSSINET,J.GUMBART,T.MIELKE,O.BERNINGHAUSEN,K.SCHULTEN, E.WESTHOF,R.GILMORE,R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ICQ" title="Link to PDB">3ICQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ICQ/index.html" title="Link to FR3D analysis. 0.3115 pairs per nucleotide">3ICQ</a></td><td align="center">E</td><td align="center">122</td><td align="center">38</td><td align="center">3.2</td><td sorttable_customkey="20110108000000">8-25-09</td><td>synthetic</td><td>KARYOPHERIN NUCLEAR STATE</td><td>A.G.COOK,N.FUKUHARA,M.JINEK,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FIR" title="Link to PDB">1FIR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FIR/index.html" title="Link to FR3D analysis. 0.3607 pairs per nucleotide">1FIR</a></td><td align="center">A</td><td align="center">61</td><td align="center">22</td><td align="center">3.3</td><td sorttable_customkey="20020501000000">1-17-01</td><td>BOS TAURUS</td><td>CRYSTAL STRUCTURE OF HIV-1 REVERSE TRANSCRIPTION PRIMER TRNA(LYS3)</td><td>P.BENAS,P.DUMAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UN6" title="Link to PDB">1UN6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UN6/index.html" title="Link to FR3D analysis. 0.4098 pairs per nucleotide">1UN6</a></td><td align="center">F</td><td align="center">122</td><td align="center">50</td><td align="center">3.1</td><td sorttable_customkey="20040811000000">11-20-03</td><td>XENOPUS LAEVIS</td><td>THE CRYSTAL STRUCTURE OF A ZINC FINGER - RNA COMPLEX REVEALS TWO MODES OF MOLECULAR RECOGNITION</td><td>D.LU,M.A.SEARLES,A.KLUG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SER" title="Link to PDB">1SER</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SER/index.html" title="Link to FR3D analysis. 0.4098 pairs per nucleotide">1SER</a></td><td align="center">T</td><td align="center">61</td><td align="center">25</td><td align="center">2.9</td><td sorttable_customkey="19960604000000">4-30-94</td><td>THERMUS THERMOPHILUS</td><td>THE 2.9 ANGSTROMS CRYSTAL STRUCTURE OF T. THERMOPHILUS SERYL-TRNA SYNTHETASE COMPLEXED WITH TRNA SER</td><td>S.BIOU,V.CUSACK,A.YAREMCHUK,M.TUKALO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2CZJ" title="Link to PDB">2CZJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CZJ/index.html" title="Link to FR3D analysis. 0.2875 pairs per nucleotide">2CZJ</a></td><td align="center">B</td><td align="center">240</td><td align="center">69</td><td align="center">3.0</td><td sorttable_customkey="20080710000000">10-31-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE TRNA DOMAIN OF TMRNA FROM THERMUS THERMOPHILUS HB8</td><td>Y.BESSHO,R.SHIBATA,S.SEKINE,K.MURAYAMA,M.SHIROUZU, S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HJW" title="Link to PDB">3HJW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HJW/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">3HJW</a></td><td align="center">DE</td><td align="center">70</td><td align="center">30</td><td align="center">2.4</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic,synthetic</td><td>STRUCTURE OF A FUNCTIONAL RIBONUCLEOPROTEIN PSEUDOURIDINE SYNTHASE BOUND TO A SUBSTRATE RNA</td><td>B.LIANG,J.ZHOU,E.KAHEN,R.M.TERNS,M.P.TERNS,H.LI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LWQ/index.html" title="Resolution 2.7A, 70 nucleotides, 28 basepairs, 0.4000 pairs per nucleotide">3LWQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LWP/index.html" title="Resolution 2.5A, 70 nucleotides, 28 basepairs, 0.4000 pairs per nucleotide">3LWP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LWV/index.html" title="Resolution 2.5A, 69 nucleotides, 27 basepairs, 0.3913 pairs per nucleotide">3LWV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LWR/index.html" title="Resolution 2.2A, 70 nucleotides, 27 basepairs, 0.3857 pairs per nucleotide">3LWR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LWO/index.html" title="Resolution 2.9A, 70 nucleotides, 27 basepairs, 0.3857 pairs per nucleotide">3LWO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HJW/3HJW_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QA6" title="Link to PDB">1QA6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QA6/index.html" title="Link to FR3D analysis. 0.4655 pairs per nucleotide">1QA6</a></td><td align="center">C</td><td align="center">116</td><td align="center">54</td><td align="center">2.8</td><td sorttable_customkey="20010105000000">5-25-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A CONSERVED RIBOSOMAL PROTEIN-RNA COMPLEX</td><td>G.L.CONN,D.E.DRAPER,E.E.LATTMAN,A.G.GITTIS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HC8/index.html" title="Resolution 2.8A, 114 nucleotides, 52 basepairs, 0.4561 pairs per nucleotide">1HC8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y39/index.html" title="Resolution 2.8A, 114 nucleotides, 51 basepairs, 0.4474 pairs per nucleotide">1Y39</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C04/index.html" title="Resolution 5.0A, 87 nucleotides, 38 basepairs, 0.4368 pairs per nucleotide">1C04</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QA6/1QA6_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MMS" title="Link to PDB">1MMS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MMS/index.html" title="Link to FR3D analysis. 0.4483 pairs per nucleotide">1MMS</a></td><td align="center">C</td><td align="center">116</td><td align="center">52</td><td align="center">2.6</td><td sorttable_customkey="20010504000000">4-17-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE RIBOSOMAL PROTEIN L11-RNA COMPLEX</td><td>B.T.WIMBERLY,R.GUYMON,J.P.MCCUTCHEON,S.W.WHITE, V.RAMAKRISHNAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OLN/index.html" title="SOLUTION NMR; THEORETICAL MODEL, 58 nucleotides, 26 basepairs, 0.4483 pairs per nucleotide">1OLN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JQ7/index.html" title="SOLUTION NMR, 580 nucleotides, 196 basepairs, 0.3379 pairs per nucleotide">2JQ7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MMS/1MMS_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HAX" title="Link to PDB">3HAX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HAX/index.html" title="Link to FR3D analysis. 0.4366 pairs per nucleotide">3HAX</a></td><td align="center">EF</td><td align="center">71</td><td align="center">31</td><td align="center">2.1</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A SUBSTRATE-BOUND GAR1-MINUS H/ACA RNP FROM PYROCOCCUS FURIOSUS</td><td>K.YE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F7Y" title="Link to PDB">1F7Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7Y/index.html" title="Link to FR3D analysis. 0.4211 pairs per nucleotide">1F7Y</a></td><td align="center">B</td><td align="center">57</td><td align="center">24</td><td align="center">2.8</td><td sorttable_customkey="20011011000000">11-22-00</td><td>synthetic</td><td>THE CRYSTAL STRUCTURE OF TWO UUCG LOOPS HIGHLIGHTS THE ROLE PLAYED BY 2'-HYDROXYL GROUPS IN ITS UNUSUAL STABILITY</td><td>E.ENNIFAR,A.NIKOULINE,A.SERGANOV,S.TISHCHENKO,N.NEVSKAYA, M.GARBER,B.EHRESMANN,C.EHRESMANN,S.NIKONOV,P.DUMAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DK1/index.html" title="Resolution 2.8A, 57 nucleotides, 23 basepairs, 0.4035 pairs per nucleotide">1DK1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KUQ/index.html" title="Resolution 2.8A, 57 nucleotides, 22 basepairs, 0.3860 pairs per nucleotide">1KUQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7Y/1F7Y_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HVY" title="Link to PDB">2HVY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HVY/index.html" title="Link to FR3D analysis. 0.3509 pairs per nucleotide">2HVY</a></td><td align="center">E</td><td align="center">57</td><td align="center">20</td><td align="center">2.3</td><td sorttable_customkey="20061209000000">9-12-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN H/ACA BOX RNP FROM PYROCOCCUS FURIOSUS</td><td>K.YE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DLC" title="Link to PDB">2DLC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DLC/index.html" title="Link to FR3D analysis. 0.3509 pairs per nucleotide">2DLC</a></td><td align="center">Y</td><td align="center">57</td><td align="center">20</td><td align="center">2.4</td><td sorttable_customkey="20071206000000">6-12-07</td><td>SACCHAROMYCES CEREVISIAE</td><td>CRYSTAL STRUCTURE OF THE TERNARY COMPLEX OF YEAST TYROSYL- TRNA SYNTHETASE</td><td>M.TSUNODA,Y.KUSAKABE,N.TANAKA,K.T.NAKAMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NRE" title="Link to PDB">2NRE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NRE/index.html" title="Link to FR3D analysis. 0.3214 pairs per nucleotide">2NRE</a></td><td align="center">F</td><td align="center">56</td><td align="center">18</td><td align="center">4.0</td><td sorttable_customkey="20080305000000">5-15-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PSEUDOUDIRINDE SYNTHASE TRUA IN COMPLEX WITH LEUCYL TRNA</td><td>S.HUR,R.M.STROUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GTF" title="Link to PDB">1GTF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GTF/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1GTF</a></td><td align="center">W</td><td align="center">44</td><td align="center">0</td><td align="center">1.8</td><td sorttable_customkey="20020504000000">4-05-02</td><td>synthetic</td><td>THE STRUCTURE OF THE TRP RNA-BINDING ATTENUATION PROTEIN (TRAP) BOUND TO A 53-NUCLEOTIDE RNA MOLECULE CONTAINING GAGUU REPEATS</td><td>N.H.HOPCROFT,A.L.WENDT,P.GOLLNICK,A.A.ANTSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C9S/index.html" title="Resolution 1.9A, 55 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1C9S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GTF/1GTF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KZL" title="Link to PDB">2KZL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KZL/index.html" title="Link to FR3D analysis. 0.3000 pairs per nucleotide">2KZL</a></td><td align="center">A</td><td align="center">440</td><td align="center">132</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20121102000000">2-23-11</td><td>BACILLUS SUBTILIS SUBSP. SUBTILIS STR. 168</td><td>SPECIFIER DOMAIN AND GA MOTIF REGION OF B. SUBTILIS TYRS T BOX LEADER RNA</td><td>J.WANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MZP" title="Link to PDB">1MZP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MZP/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">1MZP</a></td><td align="center">B</td><td align="center">55</td><td align="center">25</td><td align="center">2.6</td><td sorttable_customkey="20040901000000">1-21-03</td><td>synthetic</td><td>STRUCTURE OF THE L1 PROTUBERANCE IN THE RIBOSOME</td><td>A.NIKULIN,I.ELISEIKINA,S.TISHCHENKO,N.NEVSKAYA,N.DAVYDOVA, O.PLATONOVA,W.PIENDL,M.SELMER,A.LILJAS,R.ZIMMERMANN, M.GARBER,S.NIKONOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1P5M" title="Link to PDB">1P5M</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P5M/index.html" title="Link to FR3D analysis. 0.3438 pairs per nucleotide">1P5M</a></td><td align="center">A</td><td align="center">1210</td><td align="center">416</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030411000000">11-04-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HCV IRES DOMAIN IIA</td><td>P.J.LUKAVSKY,I.KIM,G.A.OTTO,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HGH" title="Link to PDB">2HGH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HGH/index.html" title="Link to FR3D analysis. 0.4364 pairs per nucleotide">2HGH</a></td><td align="center">B</td><td align="center">1100</td><td align="center">480</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060108000000">8-01-06</td><td>XENOPUS LAEVIS</td><td>TRANSCRIPTION FACTOR IIIA ZINC FINGERS 4-6 BOUND TO 5S RRNA 55MER (NMR STRUCTURE)</td><td>B.M.LEE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FK6" title="Link to PDB">2FK6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FK6/index.html" title="Link to FR3D analysis. 0.3774 pairs per nucleotide">2FK6</a></td><td align="center">R</td><td align="center">53</td><td align="center">20</td><td align="center">2.9</td><td sorttable_customkey="20070903000000">3-21-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNASE Z/TRNA(THR) COMPLEX</td><td>I.LI DE LA SIERRA-GALLAY,N.MATHY,O.PELLEGRINI,C.CONDON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3E5C" title="Link to PDB">3E5C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E5C/index.html" title="Link to FR3D analysis. 0.4038 pairs per nucleotide">3E5C</a></td><td align="center">A</td><td align="center">52</td><td align="center">21</td><td align="center">2.2</td><td sorttable_customkey="20080710000000">10-07-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE SMK BOX (SAM-III) RIBOSWITCH WITH SAM</td><td>C.LU</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E5F/index.html" title="Resolution 2.7A, 52 nucleotides, 21 basepairs, 0.4038 pairs per nucleotide">3E5F</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E5E/index.html" title="Resolution 2.9A, 52 nucleotides, 20 basepairs, 0.3846 pairs per nucleotide">3E5E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E5C/3E5C_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QWY" title="Link to PDB">2QWY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QWY/index.html" title="Link to FR3D analysis. 0.3974 pairs per nucleotide">2QWY</a></td><td align="center">AB</td><td align="center">156</td><td align="center">62</td><td align="center">2.8</td><td sorttable_customkey="20091001000000">1-22-08</td><td>synthetic</td><td>SAM-II RIBOSWITCH BOUND TO S-ADENOSYLMETHIONINE</td><td>S.D.GILBERT,R.P.RAMBO,D.VAN TYNE,R.T.BATEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NPN" title="Link to PDB">3NPN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NPN/index.html" title="Link to FR3D analysis. 0.3600 pairs per nucleotide">3NPN</a></td><td align="center">A</td><td align="center">50</td><td align="center">18</td><td align="center">2.8</td><td sorttable_customkey="20100610000000">10-06-10</td><td>RALSTONIA SOLANACEARUM</td><td>STRUCTURE OF THE S-ADENOSYLHOMOCYSTEINE RIBOSWITCH AT 3.0A</td><td>F.E.REYES,A.E.EDWARDS,R.T.BATEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NPQ/index.html" title="Resolution 2.2A, 150 nucleotides, 53 basepairs, 0.3533 pairs per nucleotide">3NPQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NPN/3NPN_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RMN" title="Link to PDB">1RMN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RMN/index.html" title="Link to FR3D analysis. 0.3878 pairs per nucleotide">1RMN</a></td><td align="center">A</td><td align="center">49</td><td align="center">19</td><td align="center">FLUORESCENCE TRANSFER</td><td sorttable_customkey="19970201000000">1-26-95</td><td>synthetic</td><td>A THREE-DIMENSIONAL MODEL FOR THE HAMMERHEAD RIBOZYME BASED ON FLUORESCENCE MEASUREMENTS</td><td>T.TUSCHL,C.GOHLKE,T.M.JOVIN,E.WESTHOF,F.ECKSTEIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1E8O" title="Link to PDB">1E8O</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1E8O/index.html" title="Link to FR3D analysis. 0.3673 pairs per nucleotide">1E8O</a></td><td align="center">E</td><td align="center">49</td><td align="center">18</td><td align="center">3.2</td><td sorttable_customkey="20000811000000">11-08-00</td><td>HOMO SAPIENS</td><td>CORE OF THE ALU DOMAIN OF THE MAMMALIAN SRP</td><td>O.WEICHENRIEDER,K.WILD,K.STRUB,S.CUSACK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U63" title="Link to PDB">1U63</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U63/index.html" title="Link to FR3D analysis. 0.4082 pairs per nucleotide">1U63</a></td><td align="center">B</td><td align="center">98</td><td align="center">40</td><td align="center">3.4</td><td sorttable_customkey="20051204000000">4-12-05</td><td>synthetic</td><td>THE STRUCTURE OF A RIBOSOMAL PROTEIN L1-MRNA COMPLEX</td><td>N.NEVSKAYA,S.TISHCHENKO,A.GABDOULKHAKOV,E.NIKONOVA, O.NIKONOV,A.NIKULIN,M.GARBER,S.NIKONOV,W.PIENDL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VPL" title="Link to PDB">2VPL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VPL/index.html" title="Link to FR3D analysis. 0.4043 pairs per nucleotide">2VPL</a></td><td align="center">B</td><td align="center">94</td><td align="center">38</td><td align="center">2.3</td><td sorttable_customkey="20091109000000">9-23-08</td><td>METHANOCALDOCOCCUS JANNASCHII</td><td>THE STRUCTURE OF THE COMPLEX BETWEEN THE FIRST DOMAIN OF L1 PROTEIN FROM THERMUS THERMOPHILUS AND MRNA FROM METHANOCOCCUS JANNASCHII</td><td>V.KLJASHTORNY,S.TISHCHENKO,N.NEVSKAYA,S.NIKONOV,M.GARBER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KUW" title="Link to PDB">2KUW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KUW/index.html" title="Link to FR3D analysis. 0.3939 pairs per nucleotide">2KUW</a></td><td align="center">A</td><td align="center">528</td><td align="center">208</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110705000000">5-19-10</td><td>DROSOPHILA MELANOGASTER</td><td>SOLUTION STRUCTURE OF K10 TLS RNA (A-FORM MUTANT IN LOWER HELIX)</td><td>S.L.BULLOCK,I.RINGEL,D.ISH-HOROWICZ,P.J.LUKAVSKY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KUU" title="Link to PDB">2KUU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KUU/index.html" title="Link to FR3D analysis. 0.3993 pairs per nucleotide">2KUU</a></td><td align="center">A</td><td align="center">576</td><td align="center">230</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110705000000">5-19-10</td><td>DROSOPHILA MELANOGASTER</td><td>SOLUTION STRUCTURE OF K10 TLS RNA (GC MUTANT IN UPPER HELIX)</td><td>S.L.BULLOCK,I.RINGEL,D.ISH-HOROWICZ,P.J.LUKAVSKY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KUR/index.html" title="SOLUTION NMR, 480 nucleotides, 182 basepairs, 0.3792 pairs per nucleotide">2KUR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KUV/index.html" title="SOLUTION NMR, 480 nucleotides, 169 basepairs, 0.3521 pairs per nucleotide">2KUV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KE6/index.html" title="SOLUTION NMR, 576 nucleotides, 177 basepairs, 0.3073 pairs per nucleotide">2KE6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KUU/2KUU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YMO" title="Link to PDB">1YMO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YMO/index.html" title="Link to FR3D analysis. 0.4436 pairs per nucleotide">1YMO</a></td><td align="center">A</td><td align="center">940</td><td align="center">417</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060303000000">3-15-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE P2B-P3 PSEUDOKNOT FROM HUMAN TELOMERASE RNA</td><td>C.A.THEIMER,C.A.BLOIS,J.FEIGON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K96/index.html" title="SOLUTION NMR, 940 nucleotides, 399 basepairs, 0.4245 pairs per nucleotide">2K96</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K95/index.html" title="SOLUTION NMR, 960 nucleotides, 404 basepairs, 0.4208 pairs per nucleotide">2K95</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YMO/1YMO_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S03" title="Link to PDB">1S03</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S03/index.html" title="Link to FR3D analysis. 0.4468 pairs per nucleotide">1S03</a></td><td align="center">B</td><td align="center">94</td><td align="center">42</td><td align="center">2.7</td><td sorttable_customkey="20060105000000">5-25-04</td><td>synthetic</td><td>THE STRUCTURE OF A RIBOSOMAL PROTEIN S8/SPC OPERON MRNA COMPLEX</td><td>H.J.MERIANOS,J.WANG,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CIY" title="Link to PDB">3CIY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CIY/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3CIY</a></td><td align="center">CD</td><td align="center">92</td><td align="center">46</td><td align="center">3.4</td><td sorttable_customkey="20080605000000">5-06-08</td><td>synthetic,MUS MUSCULUS</td><td>MOUSE TOLL-LIKE RECEPTOR 3 ECTODOMAIN COMPLEXED WITH DOUBLE- STRANDED RNA</td><td>L.LIU,I.BOTOS,Y.WANG,J.N.LEONARD,J.SHILOACH,D.M.SEGAL, D.R.DAVIES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XJR" title="Link to PDB">1XJR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XJR/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">1XJR</a></td><td align="center">A</td><td align="center">46</td><td align="center">20</td><td align="center">2.7</td><td sorttable_customkey="20050102000000">2-01-05</td><td>synthetic</td><td>THE STRUCTURE OF A RIGOROUSLY CONSERVED RNA ELEMENT WITHIN THE SARS VIRUS GENOME</td><td>M.P.ROBERTSON,H.IGEL,R.BAERTSCH,D.HAUSSLER,M.ARES JR., W.G.SCOTT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IAB" title="Link to PDB">3IAB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IAB/index.html" title="Link to FR3D analysis. 0.2826 pairs per nucleotide">3IAB</a></td><td align="center">R</td><td align="center">46</td><td align="center">13</td><td align="center">2.7</td><td sorttable_customkey="20120201000000">1-26-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNASE P /RNASE MRP PROTEINS POP6, POP7 IN A COMPLEX WITH THE P3 DOMAIN OF RNASE MRP RNA</td><td>A.PEREDERINA,O.A.ESAKOVA,C.QUAN,E.KHANOVA,A.S.KRASILNIKOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NUE" title="Link to PDB">2NUE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NUE/index.html" title="Link to FR3D analysis. 0.4565 pairs per nucleotide">2NUE</a></td><td align="center">C</td><td align="center">46</td><td align="center">21</td><td align="center">2.9</td><td sorttable_customkey="20080811000000">11-20-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNASE III FROM AQUIFEX AEOLICUS COMPLEXED WITH DS-RNA AT 2.9-ANGSTROM RESOLUTION</td><td>J.H.GAN,G.SHAW,J.E.TROPEA,D.S.WAUGH,D.L.COURT,X.JI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PU1" title="Link to PDB">3PU1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PU1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PU1</a></td><td align="center">R</td><td align="center">45</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A VESICULAR STOMATITIS VIRUS NUCLEOCAPSID-POLYG COMPLEX</td><td>M.LUO,T.J.GREEN,M.ROWSE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GIC" title="Link to PDB">2GIC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2GIC</a></td><td align="center">R</td><td align="center">45</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20071008000000">8-22-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A VESICULAR STOMATITIS VIRUS NUCLEOCAPSID-RNA COMPLEX</td><td>T.J.GREEN,X.ZHANG,G.W.WERTZ,M.LUO</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PU4/index.html" title="Resolution 3.0A, 45 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3PU4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HHZ/index.html" title="Resolution 3.5A, 45 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HHZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIC/2GIC_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Z2J" title="Link to PDB">1Z2J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z2J/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1Z2J</a></td><td align="center">A</td><td align="center">900</td><td align="center">400</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060206000000">6-14-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HIV-1 FRAMESHIFT INDUCING ELEMENT</td><td>D.W.STAPLE,S.E.BUTCHER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZC5/index.html" title="SOLUTION NMR, 820 nucleotides, 347 basepairs, 0.4232 pairs per nucleotide">1ZC5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JUK/index.html" title="SOLUTION NMR, 440 nucleotides, 179 basepairs, 0.4068 pairs per nucleotide">2JUK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z2J/1Z2J_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1P6V" title="Link to PDB">1P6V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P6V/index.html" title="Link to FR3D analysis. 0.2464 pairs per nucleotide">1P6V</a></td><td align="center">B</td><td align="center">69</td><td align="center">17</td><td align="center">3.2</td><td sorttable_customkey="20031208000000">8-12-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE TRNA DOMAIN OF TRANSFER-MESSENGER RNA IN COMPLEX WITH SMPB</td><td>S.GUTMANN,P.W.HAEBEL,L.METZINGER,M.SUTTER,B.FELDEN,N.BAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PU0" title="Link to PDB">3PU0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PU0/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PU0</a></td><td align="center">R</td><td align="center">45</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A VESICULAR STOMATITIS VIRUS NUCLEOCAPSID-POLYC COMPLEX</td><td>M.LUO,T.J.GREEN,M.ROWSE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2WYY" title="Link to PDB">2WYY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WYY/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2WYY</a></td><td align="center">R</td><td align="center">90</td><td align="center">0</td><td align="center">10.0</td><td sorttable_customkey="20110402000000">2-16-10</td><td>synthetic</td><td>CRYOEM MODEL OF THE VESICULAR STOMATITIS VIRUS</td><td>P.GE,J.TSAO,T.J.GREEN,M.LUO,Z.H.ZHOU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QZW" title="Link to PDB">1QZW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QZW/index.html" title="Link to FR3D analysis. 0.2444 pairs per nucleotide">1QZW</a></td><td align="center">B</td><td align="center">180</td><td align="center">44</td><td align="center">4.1</td><td sorttable_customkey="20040611000000">11-18-03</td><td>SULFOLOBUS SOLFATARICUS</td><td>CRYSTAL STRUCTURE OF THE COMPLETE CORE OF ARCHAEAL SRP AND IMPLICATIONS FOR INTER-DOMAIN COMMUNICATION</td><td>K.R.ROSENDAL,K.WILD,G.MONTOYA,I.SINNING</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PTX" title="Link to PDB">3PTX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PTX/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PTX</a></td><td align="center">R</td><td align="center">45</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A VESICULAR STOMATITIS VIRUS NUCLEOCAPSID-POLYA COMPLEX</td><td>M.LUO,T.J.GREEN,M.ROWSE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GTN" title="Link to PDB">1GTN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GTN/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1GTN</a></td><td align="center">W</td><td align="center">44</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20020504000000">4-05-02</td><td>synthetic</td><td>STRUCTURE OF THE TRP RNA-BINDING ATTENUATION PROTEIN (TRAP) BOUND TO AN RNA MOLECULE CONTAINING 11 GAGCC REPEATS</td><td>N.H.HOPCROFT,A.L.WENDT,P.GOLLNICK,A.A.ANTSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1G1X" title="Link to PDB">1G1X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G1X/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">1G1X</a></td><td align="center">DE</td><td align="center">84</td><td align="center">32</td><td align="center">2.6</td><td sorttable_customkey="20020610000000">10-30-00</td><td>synthetic,synthetic</td><td>STRUCTURE OF RIBOSOMAL PROTEINS S15, S6, S18, AND 16S RIBOSOMAL RNA</td><td>S.C.AGALAROV,G.S.PRASAD,P.M.FUNKE,C.D.STOUT,J.R.WILLIAMSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A60" title="Link to PDB">1A60</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A60/index.html" title="Link to FR3D analysis. 0.2955 pairs per nucleotide">1A60</a></td><td align="center">A</td><td align="center">1056</td><td align="center">312</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000305000000">5-27-98</td><td>TURNIP YELLOW MOSAIC VIRUS</td><td>NMR STRUCTURE OF A CLASSICAL PSEUDOKNOT: INTERPLAY OF SINGLE-AND DOUBLE-STRANDED RNA, 24 STRUCTURES</td><td>M.H.KOLK,M.VAN DER GRAAF,S.S.WIJMENGA,C.W.A.PLEIJ,H.A.HEUS, C.W.HILBERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ADT" title="Link to PDB">2ADT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ADT/index.html" title="Link to FR3D analysis. 0.5355 pairs per nucleotide">2ADT</a></td><td align="center">AB</td><td align="center">1720</td><td align="center">921</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070207000000">7-26-05</td><td>synthetic</td><td>NMR STRUCTURE OF A 30 KDA GAAA TETRALOOP-RECEPTOR COMPLEX.</td><td>J.H.DAVIS,M.TONELLI,L.G.SCOTT,L.JAEGER,J.R.WILLIAMSON, S.E.BUTCHER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JYF/index.html" title="SOLUTION NMR, 860 nucleotides, 457 basepairs, 0.5314 pairs per nucleotide">2JYF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I7E/index.html" title="SOLUTION NMR, 1720 nucleotides, 914 basepairs, 0.5314 pairs per nucleotide">2I7E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I7Z/index.html" title="SOLUTION NMR, 1720 nucleotides, 908 basepairs, 0.5279 pairs per nucleotide">2I7Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JYJ/index.html" title="SOLUTION NMR, 860 nucleotides, 445 basepairs, 0.5174 pairs per nucleotide">2JYJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JYH/index.html" title="SOLUTION NMR, 430 nucleotides, 210 basepairs, 0.4884 pairs per nucleotide">2JYH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ADT/2ADT_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FEY" title="Link to PDB">2FEY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FEY/index.html" title="Link to FR3D analysis. 0.2791 pairs per nucleotide">2FEY</a></td><td align="center">A</td><td align="center">430</td><td align="center">120</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080306000000">6-27-06</td><td>synthetic</td><td>THE STRUCTURE OF STEM LOOP IV OF TETRAHYMENA TELOMERASE RNA</td><td>Y.CHEN,J.FENDER,J.D.LEGASSIE,M.B.JARSTFER,T.M.BRYAN,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1CQL" title="Link to PDB">1CQL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CQL/index.html" title="Link to FR3D analysis. 0.3140 pairs per nucleotide">1CQL</a></td><td align="center">A</td><td align="center">430</td><td align="center">135</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010201000000">1-14-00</td><td>ESCHERICHIA COLI</td><td>NMR STRUCTURE OF SRP RNA DOMAIN IV</td><td>U.SCHMITZ,T.L.JAMES</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CQ5/index.html" title="SOLUTION NMR, 43 nucleotides, 13 basepairs, 0.3023 pairs per nucleotide">1CQ5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CQL/1CQL_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L2J" title="Link to PDB">2L2J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L2J/index.html" title="Link to FR3D analysis. 0.4440 pairs per nucleotide">2L2J</a></td><td align="center">A</td><td align="center">840</td><td align="center">373</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120310000000">10-27-10</td><td>HOMO SAPIENS</td><td>SOLUTION NMR STRUCTURE OF THE LOWER PART OF THE R/G STEM LOOP RNA</td><td>F.F.OBERSTRASS,F.H.-T.ALLAIN,R.STEFL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L2K/index.html" title="SOLUTION NMR, 840 nucleotides, 368 basepairs, 0.4381 pairs per nucleotide">2L2K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L2J/2L2J_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KX8" title="Link to PDB">2KX8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KX8/index.html" title="Link to FR3D analysis. 0.4048 pairs per nucleotide">2KX8</a></td><td align="center">A</td><td align="center">42</td><td align="center">17</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100809000000">9-08-10</td><td>HOMO SAPIENS</td><td>NMR STRUCTURE OF STEM-LOOP 4 FROM THE HUMAN 7SK SNRNA IN COMPLEX WITH ARGININE</td><td>M.A.DURNEY,V.M.D'SOUZA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MNX" title="Link to PDB">1MNX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MNX/index.html" title="Link to FR3D analysis. 0.3919 pairs per nucleotide">1MNX</a></td><td align="center">A</td><td align="center">546</td><td align="center">214</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040201000000">1-14-03</td><td>SPINACIA OLERACEA</td><td>THE SOLUTION STRUCTURE OF THE LOOP E REGION OF THE 5S RRNA FROM SPINACH CHLOROPLASTS.</td><td>P.VALLURUPALLI,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OEU" title="Link to PDB">2OEU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OEU/index.html" title="Link to FR3D analysis. 0.4068 pairs per nucleotide">2OEU</a></td><td align="center">AB</td><td align="center">59</td><td align="center">24</td><td align="center">2.0</td><td sorttable_customkey="20080802000000">2-20-07</td><td>synthetic,synthetic</td><td>FULL-LENGTH HAMMERHEAD RIBOZYME WITH MN(II) BOUND</td><td>M.MARTICK,W.G.SCOTT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GOZ/index.html" title="Resolution 2.2A, 58 nucleotides, 21 basepairs, 0.3621 pairs per nucleotide">2GOZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OEU/2OEU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BNT" title="Link to PDB">3BNT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNT/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">3BNT</a></td><td align="center">A</td><td align="center">42</td><td align="center">18</td><td align="center">2.3</td><td sorttable_customkey="20091206000000">6-24-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS MITOCHONDRIAL RIBOSOMAL DECODING SITE IN THE PRESENCE OF [CO(NH3)6]CL3 (A1555G MUTANT, BR-DERIVATIVE)</td><td>J.KONDO,E.WESTHOF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KH6" title="Link to PDB">1KH6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KH6/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">1KH6</a></td><td align="center">A</td><td align="center">42</td><td align="center">16</td><td align="center">2.9</td><td sorttable_customkey="20040204000000">4-26-02</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA TERTIARY DOMAIN ESSENTIAL TO HCV IRES-MEDIATED TRANSLATION INITIATION.</td><td>J.S.KIEFT,K.ZHOU,A.GRECH,R JUBIN,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AU4" title="Link to PDB">2AU4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AU4/index.html" title="Link to FR3D analysis. 0.2683 pairs per nucleotide">2AU4</a></td><td align="center">A</td><td align="center">41</td><td align="center">11</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080403000000">3-28-06</td><td>synthetic</td><td>CLASS I GTP APTAMER</td><td>J.M.CAROTHERS,J.H.DAVIS,J.J.CHOU,J.W.SZOSTAK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A51" title="Link to PDB">1A51</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A51/index.html" title="Link to FR3D analysis. 0.3848 pairs per nucleotide">1A51</a></td><td align="center">A</td><td align="center">369</td><td align="center">142</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000305000000">5-27-98</td><td>ESCHERICHIA COLI</td><td>LOOP D/LOOP E ARM OF E. COLI 5S RRNA, NMR, 9 STRUCTURES</td><td>A.DALLAS,P.B.MOORE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A4D/index.html" title="SOLUTION NMR, 41 nucleotides, 15 basepairs, 0.3659 pairs per nucleotide">1A4D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A51/1A51_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HUA" title="Link to PDB">2HUA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HUA/index.html" title="Link to FR3D analysis. 0.3386 pairs per nucleotide">2HUA</a></td><td align="center">A</td><td align="center">440</td><td align="center">149</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080504000000">4-17-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF CSFV IRES DOMAIN IIA</td><td>N.LOCKER,L.E.EASTON,P.J.LUKAVSKY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D1B" title="Link to PDB">2D1B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D1B/index.html" title="Link to FR3D analysis. 0.5128 pairs per nucleotide">2D1B</a></td><td align="center">AB</td><td align="center">78</td><td align="center">40</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic</td><td>SOLUTION RNA STRUCTURE MODEL OF THE HIV-1 DIMERIZATION INITIATION SITE IN THE KISSING-LOOP DIMER</td><td>S.BABA,K.TAKAHASHI,S.NOGUCHI,H.TAKAKU,Y.KOYANAGI,N.YAMAMOTO, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D1A" title="Link to PDB">2D1A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D1A/index.html" title="Link to FR3D analysis. 0.4872 pairs per nucleotide">2D1A</a></td><td align="center">AB</td><td align="center">78</td><td align="center">38</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic</td><td>SOLUTION RNA STRUCTURE MODEL OF THE HIV-1 DIMERIZATION INITIATION SITE IN THE EXTENDED-DUPLEX DIMER</td><td>S.BABA,K.TAKAHASHI,S.NOGUCHI,H.TAKAKU,Y.KOYANAGI,N.YAMAMOTO, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P22" title="Link to PDB">3P22</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P22/index.html" title="Link to FR3D analysis. 0.5474 pairs per nucleotide">3P22</a></td><td align="center">EFGH</td><td align="center">190</td><td align="center">104</td><td align="center">2.5</td><td sorttable_customkey="20100812000000">12-08-10</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE ENE, A VIRAL RNA STABILITY ELEMENT, IN COMPLEX WITH A9 RNA</td><td>R.M.MITTON-FRY,S.J.DEGREGORIO,J.WANG,T.A.STEITZ,J.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YKV" title="Link to PDB">1YKV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YKV/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1YKV</a></td><td align="center">CD</td><td align="center">98</td><td align="center">42</td><td align="center">3.3</td><td sorttable_customkey="20061002000000">2-22-05</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE DIELS-ALDER RIBOZYME COMPLEXED WITH THE PRODUCT OF THE REACTION BETWEEN N-PENTYLMALEIMIDE AND COVALENTLY ATTACHED 9-HYDROXYMETHYLANTHRACENE</td><td>A.SERGANOV,S.KEIPER,L.MALININA,V.TERESHKO,E.SKRIPKIN, C.HOBARTNER,A.POLONSKAIA,A.T.PHAN,R.WOMBACHER,R.MICURA, Z.DAUTER,A.JASCHKE,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YLS/index.html" title="Resolution 3.0A, 86 nucleotides, 34 basepairs, 0.3953 pairs per nucleotide">1YLS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YKQ/index.html" title="Resolution 3.5A, 98 nucleotides, 37 basepairs, 0.3776 pairs per nucleotide">1YKQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YKV/1YKV_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1B36" title="Link to PDB">1B36</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1B36/index.html" title="Link to FR3D analysis. 0.2789 pairs per nucleotide">1B36</a></td><td align="center">A</td><td align="center">380</td><td align="center">106</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19991112000000">12-23-98</td><td>SYNTHETIC CONSTRUCT</td><td>SOLUTION STRUCTURE OF THE HAIRPIN RIBOZYME LOOP B DOMAIN RNA, NMR, 10 STRUCTURES</td><td>S.E.BUTCHER,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KU0" title="Link to PDB">2KU0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KU0/index.html" title="Link to FR3D analysis. 0.3816 pairs per nucleotide">2KU0</a></td><td align="center">A</td><td align="center">380</td><td align="center">145</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120404000000">4-28-10</td><td>HEPATITIS C VIRUS</td><td>INHIBITOR INDUCED STRUCTURAL CHANGE IN THE HCV IRES DOMAIN IIA RNA</td><td>R.B.PAULSEN,P.P.SETH,E.E.SWAYZE,R.H.GRIFFEY,J.J.SKALICKY,T.E.CHEATHAM III,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U3K" title="Link to PDB">1U3K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U3K/index.html" title="Link to FR3D analysis. 0.3711 pairs per nucleotide">1U3K</a></td><td align="center">A</td><td align="center">380</td><td align="center">141</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20051207000000">7-12-05</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF A SUBSTRATE OF ARCHAEAL PRE-TRNA SPLICING ENDONUCLEASES</td><td>E.C.POLLOCK,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A9L" title="Link to PDB">2A9L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A9L/index.html" title="Link to FR3D analysis. 0.3684 pairs per nucleotide">2A9L</a></td><td align="center">A</td><td align="center">38</td><td align="center">14</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990506000000">6-17-98</td><td>synthetic</td><td>SOLUTION NMR STRUCTURE OF A SUBSTRATE FOR THE ARCHAEAL PRE- TRNA SPLICING ENDONUCLEASES: THE BULGE-HELIX-BULGE MOTIF, MINIMIZED AVERAGE STRUCTURE</td><td>J.L.DIENER,P.B.MOORE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A9L/index.html" title="SOLUTION NMR, 456 nucleotides, 168 basepairs, 0.3684 pairs per nucleotide">1A9L</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A9L/2A9L_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HW8" title="Link to PDB">2HW8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HW8/index.html" title="Link to FR3D analysis. 0.4722 pairs per nucleotide">2HW8</a></td><td align="center">B</td><td align="center">36</td><td align="center">17</td><td align="center">2.1</td><td sorttable_customkey="20070712000000">12-19-06</td><td>synthetic</td><td>STRUCTURE OF RIBOSOMAL PROTEIN L1-MRNA COMPLEX AT 2.1 RESOLUTION.</td><td>S.TISHCHENKO,E.NIKONOVA,A.NIKULIN,N.NEVSKAYA,S.VOLCHKOV, W.PIENDL,M.GARBER,S.NIKONOV</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZHO/index.html" title="Resolution 2.6A, 152 nucleotides, 71 basepairs, 0.4671 pairs per nucleotide">1ZHO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HW8/2HW8_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZY6" title="Link to PDB">2ZY6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZY6/index.html" title="Link to FR3D analysis. 0.3158 pairs per nucleotide">2ZY6</a></td><td align="center">A</td><td align="center">38</td><td align="center">12</td><td align="center">1.8</td><td sorttable_customkey="20110606000000">6-30-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A TRUNCATED TRNA, TPHE39A</td><td>I.TANAKA,M.YAO,Y.TANAKA,Y.KITAGO,S.YMAGATA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M5L" title="Link to PDB">1M5L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M5L/index.html" title="Link to FR3D analysis. 0.3982 pairs per nucleotide">1M5L</a></td><td align="center">A</td><td align="center">570</td><td align="center">227</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030609000000">9-18-02</td><td>synthetic</td><td>STRUCTURE OF WILD-TYPE AND MUTANT INTERNAL LOOPS FROM THE SL-1 DOMAIN OF THE HIV-1 PACKAGING SIGNAL</td><td>J.GALLEGO,J.GREATOREX,G.VARANI,A.LEVER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KTZ" title="Link to PDB">2KTZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KTZ/index.html" title="Link to FR3D analysis. 0.3684 pairs per nucleotide">2KTZ</a></td><td align="center">A</td><td align="center">380</td><td align="center">140</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120404000000">4-28-10</td><td>HEPATITIS C VIRUS</td><td>INHIBITOR INDUCED STRUCTURAL CHANGE IN THE HCV IRES DOMAIN IIA RNA</td><td>R.B.PAULSEN,P.P.SETH,E.E.SWAYZE,R.H.GRIFFEY,J.J.SKALICKY,T.E.CHEATHAM III,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KHY" title="Link to PDB">2KHY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KHY/index.html" title="Link to FR3D analysis. 0.2421 pairs per nucleotide">2KHY</a></td><td align="center">A</td><td align="center">380</td><td align="center">92</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110402000000">2-16-10</td><td>synthetic</td><td>SPECIFIER DOMAIN OF B. SUBTILIS TYRS T BOX LEADER RNA</td><td>J.WANG,T.M.HENKIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TXS" title="Link to PDB">1TXS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TXS/index.html" title="Link to FR3D analysis. 0.4447 pairs per nucleotide">1TXS</a></td><td align="center">A</td><td align="center">760</td><td align="center">338</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040510000000">10-05-04</td><td>synthetic</td><td>STEM-LOOP D OF THE CLOVERLEAF DOMAIN OF ENTEROVIRAL 5'UTR RNA</td><td>Z.DU,J.YU,N.B.ULYANOV,R.ANDINO,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Q8N" title="Link to PDB">1Q8N</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q8N/index.html" title="Link to FR3D analysis. 0.3632 pairs per nucleotide">1Q8N</a></td><td align="center">A</td><td align="center">950</td><td align="center">345</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20051103000000">3-23-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE MALACHITE GREEN RNA BINDING APTAMER</td><td>J.FLINDERS,S.C.DEFINA,D.M.BRACKETT,C.BAUGH,C.WILSON, T.DIECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1D6K" title="Link to PDB">1D6K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1D6K/index.html" title="Link to FR3D analysis. 0.3716 pairs per nucleotide">1D6K</a></td><td align="center">B</td><td align="center">740</td><td align="center">275</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20001011000000">11-22-99</td><td>ESCHERICHIA COLI</td><td>NMR SOLUTION STRUCTURE OF THE 5S RRNA E-LOOP/L25 COMPLEX</td><td>M.STOLDT,J.WOHNERT,O.OHLENSCHLAGER,M.GORLACH,L.R.BROWN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KOG" title="Link to PDB">1KOG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KOG/index.html" title="Link to FR3D analysis. 0.4459 pairs per nucleotide">1KOG</a></td><td align="center">J</td><td align="center">296</td><td align="center">132</td><td align="center">3.5</td><td sorttable_customkey="20040204000000">4-26-02</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF E. COLI THREONYL-TRNA SYNTHETASE INTERACTING WITH THE ESSENTIAL DOMAIN OF ITS MRNA OPERATOR</td><td>A.TORRES-LARRIOS,A.C.DOCK-BREGEON,P.ROMBY,B.REES, R.SANKARANARAYANAN,J.CAILLET,M.SPRINGER,C.EHRESMANN, B.EHRESMANN,D.MORAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I6U" title="Link to PDB">1I6U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I6U/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1I6U</a></td><td align="center">D</td><td align="center">74</td><td align="center">37</td><td align="center">2.6</td><td sorttable_customkey="20010308000000">8-03-01</td><td>synthetic</td><td>RNA-PROTEIN INTERACTIONS: THE CRYSTAL STRUCTURE OF RIBOSOMAL PROTEIN S8/RRNA COMPLEX FROM METHANOCOCCUS JANNASCHII</td><td>S.TISHCHENKO,A.NIKULIN,N.FOMENKOVA,N.NEVSKAYA,O.NIKONOV, P.DUMAS,H.MOINE,B.EHRESMANN,C.EHRESMANN,W.PIENDL,V.LAMZIN, M.GARBER,S.NIKONOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2P7E" title="Link to PDB">2P7E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2P7E/index.html" title="Link to FR3D analysis. 0.4667 pairs per nucleotide">2P7E</a></td><td align="center">ABCDP</td><td align="center">60</td><td align="center">28</td><td align="center">2.0</td><td sorttable_customkey="20081005000000">5-22-07</td><td>synthetic,synthetic,synthetic,synthetic</td><td>VANADATE AT THE ACTIVE SITE OF A SMALL RIBOZYME SUGGESTS A ROLE FOR WATER IN TRANSITION-STATE STABILIZATION</td><td>A.T.TORELLI,J.KRUCINSKA,J.E.WEDEKIND</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OUE/index.html" title="Resolution 2.0A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">2OUE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NPY/index.html" title="Resolution 2.6A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">2NPY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BCY/index.html" title="Resolution 2.7A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">2BCY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D2K/index.html" title="Resolution 2.6A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">2D2K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1X9C/index.html" title="Resolution 2.2A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">1X9C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZFX/index.html" title="Resolution 2.4A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">1ZFX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZFV/index.html" title="Resolution 2.4A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">1ZFV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GS8/index.html" title="Resolution 2.9A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">3GS8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2P7D/index.html" title="Resolution 2.2A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">2P7D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GS5/index.html" title="Resolution 2.8A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">3GS5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZFT/index.html" title="Resolution 2.3A, 59 nucleotides, 27 basepairs, 0.4576 pairs per nucleotide">1ZFT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FGP/index.html" title="Resolution 2.4A, 60 nucleotides, 27 basepairs, 0.4500 pairs per nucleotide">2FGP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2P7F/index.html" title="Resolution 2.4A, 60 nucleotides, 27 basepairs, 0.4500 pairs per nucleotide">2P7F</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GS1/index.html" title="Resolution 2.9A, 61 nucleotides, 27 basepairs, 0.4426 pairs per nucleotide">3GS1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BCZ/index.html" title="Resolution 2.4A, 59 nucleotides, 26 basepairs, 0.4407 pairs per nucleotide">2BCZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1X9K/index.html" title="Resolution 3.2A, 62 nucleotides, 26 basepairs, 0.4194 pairs per nucleotide">1X9K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NPZ/index.html" title="Resolution 3.4A, 61 nucleotides, 24 basepairs, 0.3934 pairs per nucleotide">2NPZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2P7E/2P7E_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1E95" title="Link to PDB">1E95</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1E95/index.html" title="Link to FR3D analysis. 0.3222 pairs per nucleotide">1E95</a></td><td align="center">A</td><td align="center">540</td><td align="center">174</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20021108000000">8-23-01</td><td>SIMIAN RETROVIRUS 1</td><td>SOLUTION STRUCTURE OF THE PSEUDOKNOT OF SRV-1 RNA, INVOLVED IN RIBOSOMAL FRAMESHIFTING</td><td>P.J.A.MICHIELS,A.VERSLEYEN,C.W.A.PLEIJ,C.W.HILBERS,H.A.HEUS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1N8X" title="Link to PDB">1N8X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N8X/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1N8X</a></td><td align="center">A</td><td align="center">720</td><td align="center">320</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030804000000">4-08-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HIV-1 STEM LOOP SL1</td><td>D.C.LAWRENCE,C.C.STOVER,J.NOZNITSKY,Z.WU,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2LA5" title="Link to PDB">2LA5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2LA5/index.html" title="Link to FR3D analysis. 0.5306 pairs per nucleotide">2LA5</a></td><td align="center">A</td><td align="center">360</td><td align="center">191</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110806000000">6-08-11</td><td>synthetic</td><td>RNA DUPLEX-QUADRUPLEX JUNCTION COMPLEX WITH FMRP RGG PEPTIDE</td><td>A.PHAN,V.KURYAVYI,J.DARNELL,A.SERGANOV,A.MAJUMDAR,S.ILIN,R.DARNELL, D.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FU2" title="Link to PDB">3FU2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FU2/index.html" title="Link to FR3D analysis. 0.5161 pairs per nucleotide">3FU2</a></td><td align="center">A</td><td align="center">93</td><td align="center">48</td><td align="center">2.9</td><td sorttable_customkey="20090303000000">3-03-09</td><td>synthetic</td><td>COCRYSTAL STRUCTURE OF A CLASS-I PREQ1 RIBOSWITCH</td><td>D.J.KLEIN,T.E.EDWARDS,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K1V/index.html" title="Resolution 2.2A, 29 nucleotides, 13 basepairs, 0.4483 pairs per nucleotide">3K1V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L1V/index.html" title="SOLUTION NMR, 720 nucleotides, 291 basepairs, 0.4042 pairs per nucleotide">2L1V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FU2/3FU2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RAW" title="Link to PDB">1RAW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RAW/index.html" title="Link to FR3D analysis. 0.2889 pairs per nucleotide">1RAW</a></td><td align="center">A</td><td align="center">360</td><td align="center">104</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990301000000">1-27-97</td><td>synthetic</td><td>ATP BINDING RNA APTAMER IN COMPLEX WITH AMP, NMR, 10 STRUCTURES</td><td>T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TPK" title="Link to PDB">2TPK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TPK/index.html" title="Link to FR3D analysis. 0.3611 pairs per nucleotide">2TPK</a></td><td align="center">A</td><td align="center">36</td><td align="center">13</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980411000000">11-04-98</td><td>synthetic</td><td>AN INVESTIGATION OF THE STRUCTURE OF THE PSEUDOKNOT WITHIN THE GENE 32 MESSENGER RNA OF BACTERIOPHAGE T2 USING HETERONUCLEAR NMR METHODS</td><td>J.A.HOLLAND,M.R.HANSEN,Z.DU,D.W.HOFFMAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FDT" title="Link to PDB">2FDT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FDT/index.html" title="Link to FR3D analysis. 0.4141 pairs per nucleotide">2FDT</a></td><td align="center">A</td><td align="center">396</td><td align="center">164</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060512000000">12-05-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A CONSERVED RNA HAIRPIN OF EEL LINE UNAL2</td><td>Y.NOMURA,S.BABA,S.NAKAZATO,T.SAKAMOTO,M.KAJIKAWA,N.OKADA, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KFO" title="Link to PDB">1KFO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KFO/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1KFO</a></td><td align="center">A</td><td align="center">36</td><td align="center">16</td><td align="center">1.6</td><td sorttable_customkey="20010712000000">12-07-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA HELIX RECOGNIZED BY A ZINC- FINGER PROTEIN: AN 18 BASE PAIR DUPLEX AT 1.6 RESOLUTION</td><td>S.LIMA,J.HILDENBRAND,A.KOROSTELEV,S.HATTMAN,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PCV" title="Link to PDB">2PCV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PCV/index.html" title="Link to FR3D analysis. 0.1943 pairs per nucleotide">2PCV</a></td><td align="center">A</td><td align="center">350</td><td align="center">68</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070805000000">5-08-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN RRNA SUBSTRATE BOUND TO THE PSEUDOURIDYLATION POCKET OF A BOX H/ACA SNORNA</td><td>H.JIN,J.P.LORIA,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DR8" title="Link to PDB">2DR8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR8/index.html" title="Link to FR3D analysis. 0.3939 pairs per nucleotide">2DR8</a></td><td align="center">B</td><td align="center">33</td><td align="center">13</td><td align="center">2.5</td><td sorttable_customkey="20070211000000">11-14-06</td><td>synthetic</td><td>COMPLEX STRUCTURE OF CCA-ADDING ENZYME WITH TRNAMINIDC AND CTP</td><td>K.TOMITA,R.ISHITANI,S.FUKAI,O.NUREKI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH1/index.html" title="Resolution 2.8A, 33 nucleotides, 13 basepairs, 0.3939 pairs per nucleotide">2ZH1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH7/index.html" title="Resolution 3.0A, 33 nucleotides, 13 basepairs, 0.3939 pairs per nucleotide">2ZH7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR7/index.html" title="Resolution 2.8A, 33 nucleotides, 13 basepairs, 0.3939 pairs per nucleotide">2DR7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZHA/index.html" title="Resolution 3.0A, 33 nucleotides, 13 basepairs, 0.3939 pairs per nucleotide">2ZHA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH9/index.html" title="Resolution 2.9A, 33 nucleotides, 13 basepairs, 0.3939 pairs per nucleotide">2ZH9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH3/index.html" title="Resolution 2.5A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH2/index.html" title="Resolution 2.7A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH6/index.html" title="Resolution 2.5A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH5/index.html" title="Resolution 2.6A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DVI/index.html" title="Resolution 2.6A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2DVI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR9/index.html" title="Resolution 2.8A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2DR9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH4/index.html" title="Resolution 2.6A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZHB/index.html" title="Resolution 3.0A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZHB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZH8/index.html" title="Resolution 2.6A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2ZH8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DRA/index.html" title="Resolution 2.5A, 34 nucleotides, 13 basepairs, 0.3824 pairs per nucleotide">2DRA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DRB/index.html" title="Resolution 2.8A, 35 nucleotides, 13 basepairs, 0.3714 pairs per nucleotide">2DRB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR5/index.html" title="Resolution 2.8A, 32 nucleotides, 10 basepairs, 0.3125 pairs per nucleotide">2DR5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DR8/2DR8_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GM0" title="Link to PDB">2GM0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GM0/index.html" title="Link to FR3D analysis. 0.3800 pairs per nucleotide">2GM0</a></td><td align="center">AB</td><td align="center">700</td><td align="center">266</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080104000000">4-25-06</td><td>synthetic</td><td>LINEAR DIMER OF STEMLOOP SL1 FROM HIV-1</td><td>N.B.ULYANOV,A.MUJEEB,Z.DU,M.TONELLI,T.G.PARSLOW,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XD0" title="Link to PDB">2XD0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XD0/index.html" title="Link to FR3D analysis. 0.3714 pairs per nucleotide">2XD0</a></td><td align="center">V</td><td align="center">35</td><td align="center">13</td><td align="center">3.0</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>A PROCESSED NON-CODING RNA REGULATES A BACTERIAL ANTIVIRAL SYSTEM</td><td>T.R.BLOWER,X.Y.PEI,F.L.SHORT,P.C.FINERAN,D.P.HUMPHREYS,B.F.LUISI, G.P.C.SALMOND</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XDB/index.html" title="Resolution 2.5A, 35 nucleotides, 12 basepairs, 0.3429 pairs per nucleotide">2XDB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XDD/index.html" title="Resolution 3.2A, 105 nucleotides, 35 basepairs, 0.3333 pairs per nucleotide">2XDD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XD0/2XD0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ULL" title="Link to PDB">1ULL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ULL/index.html" title="Link to FR3D analysis. 0.4082 pairs per nucleotide">1ULL</a></td><td align="center">A</td><td align="center">245</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980808000000">8-20-97</td><td>synthetic</td><td>RNA APTAMER COMPLEXED WITH HIV-1 REV PEPTIDE, NMR, 7 STRUCTURES</td><td>X.YE,A.GORIN,A.D.ELLINGTON,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OUY" title="Link to PDB">3OUY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OUY/index.html" title="Link to FR3D analysis. 0.3143 pairs per nucleotide">3OUY</a></td><td align="center">C</td><td align="center">70</td><td align="center">22</td><td align="center">2.7</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>HOW THE CCA-ADDING ENZYME SELECTS ADENINE OVER CYTOSINE AT POSITION 76 OF TRNA</td><td>B.C.PAN,Y.XIONG,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OVA" title="Link to PDB">3OVA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OVA/index.html" title="Link to FR3D analysis. 0.3824 pairs per nucleotide">3OVA</a></td><td align="center">C</td><td align="center">34</td><td align="center">13</td><td align="center">2.0</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>HOW THE CCA-ADDING ENZYME SELECTS ADENINE OVER CYTOSINE IN POSITION 76 OF TRNA</td><td>B.C.PAN,Y.XIONG,T.A.STEITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OVS/index.html" title="Resolution 2.8A, 68 nucleotides, 25 basepairs, 0.3676 pairs per nucleotide">3OVS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OVB/index.html" title="Resolution 1.9A, 69 nucleotides, 25 basepairs, 0.3623 pairs per nucleotide">3OVB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OV7/index.html" title="Resolution 3.0A, 68 nucleotides, 24 basepairs, 0.3529 pairs per nucleotide">3OV7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OVA/3OVA_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ET4" title="Link to PDB">1ET4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ET4/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1ET4</a></td><td align="center">AB</td><td align="center">175</td><td align="center">75</td><td align="center">2.3</td><td sorttable_customkey="20010111000000">11-13-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A VITAMIN B12 BINDING RNA APTAMER WITH LIGAND AT 2.3 A</td><td>D.SUSSMAN,C.WILSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DDY/index.html" title="Resolution 3.0A, 140 nucleotides, 49 basepairs, 0.3500 pairs per nucleotide">1DDY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ET4/1ET4_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PCW" title="Link to PDB">2PCW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PCW/index.html" title="Link to FR3D analysis. 0.3289 pairs per nucleotide">2PCW</a></td><td align="center">AB</td><td align="center">833</td><td align="center">274</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070805000000">5-08-07</td><td>synthetic,synthetic</td><td>SOLUTION STRUCTURE OF AN RRNA SUBSTRATE BOUND TO THE PSEUDOURIDYLATION POCKET OF A BOX H/ACA SNORNA</td><td>H.JIN,J.P.LORIA,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PLA" title="Link to PDB">3PLA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PLA/index.html" title="Link to FR3D analysis. 0.4023 pairs per nucleotide">3PLA</a></td><td align="center">GHIJ</td><td align="center">87</td><td align="center">35</td><td align="center">3.1</td><td sorttable_customkey="20130201000000">1-26-11</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A CATALYTICALLY ACTIVE SUBSTRATE-BOUND BOX C/D RNP FROM SULFOLOBUS SOLFATARICUS</td><td>J.LIN,S.LAI,R.JIA,A.XU,L.ZHANG,J.LU,K.YE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L3E" title="Link to PDB">2L3E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L3E/index.html" title="Link to FR3D analysis. 0.3714 pairs per nucleotide">2L3E</a></td><td align="center">A</td><td align="center">700</td><td align="center">260</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110511000000">11-17-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF P2A-J2A/B-P2B OF HUMAN TELOMERASE RNA</td><td>Q.ZHANG,N.KIM,R.D.PETERSON,Z.WANG,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FMN" title="Link to PDB">1FMN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FMN/index.html" title="Link to FR3D analysis. 0.4057 pairs per nucleotide">1FMN</a></td><td align="center">A</td><td align="center">175</td><td align="center">71</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19961107000000">7-11-96</td><td>synthetic</td><td>SOLUTION STRUCTURE OF FMN-RNA APTAMER COMPLEX, NMR, 5 STRUCTURES</td><td>P.FAN,A.K.SURI,R.FIALA,D.LIVE,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2P89" title="Link to PDB">2P89</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2P89/index.html" title="Link to FR3D analysis. 0.4510 pairs per nucleotide">2P89</a></td><td align="center">AB</td><td align="center">960</td><td align="center">433</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070304000000">4-03-07</td><td>synthetic,synthetic</td><td>SOLUTION STRUCTURE OF THE 3' PSEUDOURIDYATION POCKET OF U65 SNORNA WITH BOUND SUBSTRATE</td><td>H.WU,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L3C" title="Link to PDB">2L3C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L3C/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">2L3C</a></td><td align="center">B</td><td align="center">680</td><td align="center">280</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120310000000">10-27-10</td><td>RATTUS NORVEGICUS</td><td>SOLUTION STRUCTURE OF ADAR2 DSRBM1 BOUND TO LSL RNA</td><td>F.ALLAIN,R.STEFL,F.OBERSTRASS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ETF" title="Link to PDB">1ETF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ETF/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">1ETF</a></td><td align="center">A</td><td align="center">34</td><td align="center">14</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971203000000">3-12-97</td><td>synthetic</td><td>REV RESPONSE ELEMENT (RRE) RNA COMPLEXED WITH REV PEPTIDE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>J.L.BATTISTE,H.MAO,N.S.RAO,R.TAN,D.R.MUHANDIRAM,L.E.KAY, A.D.FRANKEL,J.R.WILLAMSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I9F/index.html" title="SOLUTION NMR, 238 nucleotides, 95 basepairs, 0.3992 pairs per nucleotide">1I9F</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ETG/index.html" title="SOLUTION NMR, 646 nucleotides, 254 basepairs, 0.3932 pairs per nucleotide">1ETG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ETF/1ETF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R2P" title="Link to PDB">1R2P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R2P/index.html" title="Link to FR3D analysis. 0.3882 pairs per nucleotide">1R2P</a></td><td align="center">A</td><td align="center">340</td><td align="center">132</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040302000000">2-03-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF DOMAIN 5 FROM THE AI5(GAMMA) GROUP II INTRON</td><td>R.K.O.SIGEL,D.G.SASHITAL,D.L.ABRAMOVITZ,A.G.PALMER III, S.E.BUTCHER,A.M.PYLE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R7W" title="Link to PDB">1R7W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R7W/index.html" title="Link to FR3D analysis. 0.3338 pairs per nucleotide">1R7W</a></td><td align="center">A</td><td align="center">680</td><td align="center">227</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060105000000">5-25-04</td><td>synthetic</td><td>NMR STRUCTURE OF THE R(GGAGGACAUCCCUCACGGGUGACCGUGGUCCUCC), DOMAIN IV STEM-LOOP B OF ENTEROVIRAL IRES WITH AUCCCU BULGE</td><td>Z.DU,N.B.ULYANOV,J.YU,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R7Z" title="Link to PDB">1R7Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R7Z/index.html" title="Link to FR3D analysis. 0.3074 pairs per nucleotide">1R7Z</a></td><td align="center">A</td><td align="center">680</td><td align="center">209</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060105000000">5-25-04</td><td>synthetic</td><td>NMR STRUCTURE OF THE R(GGAGGACAUUCCUCACGGGUGACCGUGGUCCUCC), DOMAIN IV STEM-LOOP B OF ENTEROVIRAL IRES WITH AUUCCU BULGE</td><td>Z.DU,N.B.ULYANOV,J.YU,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HMH" title="Link to PDB">1HMH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HMH/index.html" title="Link to FR3D analysis. 0.2941 pairs per nucleotide">1HMH</a></td><td align="center">AC</td><td align="center">102</td><td align="center">30</td><td align="center">2.6</td><td sorttable_customkey="19950712000000">12-07-95</td><td>engineered</td><td>THREE-DIMENSIONAL STRUCTURE OF A HAMMERHEAD RIBOZYME</td><td>H.W.PLEY,K.M.FLAHERTY,D.B.MCKAY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1P5N" title="Link to PDB">1P5N</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P5N/index.html" title="Link to FR3D analysis. 0.3721 pairs per nucleotide">1P5N</a></td><td align="center">A</td><td align="center">680</td><td align="center">253</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030411000000">11-04-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HCV IRES DOMAIN IIB</td><td>P.J.LUKAVSKY,I.KIM,G.A.OTTO,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2EUY" title="Link to PDB">2EUY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EUY/index.html" title="Link to FR3D analysis. 0.3549 pairs per nucleotide">2EUY</a></td><td align="center">A</td><td align="center">510</td><td align="center">181</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060301000000">1-03-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE INTERNAL LOOP OF HUMAN U65 H/ACA SNORNA 3' HAIRPIN</td><td>J.FEIGON,M.KHANNA,H.WU,C.JOHANSSON,M.CAIZERGUES-FERRER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NBK" title="Link to PDB">1NBK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NBK/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">1NBK</a></td><td align="center">A</td><td align="center">34</td><td align="center">14</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030312000000">12-03-03</td><td>synthetic</td><td>THE STRUCTURE OF RNA APTAMER FOR HIV TAT COMPLEXED WITH TWO ARGININAMIDE MOLECULES</td><td>A.MATSUGAMI,S.KOBAYASHI,K.OUHASHI,S.UESUGI,R.YAMAMOTO, K.TAIRA,S.NISHIKAWA,P.K.R.KUMAR,M.KATAHIRA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NMU" title="Link to PDB">3NMU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NMU/index.html" title="Link to FR3D analysis. 0.2766 pairs per nucleotide">3NMU</a></td><td align="center">DI</td><td align="center">47</td><td align="center">13</td><td align="center">2.7</td><td sorttable_customkey="20130105000000">5-25-11</td><td>PYROCOCCUS FURIOSUS,synthetic</td><td>CRYSTAL STRUCTURE OF SUBSTRATE-BOUND HALFMER BOX C/D RNP</td><td>H.LI,S.XUE,R.WANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RNK" title="Link to PDB">1RNK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RNK/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">1RNK</a></td><td align="center">A</td><td align="center">34</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970302000000">2-27-95</td><td>synthetic</td><td>THE STRUCTURE OF AN RNA PSEUDOKNOT THAT CAUSES EFFICIENT FRAMESHIFTING IN MOUSE MAMMARY TUMOR VIRUS</td><td>L.X.SHEN,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MJI" title="Link to PDB">1MJI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MJI/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">1MJI</a></td><td align="center">C</td><td align="center">68</td><td align="center">28</td><td align="center">2.5</td><td sorttable_customkey="20040901000000">1-21-03</td><td>ESCHERICHIA COLI</td><td>DETAILED ANALYSIS OF RNA-PROTEIN INTERACTIONS WITHIN THE BACTERIAL RIBOSOMAL PROTEIN L5/5S RRNA COMPLEX</td><td>A.PEREDERINA,N.NEVSKAYA,O.NIKONOV,A.NIKULIN,P.DUMAS,M.YAO, I.TANAKA,M.GARBER,G.GONGADZE,S.NIKONOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JTP" title="Link to PDB">2JTP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JTP/index.html" title="Link to FR3D analysis. 0.4015 pairs per nucleotide">2JTP</a></td><td align="center">A</td><td align="center">680</td><td align="center">273</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070210000000">10-02-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE FRAMESHIFT-INDUCING RNA STEM-LOOP IN SIV</td><td>R.J.MARCHESCHI,D.W.STAPLE,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KPV" title="Link to PDB">2KPV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KPV/index.html" title="Link to FR3D analysis. 0.3765 pairs per nucleotide">2KPV</a></td><td align="center">A</td><td align="center">340</td><td align="center">128</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100408000000">8-04-10</td><td>synthetic</td><td>NMR MODEL OF THE FIRST LET-7 MIRNA COMPLEMENTARY SITE (LCS1) IN 3'-UTR OF LIN-41 MRNA FROM C. ELEGANS</td><td>M.CEVEC,C.THIBAUDEAU,J.PLAVEC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T28" title="Link to PDB">1T28</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T28/index.html" title="Link to FR3D analysis. 0.2059 pairs per nucleotide">1T28</a></td><td align="center">A</td><td align="center">34</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041008000000">8-10-04</td><td>synthetic</td><td>HIGH RESOLUTION STRUCTURE OF A PICORNAVIRAL INTERNAL CIS- ACTING REPLICATION ELEMENT</td><td>V.THIVIYANATHAN,Y.YANG,K.KALUARACHCHI,R.REYNBRAND, D.G.GORENSTEIN,S.M.LEMON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F88" title="Link to PDB">2F88</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F88/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">2F88</a></td><td align="center">A</td><td align="center">340</td><td align="center">140</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080402000000">2-28-06</td><td>synthetic</td><td>SOLUTION NMR STRUCTURE OF DOMAIN 5 FROM THE PYAIELLA LITTORALIS (PL) GROUP II INTRON</td><td>K.T.DAYIE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EXY" title="Link to PDB">1EXY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EXY/index.html" title="Link to FR3D analysis. 0.4141 pairs per nucleotide">1EXY</a></td><td align="center">A</td><td align="center">396</td><td align="center">164</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010305000000">5-15-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HTLV-1 PEPTIDE BOUND TO ITS RNA APTAMER TARGET</td><td>F.JIANG,A.GORIN,W.HU,A.MAJUMDAR,S.BASKERVILLE,W.XU, A.ELLINGTON,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q50" title="Link to PDB">3Q50</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q50/index.html" title="Link to FR3D analysis. 0.5152 pairs per nucleotide">3Q50</a></td><td align="center">A</td><td align="center">33</td><td align="center">17</td><td align="center">2.8</td><td sorttable_customkey="20120605000000">5-18-11</td><td>synthetic</td><td>STRUCTURAL ANALYSIS OF A CLASS I PREQ1 RIBOSWITCH APTAMER IN THE METABOLITE-BOUND STATE</td><td>J.L.JENKINS,J KRUCINSKA,J.E.WEDEKIND</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q51/index.html" title="Resolution 2.9A, 32 nucleotides, 15 basepairs, 0.4688 pairs per nucleotide">3Q51</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GCA/index.html" title="Resolution 2.8A, 33 nucleotides, 15 basepairs, 0.4545 pairs per nucleotide">3GCA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q50/3Q50_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1O15" title="Link to PDB">1O15</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O15/index.html" title="Link to FR3D analysis. 0.3636 pairs per nucleotide">1O15</a></td><td align="center">A</td><td align="center">33</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040602000000">2-18-03</td><td>synthetic</td><td>THEOPHYLLINE-BINDING RNA IN COMPLEX WITH THEOPHYLLINE, NMR, REGULARIZED MEAN STRUCTURE, REFINEMENT WITH TORSION ANGLE AND BASE-BASE POSITIONAL DATABASE POTENTIALS AND DIPOLAR COUPLINGS</td><td>G.M.CLORE,J.KUSZEWSKI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EHT/index.html" title="SOLUTION NMR, 330 nucleotides, 117 basepairs, 0.3545 pairs per nucleotide">1EHT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O15/1O15_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JXV" title="Link to PDB">2JXV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JXV/index.html" title="Link to FR3D analysis. 0.3818 pairs per nucleotide">2JXV</a></td><td align="center">A</td><td align="center">330</td><td align="center">126</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100504000000">4-29-08</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A LET-7 MIRNA:LIN-41 MRNA COMPLEX FROM C. ELEGANS</td><td>M.CEVEC,C.THIBAUDEAU,J.PLAVEC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OZB" title="Link to PDB">2OZB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OZB/index.html" title="Link to FR3D analysis. 0.3939 pairs per nucleotide">2OZB</a></td><td align="center">C</td><td align="center">33</td><td align="center">13</td><td align="center">2.6</td><td sorttable_customkey="20080803000000">3-20-07</td><td>synthetic</td><td>STRUCTURE OF A HUMAN PRP31-15.5K-U4 SNRNA COMPLEX</td><td>S.LIU,R.LUEHRMANN,M.C.WAHL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F1T" title="Link to PDB">1F1T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F1T/index.html" title="Link to FR3D analysis. 0.4242 pairs per nucleotide">1F1T</a></td><td align="center">A</td><td align="center">33</td><td align="center">14</td><td align="center">2.8</td><td sorttable_customkey="20000409000000">9-04-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE MALACHITE GREEN APTAMER COMPLEXED WITH TETRAMETHYL-ROSAMINE</td><td>C.BAUGH,D.GRATE,C.WILSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1G70" title="Link to PDB">1G70</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G70/index.html" title="Link to FR3D analysis. 0.3795 pairs per nucleotide">1G70</a></td><td align="center">A</td><td align="center">448</td><td align="center">170</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010702000000">2-07-01</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1</td><td>COMPLEX OF HIV-1 RRE-IIB RNA WITH RSG-1.2 PEPTIDE</td><td>Y.GOSSER,T.HERMANN,A.MAJUMDAR,W.HU,R.FREDERICK,F.JIANG,W.XU, D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Z31" title="Link to PDB">1Z31</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z31/index.html" title="Link to FR3D analysis. 0.3438 pairs per nucleotide">1Z31</a></td><td align="center">A</td><td align="center">320</td><td align="center">110</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061003000000">3-22-05</td><td>synthetic</td><td>THE STRUCTURE OF AN ENZYME-ACTIVATING FRAGMENT OF HUMAN TELOMERASE RNA</td><td>T.C.LEEPER,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KF0" title="Link to PDB">2KF0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KF0/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2KF0</a></td><td align="center">A</td><td align="center">240</td><td align="center">96</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100907000000">7-21-09</td><td>synthetic</td><td>NMR STRUCTURE OF U6 ISL AT PH 7.0</td><td>V.VENDITTI,S.E.BUTCHER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XHP/index.html" title="SOLUTION NMR, 320 nucleotides, 120 basepairs, 0.3750 pairs per nucleotide">1XHP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KF0/2KF0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Z7F" title="Link to PDB">1Z7F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z7F/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1Z7F</a></td><td align="center">A</td><td align="center">32</td><td align="center">16</td><td align="center">2.1</td><td sorttable_customkey="20060610000000">10-18-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF 16 BASE PAIR RNA DUPLEX CONTAINING A C- A MISMATCH</td><td>C.M.GHERGHE,J.M.KRAHN,K.M.WEEKS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KAJ" title="Link to PDB">1KAJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KAJ/index.html" title="Link to FR3D analysis. 0.2812 pairs per nucleotide">1KAJ</a></td><td align="center">A</td><td align="center">32</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19961107000000">7-11-96</td><td>synthetic</td><td>CONFORMATION OF AN RNA PSEUDOKNOT FROM MOUSE MAMMARY TUMOR VIRUS, NMR, 1 STRUCTURE</td><td>H.KANG,J.V.HINES,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BO0" title="Link to PDB">3BO0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO0/index.html" title="Link to FR3D analysis. 0.3048 pairs per nucleotide">3BO0</a></td><td align="center">DEFG</td><td align="center">105</td><td align="center">32</td><td align="center">9.6</td><td sorttable_customkey="20080912000000">12-09-08</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>RIBOSOME-SECY COMPLEX</td><td>C.W.AKEY,J.F.MENETRET</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO1/index.html" title="Resolution 9.6A, 105 nucleotides, 32 basepairs, 0.3048 pairs per nucleotide">3BO1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BO0/3BO0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CZW" title="Link to PDB">3CZW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CZW/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3CZW</a></td><td align="center">X</td><td align="center">28</td><td align="center">14</td><td align="center">1.4</td><td sorttable_customkey="20080909000000">9-09-08</td><td>synthetic</td><td>X-RAY STRUCTURES OF THE (GUGGUCUGAUGAGGCC) RNA DUPLEX</td><td>W.RYPNIEWSKI,D.A.ADAMIAK,J.MILECKI,R.W.ADAMIAK</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D0M/index.html" title="Resolution 2.0A, 32 nucleotides, 14 basepairs, 0.4375 pairs per nucleotide">3D0M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CZW/3CZW_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ID5" title="Link to PDB">3ID5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ID5/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">3ID5</a></td><td align="center">D</td><td align="center">64</td><td align="center">16</td><td align="center">4.0</td><td sorttable_customkey="20110108000000">8-25-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF SULFOLOBUS SOLFATARICUS C/D RNP ASSEMBLED WITH NOP5, FIBRILLARIN, L7AE AND A SPLIT HALF C/D RNA</td><td>K.YE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DKN" title="Link to PDB">3DKN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DKN/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">3DKN</a></td><td align="center">DEF</td><td align="center">69</td><td align="center">30</td><td align="center">8.7</td><td sorttable_customkey="20090708000000">8-19-08</td><td>CANIS LUPUS FAMILIARIS,CANIS LUPUS FAMILIARIS,CANIS LUPUS FAMILIARIS</td><td>SEC61 IN THE CANINE RIBOSOME-CHANNEL COMPLEX FROM THE ENDOPLASMIC RETICULUM</td><td>J.-F.MENETRET,C.AKEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KPD" title="Link to PDB">1KPD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KPD/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">1KPD</a></td><td align="center">A</td><td align="center">32</td><td align="center">10</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970104000000">4-01-97</td><td>synthetic</td><td>A MUTANT RNA PSEUDOKNOT THAT PROMOTES RIBOSOMAL FRAMESHIFTING IN MOUSE MAMMARY TUMOR VIRUS, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>H.KANG,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T4L" title="Link to PDB">1T4L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T4L/index.html" title="Link to FR3D analysis. 0.4375 pairs per nucleotide">1T4L</a></td><td align="center">A</td><td align="center">480</td><td align="center">210</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040106000000">6-01-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF DOUBLE-STRANDED RNA BINDING DOMAIN OF S. CEREVISIAE RNASE III (RNT1P) IN COMPLEX WITH THE 5' TERMINAL RNA HAIRPIN OF SNR47 PRECURSOR</td><td>H.WU,A.HENRAS,G.CHANFREAU,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MFY" title="Link to PDB">1MFY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MFY/index.html" title="Link to FR3D analysis. 0.2540 pairs per nucleotide">1MFY</a></td><td align="center">A</td><td align="center">496</td><td align="center">126</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030609000000">9-18-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF INFLUENZA A VIRUS C4 PROMOTER</td><td>M.-K.LEE,S.-H.BAE,C.-J.PARK,H.-K.CHEONG,C.CHEONG,B.-S.CHOI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LVJ" title="Link to PDB">1LVJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LVJ/index.html" title="Link to FR3D analysis. 0.3602 pairs per nucleotide">1LVJ</a></td><td align="center">A</td><td align="center">372</td><td align="center">134</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20021112000000">12-11-02</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1</td><td>STRUCTURE OF TAR RNA COMPLEXED WITH A TAT-TAR INTERACTION NANOMOLAR INHIBITOR THAT WAS IDENTIFIED BY COMPUTATIONAL SCREENING</td><td>Z.DU,K.E.LIND,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YLG" title="Link to PDB">1YLG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YLG/index.html" title="Link to FR3D analysis. 0.3661 pairs per nucleotide">1YLG</a></td><td align="center">A</td><td align="center">620</td><td align="center">227</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050102000000">2-01-05</td><td>synthetic</td><td>NMR STRUCTURE OF THE APOB MRNA STEM-LOOP AND ITS INTERACTION WITH THE C TO U EDITING APOBEC1 COMPLEMENTARY FACTOR</td><td>C.MARIS,J.MASSE,F.H.ALLAIN,A.CHESTER,N.NAVARATNAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YNE" title="Link to PDB">1YNE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YNE/index.html" title="Link to FR3D analysis. 0.3532 pairs per nucleotide">1YNE</a></td><td align="center">A</td><td align="center">620</td><td align="center">219</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050802000000">2-08-05</td><td>synthetic</td><td>NMR STRUCTURE OF THE APOB MRNA STEM-LOOP AND ITS INTERACTION WITH THE C TO U EDITING APOBEC1 COMPLEMENTARY FACTOR</td><td>C.MARIS,J.MASSE,F.H.ALLAIN,A.CHESTER,N.NAVARATNAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JBR" title="Link to PDB">1JBR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JBR/index.html" title="Link to FR3D analysis. 0.4262 pairs per nucleotide">1JBR</a></td><td align="center">CF</td><td align="center">61</td><td align="center">26</td><td align="center">2.1</td><td sorttable_customkey="20030210000000">10-26-01</td><td>synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE RIBOTOXIN RESTRICTOCIN AND A 31- MER SRD RNA INHIBITOR</td><td>X.YANG,T.GERCZEI,L.GLOVER,C.C.CORRELL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JBT/index.html" title="Resolution 2.7A, 58 nucleotides, 22 basepairs, 0.3793 pairs per nucleotide">1JBT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JBR/1JBR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YNG" title="Link to PDB">1YNG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YNG/index.html" title="Link to FR3D analysis. 0.3286 pairs per nucleotide">1YNG</a></td><td align="center">A</td><td align="center">496</td><td align="center">163</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050802000000">2-08-05</td><td>synthetic</td><td>NMR STRUCTURE OF THE APOB MRNA STEM-LOOP AND ITS INTERACTION WITH THE C TO U EDITING APOBEC1 COMPLEMENTARY FACTOR</td><td>C.MARIS,J.MASSE,F.H.ALLAIN,A.CHESTER,N.NAVARATNAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JO7" title="Link to PDB">1JO7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JO7/index.html" title="Link to FR3D analysis. 0.2913 pairs per nucleotide">1JO7</a></td><td align="center">A</td><td align="center">992</td><td align="center">289</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020709000000">9-19-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF INFLUENZA A VIRUS PROMOTER</td><td>S.-H.BAE,H.-K.CHEONG,J.-H.LEE,C.CHEONG,M.KAINOSHO,B.-S.CHOI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1VFG" title="Link to PDB">1VFG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VFG/index.html" title="Link to FR3D analysis. 0.2581 pairs per nucleotide">1VFG</a></td><td align="center">C</td><td align="center">31</td><td align="center">8</td><td align="center">2.8</td><td sorttable_customkey="20041008000000">8-10-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF TRNA NUCLEOTIDYLTRANSFERASE COMPLEXED WITH A PRIMER TRNA AND AN INCOMING ATP ANALOG</td><td>K.TOMITA,S.FUKAI,R.ISHITANI,T.UEDA,N.TAKEUCHI,D.G.VASSYLYEV, O.NUREKI,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YNC" title="Link to PDB">1YNC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YNC/index.html" title="Link to FR3D analysis. 0.3387 pairs per nucleotide">1YNC</a></td><td align="center">A</td><td align="center">620</td><td align="center">210</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050802000000">2-08-05</td><td>synthetic</td><td>NMR STRUCTURE OF THE APOB MRNA STEM-LOOP AND ITS INTERACTION WITH THE C TO U EDITING APOBEC1 COMPLEMENTARY FACTOR</td><td>C.MARIS,J.MASSE,F.H.ALLAIN,A.CHESTER,N.NAVARATNAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EBR" title="Link to PDB">1EBR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EBR/index.html" title="Link to FR3D analysis. 0.4067 pairs per nucleotide">1EBR</a></td><td align="center">A</td><td align="center">150</td><td align="center">61</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>STRUCTURE OF RNA (5'-GGUGGGCGCAGCUUCGGCUGCGGUACCAC-3'), NMR, 5 STRUCTURES</td><td>R.D.PETERSON,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LDZ" title="Link to PDB">1LDZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LDZ/index.html" title="Link to FR3D analysis. 0.3373 pairs per nucleotide">1LDZ</a></td><td align="center">A</td><td align="center">750</td><td align="center">253</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000111000000">11-25-98</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE LEAD-DEPENDENT RIBOZYME, NMR, 25 STRUCTURES</td><td>C.G.HOOGSTRATEN,P.LEGAULT,A.PARDI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2LDZ/index.html" title="SOLUTION NMR, 30 nucleotides, 10 basepairs, 0.3333 pairs per nucleotide">2LDZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LDZ/1LDZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NA2" title="Link to PDB">1NA2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NA2/index.html" title="Link to FR3D analysis. 0.3667 pairs per nucleotide">1NA2</a></td><td align="center">A</td><td align="center">540</td><td align="center">198</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040901000000">1-21-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE P2B HAIRPIN FROM HUMAN TELOMERASE RNA</td><td>C.A.THEIMER,L.D.FINGER,L.TRANTIREK,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AJU" title="Link to PDB">1AJU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AJU/index.html" title="Link to FR3D analysis. 0.3817 pairs per nucleotide">1AJU</a></td><td align="center">A</td><td align="center">600</td><td align="center">229</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980512000000">12-17-97</td><td>synthetic</td><td>HIV-2 TAR-ARGININAMIDE COMPLEX, NMR, 20 STRUCTURES</td><td>A.S.BRODSKY,J.R.WILLIAMSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AUD" title="Link to PDB">1AUD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AUD/index.html" title="Link to FR3D analysis. 0.2290 pairs per nucleotide">1AUD</a></td><td align="center">B</td><td align="center">930</td><td align="center">213</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000102000000">2-25-98</td><td>synthetic</td><td>U1A-UTRRNA, NMR, 31 STRUCTURES</td><td>F.H.-T.ALLAIN,C.C.GUBSER,P.W.A.HOWE,K.NAGAI,D.NEUHAUS, G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=364D" title="Link to PDB">364D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/364D/index.html" title="Link to FR3D analysis. 0.4068 pairs per nucleotide">364D</a></td><td align="center">ABC</td><td align="center">59</td><td align="center">24</td><td align="center">3.0</td><td sorttable_customkey="19980201000000">1-02-98</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>.0 A STRUCTURE OF FRAGMENT I FROM E. COLI 5S RRNA</td><td>C.C.CORRELL,B.FREEBORN,P.B.MOORE,T.A.STEITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/357D/index.html" title="Resolution 3.5A, 58 nucleotides, 22 basepairs, 0.3793 pairs per nucleotide">357D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/364D/364D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EKZ" title="Link to PDB">1EKZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EKZ/index.html" title="Link to FR3D analysis. 0.4250 pairs per nucleotide">1EKZ</a></td><td align="center">B</td><td align="center">1080</td><td align="center">459</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010908000000">8-21-00</td><td>DROSOPHILA MELANOGASTER</td><td>NMR STRUCTURE OF THE COMPLEX BETWEEN THE THIRD DSRBD FROM DROSOPHILA STAUFEN AND A RNA HAIRPIN</td><td>A.RAMOS,S.GRUNERT,M.BYCROFT,D.ST JOHNSTON,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HVU" title="Link to PDB">1HVU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HVU/index.html" title="Link to FR3D analysis. 0.3000 pairs per nucleotide">1HVU</a></td><td align="center">C</td><td align="center">120</td><td align="center">36</td><td align="center">4.8</td><td sorttable_customkey="20000101000000">1-13-99</td><td>synthetic</td><td>HUMAN IMMUNODEFICIENCY VIRUS TYPE 1 REVERSE TRANSCRIPTASE COMPLEXED WITH A 33-BASE NUCLEOTIDE RNA PSEUDOKNOT</td><td>J.JAEGER,T.RESTLE,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IPY" title="Link to PDB">2IPY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IPY/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2IPY</a></td><td align="center">D</td><td align="center">60</td><td align="center">24</td><td align="center">2.8</td><td sorttable_customkey="20070901000000">1-09-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF IRON REGULATORY PROTEIN 1 IN COMPLEX WITH FERRITIN H IRE-RNA</td><td>W.E.WALDEN,A.I.SELEZNEVA,J.DUPUY,A.VOLBEDA,J.C.FONTECILLA- CAMPS,E.C.THEIL,K.VOLZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RFR" title="Link to PDB">1RFR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RFR/index.html" title="Link to FR3D analysis. 0.4317 pairs per nucleotide">1RFR</a></td><td align="center">A</td><td align="center">600</td><td align="center">259</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20051103000000">3-23-04</td><td>HUMAN COXSACKIEVIRUS B3</td><td>NMR STRUCTURE OF THE 30MER STEMLOOP-D OF COXSACKIEVIRAL RNA</td><td>O.OHLENSCHLAGER,J.WOHNERT,E.BUCCI,S.SEITZ,S.HAFNER, R.RAMACHANDRAN,R.ZELL,M.GORLACH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HWQ" title="Link to PDB">1HWQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HWQ/index.html" title="Link to FR3D analysis. 0.3667 pairs per nucleotide">1HWQ</a></td><td align="center">A</td><td align="center">600</td><td align="center">220</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020405000000">5-16-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE VS RIBOZYME SUBSTRATE STEM-LOOP</td><td>J.C.FLINDERS,T.DIECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KP7" title="Link to PDB">1KP7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KP7/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1KP7</a></td><td align="center">A</td><td align="center">30</td><td align="center">10</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20021004000000">4-10-02</td><td>synthetic</td><td>CONSERVED RNA STRUCTURE WITHIN THE HCV IRES EIF3 BINDING SITE</td><td>J.GALLEGO,R.KLINCK,A.J.COLLIER,P.T.COLE,S.J.HARRIS, G.P.HARRISON,F.ABOUL-ELA,S.WALKER,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AKX" title="Link to PDB">1AKX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AKX/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1AKX</a></td><td align="center">A</td><td align="center">30</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980711000000">11-19-97</td><td>synthetic</td><td>HIV-2 TRANS ACTIVATING REGION RNA COMPLEX WITH ARGININAMIDE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>A.S.BRODSKY,J.R.WILLIAMSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BH2" title="Link to PDB">2BH2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BH2/index.html" title="Link to FR3D analysis. 0.1724 pairs per nucleotide">2BH2</a></td><td align="center">C</td><td align="center">58</td><td align="center">10</td><td align="center">2.1</td><td sorttable_customkey="20070603000000">3-30-05</td><td>ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF E. COLI 5-METHYLURIDINE METHYLTRANSFERASE RUMA IN COMPLEX WITH  RIBOSOMAL RNA SUBSTRATE AND S-ADENOSYLHOMOCYSTEINE.</td><td>T.T.LEE,S.AGARWALLA,R.M.STROUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K64" title="Link to PDB">2K64</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K64/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2K64</a></td><td align="center">AB</td><td align="center">720</td><td align="center">320</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100207000000">7-14-09</td><td>synthetic,synthetic</td><td>NMR SOLUTION STRUCTURE OF THE D3'-HAIRPIN INCLUDING EBS1 TOGETHER WITH IBS1 OF THE GROUP II INTRON SC.AI5(GAMMA)</td><td>D.KRUSCHEL,R.K.O.SIGEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JID" title="Link to PDB">1JID</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JID/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1JID</a></td><td align="center">B</td><td align="center">26</td><td align="center">10</td><td align="center">1.8</td><td sorttable_customkey="20020710000000">10-19-01</td><td>synthetic</td><td>HUMAN SRP19 IN COMPLEX WITH HELIX 6 OF HUMAN SRP RNA</td><td>K.WILD,I.SINNING,S.CUSACK</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L1W/index.html" title="SOLUTION NMR, 29 nucleotides, 10 basepairs, 0.3448 pairs per nucleotide">1L1W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JID/1JID_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3B5F" title="Link to PDB">3B5F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B5F/index.html" title="Link to FR3D analysis. 0.4833 pairs per nucleotide">3B5F</a></td><td align="center">ABC</td><td align="center">60</td><td align="center">29</td><td align="center">2.7</td><td sorttable_customkey="20081208000000">8-12-08</td><td>synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A MINIMALLY HINGED HAIRPIN RIBOZYME INCORPORATING THE ADE38DAP MUTATION AND A 2',5' PHOSPHODIESTER LINKAGE AT THE ACTIVE SITE</td><td>C.MACELREVEY,J.KRUCINSKA,J.E.WEDEKIND</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B91/index.html" title="Resolution 2.8A, 60 nucleotides, 29 basepairs, 0.4833 pairs per nucleotide">3B91</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBK/index.html" title="Resolution 2.8A, 61 nucleotides, 29 basepairs, 0.4754 pairs per nucleotide">3BBK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B5S/index.html" title="Resolution 2.2A, 59 nucleotides, 28 basepairs, 0.4746 pairs per nucleotide">3B5S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBI/index.html" title="Resolution 2.4A, 59 nucleotides, 28 basepairs, 0.4746 pairs per nucleotide">3BBI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I2R/index.html" title="Resolution 2.8A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">3I2R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B5A/index.html" title="Resolution 2.4A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">3B5A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CR1/index.html" title="Resolution 2.2A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">3CR1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BBM/index.html" title="Resolution 2.6A, 60 nucleotides, 28 basepairs, 0.4667 pairs per nucleotide">3BBM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B58/index.html" title="Resolution 2.6A, 61 nucleotides, 28 basepairs, 0.4590 pairs per nucleotide">3B58</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I2S/index.html" title="Resolution 2.8A, 59 nucleotides, 27 basepairs, 0.4576 pairs per nucleotide">3I2S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I2Q/index.html" title="Resolution 2.9A, 59 nucleotides, 27 basepairs, 0.4576 pairs per nucleotide">3I2Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I2U/index.html" title="Resolution 2.8A, 60 nucleotides, 27 basepairs, 0.4500 pairs per nucleotide">3I2U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CQS/index.html" title="Resolution 2.8A, 61 nucleotides, 27 basepairs, 0.4426 pairs per nucleotide">3CQS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B5F/3B5F_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UUD" title="Link to PDB">1UUD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UUD/index.html" title="Link to FR3D analysis. 0.3448 pairs per nucleotide">1UUD</a></td><td align="center">B</td><td align="center">29</td><td align="center">10</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050303000000">3-15-04</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1 (CLONE</td><td>NMR STRUCTURE OF A SYNTHETIC SMALL MOLECULE, RBT203, BOUND TO HIV-1 TAR RNA</td><td>B.DAVIS,M.AFSHAR,G.VARANI,J.KARN,A.I.H.MURCHIE,G.LENTZEN, M.J.DRYSDALE,A.J.POTTER,J.BOWER,F.ABOUL-ELA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3B31" title="Link to PDB">3B31</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B31/index.html" title="Link to FR3D analysis. 0.3721 pairs per nucleotide">3B31</a></td><td align="center">AB</td><td align="center">43</td><td align="center">16</td><td align="center">2.4</td><td sorttable_customkey="20090112000000">12-25-07</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF DOMAIN III OF THE CRICKET PARALYSIS VIRUS IRES RNA</td><td>J.S.KIEFT,D.A.COSTANTINO,J.S.PFINGSTEN,R.P.RAMBO</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MJA/index.html" title="Resolution 2.8A, 43 nucleotides, 16 basepairs, 0.3721 pairs per nucleotide">3MJA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MJ3/index.html" title="Resolution 3.1A, 43 nucleotides, 15 basepairs, 0.3488 pairs per nucleotide">3MJ3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MJB/index.html" title="Resolution 2.8A, 43 nucleotides, 15 basepairs, 0.3488 pairs per nucleotide">3MJB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3B31/3B31_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GIP" title="Link to PDB">2GIP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIP/index.html" title="Link to FR3D analysis. 0.4643 pairs per nucleotide">2GIP</a></td><td align="center">A</td><td align="center">560</td><td align="center">260</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070806000000">6-20-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A PORTION OF THE 5'UTR OF HSPA MRNA FROM BRADYRHIZOBIUM JANPONICUM HAVING DELETED G83</td><td>S.CHOWDHURY,C.MARIS,F.H.ALLAIN,F.NARBERHAUS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIO/index.html" title="SOLUTION NMR, 580 nucleotides, 256 basepairs, 0.4414 pairs per nucleotide">2GIO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GIP/2GIP_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KDQ" title="Link to PDB">2KDQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KDQ/index.html" title="Link to FR3D analysis. 0.3724 pairs per nucleotide">2KDQ</a></td><td align="center">B</td><td align="center">290</td><td align="center">108</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic</td><td>SIMULTANEOUS RECOGNITION OF HIV-1 TAR RNA BULGE AND LOOP SEQUENCES BY CYCLIC PEPTIDE MIMIC OF TAT PROTEIN</td><td>A.DAVIDSON,T.C.LEEPER,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K63" title="Link to PDB">2K63</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K63/index.html" title="Link to FR3D analysis. 0.3328 pairs per nucleotide">2K63</a></td><td align="center">A</td><td align="center">580</td><td align="center">193</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100207000000">7-14-09</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE D3'-HAIRPIN INCLUDING THE EXON BINDING SITE 1 (EBS1) OF THE GROUP II INTRON SC.AI5(GAMMA)</td><td>D.KRUSCHEL,R.K.O.SIGEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EBS" title="Link to PDB">1EBS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EBS/index.html" title="Link to FR3D analysis. 0.4276 pairs per nucleotide">1EBS</a></td><td align="center">A</td><td align="center">145</td><td align="center">62</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>STRUCTURE OF RNA (5'-GGUGGGCGCAGCUUCGGCUGCGGUACCAC-3'), NMR, 5 STRUCTURES</td><td>R.D.PETERSON,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UUI" title="Link to PDB">1UUI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UUI/index.html" title="Link to FR3D analysis. 0.3793 pairs per nucleotide">1UUI</a></td><td align="center">B</td><td align="center">29</td><td align="center">11</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041202000000">2-12-04</td><td>HUMAN IMMUNODEFICIENCY VIRUS TYPE 1</td><td>NMR STRUCTURE OF A SYNTHETIC SMALL MOLECULE, RBT158, BOUND TO HIV-1 TAR RNA</td><td>B.DAVIS,M.AFSHAR,G.VARANI,J.KARN,A.I.H.MURCHIE,G.LENTZEN, M.J.DRYSDALE,A.J.POTTER,J.BOWER,F.ABOUL-ELA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NBR" title="Link to PDB">1NBR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NBR/index.html" title="Link to FR3D analysis. 0.3862 pairs per nucleotide">1NBR</a></td><td align="center">A</td><td align="center">435</td><td align="center">168</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030403000000">3-04-03</td><td>synthetic</td><td>IRON RESPONSIVE ELEMENT RNA HAIRPIN, NMR, 15 STRUCTURES</td><td>S.A.MCCALLUM,A.PARDI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K5Z" title="Link to PDB">2K5Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K5Z/index.html" title="Link to FR3D analysis. 0.3828 pairs per nucleotide">2K5Z</a></td><td align="center">A</td><td align="center">290</td><td align="center">111</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100207000000">7-14-09</td><td>synthetic</td><td>SOLUTION STRUCTURE AND DYNAMICS OF THE APICAL STEM-LOOP OF DUCK HEPATITIS B VIRUS</td><td>K.A.M.AMPT,M.TESSARI,S.S.WIJMENGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UTS" title="Link to PDB">1UTS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UTS/index.html" title="Link to FR3D analysis. 0.3233 pairs per nucleotide">1UTS</a></td><td align="center">B</td><td align="center">464</td><td align="center">150</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041202000000">2-12-04</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1</td><td>DESIGNED HIV-1 TAR BINDING LIGAND</td><td>B.DAVIS,A.I.H.MURCHIE,F.ABOUL-ELA,J.KARN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L8H" title="Link to PDB">2L8H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L8H/index.html" title="Link to FR3D analysis. 0.3517 pairs per nucleotide">2L8H</a></td><td align="center">A</td><td align="center">290</td><td align="center">102</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20121103000000">3-23-11</td><td>synthetic</td><td>CHEMICAL PROBE BOUND TO HIV TAR RNA</td><td>A.DAVIDSON,D.BEGLEY,C.LAU,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ARJ" title="Link to PDB">1ARJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ARJ/index.html" title="Link to FR3D analysis. 0.3000 pairs per nucleotide">1ARJ</a></td><td align="center">N</td><td align="center">580</td><td align="center">174</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960811000000">11-08-96</td><td>synthetic</td><td>ARG-BOUND TAR RNA, NMR</td><td>F.ABOUL-ELA,G.VARANI,J.KARN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ANR" title="Link to PDB">1ANR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ANR/index.html" title="Link to FR3D analysis. 0.2603 pairs per nucleotide">1ANR</a></td><td align="center">A</td><td align="center">580</td><td align="center">151</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971101000000">1-11-97</td><td>synthetic</td><td>CIS-ACTING RNA REGULATORY ELEMENT (HIV-1 TAR), NMR, 20 STRUCTURES</td><td>F.ABOUL-ELA,G.VARANI,J.KARN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SCL" title="Link to PDB">1SCL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SCL/index.html" title="Link to FR3D analysis. 0.3506 pairs per nucleotide">1SCL</a></td><td align="center">A</td><td align="center">174</td><td align="center">61</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970201000000">1-26-95</td><td>RATTUS NORVEGICUS</td><td>THE SARCIN-RICIN LOOP, A MODULAR RNA</td><td>A.A.SZEWCZAK,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KX5" title="Link to PDB">2KX5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KX5/index.html" title="Link to FR3D analysis. 0.3793 pairs per nucleotide">2KX5</a></td><td align="center">A</td><td align="center">290</td><td align="center">110</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic</td><td>RECOGNITION OF HIV TAR RNA BY PEPTIDE MIMETIC OF TAT PROTEIN</td><td>A.DAVIDSON,K.PATORA-KOMISARSKA,J.ROBINSON,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1L1C" title="Link to PDB">1L1C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L1C/index.html" title="Link to FR3D analysis. 0.4138 pairs per nucleotide">1L1C</a></td><td align="center">C</td><td align="center">29</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040303000000">3-27-02</td><td>BACILLUS SUBTILIS</td><td>STRUCTURE OF THE LICT BACTERIAL ANTITERMINATOR PROTEIN IN COMPLEX WITH ITS RNA TARGET</td><td>Y.YANG,N.DECLERCK,X.MANIVAL,S.AYMERICH,M.KOCHOYAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FQZ" title="Link to PDB">1FQZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FQZ/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1FQZ</a></td><td align="center">A</td><td align="center">27</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020501000000">1-17-01</td><td>synthetic</td><td>NMR VALIDATED MODEL OF DOMAIN IIID OF HEPATITIS C VIRUS INTERNAL RIBOSOME ENTRY SITE</td><td>R.KLINCK,E.WESTHOF,S.WALKER,M.AFSHAR,A.COLLIER,F.ABOUL-ELA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F84/index.html" title="SOLUTION NMR, 725 nucleotides, 274 basepairs, 0.3779 pairs per nucleotide">1F84</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FQZ/1FQZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EBQ" title="Link to PDB">1EBQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EBQ/index.html" title="Link to FR3D analysis. 0.4483 pairs per nucleotide">1EBQ</a></td><td align="center">A</td><td align="center">145</td><td align="center">65</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>STRUCTURE OF RNA (5'-GGUGGGCGCAGCUUCGGCUGCGGUACCAC-3'), NMR, 5 STRUCTURES</td><td>R.D.PETERSON,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QD3" title="Link to PDB">1QD3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QD3/index.html" title="Link to FR3D analysis. 0.3306 pairs per nucleotide">1QD3</a></td><td align="center">A</td><td align="center">493</td><td align="center">163</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20001207000000">7-12-00</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1</td><td>HIV-1 TAR RNA/NEOMYCIN B COMPLEX</td><td>C.FABER,H.STICHT,P.ROESCH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OOA" title="Link to PDB">1OOA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OOA/index.html" title="Link to FR3D analysis. 0.3103 pairs per nucleotide">1OOA</a></td><td align="center">D</td><td align="center">58</td><td align="center">18</td><td align="center">2.5</td><td sorttable_customkey="20041007000000">7-22-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF NF-KB(P50)2 COMPLEXED TO A HIGH- AFFINITY RNA APTAMER</td><td>D.B.HUANG,D.VU,L.A.CASSIDAY,J.M.ZIMMERMAN,L.J.MAHER III, G.GHOSH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JWV" title="Link to PDB">2JWV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JWV/index.html" title="Link to FR3D analysis. 0.3448 pairs per nucleotide">2JWV</a></td><td align="center">A</td><td align="center">290</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080801000000">1-08-08</td><td>synthetic</td><td>STRUCTURE OF A HIGH AFFINITY ANTI-NFKB RNA APTAMER</td><td>N.J.REITER,L.MAHER III,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AQO" title="Link to PDB">1AQO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AQO/index.html" title="Link to FR3D analysis. 0.3402 pairs per nucleotide">1AQO</a></td><td align="center">A</td><td align="center">435</td><td align="center">148</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980402000000">2-04-98</td><td>synthetic</td><td>IRON RESPONSIVE ELEMENT RNA HAIRPIN, NMR, 15 STRUCTURES</td><td>K.J.ADDESS,A.PARDI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=28SR" title="Link to PDB">28SR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/28SR/index.html" title="Link to FR3D analysis. 0.3929 pairs per nucleotide">28SR</a></td><td align="center">A</td><td align="center">28</td><td align="center">11</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000804000000">4-20-99</td><td>ESCHERICHIA COLI</td><td>NMR STRUCTURE OF THE MOST CONSERVED RNA MOTIF IN SRP RNA</td><td>U.SCHMITZ,T.L.JAMES,P.LUKAVSKY,P.WALTER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/28SP/index.html" title="SOLUTION NMR, 196 nucleotides, 68 basepairs, 0.3469 pairs per nucleotide">28SP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/28SR/28SR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZBN" title="Link to PDB">1ZBN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZBN/index.html" title="Link to FR3D analysis. 0.3393 pairs per nucleotide">1ZBN</a></td><td align="center">A</td><td align="center">280</td><td align="center">95</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060704000000">4-19-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF BIV TAR HAIRPIN COMPLEXED TO JDV TAT ARGININE-RICH MOTIF</td><td>V.CALABRO,M.D.DAUGHERTY,A.D.FRANKEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MNB" title="Link to PDB">1MNB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MNB/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1MNB</a></td><td align="center">B</td><td align="center">28</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990301000000">1-27-97</td><td>synthetic</td><td>BIV TAT PEPTIDE (RESIDUES 68-81), NMR, MINIMIZED AVERAGE STRUCTURE</td><td>J.D.PUGLISI,L.CHEN,S.BLANCHARD,A.D.FRANKEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Q96" title="Link to PDB">1Q96</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q96/index.html" title="Link to FR3D analysis. 0.4568 pairs per nucleotide">1Q96</a></td><td align="center">AC</td><td align="center">81</td><td align="center">37</td><td align="center">1.8</td><td sorttable_customkey="20050111000000">11-25-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A MUTANT OF THE SARCIN/RICIN DOMAIN FROM RAT 28S RRNA</td><td>C.C.CORRELL,J.BENEKEN,M.J.PLANTINGA,M.LUBBERS,Y.L.CHAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q93/index.html" title="Resolution 2.2A, 81 nucleotides, 37 basepairs, 0.4568 pairs per nucleotide">1Q93</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/430D/index.html" title="Resolution 2.1A, 28 nucleotides, 10 basepairs, 0.3571 pairs per nucleotide">430D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q96/1Q96_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BIV" title="Link to PDB">1BIV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BIV/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1BIV</a></td><td align="center">A</td><td align="center">140</td><td align="center">56</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971112000000">12-23-96</td><td>synthetic</td><td>BOVINE IMMUNODEFICIENCY VIRUS TAT-TAR COMPLEX, NMR, 5 STRUCTURES</td><td>X.YE,R.A.KUMAR,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2EZ6" title="Link to PDB">2EZ6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EZ6/index.html" title="Link to FR3D analysis. 0.4643 pairs per nucleotide">2EZ6</a></td><td align="center">CD</td><td align="center">56</td><td align="center">26</td><td align="center">2.0</td><td sorttable_customkey="20060702000000">2-07-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AQUIFEX AEOLICUS RNASE III (D44N) COMPLEXED WITH PRODUCT OF DOUBLE-STRANDED RNA PROCESSING</td><td>J.GAN,J.E.TROPEA,B.P.AUSTIN,D.L.COURT,D.S.WAUGH,X.JI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NUF/index.html" title="Resolution 2.5A, 56 nucleotides, 26 basepairs, 0.4643 pairs per nucleotide">2NUF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EZ6/2EZ6_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3O7V" title="Link to PDB">3O7V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O7V/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">3O7V</a></td><td align="center">A</td><td align="center">26</td><td align="center">12</td><td align="center">2.1</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN HIWI1 (V361M) PAZ DOMAIN (RESIDUES 277-399) IN COMPLEX WITH 14-MER RNA (12-BP + 2-NT OVERHANG) CONTAINING 2'-OCH3 AT ITS 3'-END</td><td>Y.TIAN,D.K.SIMANSHU,J.-B.MA,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O6E/index.html" title="Resolution 2.9A, 26 nucleotides, 12 basepairs, 0.4615 pairs per nucleotide">3O6E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O3I/index.html" title="Resolution 2.8A, 28 nucleotides, 12 basepairs, 0.4286 pairs per nucleotide">3O3I</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O7V/3O7V_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A9X" title="Link to PDB">2A9X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A9X/index.html" title="Link to FR3D analysis. 0.4250 pairs per nucleotide">2A9X</a></td><td align="center">2</td><td align="center">280</td><td align="center">119</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic</td><td>TAR RNA RECOGNITION BY A CYCLIC PEPTIDOMIMETIC OF TAT PROTEIN</td><td>T.C.LEEPER,Z.ATHANASSIOU,R.L.DIAS,J.A.ROBINSON,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KMJ" title="Link to PDB">2KMJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KMJ/index.html" title="Link to FR3D analysis. 0.4071 pairs per nucleotide">2KMJ</a></td><td align="center">A</td><td align="center">280</td><td align="center">114</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100608000000">8-18-09</td><td>HUMAN IMMUNODEFICIENCY VIRUS 2</td><td>HIGH RESOLUTION NMR SOLUTION STRUCTURE OF A COMPLEX OF HIV-2 TAR RNA AND A SYNTHETIC TRIPEPTIDE IN A 1:2 STOICHIOMETRY</td><td>J.FERNER,M.SUHARTONO,S.BREITUNG,H.R.A.JONKER,M.HENNIG, J.WOEHNERT,M.GOEBEL,H.SCHWALBE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1D4R" title="Link to PDB">1D4R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1D4R/index.html" title="Link to FR3D analysis. 0.4074 pairs per nucleotide">1D4R</a></td><td align="center">AB</td><td align="center">54</td><td align="center">22</td><td align="center">2.0</td><td sorttable_customkey="19990212000000">12-02-99</td><td>synthetic,synthetic</td><td>-MER FRAGMENT OF HUMAN SRP RNA HELIX 6</td><td>K.WILD,O.WEICHENRIEDER,G.A.LEONARD,S.CUSACK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AP5" title="Link to PDB">2AP5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AP5/index.html" title="Link to FR3D analysis. 0.4815 pairs per nucleotide">2AP5</a></td><td align="center">A</td><td align="center">27</td><td align="center">13</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060509000000">9-05-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE C27A SCYLV P1-P2 FRAMESHIFTING PSEUDOKNOT, AVERAGE STRUCTURE</td><td>P.V.CORNISH,D.P.GIEDROC</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AP0/index.html" title="SOLUTION NMR, 540 nucleotides, 230 basepairs, 0.4259 pairs per nucleotide">2AP0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AP5/2AP5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KPZ" title="Link to PDB">1KPZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KPZ/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">1KPZ</a></td><td align="center">A</td><td align="center">27</td><td align="center">10</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20021101000000">1-11-02</td><td>synthetic</td><td>PEMV-1 P1-P2 FRAMESHIFTING PSEUDOKNOT REGULARIZED AVERAGE STRUCTURE</td><td>P.L.NIXON,D.P.GIEDROC</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KPY/index.html" title="SOLUTION NMR, 405 nucleotides, 133 basepairs, 0.3284 pairs per nucleotide">1KPY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KPZ/1KPZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2X7N" title="Link to PDB">2X7N</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X7N/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2X7N</a></td><td align="center">A</td><td align="center">27</td><td align="center">12</td><td align="center">11.8</td><td sorttable_customkey="20120703000000">3-31-10</td><td>SACCHAROMYCES CEREVISIAE</td><td>MECHANISM OF EIF6S ANTI-ASSOCIATION ACTIVITY</td><td>M.GARTMANN,M.BLAU,J.-P.ARMACHE,T.MIELKE,M.TOPF,R.BECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XSG" title="Link to PDB">1XSG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XSG/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1XSG</a></td><td align="center">A</td><td align="center">27</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060412000000">12-28-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF E.COLI RNASE P RNA P4 STEM OLIGORIBONUCLEOTIDE, U69A MUTATION</td><td>M.SCHMITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XST/index.html" title="SOLUTION NMR, 27 nucleotides, 12 basepairs, 0.4444 pairs per nucleotide">1XST</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XSH/index.html" title="SOLUTION NMR, 27 nucleotides, 11 basepairs, 0.4074 pairs per nucleotide">1XSH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CD3/index.html" title="SOLUTION NMR, 27 nucleotides, 11 basepairs, 0.4074 pairs per nucleotide">2CD3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F79/index.html" title="SOLUTION NMR, 27 nucleotides, 11 basepairs, 0.4074 pairs per nucleotide">1F79</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XSU/index.html" title="SOLUTION NMR, 27 nucleotides, 11 basepairs, 0.4074 pairs per nucleotide">1XSU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CD6/index.html" title="SOLUTION NMR, 27 nucleotides, 11 basepairs, 0.4074 pairs per nucleotide">2CD6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7G/index.html" title="SOLUTION NMR, 459 nucleotides, 177 basepairs, 0.3856 pairs per nucleotide">1F7G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7I/index.html" title="SOLUTION NMR, 324 nucleotides, 124 basepairs, 0.3827 pairs per nucleotide">1F7I</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XSG/1XSG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AHT" title="Link to PDB">2AHT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AHT/index.html" title="Link to FR3D analysis. 0.4056 pairs per nucleotide">2AHT</a></td><td align="center">A</td><td align="center">540</td><td align="center">219</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070510000000">10-17-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF DOMAIN 6 FROM THE AI5(GAMMA)GROUP II INTRON</td><td>M.C.ERAT,O.ZERBE,T.FOX,R.K.O.SIGEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IXY" title="Link to PDB">2IXY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IXY/index.html" title="Link to FR3D analysis. 0.4042 pairs per nucleotide">2IXY</a></td><td align="center">A</td><td align="center">621</td><td align="center">251</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060609000000">9-06-06</td><td>HEPATITIS B VIRUS</td><td>SOLUTION STRUCTURE OF THE APICAL STEM-LOOP OF THE HUMAN HEPATITIS B VIRUS ENCAPSIDATION SIGNAL</td><td>S.FLODELL,M.PETERSEN,F.GIRARD,J.ZDUNEK,K.KIDD-LJUNGGREN, J.SCHLEUCHER,S.S.WIJMENGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DVZ" title="Link to PDB">3DVZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DVZ/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">3DVZ</a></td><td align="center">A</td><td align="center">27</td><td align="center">10</td><td align="center">1.0</td><td sorttable_customkey="20101203000000">3-24-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE SARCIN/RICIN DOMAIN FROM E. COLI 23 S RRNA</td><td>V.OLIERIC,U.RIEDER,K.LANG,A.SERGANOV,C.SCHULZE-BRIESE,R.MICURA, P.DUMAS,E.ENNIFAR</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/483D/index.html" title="Resolution 1.1A, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">483D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q9A/index.html" title="Resolution 1.0A, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">1Q9A</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/480D/index.html" title="Resolution 1.5A, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">480D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MSY/index.html" title="Resolution 1.4A, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">1MSY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DW6/index.html" title="Resolution 1.0A, 26 nucleotides, 9 basepairs, 0.3462 pairs per nucleotide">3DW6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DW4/index.html" title="Resolution 1.0A, 26 nucleotides, 9 basepairs, 0.3462 pairs per nucleotide">3DW4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DW7/index.html" title="Resolution 1.0A, 26 nucleotides, 8 basepairs, 0.3077 pairs per nucleotide">3DW7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DW5/index.html" title="Resolution 1.0A, 26 nucleotides, 8 basepairs, 0.3077 pairs per nucleotide">3DW5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DVZ/3DVZ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1L2X" title="Link to PDB">1L2X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L2X/index.html" title="Link to FR3D analysis. 0.5185 pairs per nucleotide">1L2X</a></td><td align="center">A</td><td align="center">27</td><td align="center">14</td><td align="center">1.2</td><td sorttable_customkey="20031003000000">3-22-02</td><td>synthetic</td><td>ATOMIC RESOLUTION CRYSTAL STRUCTURE OF A VIRAL RNA PSEUDOKNOT</td><td>M.EGLI,G.MINASOV,L.SU,A.RICH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/437D/index.html" title="Resolution 1.6A, 27 nucleotides, 14 basepairs, 0.5185 pairs per nucleotide">437D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L3D/index.html" title="Resolution 2.9A, 27 nucleotides, 14 basepairs, 0.5185 pairs per nucleotide">1L3D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L2X/1L2X_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BYJ" title="Link to PDB">1BYJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BYJ/index.html" title="Link to FR3D analysis. 0.3713 pairs per nucleotide">1BYJ</a></td><td align="center">A</td><td align="center">1026</td><td align="center">381</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010510000000">10-29-99</td><td>synthetic</td><td>GENTAMICIN C1A A-SITE COMPLEX</td><td>S.YOSHIZAWA,D.FOURMY,J.D.PUGLISI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PBR/index.html" title="SOLUTION NMR, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">1PBR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BYJ/1BYJ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2B63" title="Link to PDB">2B63</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B63/index.html" title="Link to FR3D analysis. 0.2222 pairs per nucleotide">2B63</a></td><td align="center">R</td><td align="center">27</td><td align="center">6</td><td align="center">3.8</td><td sorttable_customkey="20050612000000">12-06-05</td><td>synthetic</td><td>COMPLETE RNA POLYMERASE II-RNA INHIBITOR COMPLEX</td><td>H.KETTENBERGER,A.EISENFUEHR,F.BRUECKNER,M.THEIS,M.FAMULOK, P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=484D" title="Link to PDB">484D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/484D/index.html" title="Link to FR3D analysis. 0.3687 pairs per nucleotide">484D</a></td><td align="center">B</td><td align="center">594</td><td align="center">219</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000210000000">10-14-99</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HIV-1 REV PEPTIDE-RNA APTAMER COMPLEX</td><td>X.YE,A.A.GORIN,R.FREDERICK,W.HU,A.MAJUMDAR,W.XU,G.MCLENDON, A.ELLINGTON,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F7F" title="Link to PDB">1F7F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7F/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">1F7F</a></td><td align="center">A</td><td align="center">243</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000910000000">10-09-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE RNASE P RNA (M1 RNA) P4 STEM OLIGORIBONUCLEOTIDE, NMR, ENSEMBLE OF 9 STRUCTURES</td><td>M.SCHMITZ,I.TINOCO JR.</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CD1/index.html" title="SOLUTION NMR, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">2CD1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F78/index.html" title="SOLUTION NMR, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">1F78</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F6X/index.html" title="SOLUTION NMR, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">1F6X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CD5/index.html" title="SOLUTION NMR, 27 nucleotides, 10 basepairs, 0.3704 pairs per nucleotide">2CD5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7F/1F7F_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XOK" title="Link to PDB">1XOK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XOK/index.html" title="Link to FR3D analysis. 0.3611 pairs per nucleotide">1XOK</a></td><td align="center">AB</td><td align="center">36</td><td align="center">13</td><td align="center">3.0</td><td sorttable_customkey="20050401000000">1-04-05</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF ALFALFA MOSAIC VIRUS RNA 3'UTR IN COMPLEX WITH COAT PROTEIN N TERMINAL PEPTIDE</td><td>L.M.GUOGAS,D.J.FILMAN,J.M.HOGLE,L.GEHRKE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FYO" title="Link to PDB">1FYO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FYO/index.html" title="Link to FR3D analysis. 0.3615 pairs per nucleotide">1FYO</a></td><td align="center">A</td><td align="center">675</td><td align="center">244</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020203000000">3-14-01</td><td>synthetic</td><td>EUKARYOTIC DECODING REGION A-SITE RNA</td><td>S.R.LYNCH,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F6Z" title="Link to PDB">1F6Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F6Z/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">1F6Z</a></td><td align="center">A</td><td align="center">27</td><td align="center">10</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000910000000">10-09-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE RNASE P RNA (M1 RNA) P4 STEM C70U MUTANT OLIGORIBONUCLEOTIDE</td><td>M.SCHMITZ,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TOB" title="Link to PDB">1TOB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TOB/index.html" title="Link to FR3D analysis. 0.4021 pairs per nucleotide">1TOB</a></td><td align="center">A</td><td align="center">189</td><td align="center">76</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980406000000">6-16-97</td><td>synthetic</td><td>SACCHARIDE-RNA RECOGNITION IN AN AMINOGLYCOSIDE ANTIBIOTIC- RNA APTAMER COMPLEX, NMR, 7 STRUCTURES</td><td>L.JIANG,A.K.SURI,R.FIALA,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YG4" title="Link to PDB">1YG4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YG4/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1YG4</a></td><td align="center">A</td><td align="center">27</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060112000000">12-13-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE SCYLV P1-P2 FRAMESHIFTING PSEUDOKNOT, REGULARIZED AVERAGE STRUCTURE</td><td>P.V.CORNISH,M.HENNIG,D.P.GIEDROC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FYP" title="Link to PDB">1FYP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FYP/index.html" title="Link to FR3D analysis. 0.3614 pairs per nucleotide">1FYP</a></td><td align="center">A</td><td align="center">783</td><td align="center">283</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020203000000">3-14-01</td><td>synthetic</td><td>EUKARYOTIC DECODING REGION A-SITE RNA-PAROMOMYCIN COMPLEX</td><td>S.R.LYNCH,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F7H" title="Link to PDB">1F7H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F7H/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">1F7H</a></td><td align="center">A</td><td align="center">297</td><td align="center">110</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000910000000">10-09-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE RNASE P RNA (M1 RNA) P4 STEM OLIGORIBONUCLEOTIDE COMPLEXED WITH COBALT (III) HEXAMINE, NMR, ENSEMBLE OF 11 STRUCTURES</td><td>M.SCHMITZ,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YSV" title="Link to PDB">1YSV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YSV/index.html" title="Link to FR3D analysis. 0.4359 pairs per nucleotide">1YSV</a></td><td align="center">A</td><td align="center">351</td><td align="center">153</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060707000000">7-19-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE CENTRAL REGION OF THE HUMAN GLUR- B R/G PRE-MRNA</td><td>R.STEFL,F.H.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KXM" title="Link to PDB">2KXM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KXM/index.html" title="Link to FR3D analysis. 0.3704 pairs per nucleotide">2KXM</a></td><td align="center">A</td><td align="center">270</td><td align="center">100</td><td align="center">SOLID-STATE NMR</td><td sorttable_customkey="20120804000000">4-20-11</td><td>synthetic</td><td>SOLUTION NMR STRUCTURE OF THE 27 NUCLEOTIDE ENGINEERED NEOMYCIN SENSING RIBOSWITCH RNA-RIBOSTMYCIN COMPLEX</td><td>D.ELKE,W.E.JULIA,O.OLIVER,S.R.SINA,S.BEATRIX,W.JENS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=387D" title="Link to PDB">387D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/387D/index.html" title="Link to FR3D analysis. 0.1923 pairs per nucleotide">387D</a></td><td align="center">A</td><td align="center">26</td><td align="center">5</td><td align="center">3.1</td><td sorttable_customkey="20050208000000">8-26-03</td><td>synthetic</td><td>RNA PSEUDOKNOT WITH 3D DOMAIN SWAPPING</td><td>S.E.LIETZKE,C.E.KUNDROT,C.L.BARNES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=413D" title="Link to PDB">413D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/413D/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">413D</a></td><td align="center">A</td><td align="center">26</td><td align="center">12</td><td align="center">1.8</td><td sorttable_customkey="19981007000000">7-10-98</td><td>synthetic</td><td>A'-FORM RNA DOUBLE HELIX IN THE SINGLE CRYSTAL STRUCTURE OF R(UGAGCUUCGGCUC)</td><td>Y.TANAKA,S.FUJII,H.HIROAKI,T.SAKATA,T.TANAKA,S.UESUGI,K.- I.TOMITA,Y.KYOGOKU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RFK" title="Link to PDB">2RFK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RFK/index.html" title="Link to FR3D analysis. 0.3934 pairs per nucleotide">2RFK</a></td><td align="center">DEF</td><td align="center">61</td><td align="center">24</td><td align="center">2.9</td><td sorttable_customkey="20080502000000">2-05-08</td><td>PYROCOCCUS FURIOSUS,PYROCOCCUS FURIOSUS,PYROCOCCUS FURIOSUS</td><td>SUBSTRATE RNA POSITIONING IN THE ARCHAEAL H/ACA RIBONUCLEOPROTEIN COMPLEX</td><td>B.LIANG,S.XUE,R.M.TERNS,M.P.TERNS,H.LI,SOUTHEAST COLLABORATORY FOR STRUCTURAL GENOMICS (SECSG)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RP0" title="Link to PDB">2RP0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RP0/index.html" title="Link to FR3D analysis. 0.4038 pairs per nucleotide">2RP0</a></td><td align="center">A</td><td align="center">728</td><td align="center">294</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110703000000">3-31-09</td><td>synthetic</td><td>REFINED SOLUTION STRUCTURE OF THE PEMV-1 MRNA PSEUDOKNOT, 28 LOWEST ENERGY STRUCTURES</td><td>P.V.CORNISH,M.HENNIG,D.P.GIEDROC</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RP1/index.html" title="SOLUTION NMR, 26 nucleotides, 10 basepairs, 0.3846 pairs per nucleotide">2RP1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RP0/2RP0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QWB" title="Link to PDB">1QWB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QWB/index.html" title="Link to FR3D analysis. 0.3394 pairs per nucleotide">1QWB</a></td><td align="center">A</td><td align="center">442</td><td align="center">150</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-25-03</td><td>synthetic</td><td>NMR STRUCUTRE OF 5'-R(GGACACGAAAUCCCGAAGUAGUGUCC)-3' : AN RNA HAIRPIN CONTAINING THE IN VITRO SELECTED CONSENSUS SEQUENCE FOR NUCLEOLIN RBD12</td><td>L.D.FINGER,L.TRANTIREK,C.JOHANSSON,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A43" title="Link to PDB">2A43</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A43/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2A43</a></td><td align="center">A</td><td align="center">26</td><td align="center">13</td><td align="center">1.3</td><td sorttable_customkey="20070309000000">9-27-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A LUTEOVIRAL RNA PSEUDOKNOT AND MODEL FOR A MINIMAL RIBOSOMAL FRAMESHIFTING MOTIF</td><td>P.S.PALLAN,W.S.MARSHALL,J.HARP,F.C.JEWETT III,Z.WAWRZAK, B.A.BROWN II,A.RICH,M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F8T" title="Link to PDB">2F8T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F8T/index.html" title="Link to FR3D analysis. 0.4800 pairs per nucleotide">2F8T</a></td><td align="center">CD</td><td align="center">50</td><td align="center">24</td><td align="center">3.1</td><td sorttable_customkey="20080710000000">10-31-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AA-AGO WITH EXTERNALLY-BOUND SIRNA</td><td>Y.R.YUAN,H.Y.CHEN,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L5Z" title="Link to PDB">2L5Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L5Z/index.html" title="Link to FR3D analysis. 0.3773 pairs per nucleotide">2L5Z</a></td><td align="center">A</td><td align="center">546</td><td align="center">206</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120402000000">2-16-11</td><td>NEUROSPORA</td><td>NMR STRUCTURE OF THE A730 LOOP OF THE NEUROSPORA VS RIBOZYME</td><td>G.DESJARDINS,E.BONNEAU,N.GIRARD,J.BOISBOUVIER,P.LEGAULT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EI2" title="Link to PDB">1EI2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EI2/index.html" title="Link to FR3D analysis. 0.2800 pairs per nucleotide">1EI2</a></td><td align="center">A</td><td align="center">425</td><td align="center">119</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000603000000">3-06-00</td><td>synthetic</td><td>STRUCTURAL BASIS FOR RECOGNITION OF THE RNA MAJOR GROOVE IN THE TAU EXON 10 SPLICING REGULATORY ELEMENT BY AMINOGLYCOSIDE ANTIBIOTICS</td><td>L.VARANI,M.G.SPILLANTINI,M GOEDERT,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=488D" title="Link to PDB">488D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/488D/index.html" title="Link to FR3D analysis. 0.5077 pairs per nucleotide">488D</a></td><td align="center">ABCD</td><td align="center">65</td><td align="center">33</td><td align="center">3.1</td><td sorttable_customkey="20000603000000">3-06-00</td><td>synthetic,synthetic,synthetic,synthetic</td><td>CATALYTIC RNA ENZYME-PRODUCT COMPLEX</td><td>J.B.MURRAY,H.SZOKE,A.SZOKE,W.G.SCOTT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/299D/index.html" title="Resolution 3.0A, 41 nucleotides, 17 basepairs, 0.4146 pairs per nucleotide">299D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/301D/index.html" title="Resolution 3.0A, 41 nucleotides, 17 basepairs, 0.4146 pairs per nucleotide">301D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/359D/index.html" title="Resolution 2.9A, 41 nucleotides, 17 basepairs, 0.4146 pairs per nucleotide">359D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/300D/index.html" title="Resolution 3.0A, 41 nucleotides, 17 basepairs, 0.4146 pairs per nucleotide">300D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NYI/index.html" title="Resolution 2.9A, 40 nucleotides, 16 basepairs, 0.4000 pairs per nucleotide">1NYI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MME/index.html" title="Resolution 3.1A, 82 nucleotides, 32 basepairs, 0.3902 pairs per nucleotide">1MME</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q29/index.html" title="Resolution 3.0A, 40 nucleotides, 15 basepairs, 0.3750 pairs per nucleotide">1Q29</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/379D/index.html" title="Resolution 3.1A, 41 nucleotides, 11 basepairs, 0.2683 pairs per nucleotide">379D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/488D/488D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F8V" title="Link to PDB">1F8V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F8V/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1F8V</a></td><td align="center">R</td><td align="center">25</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20000612000000">12-06-00</td><td>PARIACATO VIRUS</td><td>THE STRUCTURE OF PARIACOTO VIRUS REVEALS A DODECAHEDRAL CAGE OF DUPLEX RNA</td><td>L.TANG,K.N.JOHNSON,L.A.BALL,T.LIN,M.YEAGER,J.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QC8" title="Link to PDB">1QC8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QC8/index.html" title="Link to FR3D analysis. 0.2841 pairs per nucleotide">1QC8</a></td><td align="center">A</td><td align="center">725</td><td align="center">206</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010708000000">8-31-99</td><td>synthetic</td><td>NMR STRUCTURE OF TAU EXON 10 SPLICING REGULATORY ELEMENT RNA</td><td>L.VARANI,M.G.SPILLANTINI,A.KLUG,M.GOEDERT,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KGP" title="Link to PDB">2KGP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KGP/index.html" title="Link to FR3D analysis. 0.3600 pairs per nucleotide">2KGP</a></td><td align="center">A</td><td align="center">250</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090206000000">6-02-09</td><td>synthetic</td><td>STRUCTURAL BASIS FOR STABILIZATION OF THE TAU PRE-MRNA SPLICING REGULATORY ELEMENT BY NOVANTRONE (MITOXANTRONE)</td><td>S.ZHENG,Y.CHEN,C.P.DONAHUE,M.S.WOLFE,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QUX" title="Link to PDB">2QUX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QUX/index.html" title="Link to FR3D analysis. 0.3600 pairs per nucleotide">2QUX</a></td><td align="center">F</td><td align="center">150</td><td align="center">54</td><td align="center">2.4</td><td sorttable_customkey="20080612000000">12-18-07</td><td>PSEUDOMONAS PHAGE PP7</td><td>PP7 COAT PROTEIN DIMER IN COMPLEX WITH RNA HAIRPIN</td><td>J.A.CHAO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M82" title="Link to PDB">1M82</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M82/index.html" title="Link to FR3D analysis. 0.3703 pairs per nucleotide">1M82</a></td><td align="center">A</td><td align="center">775</td><td align="center">287</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030306000000">6-03-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE COMPLEMENTARY RNA PROMOTER OF INFLUENZA A VIRUS</td><td>C.-J.PARK,S.-H.BAE,M.-K.LEE,G.VARANI,B.-S.CHOI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HJY" title="Link to PDB">3HJY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HJY/index.html" title="Link to FR3D analysis. 0.3667 pairs per nucleotide">3HJY</a></td><td align="center">CDE</td><td align="center">60</td><td align="center">22</td><td align="center">3.6</td><td sorttable_customkey="20101106000000">6-23-09</td><td>synthetic,synthetic,synthetic</td><td>STRUCTURE OF A FUNCTIONAL RIBONUCLEOPROTEIN PSEUDOURIDINE SYNTHASE BOUND TO A SUBSTRATE RNA</td><td>B.LIANG,J.ZHOU,E.KAHEN,R.M.TERNS,M.P.TERNS,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KKS" title="Link to PDB">1KKS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KKS/index.html" title="Link to FR3D analysis. 0.2917 pairs per nucleotide">1KKS</a></td><td align="center">A</td><td align="center">360</td><td align="center">105</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030103000000">3-13-02</td><td>synthetic</td><td>STRUCTURE OF THE HISTONE MRNA HAIRPIN REQUIRED FOR CELL CYCLE REGULATION OF HISTONE GENE EXPRESSION</td><td>K.ZANIER,I.LUYTEN,C.CROMBIE,B MULLER,D.SCHUEMPERLI, J.P.LINGE,M.NILGES,M.SATTLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LC6" title="Link to PDB">1LC6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LC6/index.html" title="Link to FR3D analysis. 0.3427 pairs per nucleotide">1LC6</a></td><td align="center">A</td><td align="center">960</td><td align="center">329</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040505000000">5-29-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE U6 INTRAMOLECULAR STEM-LOOP RNA</td><td>A.HUPPLER,L.J.NIKSTAD,A.M.ALLMANN,D.A.BROW,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=283D" title="Link to PDB">283D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/283D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">283D</a></td><td align="center">A</td><td align="center">24</td><td align="center">12</td><td align="center">2.3</td><td sorttable_customkey="19980609000000">9-30-96</td><td>engineered</td><td>A CURVED RNA HELIX INCORPORATING AN INTERNAL LOOP WITH G-A AND A-A NON-WATSON-CRICK BASE PAIRING</td><td>K.J.BAEYENS,H.L.DE BONDT,A.PARDI,S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HEM" title="Link to PDB">2HEM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HEM/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">2HEM</a></td><td align="center">A</td><td align="center">216</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061209000000">9-12-06</td><td>synthetic</td><td>NMR STRUCTURE AND MG2+ BINDING OF AN RNA SEGMENT THAT UNDERLIES THE L7/L12 STALK IN THE E.COLI 50S RIBOSOMAL SUBUNIT.</td><td>Q.ZHAO,U.NAGASWAMY,H.LEE,Y.XIA,X.GAO,G.FOX</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KEZ" title="Link to PDB">2KEZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KEZ/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2KEZ</a></td><td align="center">A</td><td align="center">240</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100907000000">7-21-09</td><td>synthetic</td><td>NMR STRUCTURE OF U6 ISL AT PH 8.0</td><td>V.VENDITTI,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NYB" title="Link to PDB">1NYB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NYB/index.html" title="Link to FR3D analysis. 0.3944 pairs per nucleotide">1NYB</a></td><td align="center">B</td><td align="center">360</td><td align="center">142</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041206000000">6-24-03</td><td>ENTEROBACTERIA PHAGE PHI21</td><td>SOLUTION STRUCTURE OF THE BACTERIOPHAGE PHI21 N PEPTIDE-BOXB RNA COMPLEX</td><td>C.D.CILLEY,J.R.WILLIAMSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZBL" title="Link to PDB">1ZBL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZBL/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1ZBL</a></td><td align="center">C</td><td align="center">24</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20051207000000">7-12-05</td><td>synthetic</td><td>BACILLUS HALODURANS RNASE H CATALYTIC DOMAIN MUTANT D192N IN COMPLEX WITH 12-MER RNA/DNA HYBRID</td><td>M.NOWOTNY,S.A.GAIDAMAKOV,R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SYZ" title="Link to PDB">1SYZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SYZ/index.html" title="Link to FR3D analysis. 0.3452 pairs per nucleotide">1SYZ</a></td><td align="center">A</td><td align="center">336</td><td align="center">116</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040911000000">11-09-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE S. CEREVISIAE U6 INTRAMOLECULAR STEM LOOP (ISL) RNA AT PH 5.7</td><td>N.J.REITER,H.BLAD,F.ABILDGAARD,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MIJ" title="Link to PDB">3MIJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MIJ/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3MIJ</a></td><td align="center">A</td><td align="center">24</td><td align="center">12</td><td align="center">2.6</td><td sorttable_customkey="20120402000000">2-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A TELOMERIC RNA G-QUADRUPLEX COMPLEXED WITH AN ACRIDINE-BASED LIGAND.</td><td>G.W.COLLIE,S.NEIDLE,G.N.PARKINSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RHT" title="Link to PDB">1RHT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RHT/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1RHT</a></td><td align="center">A</td><td align="center">24</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19950306000000">6-03-95</td><td>synthetic</td><td>-MER RNA HAIRPIN COAT PROTEIN BINDING SITE FOR BACTERIOPHAGE R17 (NMR, MINIMIZED AVERAGE STRUCTURE)</td><td>P.N.BORER,Y.LIN,S.WANG,M.W.ROGGENBUCK,J.M.GOTT, O.C.UHLENBECK,I.PELCZER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Q1R" title="Link to PDB">2Q1R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q1R/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2Q1R</a></td><td align="center">A</td><td align="center">24</td><td align="center">12</td><td align="center">1.1</td><td sorttable_customkey="20090610000000">10-30-07</td><td>engineered</td><td>CRYSTAL STRUCTURE ANALYSIS OF THE RNA DODECAMER CGCGAAUUAGCG, WITH A G-A MISMATCH.</td><td>F.LI,P.S.PALLAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NC0" title="Link to PDB">1NC0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NC0/index.html" title="Link to FR3D analysis. 0.3495 pairs per nucleotide">1NC0</a></td><td align="center">A</td><td align="center">432</td><td align="center">151</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050102000000">2-25-03</td><td>synthetic</td><td>U80G U6 INTRAMOLECULAR STEM-LOOP RNA FROM SACCHAROMYCES CEREVISIAE</td><td>D.G.SASHITAL,A.M.ALLMANN,S.VAN DOREN,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F4X" title="Link to PDB">2F4X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F4X/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">2F4X</a></td><td align="center">AB</td><td align="center">864</td><td align="center">360</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080402000000">2-28-06</td><td>synthetic</td><td>NMR SOLUTION OF HIV-1 LAI KISSING COMPLEX</td><td>F.KIEKEN,F.PAQUET,F.BRULE,J.PAOLETTI,G.LANCELOT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A9N" title="Link to PDB">1A9N</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A9N/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1A9N</a></td><td align="center">Q</td><td align="center">48</td><td align="center">12</td><td align="center">2.4</td><td sorttable_customkey="19991109000000">9-23-98</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF THE SPLICEOSOMAL U2B''-U2A' PROTEIN COMPLEX BOUND TO A FRAGMENT OF U2 SMALL NUCLEAR RNA</td><td>S.R.PRICE,P.R.EVANS,K.NAGAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PN4" title="Link to PDB">2PN4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PN4/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">2PN4</a></td><td align="center">AB</td><td align="center">84</td><td align="center">32</td><td align="center">2.3</td><td sorttable_customkey="20080104000000">4-01-08</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF HEPATITIS C VIRUS IRES SUBDOMAIN IIA</td><td>Q.ZHAO,Q.HAN,C.R.KISSINGER,P.A.THOMPSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PN3/index.html" title="Resolution 2.9A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2PN3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PN4/2PN4_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MT4" title="Link to PDB">1MT4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MT4/index.html" title="Link to FR3D analysis. 0.4020 pairs per nucleotide">1MT4</a></td><td align="center">A</td><td align="center">408</td><td align="center">164</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040201000000">1-14-03</td><td>synthetic</td><td>STRUCTURE OF 23S RIBOSOMAL RNA HAIRPIN 35</td><td>I.LEBARS,S.YOSHIZAWA,A.R.STENHOLM,E.GUITTET,S.DOUTHWAITE, D.FOURMY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=255D" title="Link to PDB">255D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/255D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">255D</a></td><td align="center">A</td><td align="center">24</td><td align="center">12</td><td align="center">2.0</td><td sorttable_customkey="19950702000000">2-07-95</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DOUBLE HELIX INCORPORATING A TRACK OF NON-WATSON-CRICK BASE PAIRS</td><td>S.R.HOLBROOK,C.CHEONG,I.TINOCO JR,S.-H.KIM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ANN" title="Link to PDB">2ANN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ANN/index.html" title="Link to FR3D analysis. 0.2609 pairs per nucleotide">2ANN</a></td><td align="center">B</td><td align="center">23</td><td align="center">6</td><td align="center">2.3</td><td sorttable_customkey="20071210000000">10-24-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE (I) OF NOVA-1 KH1/KH2 DOMAIN TANDEM WITH 25 NT RNA HAIRPIN</td><td>L.MALININA,M.TEPLOVA,K.MUSUNURU,A.TEPLOV,J.C.DARNELL, S.K.BURLEY,R.B.DARNELL,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ANR/index.html" title="Resolution 1.9A, 24 nucleotides, 6 basepairs, 0.2500 pairs per nucleotide">2ANR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ANN/2ANN_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SY4" title="Link to PDB">1SY4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SY4/index.html" title="Link to FR3D analysis. 0.3438 pairs per nucleotide">1SY4</a></td><td align="center">A</td><td align="center">288</td><td align="center">99</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040604000000">4-06-04</td><td>synthetic</td><td>REFINED SOLUTION STRUCTURE OF THE S. CEREVISIAE U6 INTRAMOLECULAR STEM LOOP (ISL) RNA USING RESIDUAL DIPOLAR COUPLINGS (RDCS)</td><td>N.J.REITER,L.J.NIKSTAD,A.M.ALLMAN,R.J.JOHNSON,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NOK" title="Link to PDB">2NOK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NOK/index.html" title="Link to FR3D analysis. 0.3977 pairs per nucleotide">2NOK</a></td><td align="center">ABCD</td><td align="center">88</td><td align="center">35</td><td align="center">3.0</td><td sorttable_customkey="20080102000000">2-13-07</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DOMAIN FROM HEPATITIS C VIRUS.</td><td>S.M.DIBROV,H.JOHNSTON-COS,Y.H.WENG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TFN" title="Link to PDB">1TFN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TFN/index.html" title="Link to FR3D analysis. 0.2917 pairs per nucleotide">1TFN</a></td><td align="center">A</td><td align="center">24</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971202000000">2-12-97</td><td>synthetic</td><td>STRUCTURE REFINEMENT FOR A 24-NUCLEOTIDE RNA HAIRPIN, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>D.J.KERWOOD,P.N.BORER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1E4P" title="Link to PDB">1E4P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1E4P/index.html" title="Link to FR3D analysis. 0.3833 pairs per nucleotide">1E4P</a></td><td align="center">A</td><td align="center">480</td><td align="center">184</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020401000000">1-16-01</td><td>NEUROSPORA SP.</td><td>STRUCTURE OF THE RIBOZYME SUBSTRATE HAIRPIN OF NEUROSPORA VS RNA. A CLOSE LOOK AT THE CLEAVAGE SITE</td><td>P.J.A.MICHIELS,C.H.J.SCHOUTEN,H.A.HEUS,C.W.HILBERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QH2" title="Link to PDB">2QH2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QH2/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2QH2</a></td><td align="center">A</td><td align="center">480</td><td align="center">180</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090109000000">9-25-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE CR7 TERMINAL HAIRPIN LOOP FROM HUMAN TELOMERASE RNA</td><td>J.FEIGON,C.A.THEIMER,N CHIM,K.E.BREECE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3A6P" title="Link to PDB">3A6P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3A6P/index.html" title="Link to FR3D analysis. 0.4783 pairs per nucleotide">3A6P</a></td><td align="center">DE</td><td align="center">92</td><td align="center">44</td><td align="center">2.9</td><td sorttable_customkey="20090812000000">12-08-09</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF EXPORTIN-5:RANGTP:PRE-MIRNA COMPLEX</td><td>C.OKADA,E.YAMASHITA,S.J.LEE,S.SHIBATA,J.KATAHIRA,A.NAKAGAWA, Y.YONEDA,T.TSUKIHARA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FRL" title="Link to PDB">2FRL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FRL/index.html" title="Link to FR3D analysis. 0.3457 pairs per nucleotide">2FRL</a></td><td align="center">A</td><td align="center">460</td><td align="center">159</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070202000000">2-14-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HELIX II TEMPLATE BOUNDARY ELEMENT FROM TETRAHYMENA TELOMERASE RNA</td><td>R.J.RICHARDS,C.A.THEIMER,L.D.FINGER,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QEK" title="Link to PDB">2QEK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QEK/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">2QEK</a></td><td align="center">AB</td><td align="center">46</td><td align="center">20</td><td align="center">1.8</td><td sorttable_customkey="20080605000000">5-06-08</td><td>synthetic</td><td>HIV-1 SUBTYPE F DIS RNA EXTENDED DUPLEX FORM</td><td>S.FREISZ,E.ENNIFAR,P.DUMAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ES5" title="Link to PDB">2ES5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ES5/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">2ES5</a></td><td align="center">A</td><td align="center">460</td><td align="center">200</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071201000000">1-24-06</td><td>synthetic</td><td>STRUCTURE OF THE SRE RNA</td><td>F.H.T.ALLAIN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ESE/index.html" title="SOLUTION NMR, 460 nucleotides, 200 basepairs, 0.4348 pairs per nucleotide">2ESE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B6G/index.html" title="SOLUTION NMR, 19 nucleotides, 8 basepairs, 0.4211 pairs per nucleotide">2B6G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ES5/2ES5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JTJ" title="Link to PDB">1JTJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JTJ/index.html" title="Link to FR3D analysis. 0.3021 pairs per nucleotide">1JTJ</a></td><td align="center">A</td><td align="center">437</td><td align="center">132</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20021112000000">12-11-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HIV-1LAI MUTATED SL1 HAIRPIN</td><td>F.KIEKEN,E.ARNOULT,F.BARBAULT,F.PAQUET,T.HUYNH-DINH, J.PAOLETTI,D.GENEST,G.LANCELOT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NLC" title="Link to PDB">1NLC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NLC/index.html" title="Link to FR3D analysis. 0.4783 pairs per nucleotide">1NLC</a></td><td align="center">AB</td><td align="center">46</td><td align="center">22</td><td align="center">1.9</td><td sorttable_customkey="20040105000000">5-13-03</td><td>synthetic</td><td>HIV-1 DIS(MAL) DUPLEX ZN-SOAKED</td><td>E.ENNIFAR,P.WALTER,P.DUMAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WVD/index.html" title="Resolution 2.9A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1WVD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y6T/index.html" title="Resolution 2.6A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1Y6T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y6S/index.html" title="Resolution 2.9A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1Y6S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/462D/index.html" title="Resolution 2.3A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">462D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O3Z/index.html" title="Resolution 2.6A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1O3Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y73/index.html" title="Resolution 2.9A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1Y73</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y99/index.html" title="Resolution 2.4A, 46 nucleotides, 22 basepairs, 0.4783 pairs per nucleotide">1Y99</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y95/index.html" title="Resolution 2.8A, 46 nucleotides, 21 basepairs, 0.4565 pairs per nucleotide">1Y95</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y90/index.html" title="Resolution 3.1A, 44 nucleotides, 20 basepairs, 0.4545 pairs per nucleotide">1Y90</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OIJ/index.html" title="Resolution 2.3A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">2OIJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OJ0/index.html" title="Resolution 2.6A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">2OJ0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OIY/index.html" title="Resolution 1.6A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">2OIY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3C44/index.html" title="Resolution 2.0A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">3C44</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FAR/index.html" title="Resolution 2.4A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">3FAR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DVV/index.html" title="Resolution 2.0A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">3DVV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NLC/1NLC_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BVJ" title="Link to PDB">1BVJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BVJ/index.html" title="Link to FR3D analysis. 0.3747 pairs per nucleotide">1BVJ</a></td><td align="center">A</td><td align="center">483</td><td align="center">181</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980912000000">12-09-98</td><td>HUMAN IMMUNODEFICIENCY VIRUS TYPE 1</td><td>HIV-1 RNA A-RICH HAIRPIN LOOP</td><td>E.V.PUGLISI,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MWL" title="Link to PDB">1MWL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MWL/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1MWL</a></td><td align="center">AB</td><td align="center">42</td><td align="center">18</td><td align="center">2.4</td><td sorttable_customkey="20040602000000">2-18-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GENETICIN BOUND TO THE EUBACTERIAL 16S RRNA A SITE</td><td>Q.VICENS,E.WESTHOF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BEE/index.html" title="Resolution 2.6A, 42 nucleotides, 18 basepairs, 0.4286 pairs per nucleotide">2BEE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BE0/index.html" title="Resolution 2.6A, 42 nucleotides, 18 basepairs, 0.4286 pairs per nucleotide">2BE0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PWT/index.html" title="Resolution 1.8A, 42 nucleotides, 17 basepairs, 0.4048 pairs per nucleotide">2PWT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YRJ/index.html" title="Resolution 2.7A, 45 nucleotides, 18 basepairs, 0.4000 pairs per nucleotide">1YRJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F4T/index.html" title="Resolution 3.0A, 44 nucleotides, 17 basepairs, 0.3864 pairs per nucleotide">2F4T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ET4/index.html" title="Resolution 2.4A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2ET4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ET5/index.html" title="Resolution 2.2A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2ET5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LC4/index.html" title="Resolution 2.5A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">1LC4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ET3/index.html" title="Resolution 2.8A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2ET3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ESJ/index.html" title="Resolution 2.2A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2ESJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G5Q/index.html" title="Resolution 2.7A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2G5Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J7T/index.html" title="Resolution 2.5A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">1J7T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ESI/index.html" title="Resolution 3.0A, 44 nucleotides, 16 basepairs, 0.3636 pairs per nucleotide">2ESI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MWL/1MWL_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S2F" title="Link to PDB">1S2F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S2F/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">1S2F</a></td><td align="center">A</td><td align="center">23</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060708000000">8-31-04</td><td>synthetic</td><td>AVERAGE SOLUTION STRUCTURE OF A PSEUDO-5'-SPLICE SITE FROM THE NEGATIVE REGULATOR OF SPLICING OF ROUS SARCOMA VIRUS</td><td>J.CABELLO-VILLEGAS,K.E.GILES,A.M.SOTO,P.YU,K.L.BEEMON, Y.X.WANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XPE" title="Link to PDB">1XPE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPE/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">1XPE</a></td><td align="center">AB</td><td align="center">46</td><td align="center">20</td><td align="center">1.9</td><td sorttable_customkey="20060610000000">10-18-05</td><td>synthetic</td><td>HIV-1 SUBTYPE B GENOMIC RNA DIMERIZATION INITIATION SITE</td><td>E.ENNIFAR,P.DUMAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XP7/index.html" title="Resolution 2.5A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">1XP7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B8R/index.html" title="Resolution 2.6A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">2B8R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B8S/index.html" title="Resolution 2.8A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">2B8S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPF/index.html" title="Resolution 2.3A, 46 nucleotides, 20 basepairs, 0.4348 pairs per nucleotide">1XPF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FD0/index.html" title="Resolution 1.8A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">2FD0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FCY/index.html" title="Resolution 2.2A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">2FCY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZCI/index.html" title="Resolution 1.6A, 88 nucleotides, 36 basepairs, 0.4091 pairs per nucleotide">1ZCI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y3S/index.html" title="Resolution 2.2A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">1Y3S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y3O/index.html" title="Resolution 2.7A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">1Y3O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YXP/index.html" title="Resolution 2.4A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">1YXP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FCX/index.html" title="Resolution 2.0A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">2FCX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FCZ/index.html" title="Resolution 2.0A, 88 nucleotides, 36 basepairs, 0.4091 pairs per nucleotide">2FCZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K9W/index.html" title="Resolution 3.1A, 46 nucleotides, 16 basepairs, 0.3478 pairs per nucleotide">1K9W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPE/1XPE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S34" title="Link to PDB">1S34</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S34/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">1S34</a></td><td align="center">A</td><td align="center">345</td><td align="center">135</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060708000000">8-31-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF RESIDUES 907-929 FROM ROUS SARCOMA VIRUS</td><td>J.CABELLO-VILLEGAS,K.E.GILES,A.M.SOTO,P.YU,K.L.BEEMON, Y.X.WANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TLR" title="Link to PDB">1TLR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TLR/index.html" title="Link to FR3D analysis. 0.3239 pairs per nucleotide">1TLR</a></td><td align="center">A</td><td align="center">460</td><td align="center">149</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971211000000">11-12-97</td><td>synthetic</td><td>SOLUTION STRUCTURE OF TETRALOOP RECEPTOR RNA, NMR, 20 STRUCTURES</td><td>S.E.BUTCHER,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MFK" title="Link to PDB">1MFK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MFK/index.html" title="Link to FR3D analysis. 0.3891 pairs per nucleotide">1MFK</a></td><td align="center">A</td><td align="center">460</td><td align="center">179</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030111000000">11-13-02</td><td>synthetic</td><td>STRUCTURE OF PROKARYOTIC SECIS MRNA HAIRPIN</td><td>D.FOURMY,E.GUITTET,S.YOSHIZAWA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BGZ" title="Link to PDB">1BGZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BGZ/index.html" title="Link to FR3D analysis. 0.3406 pairs per nucleotide">1BGZ</a></td><td align="center">A</td><td align="center">138</td><td align="center">47</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000101000000">1-13-99</td><td>synthetic</td><td>S8 RRNA BINDING SITE FROM E. COLI, NMR, 6 STRUCTURES</td><td>K.KALURACHCHI,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PJP" title="Link to PDB">2PJP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PJP/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">2PJP</a></td><td align="center">B</td><td align="center">23</td><td align="center">10</td><td align="center">2.3</td><td sorttable_customkey="20090610000000">10-30-07</td><td>ESCHERICHIA COLI</td><td>STRUCTURE OF THE MRNA-BINDING DOMAIN OF ELONGATION FACTOR SELB FROM E.COLI IN COMPLEX WITH SECIS RNA</td><td>N.SOLER,D.FOURMY,S.YOSHIZAWA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OW9" title="Link to PDB">1OW9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OW9/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">1OW9</a></td><td align="center">A</td><td align="center">253</td><td align="center">99</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040805000000">5-20-03</td><td>synthetic</td><td>NMR STRUCTURE OF THE ACTIVE CONFORMATION OF THE VS RIBOZYME CLEAVAGE SITE</td><td>B.HOFFMANN,G.T.MITCHELL,P.GENDRON,F.MAJOR,A.A.ANDERSEN, R.A.COLLINS,P.LEGAULT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HYS" title="Link to PDB">1HYS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HYS/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1HYS</a></td><td align="center">E</td><td align="center">23</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20030203000000">3-26-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HIV-1 REVERSE TRANSCRIPTASE IN COMPLEX WITH A POLYPURINE TRACT RNA:DNA</td><td>S.G.SARAFIANOS,K.DAS,C.TANTILLO,A.D.CLARK JR.,J.DING, J.WHITCOMB,P.L.BOYER,S.H.HUGHES,E.ARNOLD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K5I" title="Link to PDB">1K5I</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K5I/index.html" title="Link to FR3D analysis. 0.3957 pairs per nucleotide">1K5I</a></td><td align="center">A</td><td align="center">230</td><td align="center">91</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020510000000">10-17-01</td><td>synthetic</td><td>NMR STRUCTURE OF A RIBOSOMAL RNA HAIRPIN CONTAINING A CONSERVED CUCAA PENTALOOP</td><td>U.NAGASWAMY,X.GAO,S.A.MARTINIS,G.E.FOX</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PLY" title="Link to PDB">2PLY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PLY/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">2PLY</a></td><td align="center">E</td><td align="center">46</td><td align="center">20</td><td align="center">2.6</td><td sorttable_customkey="20070611000000">11-06-07</td><td>MOORELLA THERMOACETICA</td><td>STRUCTURE OF THE MRNA BINDING FRAGMENT OF ELONGATION FACTOR SELB IN COMPLEX WITH SECIS RNA.</td><td>N.SOLER,D.FOURMY,S.YOSHIZAWA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2UWM/index.html" title="Resolution 2.3A, 46 nucleotides, 19 basepairs, 0.4130 pairs per nucleotide">2UWM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WSU/index.html" title="Resolution 2.3A, 65 nucleotides, 26 basepairs, 0.4000 pairs per nucleotide">1WSU</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PLY/2PLY_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PHP" title="Link to PDB">3PHP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PHP/index.html" title="Link to FR3D analysis. 0.3478 pairs per nucleotide">3PHP</a></td><td align="center">A</td><td align="center">230</td><td align="center">80</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990910000000">10-21-98</td><td>synthetic</td><td>STRUCTURE OF THE 3' HAIRPIN OF THE TYMV PSEUDOKNOT: PREFORMATION IN RNA FOLDING</td><td>M.H.KOLK,M.VAN DER GRAAF,S.S.WIJMENGA,C.W.A.PLEIJ,H.A.HEUS, C.W.HILBERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QH3" title="Link to PDB">2QH3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QH3/index.html" title="Link to FR3D analysis. 0.3478 pairs per nucleotide">2QH3</a></td><td align="center">A</td><td align="center">460</td><td align="center">160</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090109000000">9-25-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE U64 H/ACA SNORNA 3' TERMINAL HAIRPIN LOOP</td><td>J.FEIGON,C.A.THEIMER,N.CHIM,K.E.BREECE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JU1" title="Link to PDB">1JU1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JU1/index.html" title="Link to FR3D analysis. 0.4232 pairs per nucleotide">1JU1</a></td><td align="center">AB</td><td align="center">690</td><td align="center">292</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20031108000000">8-23-02</td><td>synthetic</td><td>DIMER INITIATION SEQUENCE OF HIV-1LAI GENOMIC RNA: NMR SOLUTION STRUCTURE OF THE EXTENDED DUPLEX</td><td>F.GIRARD,F.BARBAULT,G.LANCELOT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NEM" title="Link to PDB">1NEM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NEM/index.html" title="Link to FR3D analysis. 0.4251 pairs per nucleotide">1NEM</a></td><td align="center">A</td><td align="center">207</td><td align="center">88</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010708000000">8-31-99</td><td>synthetic</td><td>SACCHARIDE-RNA RECOGNITION IN THE NEOMYCIN B / RNA APTAMER COMPLEX</td><td>L.JIANG,A.MAJUMDAR,W.HU,T.J.JAISHREE,W.XU,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BAU" title="Link to PDB">1BAU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BAU/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">1BAU</a></td><td align="center">AB</td><td align="center">46</td><td align="center">18</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>NMR STRUCTURE OF THE DIMER INITIATION COMPLEX OF HIV-1 GENOMIC RNA, MINIMIZED AVERAGE STRUCTURE</td><td>A.MUJEEB,J.L.CLEVER,T.M.BILLECI,T.L.JAMES,T.G.PARSLOW</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RO2" title="Link to PDB">2RO2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RO2/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">2RO2</a></td><td align="center">A</td><td align="center">23</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100612000000">12-30-08</td><td>synthetic</td><td>SOLUTION STRUCTURE OF DOMAIN I OF THE NEGATIVE POLARITY CCHMVD HAMMERHEAD RIBOZYME</td><td>J.GALLEGO,D.DUFOUR,S.GAGO,M.DE LA PENA,R.FLORES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BNL" title="Link to PDB">3BNL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNL/index.html" title="Link to FR3D analysis. 0.4773 pairs per nucleotide">3BNL</a></td><td align="center">AB</td><td align="center">44</td><td align="center">21</td><td align="center">2.6</td><td sorttable_customkey="20091206000000">6-24-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE BACTERIAL RIBOSOMAL DECODING A SITE IN THE PRESENCE OF [CO(NH3)6]CL3</td><td>J.KONDO,E.WESTHOF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YG3" title="Link to PDB">1YG3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YG3/index.html" title="Link to FR3D analysis. 0.3667 pairs per nucleotide">1YG3</a></td><td align="center">A</td><td align="center">540</td><td align="center">198</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060112000000">12-13-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE SCYLV P1-P2 FRAMESHIFTING PSEUDOKNOT, 20 LOWEST ENERGY STRUCTURES</td><td>P.V.CORNISH,M.HENNIG,D.P.GIEDROC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IKD" title="Link to PDB">1IKD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IKD/index.html" title="Link to FR3D analysis. 0.3121 pairs per nucleotide">1IKD</a></td><td align="center">A</td><td align="center">660</td><td align="center">206</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970104000000">4-01-97</td><td>synthetic</td><td>ACCEPTOR STEM, NMR, 30 STRUCTURES</td><td>A.RAMOS,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HTX" title="Link to PDB">3HTX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HTX/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">3HTX</a></td><td align="center">BC</td><td align="center">88</td><td align="center">40</td><td align="center">3.1</td><td sorttable_customkey="20110509000000">9-29-09</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF SMALL RNA METHYLTRANSFERASE HEN1</td><td>Y.HUANG,L.-J.JI,Q.-C.HUANG,D.G.VASSYLYEV,X.-M.CHEN,J.-B.MA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BNP" title="Link to PDB">3BNP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNP/index.html" title="Link to FR3D analysis. 0.3721 pairs per nucleotide">3BNP</a></td><td align="center">AB</td><td align="center">43</td><td align="center">16</td><td align="center">2.7</td><td sorttable_customkey="20091206000000">6-24-08</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS MITOCHONDRIAL RIBOSOMAL DECODING SITE (A1555G MUTANT)</td><td>J.KONDO,E.WESTHOF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNO/index.html" title="Resolution 2.4A, 80 nucleotides, 27 basepairs, 0.3375 pairs per nucleotide">3BNO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNN/index.html" title="Resolution 2.0A, 84 nucleotides, 28 basepairs, 0.3333 pairs per nucleotide">3BNN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNP/3BNP_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FQN" title="Link to PDB">2FQN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FQN/index.html" title="Link to FR3D analysis. 0.4186 pairs per nucleotide">2FQN</a></td><td align="center">AB</td><td align="center">43</td><td align="center">18</td><td align="center">2.3</td><td sorttable_customkey="20070202000000">2-14-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS CYTOPLASMIC RIBOSOMAL DECODING A SITE</td><td>J.KONDO,A.URZHUMTSEV,E.WESTHOF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O3W/index.html" title="Resolution 2.8A, 43 nucleotides, 18 basepairs, 0.4186 pairs per nucleotide">2O3W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G5K/index.html" title="Resolution 2.8A, 44 nucleotides, 18 basepairs, 0.4091 pairs per nucleotide">2G5K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FQN/2FQN_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JYM" title="Link to PDB">2JYM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JYM/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">2JYM</a></td><td align="center">A</td><td align="center">330</td><td align="center">135</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100504000000">4-29-08</td><td>synthetic</td><td>SOLUTION STRUCTURE OF STEM-LOOP ALPHA OF THE HEPATITIS B VIRUS POST-TRANSCRIPTIONAL REGULATORY ELEMENT</td><td>O.OHLENSCHLAGER,M.GORLACH,M.SCHWALBE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JUR" title="Link to PDB">1JUR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JUR/index.html" title="Link to FR3D analysis. 0.3838 pairs per nucleotide">1JUR</a></td><td align="center">A</td><td align="center">198</td><td align="center">76</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030401000000">1-16-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HELIX III IN XENOPUS OOCYTE 5S RRNA.</td><td>P.W.HUBER,J.P.RIFE,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K6G" title="Link to PDB">1K6G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K6G/index.html" title="Link to FR3D analysis. 0.4159 pairs per nucleotide">1K6G</a></td><td align="center">A</td><td align="center">440</td><td align="center">183</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020712000000">12-19-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF CONSERVED AGNN TETRALOOPS: INSIGHTS INTO RNT1P RNA PROCESSING</td><td>I.LEBARS,B.LAMONTAGNE,S.YOSHIZAWA,S.ABOU ELELA,D.FOURMY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F9L" title="Link to PDB">1F9L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F9L/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">1F9L</a></td><td align="center">A</td><td align="center">22</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020207000000">7-26-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A 22-NUCLEOTIDE HAIRPIN SIMILAR TO THE P5ABC REGION OF A GROUP I RIBOZYME WITH COBALT(III) HEXAMMINE COMPLEXED TO THE GAAA TETRALOOP</td><td>S.RUDISSER,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BNQ" title="Link to PDB">3BNQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNQ/index.html" title="Link to FR3D analysis. 0.3721 pairs per nucleotide">3BNQ</a></td><td align="center">AB</td><td align="center">86</td><td align="center">32</td><td align="center">2.0</td><td sorttable_customkey="20091206000000">6-24-08</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS MITOCHONDRIAL RIBOSOMAL DECODING SITE IN THE PRESENCE OF SRCL2 (A1555G MUTANT, BR-DERIVATIVE)</td><td>J.KONDO,E.WESTHOF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNS/index.html" title="Resolution 1.9A, 85 nucleotides, 31 basepairs, 0.3647 pairs per nucleotide">3BNS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNR/index.html" title="Resolution 2.1A, 85 nucleotides, 31 basepairs, 0.3647 pairs per nucleotide">3BNR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BNQ/3BNQ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FTF" title="Link to PDB">3FTF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FTF/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">3FTF</a></td><td align="center">CD</td><td align="center">44</td><td align="center">20</td><td align="center">2.8</td><td sorttable_customkey="20101203000000">3-24-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A. AEOLICUS KSGA IN COMPLEX WITH RNA AND SAH</td><td>C.TU,X.JI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FTE/index.html" title="Resolution 3.0A, 44 nucleotides, 20 basepairs, 0.4545 pairs per nucleotide">3FTE</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FTF/3FTF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GV3" title="Link to PDB">2GV3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GV3/index.html" title="Link to FR3D analysis. 0.3705 pairs per nucleotide">2GV3</a></td><td align="center">A</td><td align="center">440</td><td align="center">163</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060808000000">8-08-06</td><td>synthetic</td><td>TRANSLOCATION OF A TRNA WITH AN EXTENDED ANTICODON THROUGH THE RIBOSOME</td><td>S.S.PHELPS,C.GAUDIN,S.YOSHIZAWA,C.BENITEZ,D.FOURMY,S.JOSEPH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O3Y" title="Link to PDB">2O3Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O3Y/index.html" title="Link to FR3D analysis. 0.3953 pairs per nucleotide">2O3Y</a></td><td align="center">AB</td><td align="center">43</td><td align="center">17</td><td align="center">2.7</td><td sorttable_customkey="20070611000000">11-06-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS CYTOPLASMIC RIBOSOMAL DECODING SITE IN PRESENCE OF PAROMAMINE DERIVATIVE NB30</td><td>J.KONDO,M.HAINRICHSON,I.NUDELMAN,D.SHALLOM-SHEZIFI,T.BAASOV, E.WESTHOF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FJE" title="Link to PDB">1FJE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FJE/index.html" title="Link to FR3D analysis. 0.3541 pairs per nucleotide">1FJE</a></td><td align="center">A</td><td align="center">418</td><td align="center">148</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010301000000">1-03-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF NUCLEOLIN RBD12 IN COMPLEX WITH SNRE RNA</td><td>F.H.T.ALLAIN,P.BOUVET,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TJZ" title="Link to PDB">1TJZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TJZ/index.html" title="Link to FR3D analysis. 0.3773 pairs per nucleotide">1TJZ</a></td><td align="center">A</td><td align="center">440</td><td align="center">166</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060409000000">9-28-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE ACTIVE SITE STEM-LOOP OF THE VS RIBOZYME</td><td>J.FLINDERS,T.DIECKMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GRW" title="Link to PDB">2GRW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GRW/index.html" title="Link to FR3D analysis. 0.3447 pairs per nucleotide">2GRW</a></td><td align="center">A</td><td align="center">264</td><td align="center">91</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070603000000">3-06-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE POLIOVIRUS 3'-UTR Y-STEM</td><td>H.A.HEUS,J.ZOLL,M.TESSARI,F.J.M.VAN KUPPEVELD,W.J.G.MELCHERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GV4" title="Link to PDB">2GV4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GV4/index.html" title="Link to FR3D analysis. 0.4161 pairs per nucleotide">2GV4</a></td><td align="center">A</td><td align="center">286</td><td align="center">119</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080803000000">3-20-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE POLIOVIRUS 3'-UTR Y-STEM</td><td>H.A.HEUS,J.ZOLL,M.TESSARI,F.J.M.VAN KUPPEVELD,W.J.G.MELCHERS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N66/index.html" title="SOLUTION NMR, 440 nucleotides, 175 basepairs, 0.3977 pairs per nucleotide">1N66</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GV4/2GV4_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MEI" title="Link to PDB">3MEI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MEI/index.html" title="Link to FR3D analysis. 0.4318 pairs per nucleotide">3MEI</a></td><td align="center">AB</td><td align="center">44</td><td align="center">19</td><td align="center">2.0</td><td sorttable_customkey="20130201000000">1-26-11</td><td>synthetic</td><td>REGULATORY MOTIF FROM THE THYMIDYLATE SYNTHASE MRNA</td><td>S.DIBROV,J.MCLEAN,T.HERMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OSW" title="Link to PDB">1OSW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OSW/index.html" title="Link to FR3D analysis. 0.3577 pairs per nucleotide">1OSW</a></td><td align="center">A</td><td align="center">506</td><td align="center">181</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040805000000">5-20-03</td><td>synthetic</td><td>THE STEM OF SL1 RNA IN HIV-1: STRUCTURE AND NUCLEOCAPSID PROTEIN BINDING FOR A 1X3 INTERNAL LOOP</td><td>Y.YUAN,D.J.KERWOOD,A.C.PAOLETTI,M.F.SHUBSDA,P.N.BORER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O3V" title="Link to PDB">2O3V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O3V/index.html" title="Link to FR3D analysis. 0.3953 pairs per nucleotide">2O3V</a></td><td align="center">AB</td><td align="center">43</td><td align="center">17</td><td align="center">2.8</td><td sorttable_customkey="20070611000000">11-06-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HOMO SAPIENS CYTOPLASMIC RIBOSOMAL DECODING SITE COMPLEXED WITH PAROMAMINE DERIVATIVE NB33</td><td>J.KONDO,M.HAINRICHSON,I.NUDELMAN,D.SHALLOM-SHEZIFI,T.BAASOV, E.WESTHOF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NTB" title="Link to PDB">1NTB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTB/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1NTB</a></td><td align="center">AB</td><td align="center">40</td><td align="center">15</td><td align="center">2.9</td><td sorttable_customkey="20040105000000">5-13-03</td><td>synthetic,synthetic</td><td>.9 A CRYSTAL STRUCTURE OF STREPTOMYCIN RNA-APTAMER COMPLEX</td><td>V.TERESHKO,E.SKRIPKIN,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTA/index.html" title="Resolution 2.9A, 40 nucleotides, 14 basepairs, 0.3500 pairs per nucleotide">1NTA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTB/1NTB_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HNS" title="Link to PDB">2HNS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HNS/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">2HNS</a></td><td align="center">A</td><td align="center">440</td><td align="center">180</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080710000000">10-31-06</td><td>synthetic</td><td>STRUCTURE OF THE AAGU TETRALOOP</td><td>C.GAUDIN,F.FOURMY,S.YOSHIZAWA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KD8" title="Link to PDB">2KD8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KD8/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">2KD8</a></td><td align="center">A</td><td align="center">264</td><td align="center">108</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110701000000">1-19-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE STEM-LOOP IIID OF GBV-B IRES</td><td>V.THIVIYANATHAN,R.KULASEGRAN SHYLINI,D.G.GORENSTEIN, K.KALUARACHCHI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JSE" title="Link to PDB">2JSE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JSE/index.html" title="Link to FR3D analysis. 0.3636 pairs per nucleotide">2JSE</a></td><td align="center">A</td><td align="center">220</td><td align="center">80</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080811000000">11-20-07</td><td>synthetic</td><td>NMR REVEALS ABSENCE OF HYDROGEN BONDING IN ADJACENT UU AND AG MISMATCHES IN AN ISOLATED INTERNAL LOOP FROM RIBOSOMAL RNA.</td><td>N.SHANKAR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PJY" title="Link to PDB">1PJY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PJY/index.html" title="Link to FR3D analysis. 0.4432 pairs per nucleotide">1PJY</a></td><td align="center">A</td><td align="center">440</td><td align="center">195</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030508000000">8-05-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HIV-1 FRAMESHIFT INDUCING STEM- LOOP RNA</td><td>D.W.STAPLE,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F8S" title="Link to PDB">2F8S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F8S/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">2F8S</a></td><td align="center">CD</td><td align="center">44</td><td align="center">18</td><td align="center">3.0</td><td sorttable_customkey="20080710000000">10-31-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AA-AGO WITH EXTERNALLY-BOUND SIRNA</td><td>Y.R.YUAN,H.Y.CHEN,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DZ5" title="Link to PDB">1DZ5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DZ5/index.html" title="Link to FR3D analysis. 0.3182 pairs per nucleotide">1DZ5</a></td><td align="center">CD</td><td align="center">572</td><td align="center">182</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020503000000">3-29-00</td><td>HOMO SAPIENS</td><td>THE NMR STRUCTURE OF THE 38KDA U1A PROTEIN-PIE RNA COMPLEX REVEALS THE BASIS OF COOPERATIVITY IN REGULATION OF POLYADENYLATION BY HUMAN U1A PROTEIN</td><td>L.VARANI,S.I.GUNDERSON,I.W.MATTAJ,L.E.KAY,D.NEUHAUS,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IE1" title="Link to PDB">1IE1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IE1/index.html" title="Link to FR3D analysis. 0.3157 pairs per nucleotide">1IE1</a></td><td align="center">A</td><td align="center">396</td><td align="center">125</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020806000000">6-20-01</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF AN IN VITRO SELECTED RNA WHICH IS SEQUENCE SPECIFICALLY RECOGNIZED BY HAMSTER NUCLEOLIN RBD12.</td><td>P.BOUVET,F.H-T.ALLAIN,L.D.FINGER,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K66" title="Link to PDB">2K66</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K66/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">2K66</a></td><td align="center">A</td><td align="center">440</td><td align="center">200</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100207000000">7-14-09</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE D3'-STEM CLOSED BY A GAAA TETRALOOP OF THE GROUP II INTRON SC.AI5(GAMMA)</td><td>D.KRUSCHEL,R.K.O.SIGEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G1W" title="Link to PDB">2G1W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G1W/index.html" title="Link to FR3D analysis. 0.3273 pairs per nucleotide">2G1W</a></td><td align="center">A</td><td align="center">330</td><td align="center">108</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061104000000">4-11-06</td><td>synthetic</td><td>NMR STRUCTURE OF THE AQUIFEX AEOLICUS TMRNA PSEUDOKNOT PK1</td><td>S.NONIN-LECOMTE,F.DARDEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3LOA" title="Link to PDB">3LOA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LOA/index.html" title="Link to FR3D analysis. 0.3571 pairs per nucleotide">3LOA</a></td><td align="center">AB</td><td align="center">42</td><td align="center">15</td><td align="center">2.3</td><td sorttable_customkey="20100408000000">8-04-10</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE ANALYSIS OF THE RNA CONSTRUCT WITH TWO ADJACENT LIGAND BINDING SITES OF HELIX H44 IN 16S RIBOSOMAL RNA</td><td>S.DIBROV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K6H" title="Link to PDB">1K6H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K6H/index.html" title="Link to FR3D analysis. 0.4295 pairs per nucleotide">1K6H</a></td><td align="center">A</td><td align="center">440</td><td align="center">189</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020712000000">12-19-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF CONSERVED AGNN TETRALOOPS: INSIGHTS INTO RNT1P RNA PROCESSING</td><td>I.LEBARS,B.LAMONTAGNE,S.YOSHIZAWA,S.ABOU ELELA,D.FOURMY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2W2H" title="Link to PDB">2W2H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2W2H/index.html" title="Link to FR3D analysis. 0.1591 pairs per nucleotide">2W2H</a></td><td align="center">S</td><td align="center">44</td><td align="center">7</td><td align="center">3.2</td><td sorttable_customkey="20080912000000">12-09-08</td><td>EQUINE INFECTIOUS ANEMIA VIRUS</td><td>STRUCTURAL BASIS OF TRANSCRIPTION ACTIVATION BY THE CYCLIN T1-TAT-TAR RNA COMPLEX FROM EIAV</td><td>K.ANAND,M.GEYER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K2G" title="Link to PDB">1K2G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K2G/index.html" title="Link to FR3D analysis. 0.3636 pairs per nucleotide">1K2G</a></td><td align="center">A</td><td align="center">22</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020805000000">5-08-02</td><td>synthetic</td><td>STRUCTURAL BASIS FOR THE 3'-TERMINAL GUANOSINE RECOGNITION BY THE GROUP I INTRON</td><td>Y.KITAMURA,Y.MUTO,S.WATANABE,I.KIM,T.ITO,Y.NISHIYA, K.SAKAMOTO,T.OHTSUKI,G.KAWAI,K.WATANABE,K.HOSONO,H.TAKAKU, E.KATOH,T.YAMAZAKI,T.INOUE,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RLG" title="Link to PDB">1RLG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RLG/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">1RLG</a></td><td align="center">D</td><td align="center">44</td><td align="center">18</td><td align="center">2.7</td><td sorttable_customkey="20040106000000">6-01-04</td><td>synthetic</td><td>MOLECULAR BASIS OF BOX C/D RNA-PROTEIN INTERACTION: CO- CRYSTAL STRUCTURE OF THE ARCHAEAL SRNP INTIATION COMPLEX</td><td>T.MOORE,Y.ZHANG,M.O.FENLEY,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IE2" title="Link to PDB">1IE2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IE2/index.html" title="Link to FR3D analysis. 0.3377 pairs per nucleotide">1IE2</a></td><td align="center">A</td><td align="center">462</td><td align="center">156</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020806000000">6-20-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN IN VITRO SELECTED RNA WHICH IS SEQUENCE SPECIFICALLY RECOGNIZED BY RBD12 OF HAMSTER NUCLEOLIN.SNRE (ANTI)</td><td>P.BOUVET,F.H.-T.ALLAIN,L.D.FINGER,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FEU" title="Link to PDB">1FEU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FEU/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1FEU</a></td><td align="center">BC</td><td align="center">80</td><td align="center">32</td><td align="center">2.3</td><td sorttable_customkey="20030106000000">6-25-01</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF RIBOSOMAL PROTEIN TL5, ONE OF THE CTC FAMILY PROTEINS, COMPLEXED WITH A FRAGMENT OF 5S RRNA.</td><td>R.V.FEDOROV,V.A.MESHCHERYAKOV,G.M.GONGADZE,N.P.FOMENKOVA, N.A.NEVSKAYA,M.SELMER,M.LAURBERG,O.KRISTENSEN,S.AL- KARADAGHI,A.LILJAS,M.B.GARBER,S.V.NIKONOV</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JP0" title="Link to PDB">1JP0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JP0/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1JP0</a></td><td align="center">A</td><td align="center">588</td><td align="center">196</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020105000000">5-01-02</td><td>synthetic</td><td>NMR STRUCTURE OF THE LP5.1 HAIRPIN FROM BACILLUS RNASE P RNA REFINED WITHOUT RESIDUAL DIPOLAR COUPLINGS</td><td>T.C.LEEPER,F.J.SCHMIDT,S.R.VAN DOREN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O3X" title="Link to PDB">2O3X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O3X/index.html" title="Link to FR3D analysis. 0.4048 pairs per nucleotide">2O3X</a></td><td align="center">AB</td><td align="center">42</td><td align="center">17</td><td align="center">2.9</td><td sorttable_customkey="20070611000000">11-06-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PROKARYOTIC RIBOSOMAL DECODING SITE COMPLEXED WITH PAROMAMINE DERIVATIVE NB30</td><td>J.KONDO,M.HAINRICHSON,I.NUDELMAN,D.SHALLOM-SHEZIFI,T.BAASOV, E.WESTHOF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F4S/index.html" title="Resolution 2.8A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">2F4S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O3X/2O3X_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1URN" title="Link to PDB">1URN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1URN/index.html" title="Link to FR3D analysis. 0.2364 pairs per nucleotide">1URN</a></td><td align="center">Q</td><td align="center">55</td><td align="center">13</td><td align="center">1.9</td><td sorttable_customkey="19960803000000">3-08-96</td><td>synthetic</td><td>U1A MUTANT/RNA COMPLEX + GLYCEROL</td><td>C.OUBRIDGE,N.ITO,P.R.EVANS,C.-H.TEO,K.NAGAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SZY" title="Link to PDB">1SZY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SZY/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1SZY</a></td><td align="center">A</td><td align="center">21</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050804000000">4-20-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF ITALY1 ("INITIATOR TRNA ANTICODON LOOP FROM YEAST"), AN UNMODIFIED 21-NT RNA WITH THE SEQUENCE OF THE ANTICODON STEM-LOOP OF YEAST INITIATOR TRNA</td><td>D.C.SCHWEISGUTH,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QWA" title="Link to PDB">1QWA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QWA/index.html" title="Link to FR3D analysis. 0.4034 pairs per nucleotide">1QWA</a></td><td align="center">A</td><td align="center">357</td><td align="center">144</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-25-03</td><td>synthetic</td><td>NMR STRUCTURE OF 5'-R(GGAUGCCUCCCGAGUGCAUCC): AN RNA HAIRPIN DERIVED FROM THE MOUSE 5'ETS THAT BINDS NUCLEOLIN RBD12.</td><td>L.D.FINGER,L.TRANTIREK,C.JOHANSSON,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K8W" title="Link to PDB">1K8W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K8W/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">1K8W</a></td><td align="center">B</td><td align="center">21</td><td align="center">8</td><td align="center">1.9</td><td sorttable_customkey="20030712000000">12-31-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE E. COLI PSEUDOURIDINE SYNTHASE TRUB BOUND TO A T STEM-LOOP RNA</td><td>C.HOANG,A.R.FERRE-D'AMARE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZL3/index.html" title="Resolution 2.8A, 21 nucleotides, 8 basepairs, 0.3810 pairs per nucleotide">1ZL3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZE2/index.html" title="Resolution 3.0A, 42 nucleotides, 16 basepairs, 0.3810 pairs per nucleotide">1ZE2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AB4/index.html" title="Resolution 2.4A, 19 nucleotides, 7 basepairs, 0.3684 pairs per nucleotide">2AB4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K8W/1K8W_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=17RA" title="Link to PDB">17RA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/17RA/index.html" title="Link to FR3D analysis. 0.3413 pairs per nucleotide">17RA</a></td><td align="center">A</td><td align="center">252</td><td align="center">86</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000804000000">4-20-99</td><td>synthetic</td><td>BRANCHPOINT HELIX FROM YEAST AND BINDING SITE FOR PHAGE GA/MS2 COAT PROTEINS, NMR, 12 STRUCTURES</td><td>E.P.NIKONOWICZ,J.S.SMITH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1O9M" title="Link to PDB">1O9M</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O9M/index.html" title="Link to FR3D analysis. 0.4048 pairs per nucleotide">1O9M</a></td><td align="center">AB</td><td align="center">42</td><td align="center">17</td><td align="center">2.4</td><td sorttable_customkey="20050303000000">3-27-03</td><td>ESCHERICHIA COLI</td><td>THE COMPLEX OF A NOVEL ANTIBIOTIC WITH THE AMINOACYL SITE OF THE BACTERIAL RIBOSOME REVEALED BY X-RAY CRYSTALLOGRAPHY.</td><td>R.RUSSELL,J.B.MURRAY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F4U/index.html" title="Resolution 2.6A, 42 nucleotides, 17 basepairs, 0.4048 pairs per nucleotide">2F4U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ET8/index.html" title="Resolution 2.5A, 42 nucleotides, 17 basepairs, 0.4048 pairs per nucleotide">2ET8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1O9M/1O9M_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZKO" title="Link to PDB">2ZKO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZKO/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2ZKO</a></td><td align="center">CD</td><td align="center">42</td><td align="center">18</td><td align="center">1.7</td><td sorttable_customkey="20090210000000">10-14-08</td><td>INFLUENZA A VIRUS</td><td>STRUCTURAL BASIS FOR DSRNA RECOGNITION BY NS1 PROTEIN OF HUMAN INFLUENZA VIRUS A</td><td>Y.A.YUAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BJ2" title="Link to PDB">2BJ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BJ2/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2BJ2</a></td><td align="center">AB</td><td align="center">40</td><td align="center">20</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990202000000">2-02-99</td><td>synthetic,synthetic</td><td>RNA LOOP-LOOP COMPLEX: THE COLE1 INVERTED LOOP SEQUENCE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>A.J.LEE,D.M.CROTHERS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BJ2/index.html" title="SOLUTION NMR, 320 nucleotides, 159 basepairs, 0.4969 pairs per nucleotide">1BJ2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BJ2/2BJ2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FY1" title="Link to PDB">2FY1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FY1/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">2FY1</a></td><td align="center">B</td><td align="center">357</td><td align="center">136</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070602000000">2-06-07</td><td>HOMO SAPIENS</td><td>A DUAL MODE OF RNA RECOGNITION BY THE RBMY PROTEIN</td><td>L.SKRISOVSKA,C.BOURGOIS,R.STEFL,L.KISTER,P.WENTER,D.ELLIOT, J.STEVENIN,F.H.T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RPU" title="Link to PDB">1RPU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RPU/index.html" title="Link to FR3D analysis. 0.4524 pairs per nucleotide">1RPU</a></td><td align="center">CD</td><td align="center">42</td><td align="center">19</td><td align="center">2.5</td><td sorttable_customkey="20050101000000">1-13-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF CIRV P19 BOUND TO SIRNA</td><td>J.M.VARGASON,G.SZITTYA,J.BURGYAN,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RKJ" title="Link to PDB">1RKJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RKJ/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1RKJ</a></td><td align="center">B</td><td align="center">294</td><td align="center">98</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060304000000">4-27-04</td><td>MESOCRICETUS AURATUS</td><td>SOLUTION STRUCTURE OF THE COMPLEX FORMED BY THE TWO N- TERMINAL RNA-BINDING DOMAINS OF NUCLEOLIN AND A PRE-RRNA TARGET</td><td>C.JOHANSSON,L.D.FINGER,L.TRANTIREK,T.D.MUELLER,S.KIM, I.A.LAIRD-OFFRINGA,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BOY" title="Link to PDB">3BOY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BOY/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BOY</a></td><td align="center">D</td><td align="center">21</td><td align="center">0</td><td align="center">1.7</td><td sorttable_customkey="20090301000000">1-15-08</td><td>BACILLUS SUBTILIS</td><td>CRYSTAL STRUCTURE OF THE HUTP ANTITERMINATION COMPLEX BOUND TO THE HUT MRNA</td><td>T.S.KUMAREVEL,D.BALASUNDARESAN,J.JEYAKANTHAN,A.SHINKAI, S.YOKOYAMA,P.K.R.KUMAR,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2H2X" title="Link to PDB">2H2X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H2X/index.html" title="Link to FR3D analysis. 0.3405 pairs per nucleotide">2H2X</a></td><td align="center">A</td><td align="center">420</td><td align="center">143</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060310000000">10-03-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE TETRAHYMENA TELOMERASE RNA STEM IV TERMINAL LOOP</td><td>R.J.RICHARDS,H.WU,L.TRANTIREK,C.M.O'CONNOR,K.COLLINS, J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JOX" title="Link to PDB">1JOX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JOX/index.html" title="Link to FR3D analysis. 0.3313 pairs per nucleotide">1JOX</a></td><td align="center">A</td><td align="center">504</td><td align="center">167</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020105000000">5-01-02</td><td>synthetic</td><td>NMR STRUCTURE OF THE LP5.1 HAIRPIN FROM BACILLUS RNASE P RNA REFINED WITH RESIDUAL DIPOLAR COUPLINGS</td><td>T.C.LEEPER,F.J.SCHMIDT,S.R.VAN DOREN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1D0U" title="Link to PDB">1D0U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1D0U/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">1D0U</a></td><td align="center">A</td><td align="center">483</td><td align="center">184</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20011205000000">5-24-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN RNA BINDING SITE FOR PHAGE MS2 COAT PROTEIN</td><td>J.S.SMITH,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DH3" title="Link to PDB">3DH3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DH3/index.html" title="Link to FR3D analysis. 0.3095 pairs per nucleotide">3DH3</a></td><td align="center">EF</td><td align="center">84</td><td align="center">26</td><td align="center">3.0</td><td sorttable_customkey="20110404000000">4-28-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RLUF IN COMPLEX WITH A 22 NUCLEOTIDE RNA SUBSTRATE</td><td>A.ALIAN,A.DEGIOVANNI,R.M.STROUD,J.S.FINER-MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RPK" title="Link to PDB">2RPK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RPK/index.html" title="Link to FR3D analysis. 0.3500 pairs per nucleotide">2RPK</a></td><td align="center">A</td><td align="center">20</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100612000000">12-30-08</td><td>synthetic</td><td>SOLUTION STRUCTURE OF DOMAIN II OF THE POSITIVE POLARITY CCHMVD HAMMERHEAD RIBOZYME</td><td>J.GALLEGO,D.DUFOUR,M.DE LA PENA,S.GAGO,R.FLORES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TOB" title="Link to PDB">2TOB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TOB/index.html" title="Link to FR3D analysis. 0.3462 pairs per nucleotide">2TOB</a></td><td align="center">A</td><td align="center">260</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20001006000000">6-22-99</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE TOBRAMYCIN-RNA APTAMER COMPLEX, NMR, 13 STRUCTURES</td><td>L.JIANG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BN0" title="Link to PDB">1BN0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BN0/index.html" title="Link to FR3D analysis. 0.3773 pairs per nucleotide">1BN0</a></td><td align="center">A</td><td align="center">220</td><td align="center">83</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>SL3 HAIRPIN FROM THE PACKAGING SIGNAL OF HIV-1, NMR, 11 STRUCTURES</td><td>L.PAPPALARDO,D.J.KERWOOD,I.PELCZER,P.N.BORER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HLX" title="Link to PDB">1HLX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HLX/index.html" title="Link to FR3D analysis. 0.3850 pairs per nucleotide">1HLX</a></td><td align="center">A</td><td align="center">400</td><td align="center">154</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960309000000">9-15-95</td><td>synthetic</td><td>P1 HELIX NUCLEIC ACIDS (DNA/RNA) RIBONUCLEIC ACID</td><td>F.H.-T.ALLAIN,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A1T" title="Link to PDB">1A1T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A1T/index.html" title="Link to FR3D analysis. 0.3960 pairs per nucleotide">1A1T</a></td><td align="center">B</td><td align="center">500</td><td align="center">198</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990506000000">6-17-98</td><td>synthetic</td><td>STRUCTURE OF THE HIV-1 NUCLEOCAPSID PROTEIN BOUND TO THE SL3 PSI-RNA RECOGNITION ELEMENT, NMR, 25 STRUCTURES</td><td>R.N.DE GUZMAN,Z.R.WU,C.C.STALLING,L.PAPPALARDO,P.N.BORER, M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O33" title="Link to PDB">2O33</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O33/index.html" title="Link to FR3D analysis. 0.3975 pairs per nucleotide">2O33</a></td><td align="center">A</td><td align="center">400</td><td align="center">159</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070602000000">2-06-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF U2 SNRNA STEM I FROM S. CEREVISIAE</td><td>D.G.SASHITAL,V.VENDITTI,C.A.ANGERS,G.CORNILESCU,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EC6" title="Link to PDB">1EC6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EC6/index.html" title="Link to FR3D analysis. 0.2250 pairs per nucleotide">1EC6</a></td><td align="center">C</td><td align="center">40</td><td align="center">9</td><td align="center">2.4</td><td sorttable_customkey="20010902000000">2-21-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF NOVA-2 KH3 K-HOMOLOGY RNA-BINDING DOMAIN BOUND TO 20-MER RNA HAIRPIN</td><td>H.A.LEWIS,K.MUSUNURU,K.B.JENSEN,C.EDO,H.CHEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2I82" title="Link to PDB">2I82</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I82/index.html" title="Link to FR3D analysis. 0.3544 pairs per nucleotide">2I82</a></td><td align="center">E</td><td align="center">79</td><td align="center">28</td><td align="center">2.0</td><td sorttable_customkey="20070911000000">11-21-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PSEUDOURIDINE SYNTHASE RLUA: INDIRECT SEQUENCE READOUT THROUGH PROTEIN-INDUCED RNA STRUCTURE</td><td>C.HOANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NJ6" title="Link to PDB">3NJ6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NJ6/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">3NJ6</a></td><td align="center">A</td><td align="center">20</td><td align="center">8</td><td align="center">0.9</td><td sorttable_customkey="20120108000000">8-25-10</td><td>synthetic</td><td>.95 A RESOLUTION X-RAY STRUCTURE OF (GGCAGCAGCC)2</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RPT" title="Link to PDB">2RPT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RPT/index.html" title="Link to FR3D analysis. 0.3900 pairs per nucleotide">2RPT</a></td><td align="center">A</td><td align="center">200</td><td align="center">78</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110108000000">8-25-09</td><td>synthetic</td><td>STRUCTURE OF THE CC MISMATCH FROM THE THYMIDYLATE SYNTHASE BINDING SITE 1 HAIRPIN AND ANALYSIS OF ITS INTERACTION WITH PAROMOMYCIN</td><td>T.J.TAVARES,P.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U2A" title="Link to PDB">1U2A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U2A/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1U2A</a></td><td align="center">A</td><td align="center">20</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980403000000">3-04-98</td><td>SACCHAROMYCES CEREVISIAE</td><td>STEM LOOP IIA FROM U2SNRNA OF SACCHAROMYCES CEREVISIAE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>S.C.STALLINGS,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EGK" title="Link to PDB">1EGK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EGK/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1EGK</a></td><td align="center">A</td><td align="center">40</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20020206000000">6-26-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A NUCLEIC ACID FOUR-WAY JUNCTION</td><td>J.NOWAKOWSKI,P.J.SHIM,C.D.STOUT,G.F.JOYCE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2U2A" title="Link to PDB">2U2A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2U2A/index.html" title="Link to FR3D analysis. 0.3500 pairs per nucleotide">2U2A</a></td><td align="center">A</td><td align="center">20</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980403000000">3-04-98</td><td>SACCHAROMYCES CEREVISIAE</td><td>STEM LOOP IIA FROM U2 SNRNA OF SACCHAROMYCES CEREVISIAE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>S.C.STALLINGS,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=361D" title="Link to PDB">361D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/361D/index.html" title="Link to FR3D analysis. 0.2250 pairs per nucleotide">361D</a></td><td align="center">AB</td><td align="center">40</td><td align="center">9</td><td align="center">3.0</td><td sorttable_customkey="19980107000000">7-01-98</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF DOMAIN E OF THERMUS FLAVUS 5S RRNA: A HELICAL RNA-STRUCTURE INCLUDING A TETRALOOP</td><td>M.PERBANDT,A.NOLTE,S.LORENZ,V.A.ERDMANN,C.BETZEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MFJ" title="Link to PDB">1MFJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MFJ/index.html" title="Link to FR3D analysis. 0.3800 pairs per nucleotide">1MFJ</a></td><td align="center">A</td><td align="center">200</td><td align="center">76</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020611000000">11-06-02</td><td>synthetic</td><td>' STEM-LOOP FROM HUMAN U4 SNRNA</td><td>L.R.COMOLLI,N.B.ULYANOV,T.L.JAMES,W.H.GMEINER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JPP" title="Link to PDB">2JPP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JPP/index.html" title="Link to FR3D analysis. 0.3500 pairs per nucleotide">2JPP</a></td><td align="center">D</td><td align="center">400</td><td align="center">140</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080908000000">8-21-07</td><td>PSEUDOMONAS FLUORESCENS</td><td>STRUCTURAL BASIS OF RSMA/CSRA RNA RECOGNITION: STRUCTURE OF RSME BOUND TO THE SHINE-DALGARNO SEQUENCE OF HCNA MRNA</td><td>M.SCHUBERT,K.LAPOUGE,O.DUSS,F.C.OBERSTRASS,I.JELESAROV, D.HAAS,F.H.-T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Y8W" title="Link to PDB">2Y8W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y8W/index.html" title="Link to FR3D analysis. 0.3158 pairs per nucleotide">2Y8W</a></td><td align="center">B</td><td align="center">19</td><td align="center">6</td><td align="center">1.8</td><td sorttable_customkey="20120605000000">5-18-11</td><td>synthetic</td><td>STRUCTURE OF CRISPR ENDORIBONUCLEASE CSE3 BOUND TO 20 NT RNA</td><td>D.G.SASHITAL,M.JINEK,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SLO" title="Link to PDB">1SLO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SLO/index.html" title="Link to FR3D analysis. 0.3684 pairs per nucleotide">1SLO</a></td><td align="center">A</td><td align="center">19</td><td align="center">7</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960712000000">12-07-96</td><td>CAENORHABDITIS ELEGANS</td><td>FIRST STEM LOOP OF THE SL1 RNA FROM CAENORHABDITIS ELEGANS, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>N.L.GREENBAUM,I.RADHAKRISHNAN,D.J.PATEL,D.HIRSH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SLP/index.html" title="SOLUTION NMR, 304 nucleotides, 108 basepairs, 0.3553 pairs per nucleotide">1SLP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SLO/1SLO_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QKB" title="Link to PDB">2QKB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QKB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2QKB</a></td><td align="center">C</td><td align="center">19</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20080111000000">11-13-07</td><td>HOMO SAPIENS</td><td>HUMAN RNASE H CATALYTIC DOMAIN MUTANT D210N IN COMPLEX WITH 20-MER RNA/DNA HYBRID</td><td>M.NOWOTNY,S.A.GAIDAMAKOV,R.GHIRLANDO,S.M.CERRITELLI, R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QC0" title="Link to PDB">1QC0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QC0/index.html" title="Link to FR3D analysis. 0.4912 pairs per nucleotide">1QC0</a></td><td align="center">CD</td><td align="center">57</td><td align="center">28</td><td align="center">1.6</td><td sorttable_customkey="19991111000000">11-11-99</td><td>synthetic,synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A 19 BASE PAIR COPY CONTROL RELATED RNA DUPLEX</td><td>P.S.KLOSTERMAN,S.A.SHAH,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DFU" title="Link to PDB">1DFU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DFU/index.html" title="Link to FR3D analysis. 0.3947 pairs per nucleotide">1DFU</a></td><td align="center">MN</td><td align="center">38</td><td align="center">15</td><td align="center">1.8</td><td sorttable_customkey="19990212000000">12-02-99</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF E.COLI RIBOSOMAL PROTEIN L25 COMPLEXED WITH A 5S RRNA FRAGMENT AT 1.8 A RESOLUTION</td><td>M.LU,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RLU" title="Link to PDB">2RLU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RLU/index.html" title="Link to FR3D analysis. 0.3684 pairs per nucleotide">2RLU</a></td><td align="center">A</td><td align="center">190</td><td align="center">70</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100202000000">2-26-08</td><td>synthetic</td><td>THE THREE DIMENSIONAL STRUCTURE OF THE MOORELLA THERMOACETICA SELENOCYSTEINE INSERTION SEQUENCE RNA HAIRPIN AND ITS INTERACTION WITH THE ELONGATION FACTOR SELB</td><td>A.V.BERIBISKY,T.J.TAVARES,A.N.AMBORSKI,M.MOTAMED, A.E.JOHNSON,T.L.MARK,P.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I3X" title="Link to PDB">1I3X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I3X/index.html" title="Link to FR3D analysis. 0.4105 pairs per nucleotide">1I3X</a></td><td align="center">A</td><td align="center">285</td><td align="center">117</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010404000000">4-04-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE A LOOP OF 23S RIBOSOMAL RNA</td><td>S.C.BLANCHARD,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ESY" title="Link to PDB">1ESY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ESY/index.html" title="Link to FR3D analysis. 0.3553 pairs per nucleotide">1ESY</a></td><td align="center">A</td><td align="center">380</td><td align="center">135</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020705000000">5-31-00</td><td>synthetic</td><td>NMR STRUCTURE OF STEM LOOP SL2 OF THE HIV-1 PSI RNA PACKAGING SIGNAL REVEALS A NOVEL A-U-A BASE-TRIPLE PLATFORM</td><td>G.K.AMARASINGHE,R.N.DE GUZMAN,R.B.TURNER,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ZI0" title="Link to PDB">2ZI0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ZI0/index.html" title="Link to FR3D analysis. 0.4737 pairs per nucleotide">2ZI0</a></td><td align="center">CD</td><td align="center">38</td><td align="center">18</td><td align="center">2.8</td><td sorttable_customkey="20091007000000">7-22-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF TAV2B/SIRNA COMPLEX</td><td>Y.A.YUAN,H.-Y.CHEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BQ5" title="Link to PDB">2BQ5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BQ5/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">2BQ5</a></td><td align="center">R</td><td align="center">36</td><td align="center">12</td><td align="center">2.9</td><td sorttable_customkey="20071003000000">3-22-06</td><td>BACTERIOPHAGE MS2</td><td>MS2 (N87AE89K MUTANT) - RNA HAIRPIN COMPLEX</td><td>W.T.HORN,K.TARS,E.GRAHN,C.HELGSTRAND,A.J.BARON,H.LAGO, C.J.ADAMS,D.S.PEABODY,S.E.V.PHILLIPS,N.J.STONEHOUSE, L.LILJAS,P.G.STOCKLEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZDK/index.html" title="Resolution 2.9A, 38 nucleotides, 12 basepairs, 0.3158 pairs per nucleotide">1ZDK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BQ5/2BQ5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2B7G" title="Link to PDB">2B7G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B7G/index.html" title="Link to FR3D analysis. 0.3684 pairs per nucleotide">2B7G</a></td><td align="center">A</td><td align="center">190</td><td align="center">70</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071201000000">1-24-06</td><td>synthetic</td><td>STRUCTURE OF THE SMAUG RECOGNITION RNA ELEMENT</td><td>P.E.JOHNSON,L.W.DONALDSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F27" title="Link to PDB">1F27</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F27/index.html" title="Link to FR3D analysis. 0.4667 pairs per nucleotide">1F27</a></td><td align="center">AB</td><td align="center">30</td><td align="center">14</td><td align="center">1.3</td><td sorttable_customkey="20001206000000">6-12-00</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A BIOTIN-BINDING RNA PSEUDOKNOT</td><td>J.NIX,D.SUSSMAN,C.WILSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XEB" title="Link to PDB">2XEB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XEB/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2XEB</a></td><td align="center">AB</td><td align="center">330</td><td align="center">132</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120205000000">5-26-10</td><td>HOMO SAPIENS,HOMO SAPIENS</td><td>NMR STRUCTURE OF THE PROTEIN-UNBOUND SPLICEOSOMAL U4 SNRNA 5' STEM LOOP</td><td>M.FALB,I.AMATA,F.GABEL,B.SIMON,T.CARLOMAGNO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Q2R" title="Link to PDB">1Q2R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q2R/index.html" title="Link to FR3D analysis. 0.3421 pairs per nucleotide">1Q2R</a></td><td align="center">EF</td><td align="center">38</td><td align="center">13</td><td align="center">2.9</td><td sorttable_customkey="20030909000000">9-09-03</td><td>synthetic</td><td>CHEMICAL TRAPPING AND CRYSTAL STRUCTURE OF A CATALYTIC TRNA GUANINE TRANSGLYCOSYLASE COVALENT INTERMEDIATE</td><td>W.XIE,X.LIU,R.H.HUANG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q2S/index.html" title="Resolution 3.2A, 38 nucleotides, 13 basepairs, 0.3421 pairs per nucleotide">1Q2S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q2R/1Q2R_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UUU" title="Link to PDB">1UUU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UUU/index.html" title="Link to FR3D analysis. 0.3263 pairs per nucleotide">1UUU</a></td><td align="center">A</td><td align="center">285</td><td align="center">93</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000102000000">2-25-98</td><td>synthetic</td><td>STRUCTURE OF AN RNA HAIRPIN LOOP WITH A 5'-CGUUUCG-3' LOOP MOTIF BY HETERONUCLEAR NMR SPECTROSCOPY AND DISTANCE GEOMETRY, 15 STRUCTURES</td><td>C.SICH,O.OHLENSCHLAGER,R.RAMACHANDRAN,M.GORLACH,L.R.BROWN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R9F" title="Link to PDB">1R9F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R9F/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1R9F</a></td><td align="center">BC</td><td align="center">38</td><td align="center">19</td><td align="center">1.9</td><td sorttable_customkey="20060301000000">1-27-04</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF P19 COMPLEXED WITH 19-BP SMALL INTERFERING RNA</td><td>K.YE,L.MALININA,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ATO" title="Link to PDB">1ATO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ATO/index.html" title="Link to FR3D analysis. 0.3158 pairs per nucleotide">1ATO</a></td><td align="center">A</td><td align="center">190</td><td align="center">60</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971211000000">11-12-97</td><td>HEPATITIS DELTA VIRUS</td><td>THE STRUCTURE OF THE ISOLATED, CENTRAL HAIRPIN OF THE HDV ANTIGENOMIC RIBOZYME, NMR, 10 STRUCTURES</td><td>M.H.KOLK,H.A.HEUS,C.W.HILBERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CZ3" title="Link to PDB">3CZ3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CZ3/index.html" title="Link to FR3D analysis. 0.4737 pairs per nucleotide">3CZ3</a></td><td align="center">GH</td><td align="center">76</td><td align="center">36</td><td align="center">3.2</td><td sorttable_customkey="20090505000000">5-05-09</td><td>synthetic,TOMATO ASPERMY VIRUS</td><td>CRYSTAL STRUCTURE OF TOMATO ASPERMY VIRUS 2B IN COMPLEX WITH SIRNA</td><td>J.B.MA,F.LI,S.W.DING,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T0E" title="Link to PDB">1T0E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T0E/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1T0E</a></td><td align="center">AC</td><td align="center">35</td><td align="center">15</td><td align="center">1.7</td><td sorttable_customkey="20050306000000">6-15-04</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF 2-AMINOPURINE LABELLED BACTERIAL DECODING SITE RNA</td><td>S.SHANDRICK,Q.ZHAO,Q.HAN,B.K.AYIDA,M.TAKAHASHI,G.C.WINTERS, K.B.SIMONSEN,D.VOURLOUMIS,T.HERMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OL9" title="Link to PDB">3OL9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OL9/index.html" title="Link to FR3D analysis. 0.4429 pairs per nucleotide">3OL9</a></td><td align="center">JKLNOP</td><td align="center">140</td><td align="center">62</td><td align="center">2.2</td><td sorttable_customkey="20111012000000">12-22-10</td><td>synthetic,synthetic,synthetic</td><td>POLIOVIRUS POLYMERASE ELONGATION COMPLEX WITH 3'-DEOXY-CTP</td><td>P.GONG,O.B.PEERSEN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OLA/index.html" title="Resolution 2.5A, 140 nucleotides, 62 basepairs, 0.4429 pairs per nucleotide">3OLA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OLB/index.html" title="Resolution 2.4A, 140 nucleotides, 60 basepairs, 0.4286 pairs per nucleotide">3OLB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OL6/index.html" title="Resolution 2.5A, 140 nucleotides, 59 basepairs, 0.4214 pairs per nucleotide">3OL6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OL9/3OL9_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BT7" title="Link to PDB">3BT7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BT7/index.html" title="Link to FR3D analysis. 0.3056 pairs per nucleotide">3BT7</a></td><td align="center">D</td><td align="center">36</td><td align="center">11</td><td align="center">2.4</td><td sorttable_customkey="20100504000000">4-29-08</td><td>synthetic</td><td>STRUCTURE OF E. COLI 5-METHYLURIDINE METHYLTRANSFERASE TRMA IN COMPLEX WITH 19 NUCLEOTIDE T-ARM ANALOGUE</td><td>A.ALIAN,R.M.STROUD,J.FINER-MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NZ1" title="Link to PDB">1NZ1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NZ1/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">1NZ1</a></td><td align="center">A</td><td align="center">391</td><td align="center">153</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040105000000">5-13-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE S. CEREVISIAE U6 INTRAMOLECULAR STEM-LOOP CONTAINING AN SP PHOSPHOROTHIOATE AT NUCLEOTIDE U80</td><td>N.J.REITER,L.J.NIKSTAD,A.M.ALLMAN,R.J.JOHNSON,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KS8" title="Link to PDB">3KS8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KS8/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3KS8</a></td><td align="center">EF</td><td align="center">36</td><td align="center">18</td><td align="center">2.4</td><td sorttable_customkey="20101201000000">1-12-10</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF RESTON EBOLAVIRUS VP35 RNA BINDING DOMAIN IN COMPLEX WITH 18BP DSRNA</td><td>C.R.KIMBERLIN,Z.A.BORNHOLDT,S.LI,V.L.WOODS,I.J.MACRAE, E.O.SAPHIRE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QH4" title="Link to PDB">2QH4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QH4/index.html" title="Link to FR3D analysis. 0.3361 pairs per nucleotide">2QH4</a></td><td align="center">A</td><td align="center">360</td><td align="center">121</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090109000000">9-25-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE U85 C/D-H/ACA SCARNA 5' TERMINAL HAIRPIN LOOP</td><td>J.FEIGON,C.A.THEIMER,N.CHIM,K.E.BREECE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=5MSF" title="Link to PDB">5MSF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/5MSF/index.html" title="Link to FR3D analysis. 0.3235 pairs per nucleotide">5MSF</a></td><td align="center">R</td><td align="center">34</td><td align="center">11</td><td align="center">2.8</td><td sorttable_customkey="19981111000000">11-11-98</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2 PROTEIN CAPSID/RNA COMPLEX</td><td>S.ROWSELL,N.J.STONEHOUSE,M.A.CONVERY,C.J.ADAMS, A.D.ELLINGTON,I.HIRAO,D.S.PEABODY,P.G.STOCKLEY, S.E.V.PHILLIPS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SI3" title="Link to PDB">1SI3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SI3/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1SI3</a></td><td align="center">B</td><td align="center">18</td><td align="center">6</td><td align="center">2.6</td><td sorttable_customkey="20060105000000">5-25-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PAZ DOMAIN OF HUMAN EIF2C1 IN COMPLEX WITH A 9-MER SIRNA-LIKE DUPLEX</td><td>K.YE,J.B.MA,D.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F6U" title="Link to PDB">1F6U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F6U/index.html" title="Link to FR3D analysis. 0.3028 pairs per nucleotide">1F6U</a></td><td align="center">B</td><td align="center">360</td><td align="center">109</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000910000000">10-09-00</td><td>HUMAN IMMUNODEFICIENCY VIRUS 1</td><td>NMR STRUCTURE OF THE HIV-1 NUCLEOCAPSID PROTEIN BOUND TO STEM-LOOP SL2 OF THE PSI-RNA PACKAGING SIGNAL. IMPLICATIONS FOR GENOME RECOGNITION</td><td>G.K.AMARASINGHE,R.N.DE GUZMAN,R.B.TURNER,K.J.CHANCELLOR, M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F5U" title="Link to PDB">1F5U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F5U/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1F5U</a></td><td align="center">AB</td><td align="center">36</td><td align="center">16</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010908000000">8-21-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE KISSING DIMER OF H3 GACG STEM- LOOP IN THE 5'-END DIMERIZATION SIGNAL OF MOLONEY MURINE LEUKEMIA VIRUS GENOMIC RNA</td><td>C.-H.KIM,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QK9" title="Link to PDB">2QK9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QK9/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2QK9</a></td><td align="center">B</td><td align="center">18</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20080111000000">11-13-07</td><td>HOMO SAPIENS</td><td>HUMAN RNASE H CATALYTIC DOMAIN MUTANT D210N IN COMPLEX WITH 18-MER RNA/DNA HYBRID</td><td>M.NOWOTNY,S.A.GAIDAMAKOV,R.GHIRLANDO,S.M.CERRITELLI, R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Y8Y" title="Link to PDB">2Y8Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y8Y/index.html" title="Link to FR3D analysis. 0.2778 pairs per nucleotide">2Y8Y</a></td><td align="center">B</td><td align="center">18</td><td align="center">5</td><td align="center">1.4</td><td sorttable_customkey="20120605000000">5-18-11</td><td>synthetic</td><td>STRUCTURE B OF CRISPR ENDORIBONUCLEASE CSE3 BOUND TO 19 NT RNA</td><td>D.G.SASHITAL,M.JINEK,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZEV" title="Link to PDB">1ZEV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZEV/index.html" title="Link to FR3D analysis. 0.3611 pairs per nucleotide">1ZEV</a></td><td align="center">AB</td><td align="center">36</td><td align="center">13</td><td align="center">1.6</td><td sorttable_customkey="20070110000000">10-25-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A PATHOGENIC RNA: CUG REPEATS</td><td>B.H.MOOERS,J.S.LOGUE,J.A.BERGLUND</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GM7/index.html" title="Resolution 1.6A, 36 nucleotides, 12 basepairs, 0.3333 pairs per nucleotide">3GM7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZEV/1ZEV_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GVO" title="Link to PDB">2GVO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GVO/index.html" title="Link to FR3D analysis. 0.2989 pairs per nucleotide">2GVO</a></td><td align="center">A</td><td align="center">378</td><td align="center">113</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070805000000">5-08-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A PURINE RICH HEXALOOP HAIRPIN BELONGING TO PGY/MDR1 MRNA AND TARGETED BY ANTISENSE OLIGONUCLEOTIDES</td><td>F.JOLI,N.BOUCHEMAL,A.LAIGLE,B.HARTMANN,E.HANTZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D2L" title="Link to PDB">2D2L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D2L/index.html" title="Link to FR3D analysis. 0.4576 pairs per nucleotide">2D2L</a></td><td align="center">ABCD</td><td align="center">59</td><td align="center">27</td><td align="center">2.5</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic,synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A MINIMAL, ALL-RNA HAIRPIN RIBOZYME WITH A PROPYL LINKER (C3) AT POSITION U39</td><td>S.ALAM,V.GRUM-TOKARS,J.KRUCINSKA,M.L.KUNDRACIK,J.E.WEDEKIND</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AJF" title="Link to PDB">1AJF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AJF/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1AJF</a></td><td align="center">A</td><td align="center">18</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE P5B STEM LOOP FROM A GROUP I INTRON COMPLEXED WITH COBALT (III) HEXAMMINE, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>J.S.KIEFT,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Z30" title="Link to PDB">1Z30</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z30/index.html" title="Link to FR3D analysis. 0.3889 pairs per nucleotide">1Z30</a></td><td align="center">A</td><td align="center">270</td><td align="center">105</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070204000000">4-26-05</td><td>synthetic</td><td>NMR STRUCTURE OF THE APICAL PART OF STEMLOOP D FROM CLOVERLEAF 1 OF BOVINE ENTEROVIRUS 1 RNA</td><td>Y.IHLE,O.OHLENSCHLAGER,E.DUCHARDT,R.RAMACHANDRAN,M.GORLACH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GJW" title="Link to PDB">2GJW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GJW/index.html" title="Link to FR3D analysis. 0.4507 pairs per nucleotide">2GJW</a></td><td align="center">EFH</td><td align="center">71</td><td align="center">32</td><td align="center">2.9</td><td sorttable_customkey="20061209000000">9-12-06</td><td>synthetic,synthetic,synthetic</td><td>RNA RECOGNITION AND CLEAVAGE BY AN SPLICING ENDONUCLEASE</td><td>S.XUE,K.CALVIN,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OE5" title="Link to PDB">2OE5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OE5/index.html" title="Link to FR3D analysis. 0.4242 pairs per nucleotide">2OE5</a></td><td align="center">AB</td><td align="center">33</td><td align="center">14</td><td align="center">1.5</td><td sorttable_customkey="20080102000000">2-13-07</td><td>synthetic,synthetic</td><td>.5 A X-RAY CRYSTAL STRUCTURE OF APRAMYCIN COMPLEX WITH RNA FRAGMENT GGCGUCGCUAGUACCG/GGUACUAAAAGUCGCCC CONTAINING THE HUMAN RIBOSOMAL DECODING A SITE: RNA CONSTRUCT WITH 3'- OVERHANG</td><td>T.HERMANN,V.TERESHKO,E.SKRIPKIN,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OE8/index.html" title="Resolution 1.8A, 33 nucleotides, 14 basepairs, 0.4242 pairs per nucleotide">2OE8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OE6/index.html" title="Resolution 2.4A, 33 nucleotides, 14 basepairs, 0.4242 pairs per nucleotide">2OE6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OE5/2OE5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JR4" title="Link to PDB">2JR4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JR4/index.html" title="Link to FR3D analysis. 0.3155 pairs per nucleotide">2JR4</a></td><td align="center">A</td><td align="center">187</td><td align="center">59</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20081207000000">7-24-07</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF E.COLI TRNA-VAL3 WITH NO MODIFICATIONS</td><td>F.A.P.VENDEIX,A.DZIERGOWSKA,E.M.GUSTILO,W.D.GRAHAM,B.SPROAT, A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3QRR" title="Link to PDB">3QRR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QRR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3QRR</a></td><td align="center">B</td><td align="center">17</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20130105000000">5-25-11</td><td>synthetic</td><td>STRUCTURE OF THERMUS THERMOPHILUS CSE3 BOUND TO AN RNA REPRESENTING A PRODUCT COMPLEX</td><td>M.J.SCHELLENBERG,E.G.GESNER,GARSIDE E.L.,A.M.MACMILLAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D18" title="Link to PDB">2D18</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D18/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">2D18</a></td><td align="center">AB</td><td align="center">374</td><td align="center">153</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic</td><td>SOLUTION RNA STRUCTURE OF LOOP REGION OF THE HIV-1 DIMERIZATION INITIATION SITE IN THE EXTENDED-DUPLEX DIMER</td><td>S.BABA,K.TAKAHASHI,S.NOGUCHI,H.TAKAKU,Y.KOYANAGI,N.YAMAMOTO, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KPD" title="Link to PDB">2KPD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KPD/index.html" title="Link to FR3D analysis. 0.2471 pairs per nucleotide">2KPD</a></td><td align="center">A</td><td align="center">170</td><td align="center">42</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120606000000">6-30-10</td><td>synthetic</td><td>STRUCTURE DETERMINATION OF THE TOP-LOOP OF THE CONSERVED 3 TERMINAL SECONDARY STRUCTURE IN THE GENOME OF YFV-MUTANT</td><td>E.LESCRINIER,N.DYUBANKOVA,K.NAUWELAERTS,R.JONES,P.HERDEWIJN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRP" title="Link to PDB">2KRP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRP/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">2KRP</a></td><td align="center">A</td><td align="center">17</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE E. COLI TRNA-ARG2(ACG) ANTICODON STEM AND LOOP</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KPC" title="Link to PDB">2KPC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KPC/index.html" title="Link to FR3D analysis. 0.3294 pairs per nucleotide">2KPC</a></td><td align="center">A</td><td align="center">170</td><td align="center">56</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120606000000">6-30-10</td><td>synthetic</td><td>STRUCTURE DETERMINATION OF THE TOP-LOOP OF THE CONSERVED 3 TERMINAL SECONDARY STRUCTURE IN THE GENOME OF YFV</td><td>E.LESCRINIER,N.DYUBANKOVA,K.NAUWELAERTS,R.JONES,P.HERDEWIJN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YN2" title="Link to PDB">1YN2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YN2/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">1YN2</a></td><td align="center">A</td><td align="center">187</td><td align="center">66</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071201000000">1-24-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE NEUROSPORA VS RIBOZYME STEM-LOOP V IN THE PRESENCE OF MGCL2 WITH MODELING OF BOUND MANGANESE IONS</td><td>D.O.CAMPBELL,P.LEGAULT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YN1/index.html" title="SOLUTION NMR, 187 nucleotides, 66 basepairs, 0.3529 pairs per nucleotide">1YN1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TBK/index.html" title="SOLUTION NMR, 187 nucleotides, 65 basepairs, 0.3476 pairs per nucleotide">1TBK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YN2/1YN2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KVN" title="Link to PDB">2KVN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KVN/index.html" title="Link to FR3D analysis. 0.2980 pairs per nucleotide">2KVN</a></td><td align="center">A</td><td align="center">510</td><td align="center">152</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic</td><td>PHI29 E-LOOP HAIRPIN</td><td>S.M.HARRIS,S.J.SCHROEDER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WKS" title="Link to PDB">1WKS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WKS/index.html" title="Link to FR3D analysis. 0.4034 pairs per nucleotide">1WKS</a></td><td align="center">A</td><td align="center">357</td><td align="center">144</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20051006000000">6-22-04</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN RNA STEM-LOOP DERIVED FROM THE 3' CONSERVED REGION OF EEL LINE UNAL2</td><td>S.BABA,M.KAJIKAWA,N.OKADA,G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JLT" title="Link to PDB">2JLT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLT/index.html" title="Link to FR3D analysis. 0.4688 pairs per nucleotide">2JLT</a></td><td align="center">AB</td><td align="center">32</td><td align="center">15</td><td align="center">2.9</td><td sorttable_customkey="20100608000000">8-18-09</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF AN RNA KISSING COMPLEX</td><td>C.DIPRIMO,S.FRIBOURG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R3E" title="Link to PDB">1R3E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R3E/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1R3E</a></td><td align="center">DE</td><td align="center">50</td><td align="center">20</td><td align="center">2.1</td><td sorttable_customkey="20030411000000">11-04-03</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF TRNA PSEUDOURIDINE SYNTHASE TRUB AND ITS RNA COMPLEX: RNA-PROTEIN RECOGNITION THROUGH A COMBINATION OF RIGID DOCKING AND INDUCED FIT</td><td>H.PAN,S.AGARWALLA,D.T.MOUSTAKAS,J.FINER-MOORE,R.M.STROUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JBS" title="Link to PDB">1JBS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JBS/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">1JBS</a></td><td align="center">D</td><td align="center">34</td><td align="center">12</td><td align="center">2.0</td><td sorttable_customkey="20030210000000">10-26-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RIBOTOXIN RESTRICTOCIN AND A 29-MER SRD RNA ANALOG</td><td>X.YANG,T.GERCZEI,L.GLOVER,C.C.CORRELL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BZ3" title="Link to PDB">1BZ3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BZ3/index.html" title="Link to FR3D analysis. 0.2941 pairs per nucleotide">1BZ3</a></td><td align="center">A</td><td align="center">17</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>STABILIZATION OF THE ANTICODON STEM-LOOP OF TRNALYS,3 BY AN A+C BASE PAIR AND BY PSEUDOURIDINE, NMR, 1 STRUCTURE</td><td>P.C.DURANT,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1E7K" title="Link to PDB">1E7K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1E7K/index.html" title="Link to FR3D analysis. 0.4118 pairs per nucleotide">1E7K</a></td><td align="center">D</td><td align="center">34</td><td align="center">14</td><td align="center">2.9</td><td sorttable_customkey="20020702000000">2-19-01</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF THE SPLICEOSOMAL 15.5KD PROTEIN BOUND TO A U4 SNRNA FRAGMENT</td><td>I.VIDOVIC,S.NOTTROTT,K.HARTHMUTH,R.LUHRMANN,R.FICNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OL7" title="Link to PDB">3OL7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OL7/index.html" title="Link to FR3D analysis. 0.4429 pairs per nucleotide">3OL7</a></td><td align="center">BCDFGH</td><td align="center">140</td><td align="center">62</td><td align="center">2.7</td><td sorttable_customkey="20111012000000">12-22-10</td><td>synthetic,synthetic,synthetic</td><td>POLIOVIRUS POLYMERASE ELONGATION COMPLEX WITH CTP</td><td>P.GONG,O.B.PEERSEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KKA" title="Link to PDB">1KKA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KKA/index.html" title="Link to FR3D analysis. 0.3456 pairs per nucleotide">1KKA</a></td><td align="center">A</td><td align="center">136</td><td align="center">47</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030507000000">7-17-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE UNMODIFIED ANTICODON STEM-LOOP FROM E. COLI TRNA(PHE)</td><td>J.CABELLO-VILLEGAS,M.E.WINKLER,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BZ2" title="Link to PDB">1BZ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BZ2/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">1BZ2</a></td><td align="center">A</td><td align="center">17</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>STABILIZATION OF THE ANTICODON STEM-LOOP OF TRNALYS,3 BY AN A+C BASE PAIR AND BY PSEUDOURIDINE, NMR, 1 STRUCTURE</td><td>P.C.DURANT,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Y9H" title="Link to PDB">2Y9H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Y9H/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">2Y9H</a></td><td align="center">B</td><td align="center">17</td><td align="center">6</td><td align="center">2.5</td><td sorttable_customkey="20120605000000">5-18-11</td><td>synthetic</td><td>STRUCTURE A OF CRISPR ENDORIBONUCLEASE CSE3 BOUND TO 19 NT RNA</td><td>D.G.SASHITAL,M.JINEK,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J4Y" title="Link to PDB">1J4Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J4Y/index.html" title="Link to FR3D analysis. 0.3529 pairs per nucleotide">1J4Y</a></td><td align="center">A</td><td align="center">17</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030507000000">7-17-02</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE UNMODIFIED ANTICODON STEM-LOOP FROM E. COLI TRNA(PHE)</td><td>J.CABELLO-VILLEGAS,M.E.WINKLER,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D19" title="Link to PDB">2D19</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D19/index.html" title="Link to FR3D analysis. 0.4385 pairs per nucleotide">2D19</a></td><td align="center">AB</td><td align="center">374</td><td align="center">164</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic</td><td>SOLUTION RNA STRUCTURE OF LOOP REGION OF THE HIV-1 DIMERIZATION INITIATION SITE IN THE KISSING-LOOP DIMER</td><td>S.BABA,K.TAKAHASHI,S.NOGUCHI,H.TAKAKU,Y.KOYANAGI,N.YAMAMOTO, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ATV" title="Link to PDB">1ATV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ATV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1ATV</a></td><td align="center">A</td><td align="center">68</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000102000000">2-25-98</td><td>synthetic</td><td>HAIRPIN WITH AGAA TETRALOOP, NMR, 4 STRUCTURES</td><td>H.KANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BU1" title="Link to PDB">2BU1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BU1/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2BU1</a></td><td align="center">R</td><td align="center">32</td><td align="center">12</td><td align="center">2.2</td><td sorttable_customkey="20060608000000">8-18-05</td><td>BACTERIOPHAGE MS2</td><td>MS2-RNA HAIRPIN (5BRU-5) COMPLEX</td><td>E.GRAHN,T.MOSS,C.HELGSTRAND,K.FRIDBORG,M.SUNDARAM,K.TARS, H.LAGO,N.J.STONEHOUSE,D.R.DAVIS,P.G.STOCKLEY,L.LILJAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C4Q/index.html" title="Resolution 2.4A, 32 nucleotides, 12 basepairs, 0.3750 pairs per nucleotide">2C4Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C4Y/index.html" title="Resolution 2.7A, 28 nucleotides, 10 basepairs, 0.3571 pairs per nucleotide">2C4Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZDI/index.html" title="Resolution 2.7A, 29 nucleotides, 9 basepairs, 0.3103 pairs per nucleotide">1ZDI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IZ8/index.html" title="Resolution 3.3A, 29 nucleotides, 9 basepairs, 0.3103 pairs per nucleotide">2IZ8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B2E/index.html" title="Resolution 3.1A, 30 nucleotides, 9 basepairs, 0.3000 pairs per nucleotide">2B2E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AQ4/index.html" title="Resolution 3.0A, 30 nucleotides, 9 basepairs, 0.3000 pairs per nucleotide">1AQ4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B2G/index.html" title="Resolution 3.0A, 31 nucleotides, 9 basepairs, 0.2903 pairs per nucleotide">2B2G</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AQ3/index.html" title="Resolution 2.8A, 28 nucleotides, 8 basepairs, 0.2857 pairs per nucleotide">1AQ3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BU1/2BU1_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L6I" title="Link to PDB">2L6I</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L6I/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2L6I</a></td><td align="center">A</td><td align="center">432</td><td align="center">162</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20121103000000">3-23-11</td><td>synthetic</td><td>SOLUTION STRUCTURE OF CORONAVIRAL STEM-LOOP 2 (SL2)</td><td>C.LEE,L.LI,D.P.GIEDROC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O5I" title="Link to PDB">2O5I</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O5I/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2O5I</a></td><td align="center">H</td><td align="center">32</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20070307000000">7-03-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE T. THERMOPHILUS RNA POLYMERASE ELONGATION COMPLEX</td><td>D.G.VASSYLYEV,T.H.TAHIROV,M.N.VASSYLYEVA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O5J/index.html" title="Resolution 3.0A, 32 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2O5J</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PPB/index.html" title="Resolution 3.0A, 32 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2PPB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O5I/2O5I_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JTW" title="Link to PDB">1JTW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JTW/index.html" title="Link to FR3D analysis. 0.2902 pairs per nucleotide">1JTW</a></td><td align="center">A</td><td align="center">224</td><td align="center">65</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20011212000000">12-12-01</td><td>synthetic</td><td>STRUCTURE OF SL4 RNA FROM THE HIV-1 PACKAGING SIGNAL</td><td>D.J.KERWOOD,M.J.CAVALUZZI,P.N.BORER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2D17" title="Link to PDB">2D17</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2D17/index.html" title="Link to FR3D analysis. 0.4273 pairs per nucleotide">2D17</a></td><td align="center">AB</td><td align="center">330</td><td align="center">141</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050111000000">11-01-05</td><td>synthetic,synthetic</td><td>SOLUTION RNA STRUCTURE OF STEM-BULGE-STEM REGION OF THE HIV- 1 DIMERIZATION INITIATION SITE</td><td>S.BABA,K.TAKAHASHI,S.NOGUCHI,H.TAKAKU,Y.KOYANAGI,N.YAMAMOTO, G.KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3L26" title="Link to PDB">3L26</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3L26/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3L26</a></td><td align="center">C</td><td align="center">16</td><td align="center">8</td><td align="center">2.4</td><td sorttable_customkey="20120201000000">1-26-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ZAIRE EBOLA VP35 INTERFERON INHIBITORY DOMAIN BOUND TO 8 BP DSRNA</td><td>D.W.LEUNG,K.C.PRINS,D.M.BOREK,M.FARAHBAKHSH,J.M.TUFARIELLO, P.RAMANAN,J.C.NIX,L.A.HELGESON,Z.OTWINOWSKI,R.B.HONZATKO, C.F.BASLER,G.K.AMARASINGHE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=406D" title="Link to PDB">406D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/406D/index.html" title="Link to FR3D analysis. 0.4688 pairs per nucleotide">406D</a></td><td align="center">AE</td><td align="center">32</td><td align="center">15</td><td align="center">1.8</td><td sorttable_customkey="19990502000000">2-05-99</td><td>engineered</td><td>'-R(*CP*AP*CP*CP*GP*GP*AP*UP*GP*GP*UP*(BRO) UP*CP*GP*GP*UP*G)-3'</td><td>S.A.SHAH,A.T.BRUNGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RXA" title="Link to PDB">1RXA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RXA/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1RXA</a></td><td align="center">A</td><td align="center">16</td><td align="center">8</td><td align="center">2.9</td><td sorttable_customkey="19970810000000">10-20-96</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF R(CCCCGGGG) IN TWO DISTINCT LATTICES</td><td>S.PORTMANN,N.USMAN,M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=405D" title="Link to PDB">405D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/405D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">405D</a></td><td align="center">AB</td><td align="center">32</td><td align="center">16</td><td align="center">2.5</td><td sorttable_customkey="20000101000000">1-13-99</td><td>synthetic</td><td>STRUCTURE OF A 16-MER RNA DUPLEX R(GCAGACUUAAAUCUGC)2 WITH WOBBLE LIKE A.C MISMATCHES</td><td>M.SUNDARALINGAM B.PAN,S.N.MITRA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JWC" title="Link to PDB">1JWC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JWC/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1JWC</a></td><td align="center">A</td><td align="center">16</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030803000000">3-20-02</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE RNA HAIRPIN BINDING SITE FOR THE HISTONE STEM-LOOP BINDING PROTEIN</td><td>E.S.DEJONG,W.F.MARZLUFF,E.P.NIKONOWICZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JU7/index.html" title="SOLUTION NMR, 160 nucleotides, 55 basepairs, 0.3438 pairs per nucleotide">1JU7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JWC/1JWC_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BZU" title="Link to PDB">1BZU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BZU/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1BZU</a></td><td align="center">A</td><td align="center">16</td><td align="center">4</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>STABILIZATION OF THE ANTICODON STEM-LOOP OF TRNALYS, 3 BY AN A+C BASE PAIR AND BY PSEUDOURIDINE, NMR, 1 STRUCTURE</td><td>P.C.DURANT,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRQ" title="Link to PDB">2KRQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRQ/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2KRQ</a></td><td align="center">A</td><td align="center">16</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE TRNA-ARG2 (ICG) ASL.</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KYD" title="Link to PDB">2KYD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KYD/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2KYD</a></td><td align="center">AB</td><td align="center">640</td><td align="center">320</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic</td><td>RDC AND RCSA REFINEMENT OF AN A-FORM RNA: IMPROVEMENTS IN MAJOR GROOVE WIDTH</td><td>B.S.TOLBERT,M.F.SUMMERS,Y.MIYAZAKI,S.BARTON,B.KINDE,P.STARK,R.SINGH, A.BAX,D.CASE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRZ" title="Link to PDB">2KRZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRZ/index.html" title="Link to FR3D analysis. 0.2437 pairs per nucleotide">2KRZ</a></td><td align="center">A</td><td align="center">160</td><td align="center">39</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110608000000">8-18-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HUMAN MITOCHONDRIAL TRNAMET</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1N53" title="Link to PDB">1N53</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N53/index.html" title="Link to FR3D analysis. 0.3200 pairs per nucleotide">1N53</a></td><td align="center">AB</td><td align="center">25</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050504000000">4-29-03</td><td>synthetic,synthetic</td><td>SOLUTION STRUCTURE OF B. SUBTILIS T BOX ANTITERMINATOR RNA</td><td>M.S.GERDEMAN,T.M.HENKIN,J.V.HINES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BZT" title="Link to PDB">1BZT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BZT/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1BZT</a></td><td align="center">A</td><td align="center">16</td><td align="center">4</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010304000000">4-27-99</td><td>synthetic</td><td>STABILIZATION OF THE ANTICODON STEM-LOOP OF TRNALYS,3 BY AN A+C BASE PAIR AND BY PSEUDOURIDINE, NMR, 1 STRUCTURE</td><td>P.C.DURANT,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LU3" title="Link to PDB">1LU3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LU3/index.html" title="Link to FR3D analysis. 0.1875 pairs per nucleotide">1LU3</a></td><td align="center">A</td><td align="center">16</td><td align="center">3</td><td align="center">16.8</td><td sorttable_customkey="20040206000000">6-26-02</td><td>SACCHAROMYCES CEREVISIAE</td><td>SEPARATE FITTING OF THE ANTICODON LOOP REGION OF TRNA (NUCLEOTIDE 26-42) IN THE LOW RESOLUTION CRYO-EM MAP OF AN EF-TU TERNARY COMPLEX (GDP AND KIRROMYCIN) BOUND TO E. COLI 70S RIBOSOME</td><td>M.VALLE,J.SENGUPTA,N.K.SWAMI,R.A.GRASSUCCI,N.BURKHARDT, K.H.NIERHAUS,R.K.AGRAWAL,J.FRANK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RN1" title="Link to PDB">2RN1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RN1/index.html" title="Link to FR3D analysis. 0.5221 pairs per nucleotide">2RN1</a></td><td align="center">AB</td><td align="center">544</td><td align="center">284</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20091109000000">9-23-08</td><td>synthetic,synthetic</td><td>LIQUID CRYSTAL SOLUTION STRUCTURE OF THE KISSING COMPLEX FORMED BY THE APICAL LOOP OF THE HIV TAR RNA AND A HIGH AFFINITY RNA APTAMER OPTIMIZED BY SELEX</td><td>H.VAN MELCKEBEKE,M.DEVANY,C.DI PRIMO,F.BEAURAIN,J.TOULME, D.L.BRYCE,J.BOISBOUVIER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1U1Y" title="Link to PDB">1U1Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1U1Y/index.html" title="Link to FR3D analysis. 0.3548 pairs per nucleotide">1U1Y</a></td><td align="center">R</td><td align="center">31</td><td align="center">11</td><td align="center">2.9</td><td sorttable_customkey="20040712000000">12-07-04</td><td>ENTEROBACTERIO PHAGE MS2</td><td>CRYSTAL STRUCTURE OF A COMPLEX BETWEEN WT BACTERIOPHAGE MS2 COAT PROTEIN AND AN F5 APTAMER RNA STEMLOOP WITH 2AMINOPURINE SUBSTITUTED AT THE-10 POSITION</td><td>W.T.HORN,M.A.CONVERY,N.J.STONEHOUSE,C.J.ADAMS,L.LILJAS, S.E.PHILLIPS,P.G.STOCKLEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=420D" title="Link to PDB">420D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/420D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">420D</a></td><td align="center">AB</td><td align="center">32</td><td align="center">16</td><td align="center">1.9</td><td sorttable_customkey="20000101000000">1-13-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A 16-MER RNA DUPLEX WITH NON-ADJACENT A(ANTI).G(SYN) MISMATCHES</td><td>B.PAN,S.N.MITRA,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KIS" title="Link to PDB">1KIS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KIS/index.html" title="Link to FR3D analysis. 0.4062 pairs per nucleotide">1KIS</a></td><td align="center">AB</td><td align="center">32</td><td align="center">13</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980310000000">10-15-97</td><td>synthetic,synthetic</td><td>TAR-TAR "KISSING" HAIRPIN COMPLEX DERIVED FROM THE HIV GENOME, NMR, 1 STRUCTURE</td><td>K.Y.CHANG,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PN9" title="Link to PDB">2PN9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PN9/index.html" title="Link to FR3D analysis. 0.4200 pairs per nucleotide">2PN9</a></td><td align="center">AB</td><td align="center">300</td><td align="center">126</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090610000000">10-30-07</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF A KISSING COMPLEX FORMED BETWEEN THE TAR RNA ELEMENT OF HIV-1 AND A LNA MODIFIED APTAMER</td><td>I.LEBARS,T.RICHARD,C.DI PRIMO,J.-J.TOULME</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OOM/index.html" title="SOLUTION NMR, 360 nucleotides, 149 basepairs, 0.4139 pairs per nucleotide">2OOM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PN9/2PN9_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T0D" title="Link to PDB">1T0D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T0D/index.html" title="Link to FR3D analysis. 0.4375 pairs per nucleotide">1T0D</a></td><td align="center">AB</td><td align="center">64</td><td align="center">28</td><td align="center">2.2</td><td sorttable_customkey="20050306000000">6-15-04</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF 2-AMINOPURINE LABELLED BACTERIAL DECODING SITE RNA</td><td>S.SHANDRICK,Q.ZHAO,Q.HAN,B.K.AYIDA,M.TAKAHASHI,G.C.WINTERS, K.B.SIMONSEN,D.VOURLOUMIS,T.HERMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XWU" title="Link to PDB">1XWU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XWU/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">1XWU</a></td><td align="center">A</td><td align="center">16</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060302000000">2-15-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF ACAUAGA LOOP</td><td>T.SAKAMOTO,A.OGURO,G.KAWAI,T.OHTSU,Y.NAKAMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZBH" title="Link to PDB">1ZBH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZBH/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">1ZBH</a></td><td align="center">F</td><td align="center">32</td><td align="center">10</td><td align="center">3.0</td><td sorttable_customkey="20080209000000">9-26-06</td><td>synthetic</td><td>'-END SPECIFIC RECOGNITION OF HISTONE MRNA STEM-LOOP BY 3'- EXONUCLEASE</td><td>Y.CHENG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OL8" title="Link to PDB">3OL8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OL8/index.html" title="Link to FR3D analysis. 0.4348 pairs per nucleotide">3OL8</a></td><td align="center">JKLNOP</td><td align="center">138</td><td align="center">60</td><td align="center">2.8</td><td sorttable_customkey="20111012000000">12-22-10</td><td>synthetic,synthetic,synthetic</td><td>POLIOVIRUS POLYMERASE ELONGATION COMPLEX WITH CTP-MN</td><td>P.GONG,O.B.PEERSEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AGV" title="Link to PDB">3AGV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AGV/index.html" title="Link to FR3D analysis. 0.0800 pairs per nucleotide">3AGV</a></td><td align="center">SU</td><td align="center">25</td><td align="center">2</td><td align="center">2.1</td><td sorttable_customkey="20101011000000">11-10-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A HUMAN IGG-APTAMER COMPLEX</td><td>Y.NOMURA,S.SUGIYAMA,T.SAKAMOTO,S.MIYAKAWA,H.ADACHI,K.TAKANO, S.MURAKAMI,T.INOUE,Y.MORI,Y.NAKAMURA,H.MATSUMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PO2" title="Link to PDB">3PO2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PO2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PO2</a></td><td align="center">P</td><td align="center">15</td><td align="center">0</td><td align="center">3.3</td><td sorttable_customkey="20110203000000">3-02-11</td><td>synthetic</td><td>ARRESTED RNA POLYMERASE II ELONGATION COMPLEX</td><td>A.C.M.CHEUNG,P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZX7" title="Link to PDB">1ZX7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZX7/index.html" title="Link to FR3D analysis. 0.4224 pairs per nucleotide">1ZX7</a></td><td align="center">ABGH</td><td align="center">116</td><td align="center">49</td><td align="center">2.1</td><td sorttable_customkey="20060809000000">9-20-05</td><td>synthetic,synthetic</td><td>MOLECULAR RECOGNITION OF RNA BY NEOMYCIN AND A RESTRICTED NEOMYCIN DERIVATIVE</td><td>Q.ZHAO,F.ZHAO,K.F.BLOUNT,Q.HAN,Y.TOR,T.HERMANN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A04/index.html" title="Resolution 3.0A, 58 nucleotides, 23 basepairs, 0.3966 pairs per nucleotide">2A04</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZZ5/index.html" title="Resolution 3.0A, 58 nucleotides, 23 basepairs, 0.3966 pairs per nucleotide">1ZZ5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZX7/1ZX7_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YZD" title="Link to PDB">1YZD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YZD/index.html" title="Link to FR3D analysis. 0.4667 pairs per nucleotide">1YZD</a></td><td align="center">AB</td><td align="center">30</td><td align="center">14</td><td align="center">2.4</td><td sorttable_customkey="20060610000000">10-18-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DUPLEX CONTAINING A SITE SPECIFIC 2'-AMINE SUBSTITUTION AT A C-G WATSON-CRICK BASE PAIR</td><td>C.M.GHERGHE,J.M.KRAHN,K.M.WEEKS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Z79/index.html" title="Resolution 2.5A, 30 nucleotides, 14 basepairs, 0.4667 pairs per nucleotide">1Z79</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YY0/index.html" title="Resolution 3.2A, 30 nucleotides, 14 basepairs, 0.4667 pairs per nucleotide">1YY0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H1M/index.html" title="Resolution 2.9A, 30 nucleotides, 12 basepairs, 0.4000 pairs per nucleotide">2H1M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YZD/1YZD_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T0K" title="Link to PDB">1T0K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T0K/index.html" title="Link to FR3D analysis. 0.4074 pairs per nucleotide">1T0K</a></td><td align="center">CD</td><td align="center">27</td><td align="center">11</td><td align="center">3.2</td><td sorttable_customkey="20050807000000">7-20-04</td><td>synthetic,synthetic</td><td>JOINT X-RAY AND NMR REFINEMENT OF YEAST L30E-MRNA COMPLEX</td><td>J.A.CHAO,J.R.WILLIAMSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ATW" title="Link to PDB">1ATW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ATW/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1ATW</a></td><td align="center">A</td><td align="center">45</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000102000000">2-25-98</td><td>synthetic</td><td>HAIRPIN WITH AGAU TETRALOOP, NMR, 3 STRUCTURES</td><td>H.KANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L9E" title="Link to PDB">2L9E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L9E/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">2L9E</a></td><td align="center">A</td><td align="center">15</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110203000000">3-02-11</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HUMAN ANTI-CODON STEM AND LOOP(HASL) OF TRANSFER RNA LYSINE 3 (TRNALYS3)</td><td>F.A.P.VENDEIX,F.V.MURPHY IV,W.CANTARA,G.LESZCZYNSKA,E.M.GUSTILO, B.SPROAT,A.A.P.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HK2" title="Link to PDB">3HK2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HK2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HK2</a></td><td align="center">D</td><td align="center">30</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20090610000000">10-06-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF T. THERMOPHILUS ARGONAUTE N478 MUTANT PROTEIN COMPLEXED WITH DNA GUIDE STRAND AND 19-NT RNA TARGET STRAND</td><td>Y.WANG,H.LI,G.SHENG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3DD2" title="Link to PDB">3DD2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3DD2/index.html" title="Link to FR3D analysis. 0.2000 pairs per nucleotide">3DD2</a></td><td align="center">B</td><td align="center">15</td><td align="center">3</td><td align="center">1.9</td><td sorttable_customkey="20081111000000">11-11-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA APTAMER BOUND TO HUMAN THROMBIN</td><td>S.B.LONG,B.A.SULLENGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IZN" title="Link to PDB">2IZN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IZN/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">2IZN</a></td><td align="center">R</td><td align="center">26</td><td align="center">8</td><td align="center">2.6</td><td sorttable_customkey="20080307000000">7-27-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2-RNA HAIRPIN (G-10) COMPLEX</td><td>C.HELGSTRAND,E.GRAHN,T.MOSS,N.J.STONEHOUSE,K.TARS, P.G.STOCKLEY,L.LILJAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AZ2" title="Link to PDB">2AZ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AZ2/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2AZ2</a></td><td align="center">CD</td><td align="center">30</td><td align="center">12</td><td align="center">2.6</td><td sorttable_customkey="20051110000000">10-11-05</td><td>synthetic</td><td>FLOCK HOUSE VIRUS B2-DSRNA COMPLEX (P4122)</td><td>J.A.CHAO,J.H.LEE,B.R.CHAPADOS,E.W.DEBLER,A.SCHNEEMANN, J.R.WILLIAMSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AZ0/index.html" title="Resolution 2.6A, 30 nucleotides, 12 basepairs, 0.4000 pairs per nucleotide">2AZ0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AZ2/2AZ2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XWP" title="Link to PDB">1XWP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XWP/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1XWP</a></td><td align="center">A</td><td align="center">15</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060302000000">2-15-05</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AUCGCA LOOP</td><td>T.SAKAMOTO,A.OGURO,G.KAWAI,T.OHTSU,Y.NAKAMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QFQ" title="Link to PDB">1QFQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QFQ/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1QFQ</a></td><td align="center">A</td><td align="center">435</td><td align="center">145</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000804000000">4-20-99</td><td>ENTEROBACTERIA PHAGE LAMBDA</td><td>BACTERIOPHAGE LAMBDA N-PROTEIN-NUTBOXB-RNA COMPLEX</td><td>M.SCHAERPF,H.STICHT,P.ROESCH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HJI/index.html" title="SOLUTION NMR, 300 nucleotides, 100 basepairs, 0.3333 pairs per nucleotide">1HJI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QFQ/1QFQ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=397D" title="Link to PDB">397D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/397D/index.html" title="Link to FR3D analysis. 0.4074 pairs per nucleotide">397D</a></td><td align="center">AB</td><td align="center">27</td><td align="center">11</td><td align="center">1.3</td><td sorttable_customkey="19981109000000">9-11-98</td><td>synthetic,synthetic</td><td>A 1.3 A RESOLUTION CRYSTAL STRUCTURE OF THE HIV-1 TRANS- ACTIVATION RESPONSE REGION RNA STEM REVEALS A METAL ION- DEPENDENT BULGE CONFORMATION</td><td>J.A.IPPOLITO,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1CSL" title="Link to PDB">1CSL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CSL/index.html" title="Link to FR3D analysis. 0.3929 pairs per nucleotide">1CSL</a></td><td align="center">AB</td><td align="center">28</td><td align="center">11</td><td align="center">1.6</td><td sorttable_customkey="20010202000000">2-14-00</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE RRE HIGH AFFINITY SITE</td><td>J.A.IPPOLITO,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SDS" title="Link to PDB">1SDS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SDS/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1SDS</a></td><td align="center">D</td><td align="center">45</td><td align="center">0</td><td align="center">1.8</td><td sorttable_customkey="20050605000000">5-18-04</td><td>synthetic</td><td>STRUCTURE OF PROTEIN L7AE BOUND TO A K-TURN DERIVED FROM AN ARCHAEAL BOX H/ACA SRNA</td><td>T.HAMMA,A.FERRE-D'AMARE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2C4Z" title="Link to PDB">2C4Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C4Z/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">2C4Z</a></td><td align="center">R</td><td align="center">26</td><td align="center">10</td><td align="center">2.6</td><td sorttable_customkey="20060501000000">1-05-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2-RNA HAIRPIN (2SU -5-6) COMPLEX</td><td>E.GRAHN,T.MOSS,C.HELGSTRAND,K.FRIDBORG,M.SUNDARAM,K.TARS, H.LAGO,N.J.STONEHOUSE,D.R.DAVIS,P.G.STOCKLEY,L.LILJAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C51/index.html" title="Resolution 2.8A, 28 nucleotides, 9 basepairs, 0.3214 pairs per nucleotide">2C51</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BNY/index.html" title="Resolution 3.0A, 28 nucleotides, 9 basepairs, 0.3214 pairs per nucleotide">2BNY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C4Z/2C4Z_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1Q75" title="Link to PDB">1Q75</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Q75/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1Q75</a></td><td align="center">A</td><td align="center">300</td><td align="center">120</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041112000000">12-23-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE DYSKERATOSIS CONGENITA MUTANT P2B HAIRPIN FROM HUMAN TELOMERASE RNA</td><td>C.A.THEIMER,L.D.FINGER,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OQ0" title="Link to PDB">1OQ0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OQ0/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1OQ0</a></td><td align="center">A</td><td align="center">300</td><td align="center">120</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040105000000">5-13-03</td><td>synthetic</td><td>P6.1 STEM LOOP FROM THE ACTIVATION DOMAIN OF HTR</td><td>T.LEEPER,N.LEULLIOT,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F8K" title="Link to PDB">2F8K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F8K/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2F8K</a></td><td align="center">B</td><td align="center">15</td><td align="center">6</td><td align="center">2.0</td><td sorttable_customkey="20071201000000">1-24-06</td><td>synthetic</td><td>SEQUENCE SPECIFIC RECOGNITION OF RNA HAIRPINS BY THE SAM DOMAIN OF VTS1</td><td>T.AVIV,Z.LIN,G.BEN-ARI,C.A.SMIBERT,F.SICHERI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A4T" title="Link to PDB">1A4T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A4T/index.html" title="Link to FR3D analysis. 0.3833 pairs per nucleotide">1A4T</a></td><td align="center">A</td><td align="center">300</td><td align="center">115</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000504000000">4-29-98</td><td>synthetic</td><td>SOLUTION STRUCTURE OF PHAGE P22 N PEPTIDE-BOX B RNA COMPLEX, NMR, 20 STRUCTURES</td><td>Z.CAI,A.A.GORIN,R.FREDERICK,X.YE,W.HU,A.MAJUMDAR,A.KETTANI, D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OG8" title="Link to PDB">3OG8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OG8/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3OG8</a></td><td align="center">CD</td><td align="center">28</td><td align="center">14</td><td align="center">2.4</td><td sorttable_customkey="20100311000000">11-03-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN RIG-I CTD BOUND TO A 14-BP BLUNT-ENDED DSRNA</td><td>P.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KOD" title="Link to PDB">1KOD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KOD/index.html" title="Link to FR3D analysis. 0.2400 pairs per nucleotide">1KOD</a></td><td align="center">AB</td><td align="center">25</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960811000000">11-08-96</td><td>synthetic,synthetic</td><td>RNA APTAMER COMPLEXED WITH CITRULLINE, NMR</td><td>Y.S.YANG,M.KOCHOYAN,P.BURGSTALLER,E.WESTHOF,M.FAMULOK</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KOC/index.html" title="SOLUTION NMR, 25 nucleotides, 6 basepairs, 0.2400 pairs per nucleotide">1KOC</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KOD/1KOD_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1G2J" title="Link to PDB">1G2J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G2J/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1G2J</a></td><td align="center">A</td><td align="center">14</td><td align="center">6</td><td align="center">2.0</td><td sorttable_customkey="20000612000000">12-06-00</td><td>synthetic</td><td>RNA OCTAMER R(CCCP*GGGG) CONTAINING PHENYL RIBONUCLEOTIDE</td><td>G.MINASOV,J.MATULIC-ADAMIC,C.J.WILDS,P.HAEBERLI,N.USMAN, L.BEIGELMAN,M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F85" title="Link to PDB">1F85</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F85/index.html" title="Link to FR3D analysis. 0.2429 pairs per nucleotide">1F85</a></td><td align="center">A</td><td align="center">280</td><td align="center">68</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010511000000">11-17-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF HCV IRES RNA DOMAIN IIIE</td><td>P.J.LUKAVSKY,G.A.OTTO,A.M.LANCASTER,P.SARNOW,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TFY" title="Link to PDB">1TFY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TFY/index.html" title="Link to FR3D analysis. 0.4110 pairs per nucleotide">1TFY</a></td><td align="center">FI</td><td align="center">73</td><td align="center">30</td><td align="center">3.2</td><td sorttable_customkey="20041008000000">8-10-04</td><td>synthetic,synthetic,synthetic,synthetic</td><td>HOW CCA IS ADDED TO THE 3' END OF IMMATURE TRNA WITHOUT THE USE OF AN OLIGONUCLEOTIDE TEMPLATE</td><td>Y.XIONG,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ROQ" title="Link to PDB">1ROQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ROQ/index.html" title="Link to FR3D analysis. 0.4143 pairs per nucleotide">1ROQ</a></td><td align="center">A</td><td align="center">140</td><td align="center">58</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050605000000">5-18-04</td><td>synthetic</td><td>EXTENDING THE FAMILY OF UNCG-LIKE TETRALOOP MOTIFS: NMR STRUCTURE OF A CACG TETRALOOP FROM COXSACKIEVIRUS B3</td><td>Z.DU,J.YU,R.ANDINO,T.L.JAMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MY9" title="Link to PDB">1MY9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MY9/index.html" title="Link to FR3D analysis. 0.3929 pairs per nucleotide">1MY9</a></td><td align="center">AB</td><td align="center">28</td><td align="center">11</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030710000000">10-07-03</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A K+ CATION STABILIZED DIMERIC RNA QUADRUPLEX CONTAINING TWO G:G(:A):G:G(:A) HEXADS, G:G:G:G TETRADS AND UUUU LOOPS</td><td>H.LIU,A.MATSUGAMI,M.KATAHIRA,S.UESUGI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A3M" title="Link to PDB">1A3M</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A3M/index.html" title="Link to FR3D analysis. 0.3741 pairs per nucleotide">1A3M</a></td><td align="center">AB</td><td align="center">540</td><td align="center">202</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000504000000">4-29-98</td><td>synthetic,synthetic</td><td>PAROMOMYCIN BINDING INDUCES A LOCAL CONFORMATIONAL CHANGE IN THE A SITE OF 16S RRNA, NMR, 20 STRUCTURES</td><td>D.FOURMY,S.YOSHIZAWA,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2B3J" title="Link to PDB">2B3J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B3J/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">2B3J</a></td><td align="center">E</td><td align="center">28</td><td align="center">7</td><td align="center">2.0</td><td sorttable_customkey="20071201000000">1-24-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF STAPHYLOCOCCUS AUREUS TRNA ADENOSINE DEAMINASE, TADA, IN COMPLEX WITH RNA</td><td>H.C.LOSEY,A.J.RUTHENBURG,G.L.VERDINE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XLJ" title="Link to PDB">2XLJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XLJ/index.html" title="Link to FR3D analysis. 0.3571 pairs per nucleotide">2XLJ</a></td><td align="center">B</td><td align="center">14</td><td align="center">5</td><td align="center">2.6</td><td sorttable_customkey="20111009000000">9-22-10</td><td>PSEUDOMONAS AERUGINOSA</td><td>CRYSTAL STRUCTURE OF THE CSY4-CRRNA COMPLEX, HEXAGONAL FORM</td><td>R.E.HAURWITZ,M.JINEK,B.WIEDENHEFT,K.ZHOU,J.A.DOUDNA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XLK/index.html" title="Resolution 1.8A, 28 nucleotides, 9 basepairs, 0.3214 pairs per nucleotide">2XLK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XLJ/2XLJ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=433D" title="Link to PDB">433D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/433D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">433D</a></td><td align="center">AB</td><td align="center">28</td><td align="center">14</td><td align="center">2.1</td><td sorttable_customkey="19980412000000">12-04-98</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A 14 BASE PAIR RNA DUPLEX WITH NONSYMMETRICAL TANDEM G.U WOBBLE BASE PAIRS</td><td>J.TRIKHA,D.J.FILMAN,J.M.HOGLE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BX2" title="Link to PDB">2BX2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BX2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2BX2</a></td><td align="center">R</td><td align="center">14</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20060210000000">10-14-05</td><td>synthetic</td><td>CATALYTIC DOMAIN OF E. COLI RNASE E</td><td>M.J.MARCAIDA,A.J.CALLAGHAN,W.G.SCOTT,B.F.LUISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2I91" title="Link to PDB">2I91</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I91/index.html" title="Link to FR3D analysis. 0.3182 pairs per nucleotide">2I91</a></td><td align="center">CD</td><td align="center">22</td><td align="center">7</td><td align="center">2.6</td><td sorttable_customkey="20070510000000">10-17-06</td><td>synthetic,synthetic</td><td>KDA RO AUTOANTIGEN IN COMPLEX WITH A FRAGMENT OF MISFOLDED RNA</td><td>K.M.REINISCH,A.J.STEIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IK1" title="Link to PDB">1IK1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IK1/index.html" title="Link to FR3D analysis. 0.3571 pairs per nucleotide">1IK1</a></td><td align="center">A</td><td align="center">14</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020607000000">7-18-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN RNA HAIRPIN FROM HRV-14</td><td>H.HUANG,A.ALEXANDROV,X.CHEN,T.W.BARNES III,H.ZHANG,K.DUTTA, S.M.PASCAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2EVY" title="Link to PDB">2EVY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EVY/index.html" title="Link to FR3D analysis. 0.3571 pairs per nucleotide">2EVY</a></td><td align="center">A</td><td align="center">252</td><td align="center">90</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071008000000">8-22-06</td><td>synthetic</td><td>GNYA TETRANUCLEOTIDE LOOPS FOUND IN POLIOVIRUS ORIL BY IN VIVO SELEX (UN)EXPECTEDLY FORM A YNMG-LIKE STRUCTURE</td><td>W.J.G.MELCHERS,J.ZOLL,M.TESSARI,D.V.BAKHMUTOV,A.P.GMYL, V.I.AGOL,H.A.HEUS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P59" title="Link to PDB">3P59</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P59/index.html" title="Link to FR3D analysis. 0.3723 pairs per nucleotide">3P59</a></td><td align="center">ABCDEFGH</td><td align="center">94</td><td align="center">35</td><td align="center">2.2</td><td sorttable_customkey="20110604000000">4-06-11</td><td>synthetic,synthetic</td><td>FIRST CRYSTAL STRUCTURE OF A RNA NANOSQUARE</td><td>S.DIBROV,T.HERMANN,J.MCLEAN,J.PARSONS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3OIN" title="Link to PDB">3OIN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OIN/index.html" title="Link to FR3D analysis. 0.2857 pairs per nucleotide">3OIN</a></td><td align="center">C</td><td align="center">14</td><td align="center">4</td><td align="center">1.9</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF SACCHAROMYCES CEREVISIAE NEP1/EMG1 BOUND TO S- ADENOSYLHOMOCYSTEINE AND 1 MOLECULE OF COGNATE RNA</td><td>S.R.THOMAS,N.LARONDE-LEBLANC</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OIJ/index.html" title="Resolution 3.0A, 28 nucleotides, 8 basepairs, 0.2857 pairs per nucleotide">3OIJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3OIN/3OIN_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HJF" title="Link to PDB">3HJF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HJF/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HJF</a></td><td align="center">Y</td><td align="center">14</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20090610000000">10-06-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF T. THERMOPHILUS ARGONAUTE E546 MUTANT PROTEIN COMPLEXED WITH DNA GUIDE STRAND AND 15-NT RNA TARGET STRAND</td><td>Y.WANG,H.LI,G.SHENG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K4B" title="Link to PDB">1K4B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K4B/index.html" title="Link to FR3D analysis. 0.3571 pairs per nucleotide">1K4B</a></td><td align="center">A</td><td align="center">280</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020712000000">12-19-01</td><td>synthetic</td><td>STRUCTURE OF AGUU RNA TETRALOOP, NMR, 20 STRUCTURES</td><td>H.WU,P.K.YANG,S.E.BUTCHER,S.KANG,G.CHANFREAU,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=6MSF" title="Link to PDB">6MSF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/6MSF/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">6MSF</a></td><td align="center">RS</td><td align="center">24</td><td align="center">8</td><td align="center">2.8</td><td sorttable_customkey="19980807000000">7-08-98</td><td>ENTEROBACTERIO PHAGE MS2,synthetic</td><td>F6 APTAMER MS2 COAT PROTEIN COMPLEX</td><td>M.A.CONVERY,S.ROWSELL,N.J.STONEHOUSE,A.D.ELLINGTON,I.HIRAO, J.B.MURRAY,D.S.PEABODY,S.E.V.PHILLIPS,P.G.STOCKLEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1C0O" title="Link to PDB">1C0O</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C0O/index.html" title="Link to FR3D analysis. 0.4098 pairs per nucleotide">1C0O</a></td><td align="center">A</td><td align="center">266</td><td align="center">109</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010708000000">8-31-99</td><td>TETRAHYMENA THERMOPHILA</td><td>SOLUTION STRUCTURE OF THE P5 HAIRPIN FROM A GROUP I INTRON COMPLEXED WITH COBALT (III) HEXAMMINE, NMR, 19 CONVERGED STRUCTURES</td><td>G.COLMENAREJO,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FHK" title="Link to PDB">1FHK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FHK/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">1FHK</a></td><td align="center">A</td><td align="center">210</td><td align="center">80</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020903000000">3-21-01</td><td>synthetic</td><td>NMR STRUCTURE OF THE 690 LOOP OF 16 S RRNA OF E. COLI</td><td>S.V.MOROSYUK,P.R.CUNNINGHAM,J.SANTALUCIA JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QKK" title="Link to PDB">2QKK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QKK/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2QKK</a></td><td align="center">C</td><td align="center">83</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20080111000000">11-13-07</td><td>HOMO SAPIENS</td><td>HUMAN RNASE H CATALYTIC DOMAIN MUTANT D210N IN COMPLEX WITH 14-MER RNA/DNA HYBRID</td><td>M.NOWOTNY,S.A.GAIDAMAKOV,R.GHIRLANDO,S.M.CERRITELLI, R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MHK" title="Link to PDB">1MHK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MHK/index.html" title="Link to FR3D analysis. 0.3462 pairs per nucleotide">1MHK</a></td><td align="center">LS</td><td align="center">26</td><td align="center">9</td><td align="center">2.5</td><td sorttable_customkey="20020609000000">9-06-02</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE ANALYSIS OF A 26MER RNA MOLECULE, REPRESENTING A NEW RNA MOTIF, THE HOOK-TURN</td><td>S.SZEP,J.WANG,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DUQ" title="Link to PDB">1DUQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DUQ/index.html" title="Link to FR3D analysis. 0.4231 pairs per nucleotide">1DUQ</a></td><td align="center">AB</td><td align="center">104</td><td align="center">44</td><td align="center">2.1</td><td sorttable_customkey="20010405000000">5-16-00</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE REV BINDING ELEMENT OF HIV-1</td><td>L.-W.HUNG,E.L.HOLBROOK,S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K4A" title="Link to PDB">1K4A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K4A/index.html" title="Link to FR3D analysis. 0.4071 pairs per nucleotide">1K4A</a></td><td align="center">A</td><td align="center">280</td><td align="center">114</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020712000000">12-19-01</td><td>synthetic</td><td>STRUCTURE OF AGAA RNA TETRALOOP, NMR, 20 STRUCTURES</td><td>H.WU,P.K.YANG,S.E.BUTCHER,S.KANG,G.CHANFREAU,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R93" title="Link to PDB">2R93</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R93/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2R93</a></td><td align="center">R</td><td align="center">14</td><td align="center">6</td><td align="center">4.0</td><td sorttable_customkey="20090311000000">11-27-07</td><td>synthetic</td><td>ELONGATION COMPLEX OF RNA POLYMERASE II WITH A HEPATITIS DELTA VIRUS-DERIVED RNA STEM LOOP</td><td>E.LEHMANN,F.BRUECKNER,P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RNA" title="Link to PDB">1RNA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RNA/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1RNA</a></td><td align="center">AB</td><td align="center">28</td><td align="center">14</td><td align="center">2.2</td><td sorttable_customkey="19920304000000">4-15-91</td><td>synthetic</td><td>CRYSTALLOGRAPHIC STRUCTURE OF AN RNA HELIX: [U(U-A)6A]2</td><td>A.C.DOCK-BREGEON,B.CHEVRIER,A.PODJARNY,J.JOHNSON,J.S.DE BEAR,G.R.GOUGH,P.T.GILHAM,D.MORAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=7MSF" title="Link to PDB">7MSF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/7MSF/index.html" title="Link to FR3D analysis. 0.3200 pairs per nucleotide">7MSF</a></td><td align="center">R</td><td align="center">25</td><td align="center">8</td><td align="center">2.8</td><td sorttable_customkey="19981111000000">11-11-98</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2 PROTEIN CAPSID/RNA COMPLEX</td><td>S.ROWSELL,N.J.STONEHOUSE,M.A.CONVERY,C.J.ADAMS, A.D.ELLINGTON,I.HIRAO,D.S.PEABODY,P.G.STOCKLEY, S.E.V.PHILLIPS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=333D" title="Link to PDB">333D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/333D/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">333D</a></td><td align="center">A</td><td align="center">14</td><td align="center">6</td><td align="center">2.5</td><td sorttable_customkey="19980410000000">10-16-97</td><td>synthetic</td><td>THE CRYSTAL STRUCTURE OF AN RNA OLIGOMER INCORPORATING TANDEM ADENOSINE-INOSINE MISMATCHES</td><td>R.J.CARTER,K.J.BAEYENS,J.SANTALUCIA JR.,D.H.TURNER, S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1L3Z" title="Link to PDB">1L3Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1L3Z/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1L3Z</a></td><td align="center">A</td><td align="center">14</td><td align="center">6</td><td align="center">2.0</td><td sorttable_customkey="20050102000000">2-25-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE ANALYSIS OF AN RNA HEPTAMER</td><td>K.SHI,B.PAN,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2C50" title="Link to PDB">2C50</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C50/index.html" title="Link to FR3D analysis. 0.2963 pairs per nucleotide">2C50</a></td><td align="center">R</td><td align="center">27</td><td align="center">8</td><td align="center">2.6</td><td sorttable_customkey="20060501000000">1-05-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2-RNA HAIRPIN (A -5) COMPLEX</td><td>E.GRAHN,T.MOSS,C.HELGSTRAND,K.FRIDBORG,M.SUNDARAM,K.TARS, H.LAGO,N.J.STONEHOUSE,D.R.DAVIS,P.G.STOCKLEY,L.LILJAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IZ9" title="Link to PDB">2IZ9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IZ9/index.html" title="Link to FR3D analysis. 0.3214 pairs per nucleotide">2IZ9</a></td><td align="center">R</td><td align="center">28</td><td align="center">9</td><td align="center">2.9</td><td sorttable_customkey="20080307000000">7-27-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>MS2-RNA HAIRPIN (4ONE-5) COMPLEX</td><td>E.GRAHN,N.J.STONEHOUSE,C.J.ADAMS,K.FRIDBORG,L.BEIGELMAN, J.MATULIC-ADAMIC,S.L.WARRINER,P.G.STOCKLEY,L.LILJAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KOC" title="Link to PDB">2KOC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KOC/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2KOC</a></td><td align="center">A</td><td align="center">280</td><td align="center">120</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090112000000">12-01-09</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF A 14-MER HAIRPIN RNA WITH CUUCGG TETRALOOP</td><td>S.NOZINOVIC,B.FUERTIG,H.R.A.JONKER,C.RICHTER,H.SCHWALBE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1HS2" title="Link to PDB">1HS2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS2/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">1HS2</a></td><td align="center">A</td><td align="center">13</td><td align="center">4</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030508000000">8-29-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF RNA HAIRPIN LOOP UUAAGU AS PART OF HAIRPIN R(GCGUUAAGUCGCA)</td><td>H.ZHANG,M.A.FOUNTAIN,T.R.KRUGH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS8/index.html" title="SOLUTION NMR, 13 nucleotides, 4 basepairs, 0.3077 pairs per nucleotide">1HS8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS3/index.html" title="SOLUTION NMR, 13 nucleotides, 4 basepairs, 0.3077 pairs per nucleotide">1HS3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS4/index.html" title="SOLUTION NMR, 13 nucleotides, 4 basepairs, 0.3077 pairs per nucleotide">1HS4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS1/index.html" title="SOLUTION NMR, 13 nucleotides, 4 basepairs, 0.3077 pairs per nucleotide">1HS1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HS2/1HS2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NUJ" title="Link to PDB">1NUJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NUJ/index.html" title="Link to FR3D analysis. 0.3854 pairs per nucleotide">1NUJ</a></td><td align="center">CDEF</td><td align="center">96</td><td align="center">37</td><td align="center">1.8</td><td sorttable_customkey="20040708000000">8-19-03</td><td>synthetic,synthetic</td><td>THE LEADZYME STRUCTURE BOUND TO MG(H20)6(II) AT 1.8 A RESOLUTION</td><td>J.E.WEDEKIND,D.B.MCKAY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NUV/index.html" title="Resolution 1.8A, 96 nucleotides, 37 basepairs, 0.3854 pairs per nucleotide">1NUV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NUJ/1NUJ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CGS" title="Link to PDB">3CGS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CGS/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">3CGS</a></td><td align="center">AB</td><td align="center">24</td><td align="center">10</td><td align="center">1.6</td><td sorttable_customkey="20080107000000">7-01-08</td><td>synthetic,synthetic</td><td>X-RAY STRUCTURE CONTAINING THE PSEUDOURIDYLATED U2 SNRNA AND MAMMALIAN INTRON BRANCH SITE CONSENSUS SEQUENCES</td><td>Y.LIN,C.L.KIELKOPF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1VOP" title="Link to PDB">1VOP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VOP/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1VOP</a></td><td align="center">A</td><td align="center">429</td><td align="center">165</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19981212000000">12-24-97</td><td>synthetic</td><td>CONSERVED RNA COMPONENT OF THE PEPTIDYL TRANSFERASE CENTER, NMR, 33 STRUCTURES</td><td>E.V.PUGLISI,J.D.PUGLISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JZC" title="Link to PDB">1JZC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JZC/index.html" title="Link to FR3D analysis. 0.2308 pairs per nucleotide">1JZC</a></td><td align="center">A</td><td align="center">13</td><td align="center">3</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030710000000">10-07-03</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF THE MUTANT 5'AUG3' TRILOOP IN THE RNA PROMOTER REGION OF THE BROME MOSAIC VIRUS GENOMIC (+)- RNA</td><td>C.-H.KIM,C.C.KAO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HM9" title="Link to PDB">3HM9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HM9/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HM9</a></td><td align="center">Y</td><td align="center">13</td><td align="center">0</td><td align="center">3.3</td><td sorttable_customkey="20090610000000">10-06-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF T. THERMOPHILUS ARGONAUTE COMPLEXED WITH DNA GUIDE STRAND AND 19-NT RNA TARGET STRAND</td><td>Y.WANG,H.LI,G.SHENG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I4C" title="Link to PDB">1I4C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I4C/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1I4C</a></td><td align="center">A</td><td align="center">13</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020904000000">4-21-01</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF THE MINOR FAMILY OF THE MUTANT STEM LOOP C 5'UUA3' TRILOOP OF BROME MOSAIC VIRUS (+) STRAND RNA</td><td>I.TINOCO JR.,C.-H.KIM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AM0" title="Link to PDB">1AM0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AM0/index.html" title="Link to FR3D analysis. 0.1484 pairs per nucleotide">1AM0</a></td><td align="center">A</td><td align="center">128</td><td align="center">19</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19981107000000">7-23-97</td><td>synthetic</td><td>AMP RNA APTAMER COMPLEX, NMR, 8 STRUCTURES</td><td>F.JIANG,R.A.KUMAR,R.A.JONES,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FUF" title="Link to PDB">1FUF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FUF/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">1FUF</a></td><td align="center">AB</td><td align="center">26</td><td align="center">12</td><td align="center">1.7</td><td sorttable_customkey="20010911000000">11-09-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A 14BP RNA OLIGONUCLEOTIDE CONTAINING DOUBLE UU BULGES: A NOVEL INTRAMOLECULAR U*(AU) BASE TRIPLE</td><td>J.DENG,Y.XIONG,C.SUDARSANAKUMAR,K.SHI,M.SUNDARALINGAM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IK5/index.html" title="Resolution 1.8A, 26 nucleotides, 12 basepairs, 0.4615 pairs per nucleotide">1IK5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FUF/1FUF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CGP" title="Link to PDB">3CGP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CGP/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">3CGP</a></td><td align="center">AB</td><td align="center">24</td><td align="center">10</td><td align="center">1.6</td><td sorttable_customkey="20080107000000">7-01-08</td><td>synthetic,synthetic</td><td>X-RAY STRUCTURE OF A PSEUDOURIDINE-CONTAINING YEAST SPLICEOSOMAL U2 SNRNA-INTRON BRANCH SITE DUPLEX BOUND TO IODIDE IONS</td><td>Y.LIN,C.L.KIELKOPF</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CGQ/index.html" title="Resolution 2.5A, 24 nucleotides, 10 basepairs, 0.4167 pairs per nucleotide">3CGQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CGP/3CGP_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IZM" title="Link to PDB">2IZM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IZM/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">2IZM</a></td><td align="center">R</td><td align="center">26</td><td align="center">8</td><td align="center">2.7</td><td sorttable_customkey="20070307000000">7-03-07</td><td>BACTERIOPHAGE MS2</td><td>MS2-RNA HAIRPIN (C-10) COMPLEX</td><td>C.HELGSTRAND,E.GRAHN,T.MOSS,N.J.STONEHOUSE,K.TARS, P.G.STOCKLEY,L.LILJAS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZDH/index.html" title="Resolution 2.7A, 26 nucleotides, 8 basepairs, 0.3077 pairs per nucleotide">1ZDH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IZM/2IZM_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TFW" title="Link to PDB">1TFW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TFW/index.html" title="Link to FR3D analysis. 0.3725 pairs per nucleotide">1TFW</a></td><td align="center">EH</td><td align="center">51</td><td align="center">19</td><td align="center">2.2</td><td sorttable_customkey="20041008000000">8-10-04</td><td>synthetic,synthetic,synthetic,synthetic</td><td>HOW CCA IS ADDED TO THE 3' END OF IMMATURE TRNA WITHOUT THE USE OF AN OLIGONUCLEOTIDE TEMPLATE</td><td>Y.XIONG,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EFS" title="Link to PDB">1EFS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EFS/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1EFS</a></td><td align="center">B</td><td align="center">130</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000603000000">3-06-00</td><td>synthetic</td><td>CONFORMATION OF A DNA-RNA HYBRID</td><td>V.LARUE,E.HANTZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IX1" title="Link to PDB">2IX1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IX1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2IX1</a></td><td align="center">B</td><td align="center">13</td><td align="center">0</td><td align="center">2.7</td><td sorttable_customkey="20060510000000">10-05-06</td><td>synthetic</td><td>RNASE II D209N MUTANT</td><td>C.FRAZAO,C.E.MCVEY,M.AMBLAR,A.BARBAS,C.VONRHEIN, C.M.ARRAIANO,M.A.CARRONDO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I46" title="Link to PDB">1I46</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I46/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">1I46</a></td><td align="center">A</td><td align="center">13</td><td align="center">4</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020904000000">4-21-01</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF THE MUTANT STEM LOOP C 5'GUA3' TRILOOP OF BROME MOSAIC VIRUS (+) STRAND RNA</td><td>C.-H.KIM,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KM8" title="Link to PDB">2KM8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KM8/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2KM8</a></td><td align="center">A</td><td align="center">130</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120407000000">7-28-10</td><td>synthetic</td><td>INTERDOMAIN RRM PACKING CONTRIBUTES TO RNA RECOGNITION IN THE RNA15, HRP1, ANCHOR RNA 3' PROCESSING TERNARY COMPLEX</td><td>T.C.LEEPER,G.VARANI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K8S" title="Link to PDB">1K8S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K8S/index.html" title="Link to FR3D analysis. 0.4800 pairs per nucleotide">1K8S</a></td><td align="center">AB</td><td align="center">25</td><td align="center">12</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020211000000">11-14-01</td><td>synthetic,synthetic</td><td>BULGED ADENOSINE IN AN RNA DUPLEX</td><td>V.THIVIYANATHAN,A.B.GULIAEV,N.B.LEONTIS,D.G.GORENSTEIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I4B" title="Link to PDB">1I4B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I4B/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1I4B</a></td><td align="center">A</td><td align="center">13</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020904000000">4-21-01</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF THE MAJOR FAMILY OF THE MUTANT STEM LOOP C 5'UUA3' TRILOOP OF BROME MOSAIC VIRUS (+) STRAND RNA</td><td>I.TINOCO JR.,C.-H.KIM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R22" title="Link to PDB">2R22</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R22/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2R22</a></td><td align="center">AB</td><td align="center">26</td><td align="center">13</td><td align="center">1.4</td><td sorttable_customkey="20090109000000">9-25-07</td><td>synthetic</td><td>STRUCTURE OF THE NATIVE RNA TRIDECAMER R(GCGUUUGAAACGC) AT 1.5 A (NATMN)</td><td>Y.TIMSIT,S.BOMBARD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CGR" title="Link to PDB">3CGR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CGR/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">3CGR</a></td><td align="center">AB</td><td align="center">24</td><td align="center">10</td><td align="center">2.1</td><td sorttable_customkey="20080107000000">7-01-08</td><td>synthetic,synthetic</td><td>X-RAY STRUCTURE CONTAINING THE PSEUDOURIDYLATED U2 SNRNA AND INTRON BRANCH SITE CONSENSUS SEQUENCES</td><td>Y.LIN,C.L.KIELKOPF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MQK" title="Link to PDB">3MQK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MQK/index.html" title="Link to FR3D analysis. 0.2273 pairs per nucleotide">3MQK</a></td><td align="center">DE</td><td align="center">22</td><td align="center">5</td><td align="center">2.8</td><td sorttable_customkey="20120512000000">12-29-10</td><td>synthetic,synthetic</td><td>CBF5-NOP10-GAR1 COMPLEX BINDING WITH 17MER RNA CONTAINING ACA TRINUCLEOTIDE</td><td>J.ZHOU,B.LIANG,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GTP" title="Link to PDB">3GTP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTP/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3GTP</a></td><td align="center">R</td><td align="center">13</td><td align="center">0</td><td align="center">3.9</td><td sorttable_customkey="20090906000000">6-09-09</td><td>synthetic</td><td>BACKTRACKED RNA POLYMERASE II COMPLEX WITH 24MER RNA</td><td>D.WANG,D.A.BUSHNELL,X.HUANG,K.D.WESTOVER,M.LEVITT, R.D.KORNBERG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ESH" title="Link to PDB">1ESH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ESH/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">1ESH</a></td><td align="center">A</td><td align="center">13</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20011204000000">4-24-00</td><td>synthetic</td><td>THE SOLUTION STRUCTURE OF THE STEM LOOP C 5'AUA3' TRILOOP OF BROME MOSAIC VIRUS (+) STRAND RNA</td><td>C.-H.KIM,C.C.KAO,I.TINOCO JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BR3" title="Link to PDB">1BR3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BR3/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1BR3</a></td><td align="center">A</td><td align="center">13</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20001102000000">2-23-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN 82-NUCLEOTIDE RNA-DNA COMPLEX FORMED BY THE 10-23 DNA ENZYME</td><td>J.NOWAKOWSKI,P.J.SHIM,G.S.PRASAD,C.D.STOUT,G.F.JOYCE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=429D" title="Link to PDB">429D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/429D/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">429D</a></td><td align="center">CD</td><td align="center">48</td><td align="center">16</td><td align="center">2.7</td><td sorttable_customkey="19990103000000">3-01-99</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A LEADZYME; METAL BINDING AND IMPLICATIONS FOR CATALYSIS</td><td>J. E.WEDEKIND,D. B.MCKAY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ELH" title="Link to PDB">1ELH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ELH/index.html" title="Link to FR3D analysis. 0.3933 pairs per nucleotide">1ELH</a></td><td align="center">AB</td><td align="center">150</td><td align="center">59</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960701000000">1-31-94</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>NMR ANALYSIS OF HELIX I FROM THE 5S RNA OF ESCHERICHIA COLI</td><td>S.WHITE,P.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GJE" title="Link to PDB">2GJE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GJE/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2GJE</a></td><td align="center">RS</td><td align="center">18</td><td align="center">0</td><td align="center">3.4</td><td sorttable_customkey="20060509000000">9-05-06</td><td>synthetic,synthetic</td><td>STRUCTURE OF A GUIDERNA-BINDING PROTEIN COMPLEX BOUND TO A GRNA</td><td>M.A.SCHUMACHER,E.KARAMOOZ,A.ZIKOVA,L.TRANTIREK,J.LUKES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XLI" title="Link to PDB">2XLI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XLI/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">2XLI</a></td><td align="center">B</td><td align="center">13</td><td align="center">5</td><td align="center">2.3</td><td sorttable_customkey="20111009000000">9-22-10</td><td>PSEUDOMONAS AERUGINOSA</td><td>CRYSTAL STRUCTURE OF THE CSY4-CRRNA COMPLEX, MONOCLINIC FORM</td><td>R.E.HAURWITZ,M.JINEK,B.WIEDENHEFT,K.ZHOU,J.A.DOUDNA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I9X" title="Link to PDB">1I9X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I9X/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">1I9X</a></td><td align="center">AB</td><td align="center">26</td><td align="center">12</td><td align="center">2.2</td><td sorttable_customkey="20010509000000">9-05-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF BPS-U2 SNRNA DUPLEX</td><td>J.A.BERGLUND,M.ROSBASH,S.C.SCHULTZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3LRN" title="Link to PDB">3LRN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LRN/index.html" title="Link to FR3D analysis. 0.4615 pairs per nucleotide">3LRN</a></td><td align="center">CD</td><td align="center">26</td><td align="center">12</td><td align="center">2.6</td><td sorttable_customkey="20100206000000">6-02-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN RIG-I CTD BOUND TO A 14 BP GC 5' PPP DSRNA</td><td>P.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ATW" title="Link to PDB">2ATW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ATW/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2ATW</a></td><td align="center">B</td><td align="center">24</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20051110000000">10-11-05</td><td>synthetic</td><td>STRUCTURE OF A MYCOBACTERIUM TUBERCULOSIS NUSA-RNA COMPLEX</td><td>B.BEUTH,S.PENNELL,K.B.ARNVIG,S.R.MARTIN,I.A.TAYLOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZSE" title="Link to PDB">1ZSE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZSE/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1ZSE</a></td><td align="center">R</td><td align="center">12</td><td align="center">4</td><td align="center">3.0</td><td sorttable_customkey="20060905000000">5-09-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>RNA STEMLOOP FROM BACTERIOPHAGE QBETA COMPLEXED WITH AN N87S MUTANT MS2 CAPSID</td><td>W.T.HORN,K.TARS,E.GRAHN,C.HELGSTRAND,A.J.BARON,H.LAGO, C.J.ADAMS,D.S.PEABODY,S.E.V.PHILLIPS,N.J.STONEHOUSE, L.LILJAS,P.G.STOCKLEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BS1/index.html" title="Resolution 2.8A, 23 nucleotides, 7 basepairs, 0.3043 pairs per nucleotide">2BS1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BS0/index.html" title="Resolution 2.5A, 21 nucleotides, 6 basepairs, 0.2857 pairs per nucleotide">2BS0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZSE/1ZSE_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Z2Q" title="Link to PDB">2Z2Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z2Q/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2Z2Q</a></td><td align="center">R</td><td align="center">12</td><td align="center">0</td><td align="center">2.7</td><td sorttable_customkey="20090307000000">7-15-08</td><td>FLOCK HOUSE VIRUS</td><td>CRYSTAL STRUCTURE OF FLOCK HOUSE VIRUS</td><td>J.E.JOHNSON,Z.CHEN,V.REDDY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q25/index.html" title="Resolution 3.5A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2Q25</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q23/index.html" title="Resolution 3.2A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2Q23</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q26/index.html" title="Resolution 3.5A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2Q26</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Z2Q/2Z2Q_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=354D" title="Link to PDB">354D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/354D/index.html" title="Link to FR3D analysis. 0.3636 pairs per nucleotide">354D</a></td><td align="center">AB</td><td align="center">22</td><td align="center">8</td><td align="center">1.5</td><td sorttable_customkey="19990211000000">11-26-97</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>STRUCTURE OF LOOP E FROM E. COLI 5S RRNA</td><td>C.C.CORRELL,B.FREEBORN,P.B.MOORE,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GTL" title="Link to PDB">3GTL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTL/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3GTL</a></td><td align="center">R</td><td align="center">12</td><td align="center">0</td><td align="center">3.4</td><td sorttable_customkey="20090906000000">6-09-09</td><td>synthetic</td><td>BACKTRACKED RNA POLYMERASE II COMPLEX WITH 13MER WITH G<>U MISMATCH</td><td>D.WANG,D.A.BUSHNELL,X.HUANG,K.D.WESTOVER,M.LEVITT, R.D.KORNBERG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTJ/index.html" title="Resolution 3.4A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3GTJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTG/index.html" title="Resolution 3.8A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3GTG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTO/index.html" title="Resolution 4.0A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3GTO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTL/3GTL_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3CJZ" title="Link to PDB">3CJZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3CJZ/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">3CJZ</a></td><td align="center">AB</td><td align="center">24</td><td align="center">10</td><td align="center">1.8</td><td sorttable_customkey="20090210000000">10-14-08</td><td>synthetic</td><td>EFFECTS OF N2,N2-DIMETHYLGUANOSINE ON RNA STRUCTURE AND STABILITY: CRYSTAL STRUCTURE OF AN RNA DUPLEX WITH TANDEM M22G:A PAIRS</td><td>P.S.PALLAN,C.KREUTZ,S.BOSIO,R.MICURA,M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YYW" title="Link to PDB">1YYW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YYW/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1YYW</a></td><td align="center">EF</td><td align="center">96</td><td align="center">48</td><td align="center">2.8</td><td sorttable_customkey="20061011000000">11-22-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNASE III FROM AQUIFEX AEOLICUS COMPLEXED WITH DOUBLE STRANDED RNA AT 2.8-ANGSTROM RESOLUTION</td><td>J.GAN,J.E.TROPEA,B.P.AUSTIN,D.L.COURT,D.S.WAUGH,X.JI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BSU" title="Link to PDB">3BSU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BSU</a></td><td align="center">D</td><td align="center">24</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20100103000000">3-25-08</td><td>HOMO SAPIENS</td><td>HYBRID-BINDING DOMAIN OF HUMAN RNASE H1 IN COMPLEX WITH 12- MER RNA/DNA</td><td>M.NOWOTNY,S.M.CERRITELLI,R.GHIRLANDO,S.A.GAIDAMAKOV, R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R20" title="Link to PDB">2R20</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R20/index.html" title="Link to FR3D analysis. 0.4583 pairs per nucleotide">2R20</a></td><td align="center">AB</td><td align="center">24</td><td align="center">11</td><td align="center">1.3</td><td sorttable_customkey="20090109000000">9-25-07</td><td>synthetic</td><td>STRUCTURE OF THE RNA BROMINATED TRIDECAMER R(GCGUU- 5BUGAAACGC) AT 1.3 A (BR2)</td><td>Y.TIMSIT,S.BOMBARD</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R21/index.html" title="Resolution 1.6A, 24 nucleotides, 11 basepairs, 0.4583 pairs per nucleotide">2R21</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R1S/index.html" title="Resolution 1.4A, 24 nucleotides, 11 basepairs, 0.4583 pairs per nucleotide">2R1S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R20/2R20_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RQJ" title="Link to PDB">2RQJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RQJ/index.html" title="Link to FR3D analysis. 0.6759 pairs per nucleotide">2RQJ</a></td><td align="center">AB</td><td align="center">216</td><td align="center">146</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100511000000">11-17-09</td><td>synthetic</td><td>QUADRUPLEX STRUCTURE OF AN RNA APTAMER AGAINST BOVINE PRION PROTEIN</td><td>M.KATAHIRA,T.MASHIMA,A.MATSUGAMI,F.NISHIKAWA,S.NISHIKAWA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZIG" title="Link to PDB">1ZIG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZIG/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">1ZIG</a></td><td align="center">A</td><td align="center">120</td><td align="center">50</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971203000000">3-12-97</td><td>synthetic</td><td>GAGA RNA TETRALOOP, NMR, 10 STRUCTURES</td><td>F.M.JUCKER,H.A.HEUS,P.F.YIP,E.MOORS,A.PARDI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZIH/index.html" title="SOLUTION NMR, 120 nucleotides, 44 basepairs, 0.3667 pairs per nucleotide">1ZIH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZIF/index.html" title="SOLUTION NMR, 120 nucleotides, 35 basepairs, 0.2917 pairs per nucleotide">1ZIF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZIG/1ZIG_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AFX" title="Link to PDB">1AFX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AFX/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1AFX</a></td><td align="center">A</td><td align="center">156</td><td align="center">60</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980305000000">5-15-97</td><td>synthetic</td><td>UGAA EUKARYOTIC RIBOSOMAL RNA TETRALOOP, NMR, 13 STRUCTURES</td><td>S.E.BUTCHER,T.DIECKMANN,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SDR" title="Link to PDB">1SDR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SDR/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1SDR</a></td><td align="center">AB</td><td align="center">48</td><td align="center">24</td><td align="center">2.6</td><td sorttable_customkey="19970302000000">2-27-95</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DODECAMER CONTAINING THE ESCHERICHIA COLI SHINE-DALGARNO SEQUENCE</td><td>H.SCHINDELIN,M.ZHANG,R.BALD,J.-P.FUERSTE,V.A.ERDMANN, U.HEINEMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AJL" title="Link to PDB">1AJL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AJL/index.html" title="Link to FR3D analysis. 0.3158 pairs per nucleotide">1AJL</a></td><td align="center">AB</td><td align="center">19</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>TETRAHYMENA THERMOPHILA,TETRAHYMENA THERMOPHILA</td><td>FIVE-NUCLEOTIDE BULGE LOOP FROM TETRAHYMENA THERMOPHILA GROUP I INTRON</td><td>K.J.LUEBKE,S.M.LANDRY,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=157D" title="Link to PDB">157D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/157D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">157D</a></td><td align="center">AB</td><td align="center">24</td><td align="center">12</td><td align="center">1.8</td><td sorttable_customkey="19960705000000">5-31-94</td><td>synthetic</td><td>CRYSTAL AND MOLECULAR STRUCTURE OF R(CGCGAAUUAGCG): AN RNA DUPLEX CONTAINING TWO G(ANTI).A(ANTI) BASE-PAIRS</td><td>G.A.LEONARD,K.E.MCAULEY-HECHT,S.EBEL,D.M.LOUGH,T.BROWN, W.N.HUNTER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YYO/index.html" title="Resolution 2.9A, 48 nucleotides, 24 basepairs, 0.5000 pairs per nucleotide">1YYO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YYK/index.html" title="Resolution 2.5A, 48 nucleotides, 24 basepairs, 0.5000 pairs per nucleotide">1YYK</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AL5/index.html" title="SOLUTION NMR, 288 nucleotides, 144 basepairs, 0.5000 pairs per nucleotide">1AL5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CX5/index.html" title="SOLUTION NMR, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1CX5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AC3/index.html" title="SOLUTION NMR, 96 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1AC3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/157D/157D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=353D" title="Link to PDB">353D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/353D/index.html" title="Link to FR3D analysis. 0.4583 pairs per nucleotide">353D</a></td><td align="center">AB</td><td align="center">24</td><td align="center">11</td><td align="center">2.4</td><td sorttable_customkey="19971011000000">11-10-97</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF DOMAIN A OF THERMUS FLAVUS 5S RRNA AND THE CONTRIBUTION OF WATER MOLECULES TO ITS STRUCTURE</td><td>C.BETZEL,S.LORENZ,J.P.FURSTE,R.BALD,M.ZHANG,T.R.SCHNEIDER, K.S.WILSON,V.A.ERDMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KBP" title="Link to PDB">2KBP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KBP/index.html" title="Link to FR3D analysis. 0.4083 pairs per nucleotide">2KBP</a></td><td align="center">AB</td><td align="center">240</td><td align="center">98</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20101203000000">3-24-09</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A G-QUADRUPLEX OF HUMAN TELOMERIC RNA</td><td>A.T.PHAN,H.MARTADINATA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K7E" title="Link to PDB">2K7E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K7E/index.html" title="Link to FR3D analysis. 0.4455 pairs per nucleotide">2K7E</a></td><td align="center">AB</td><td align="center">220</td><td align="center">98</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080710000000">10-07-08</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF THE HUMAN TRNALYS3 BOUND TO THE HIV GENOME LOOP I</td><td>Y.BILBILLE,F.P.A.VENDEIX,R.GUENTHER,A.MALKIEWICZ,X.ARIZA, J.VILARRASA,P.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FTM" title="Link to PDB">3FTM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FTM/index.html" title="Link to FR3D analysis. 0.3913 pairs per nucleotide">3FTM</a></td><td align="center">AB</td><td align="center">46</td><td align="center">18</td><td align="center">2.7</td><td sorttable_customkey="20101202000000">2-24-09</td><td>synthetic,synthetic</td><td>CLASS II LIGASE RIBOZYME PRODUCT-TEMPLATE DUPLEX, STRUCTURE 1</td><td>J.N.PITT,A.R.FERRE-D'AMARE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=280D" title="Link to PDB">280D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/280D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">280D</a></td><td align="center">AB</td><td align="center">48</td><td align="center">24</td><td align="center">2.4</td><td sorttable_customkey="19980609000000">9-30-96</td><td>engineered</td><td>THE STRUCTURE OF AN RNA DODECAMER SHOWS HOW TANDEM U-U BASE PAIRS INCREASE THE RANGE OF STABLE RNA STRUCTURES AND THE DIVERSITY OF RECOGNITION SITES</td><td>S.E.LIETZKE,C.L.BARNES,C.E.KUNDROT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/205D/index.html" title="Resolution 2.6A, 24 nucleotides, 12 basepairs, 0.5000 pairs per nucleotide">205D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/280D/280D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AJT" title="Link to PDB">1AJT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AJT/index.html" title="Link to FR3D analysis. 0.2105 pairs per nucleotide">1AJT</a></td><td align="center">AB</td><td align="center">19</td><td align="center">4</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980808000000">8-20-97</td><td>synthetic,synthetic</td><td>FIVE-NUCLEOTIDE BULGE LOOP FROM TETRAHYMENA THERMOPHILA GROUP I INTRON, NMR, 1 STRUCTURE</td><td>K.J.LUEBKE,S.M.LANDRY,I.TINOCO JUNIOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RNG" title="Link to PDB">1RNG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RNG/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1RNG</a></td><td align="center">A</td><td align="center">60</td><td align="center">24</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19950712000000">12-07-95</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE CUUG HAIRPIN: A NOVEL RNA TETRALOOP MOTIF</td><td>F.M.JUCKER,A.PARDI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=422D" title="Link to PDB">422D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/422D/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">422D</a></td><td align="center">AB</td><td align="center">24</td><td align="center">8</td><td align="center">2.6</td><td sorttable_customkey="20030306000000">6-03-03</td><td>engineered</td><td>'-R(*GP*AP*UP*CP*AP*CP*UP*UP*CP*GP*GP*U)-3'</td><td>S.ESWARAMOORTHY,S.T.RAO,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1B7F" title="Link to PDB">1B7F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1B7F/index.html" title="Link to FR3D analysis. 0.0833 pairs per nucleotide">1B7F</a></td><td align="center">PQ</td><td align="center">24</td><td align="center">2</td><td align="center">2.6</td><td sorttable_customkey="19990305000000">5-03-99</td><td>synthetic</td><td>SXL-LETHAL PROTEIN/RNA COMPLEX</td><td>N.HANDA,O.NUREKI,K.KURIMOTO,I.KIM,H.SAKAMOTO,Y.SHIMURA, Y.MUTO,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QBP" title="Link to PDB">1QBP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QBP/index.html" title="Link to FR3D analysis. 0.4028 pairs per nucleotide">1QBP</a></td><td align="center">AB</td><td align="center">72</td><td align="center">29</td><td align="center">2.1</td><td sorttable_customkey="20010211000000">11-26-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A BROMINATED RNA HELIX WITH FOUR MISMATCHED BASE PAIRS</td><td>A.C.ANDERSON,R.H.O'NEIL,D.J.FILMAN,C.A.FREDERICK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PY9" title="Link to PDB">2PY9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PY9/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2PY9</a></td><td align="center">E</td><td align="center">24</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20071206000000">6-12-07</td><td>synthetic</td><td>PROTEIN-RNA INTERACTION INVOLVING KH1 DOMAIN FROM HUMAN POLY(C)-BINDING PROTEIN-2</td><td>T.L.JAMES,Z.DU,J.K.LEE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2F87" title="Link to PDB">2F87</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2F87/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">2F87</a></td><td align="center">A</td><td align="center">156</td><td align="center">52</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060808000000">8-08-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A GAAG TETRALOOP IN SRP RNA FROM PYROCOCCUS FURIOSUS</td><td>K.OKADA,M.TAKAHASHI,T.SAKAMOTO,K.NAKAMURA,A.KANAI,G KAWAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZBI" title="Link to PDB">1ZBI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZBI/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1ZBI</a></td><td align="center">C</td><td align="center">12</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20051207000000">7-12-05</td><td>synthetic</td><td>BACILLUS HALODURANS RNASE H CATALYTIC DOMAIN MUTANT D132N IN COMPLEX WITH 12-MER RNA/DNA HYBRID</td><td>M.NOWOTNY,S.A.GAIDAMAKOV,R.J.CROUCH,W.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KD5" title="Link to PDB">1KD5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KD5/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">1KD5</a></td><td align="center">AB</td><td align="center">22</td><td align="center">10</td><td align="center">1.6</td><td sorttable_customkey="20030403000000">3-04-03</td><td>synthetic</td><td>THE CRYSTAL STRUCTURE OF R(GGUCACAGCCC)2 METAL FREE FORM</td><td>V.KACER,S.A.SCARINGE,J.N.SCARSDALE,J.P.RIFE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KD4/index.html" title="Resolution 1.9A, 22 nucleotides, 10 basepairs, 0.4545 pairs per nucleotide">1KD4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KD3/index.html" title="Resolution 1.8A, 22 nucleotides, 10 basepairs, 0.4545 pairs per nucleotide">1KD3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KD5/1KD5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2NUG" title="Link to PDB">2NUG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NUG/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">2NUG</a></td><td align="center">CDEF</td><td align="center">44</td><td align="center">20</td><td align="center">1.7</td><td sorttable_customkey="20080811000000">11-20-07</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF RNASE III FROM AQUIFEX AEOLICUS COMPLEXED WITH DS-RNA AT 1.7-ANGSTROM RESOLUTION</td><td>J.H.GAN,G.SHAW,J.E.TROPEA,D.S.WAUGH,D.L.COURT,X.JI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QCU" title="Link to PDB">1QCU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QCU/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1QCU</a></td><td align="center">AB</td><td align="center">44</td><td align="center">22</td><td align="center">1.2</td><td sorttable_customkey="19991111000000">11-11-99</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF AN 18 BASE PAIR COPY CONTROL RELATED RNA DUPLEX</td><td>P.S.KLOSTERMAN,S.A.SHAH,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HO1" title="Link to PDB">3HO1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HO1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HO1</a></td><td align="center">Y</td><td align="center">11</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20090610000000">10-06-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF T. THERMOPHILUS ARGONAUTE N546 MUTANT PROTEIN COMPLEXED WITH DNA GUIDE STRAND AND 12-NT RNA TARGET STRAND</td><td>Y.WANG,H.LI,G.SHENG,D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HVR/index.html" title="Resolution 3.2A, 21 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HVR</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HO1/3HO1_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3LRR" title="Link to PDB">3LRR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LRR/index.html" title="Link to FR3D analysis. 0.4091 pairs per nucleotide">3LRR</a></td><td align="center">CD</td><td align="center">22</td><td align="center">9</td><td align="center">2.1</td><td sorttable_customkey="20100206000000">6-02-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN RIG-I CTD BOUND TO A 12 BP AU RICH 5' PPP DSRNA</td><td>P.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HOU" title="Link to PDB">3HOU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HOU</a></td><td align="center">3</td><td align="center">22</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20110407000000">7-28-09</td><td>synthetic</td><td>COMPLETE RNA POLYMERASE II ELONGATION COMPLEX I WITH A T-U MISMATCH</td><td>J.F.SYDOW,F.BRUECKNER,A.C.M.CHEUNG,G.E.DAMSMA,S.DENGL, E.LEHMANN,D.VASSYLYEV,P.CRAMER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOW/index.html" title="Resolution 3.6A, 11 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HOW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOZ/index.html" title="Resolution 3.6A, 11 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HOZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOU/3HOU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BMV" title="Link to PDB">1BMV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BMV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1BMV</a></td><td align="center">M</td><td align="center">11</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="19890910000000">10-09-89</td><td>BEAN POD MOTTLE VIRUS</td><td>PROTEIN-RNA INTERACTIONS IN AN ICOSAHEDRAL VIRUS AT 3.0 ANGSTROMS RESOLUTION</td><td>Z.CHEN,C.STAUFFACHER,Y.LI,T.SCHMIDT,W.BOMU,G.KAMER,M.SHANKS, G.LOMONOSSOFF,J.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I3Y" title="Link to PDB">1I3Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I3Y/index.html" title="Link to FR3D analysis. 0.3862 pairs per nucleotide">1I3Y</a></td><td align="center">A</td><td align="center">378</td><td align="center">146</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010404000000">4-04-01</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE A LOOP OF 23S RIBOSOMAL RNA.</td><td>J.D.PUGLISI,S.C.BLANCHARD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NCU" title="Link to PDB">3NCU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NCU/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">3NCU</a></td><td align="center">CD</td><td align="center">22</td><td align="center">10</td><td align="center">2.5</td><td sorttable_customkey="20120606000000">6-30-10</td><td>synthetic</td><td>STRUCTURAL AND FUNCTIONAL INSIGHTS INTO PATTERN RECOGNITION BY THE INNATE IMMUNE RECEPTOR RIG-I</td><td>G.SHENG,H.LI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KXZ/index.html" title="SOLUTION NMR, 154 nucleotides, 70 basepairs, 0.4545 pairs per nucleotide">2KXZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KY0/index.html" title="SOLUTION NMR, 220 nucleotides, 100 basepairs, 0.4545 pairs per nucleotide">2KY0</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KY1/index.html" title="SOLUTION NMR, 220 nucleotides, 100 basepairs, 0.4545 pairs per nucleotide">2KY1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NCU/3NCU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LNT" title="Link to PDB">1LNT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LNT/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1LNT</a></td><td align="center">AB</td><td align="center">22</td><td align="center">11</td><td align="center">1.7</td><td sorttable_customkey="20030306000000">6-03-03</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE HIGHLY CONSERVED RNA INTERNAL LOOP OF SRP</td><td>J.DENG,Y.XIONG,B.PAN,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FS0" title="Link to PDB">3FS0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FS0/index.html" title="Link to FR3D analysis. 0.3810 pairs per nucleotide">3FS0</a></td><td align="center">AB</td><td align="center">21</td><td align="center">8</td><td align="center">2.3</td><td sorttable_customkey="20101202000000">2-24-09</td><td>synthetic,synthetic</td><td>CLASS II LIGASE RIBOZYME PRODUCT-TEMPLATE DUPLEX, STRUCTURE 2</td><td>J.N.PITT,A.R.FERRE-D'AMARE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3M3Y" title="Link to PDB">3M3Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3M3Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3M3Y</a></td><td align="center">R</td><td align="center">11</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20101205000000">5-12-10</td><td>synthetic</td><td>RNA POLYMERASE II ELONGATION COMPLEX C</td><td>D.WANG,G.ZHU,X.HUANG,S.J.LIPPARD</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTM/index.html" title="Resolution 3.8A, 11 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3GTM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GTQ/index.html" title="Resolution 3.8A, 11 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3GTQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3M3Y/3M3Y_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Q1O" title="Link to PDB">2Q1O</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q1O/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">2Q1O</a></td><td align="center">AB</td><td align="center">44</td><td align="center">20</td><td align="center">1.1</td><td sorttable_customkey="20090610000000">10-30-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE ANALYSIS OF THE RNA DODECAMER CGC-NF2- AAUUGGCG, WITH AN INCORPORATED 2,4-DIFLUOROTOLUYL RESIDUE (NF2)</td><td>F.LI,P.S.PALLAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G92/index.html" title="Resolution 1.6A, 22 nucleotides, 10 basepairs, 0.4545 pairs per nucleotide">2G92</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q1O/2Q1O_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1TUT" title="Link to PDB">1TUT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1TUT/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1TUT</a></td><td align="center">AB</td><td align="center">22</td><td align="center">11</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060412000000">12-28-04</td><td>synthetic,synthetic</td><td>J4/5 LOOP FROM THE CANDIDA ALBICANS AND CANDIDA DUBLINIENSIS GROUP I INTRONS</td><td>B.M.ZNOSKO,S.D.KENNEDY,P.C.WILLE,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YZ9" title="Link to PDB">1YZ9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YZ9/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">1YZ9</a></td><td align="center">CDEF</td><td align="center">44</td><td align="center">20</td><td align="center">2.1</td><td sorttable_customkey="20061011000000">11-22-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNASE III MUTANT E110Q FROM AQUIFEX AEOLICUS COMPLEXED WITH DOUBLE STRANDED RNA AT 2.1- ANGSTROM RESOLUTION</td><td>J.GAN,J.E.TROPEA,B.P.AUSTIN,D.L.COURT,D.S.WAUGH,X.JI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WTS" title="Link to PDB">1WTS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WTS/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">1WTS</a></td><td align="center">A</td><td align="center">11</td><td align="center">5</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010301000000">1-27-99</td><td>synthetic</td><td>HELIX 45 (16S RRNA) FROM B. STEAROTHERMOPHILUS, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>J.P.RIFE,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IBK" title="Link to PDB">3IBK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IBK/index.html" title="Link to FR3D analysis. 0.5455 pairs per nucleotide">3IBK</a></td><td align="center">AB</td><td align="center">22</td><td align="center">12</td><td align="center">2.2</td><td sorttable_customkey="20101205000000">5-12-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A TELOMERIC RNA QUADRUPLEX</td><td>G.W.COLLIE,S.NEIDLE,G.N.PARKINSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IEV" title="Link to PDB">3IEV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IEV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3IEV</a></td><td align="center">D</td><td align="center">11</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20110108000000">8-25-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ERA IN COMPLEX WITH MGGNP AND THE 3' END OF 16S RRNA</td><td>C.TU,X.JI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AWQ" title="Link to PDB">2AWQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AWQ/index.html" title="Link to FR3D analysis. 0.2375 pairs per nucleotide">2AWQ</a></td><td align="center">A</td><td align="center">160</td><td align="center">38</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070501000000">1-17-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF PSEUDOURIDINE-32 MODIFIED ANTICODON STEM-LOOP OF E. COLI TRNAPHE</td><td>J.CABELLO-VILLEGAS,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRW" title="Link to PDB">2KRW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRW/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">2KRW</a></td><td align="center">A</td><td align="center">176</td><td align="center">55</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE E COLI TRNA-ARG1 (ACG) CONTAINING THE 2- THIOCYTIDINE MODIFICATION IN POSITION 32</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ASB" title="Link to PDB">2ASB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ASB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2ASB</a></td><td align="center">B</td><td align="center">11</td><td align="center">0</td><td align="center">1.5</td><td sorttable_customkey="20051110000000">10-11-05</td><td>synthetic</td><td>STRUCTURE OF A MYCOBACTERIUM TUBERCULOSIS NUSA-RNA COMPLEX</td><td>B.BEUTH,S.PENNELL,K.B.ARNVIG,S.R.MARTIN,I.A.TAYLOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1K1G" title="Link to PDB">1K1G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1K1G/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1K1G</a></td><td align="center">B</td><td align="center">110</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010711000000">11-07-01</td><td>HOMO SAPIENS</td><td>STRUCTURAL BASIS FOR RECOGNITION OF THE INTRON BRANCH SITE RNA BY SPLICING FACTOR 1</td><td>Z.LIU,I.LUYTEN,M.J.BOTTOMLEY,A.C.MESSIAS,S.HOUNGNINOU- MOLANGO,R.SPRANGERS,K.ZANIER,A.KRAMER,M.SATTLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JXS" title="Link to PDB">2JXS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JXS/index.html" title="Link to FR3D analysis. 0.4762 pairs per nucleotide">2JXS</a></td><td align="center">AB</td><td align="center">210</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090805000000">5-20-08</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF RNA DUPLEX CONTAINING SINGLE ADENOSINE BULGE</td><td>L.POPENDA,R.W.ADAMIAK,Z.GDANIEC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AEV" title="Link to PDB">3AEV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AEV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3AEV</a></td><td align="center">C</td><td align="center">11</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20110904000000">4-21-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A/EIF2ALPHA-ADIM2P-RRNA COMPLEX FROM PYROCOCCUS HORIKOSHII OT3</td><td>M.TANOKURA,M.Z.JIA,K.NAGATA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LMV" title="Link to PDB">1LMV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LMV/index.html" title="Link to FR3D analysis. 0.4526 pairs per nucleotide">1LMV</a></td><td align="center">AB</td><td align="center">190</td><td align="center">86</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040311000000">11-27-02</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>SOLUTION STRUCTURE OF THE UNMODIFIED U2 SNRNA-INTRON BRANCH SITE HELIX FROM S. CEREVISIAE</td><td>M.I.NEWBY,N.L.GREENBAUM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JSG" title="Link to PDB">2JSG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JSG/index.html" title="Link to FR3D analysis. 0.3352 pairs per nucleotide">2JSG</a></td><td align="center">A</td><td align="center">176</td><td align="center">59</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070708000000">8-07-07</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF E.COLI TRNA-VAL3 WITH 1 MODIFICATION (M6A37)</td><td>F.A.P.VENDEIX,A.DZIERGOWSKA,E.M.GUSTILO,W.D.GRAHAM,B.SPROAT, A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NNH" title="Link to PDB">3NNH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NNH/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3NNH</a></td><td align="center">EF</td><td align="center">20</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20120310000000">10-27-10</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF THE CUGBP1 RRM1 WITH GUUGUUUUGUUU RNA</td><td>M.TEPLOVA,J.SONG,H.GAW,A TEPLOV,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FEQ" title="Link to PDB">1FEQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FEQ/index.html" title="Link to FR3D analysis. 0.3352 pairs per nucleotide">1FEQ</a></td><td align="center">A</td><td align="center">176</td><td align="center">59</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20011011000000">11-22-00</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF TRNA(LYS3) WITH T6A MODIFICATION AT POSITION 37</td><td>J.W.STUART,Z.GDANIEC,R.H.GUENTHER,M.MARSZALEK,E.SOCHACKA, A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HGA" title="Link to PDB">3HGA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HGA/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">3HGA</a></td><td align="center">X</td><td align="center">10</td><td align="center">4</td><td align="center">1.3</td><td sorttable_customkey="20100608000000">8-18-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF 4-SE-URIDINE DERIVATIZED RNA</td><td>JIA SHENG,SOARES ALEXEI,ZHEN HUANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RRR" title="Link to PDB">1RRR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RRR/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1RRR</a></td><td align="center">AB</td><td align="center">200</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19991004000000">4-22-98</td><td>synthetic,synthetic</td><td>RNA DUPLEX CONTAINING A PURINE-RICH STRAND, NMR, 10 STRUCTURES</td><td>J.I.GYI,A.N.LANE,G.L.CONN,T.BROWN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1A34/index.html" title="Resolution 1.8A, 20 nucleotides, 9 basepairs, 0.4500 pairs per nucleotide">1A34</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E2I/index.html" title="Resolution 3.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2E2I</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/404D/index.html" title="Resolution 2.5A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">404D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NVQ/index.html" title="Resolution 2.9A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2NVQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JA8/index.html" title="Resolution 3.8A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JA8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E2H/index.html" title="Resolution 4.0A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2E2H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OO7/index.html" title="SOLUTION NMR, 40 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1OO7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2YU9/index.html" title="Resolution 3.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2YU9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JB8/index.html" title="Resolution 2.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1JB8</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PJG/index.html" title="Resolution 1.1A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1PJG</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOY/index.html" title="Resolution 3.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HOY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EFO/index.html" title="Resolution 2.3A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1EFO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JA7/index.html" title="Resolution 3.8A, 20 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JA7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VUM/index.html" title="Resolution 3.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2VUM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOV/index.html" title="Resolution 3.5A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HOV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NVT/index.html" title="Resolution 3.4A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2NVT</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7Z/index.html" title="Resolution 3.8A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2R7Z</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I4N/index.html" title="Resolution 3.9A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3I4N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R9S/index.html" title="Resolution 4.2A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1R9S</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SFO/index.html" title="Resolution 3.6A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1SFO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PJO/index.html" title="Resolution 1.1A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1PJO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y1W/index.html" title="Resolution 4.0A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1Y1W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JA6/index.html" title="Resolution 4.0A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JA6</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HOX/index.html" title="Resolution 3.6A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3HOX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RRD/index.html" title="SOLUTION NMR, 100 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1RRD</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G4Q/index.html" title="Resolution 1.1A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1G4Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NVZ/index.html" title="Resolution 4.3A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2NVZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I4M/index.html" title="Resolution 3.7A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3I4M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R9T/index.html" title="Resolution 3.5A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1R9T</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2NVX/index.html" title="Resolution 3.6A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2NVX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3M4O/index.html" title="Resolution 3.6A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3M4O</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1Y77/index.html" title="Resolution 4.5A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1Y77</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JA5/index.html" title="Resolution 3.8A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JA5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RRR/1RRR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MSW" title="Link to PDB">1MSW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MSW/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1MSW</a></td><td align="center">R</td><td align="center">10</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20030311000000">11-15-02</td><td>synthetic</td><td>STRUCTURAL BASIS FOR THE TRANSITION FROM INITIATION TO ELONGATION TRANSCRIPTION IN T7 RNA POLYMERASE</td><td>Y.W.YIN,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XV0" title="Link to PDB">1XV0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XV0/index.html" title="Link to FR3D analysis. 0.4418 pairs per nucleotide">1XV0</a></td><td align="center">AB</td><td align="center">627</td><td align="center">277</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040911000000">11-09-04</td><td>synthetic,synthetic</td><td>SOLUTION NMR STRUCTURE OF RNA INTERNAL LOOP WITH THREE CONSECUTIVE SHEARED GA PAIRS IN 5'GGUGGAGGCU/3'PCCGAAGCCG</td><td>G.CHEN,B.M.ZNOSKO,S.D.KENNEDY,T.R.KRUGH,D.H.TURNER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DD2/index.html" title="SOLUTION NMR, 513 nucleotides, 217 basepairs, 0.4230 pairs per nucleotide">2DD2</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DD1/index.html" title="SOLUTION NMR, 456 nucleotides, 192 basepairs, 0.4211 pairs per nucleotide">2DD1</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DD3/index.html" title="SOLUTION NMR, 342 nucleotides, 137 basepairs, 0.4006 pairs per nucleotide">2DD3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XV0/1XV0_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DI2" title="Link to PDB">1DI2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DI2/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1DI2</a></td><td align="center">CD</td><td align="center">20</td><td align="center">10</td><td align="center">1.9</td><td sorttable_customkey="19990212000000">12-02-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A DSRNA-BINDING DOMAIN COMPLEXED WITH DSRNA: MOLECULAR BASIS OF DOUBLE-STRANDED RNA-PROTEIN INTERACTIONS</td><td>J.M.RYTER,S.C.SCHULTZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RC7/index.html" title="Resolution 2.1A, 40 nucleotides, 20 basepairs, 0.5000 pairs per nucleotide">1RC7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADL/index.html" title="Resolution 2.2A, 20 nucleotides, 9 basepairs, 0.4500 pairs per nucleotide">3ADL</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FIX/index.html" title="Resolution 2.3A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1FIX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DI2/1DI2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3R1D" title="Link to PDB">3R1D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R1D/index.html" title="Link to FR3D analysis. 0.3600 pairs per nucleotide">3R1D</a></td><td align="center">AB</td><td align="center">50</td><td align="center">18</td><td align="center">1.4</td><td sorttable_customkey="20110106000000">6-01-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GC(8BRG)GCGGCGGC DUPLEX</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K49" title="Link to PDB">3K49</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K49/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K49</a></td><td align="center">B</td><td align="center">30</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20110310000000">10-27-09</td><td>synthetic</td><td>PUF3 RNA BINDING DOMAIN BOUND TO COX17 RNA 3' UTR RECOGNITION SEQUENCE SITE B</td><td>D.ZHU,C.R.STUMPF,J.M.KRAHN,M.WICKENS,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K4E/index.html" title="Resolution 3.2A, 30 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3K4E</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K49/3K49_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1F5G" title="Link to PDB">1F5G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F5G/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1F5G</a></td><td align="center">AB</td><td align="center">520</td><td align="center">260</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20020206000000">6-26-00</td><td>synthetic</td><td>THE G7(SYN)-G4(ANTI) STRUCTURE OF R(GCAGGCGUGC)2</td><td>M.E.BURKARD,D.H.TURNER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F5H/index.html" title="SOLUTION NMR, 520 nucleotides, 256 basepairs, 0.4923 pairs per nucleotide">1F5H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1F5G/1F5G_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JXQ" title="Link to PDB">2JXQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JXQ/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2JXQ</a></td><td align="center">AB</td><td align="center">200</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090805000000">5-20-08</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF RNA DUPLEX</td><td>L.POPENDA,R.W.ADAMIAK,Z.GDANIEC</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BSX" title="Link to PDB">3BSX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSX/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BSX</a></td><td align="center">C</td><td align="center">19</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20080804000000">4-08-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF HUMAN PUMILIO 1 IN COMPLEX WITH PUF5 RNA</td><td>Y.K.GUPTA,D.T.NAIR,R.P.WHARTON,A.K.AGGARWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M8Y" title="Link to PDB">1M8Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1M8Y</a></td><td align="center">C</td><td align="center">20</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20040109000000">9-25-02</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO1 IN COMPLEX WITH NRE2-10 RNA</td><td>X.WANG,J.MCLACHLAN,P.D.ZAMORE,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2YJY/index.html" title="Resolution 2.6A, 20 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2YJY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8Y/1M8Y_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2C0B" title="Link to PDB">2C0B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C0B/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2C0B</a></td><td align="center">R</td><td align="center">10</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20060210000000">10-14-05</td><td>synthetic</td><td>CATALYTIC DOMAIN OF E. COLI RNASE E IN COMPLEX WITH 13-MER RNA</td><td>M.J.MARCAIDA,A.J.CALLAGHAN,W.G.SCOTT,B.F.LUISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DQH" title="Link to PDB">1DQH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DQH/index.html" title="Link to FR3D analysis. 0.4737 pairs per nucleotide">1DQH</a></td><td align="center">AB</td><td align="center">19</td><td align="center">9</td><td align="center">1.7</td><td sorttable_customkey="20010111000000">11-13-00</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF HELIX II OF THE X. LAEVIS SOMATIC 5S RRNA WITH A CYTOSINE BULGE IN TWO CONFORMATIONS</td><td>Y.XIONG,M.SUNDARALINGAM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DQF/index.html" title="Resolution 2.2A, 19 nucleotides, 9 basepairs, 0.4737 pairs per nucleotide">1DQF</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DQH/1DQH_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1IDV" title="Link to PDB">1IDV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1IDV/index.html" title="Link to FR3D analysis. 0.2400 pairs per nucleotide">1IDV</a></td><td align="center">A</td><td align="center">100</td><td align="center">24</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010510000000">10-05-01</td><td>synthetic</td><td>NMR STRUCTURE OF HCV IRES RNA DOMAIN IIIC</td><td>K.KALUARACHCHI,R.RIJNBRAND,S.M.LEMON,D.G.GORENSTEIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G1G" title="Link to PDB">2G1G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G1G/index.html" title="Link to FR3D analysis. 0.3125 pairs per nucleotide">2G1G</a></td><td align="center">A</td><td align="center">320</td><td align="center">100</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070806000000">6-20-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE ANTICODON LOOP OF S. POMBE TRNAI INCLUDING THE NATURALLY OCCURRING N6-THREONYL ADENINE</td><td>E.LESCRINIER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=219D" title="Link to PDB">219D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/219D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">219D</a></td><td align="center">B</td><td align="center">10</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960310000000">10-15-95</td><td>synthetic</td><td>DNA/RNA HYBRID DUPLEX (5'-D(*GP*CP*TP*AP*TP*AP*APS*TP*GP*G)- 3')(DOT) (5'-R(*CP*CP*AP*UP*UP*AP*UP*AP*GP*C)-3') WITH A PHOSPHOROTHIOATE MOIETY</td><td>C.GONZALEZ,T.L.JAMES</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KP4/index.html" title="SOLUTION NMR, 100 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2KP4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KP3/index.html" title="SOLUTION NMR, 100 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2KP3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/219D/219D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ADI" title="Link to PDB">3ADI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ADI/index.html" title="Link to FR3D analysis. 0.4500 pairs per nucleotide">3ADI</a></td><td align="center">DE</td><td align="center">20</td><td align="center">9</td><td align="center">3.2</td><td sorttable_customkey="20120205000000">5-26-10</td><td>synthetic,synthetic</td><td>STRUCTURE OF ARABIDOPSIS HYL1 AND ITS MOLECULAR IMPLICATIONS FOR MIRNA PROCESSING</td><td>Y.A.YUAN,H.Y.CHEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BBV" title="Link to PDB">2BBV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BBV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2BBV</a></td><td align="center">N</td><td align="center">10</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="19960708000000">8-31-94</td><td>BLACK BEETLE VIRUS</td><td>THE REFINED THREE-DIMENSIONAL STRUCTURE OF AN INSECT VIRUS AT 2.8 ANGSTROMS RESOLUTION</td><td>J.-P.WERY,V.S.REDDY,M.V.HOSUR,J.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R92" title="Link to PDB">2R92</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R92/index.html" title="Link to FR3D analysis. 0.4211 pairs per nucleotide">2R92</a></td><td align="center">PT</td><td align="center">19</td><td align="center">8</td><td align="center">3.8</td><td sorttable_customkey="20090311000000">11-27-07</td><td>synthetic,synthetic</td><td>ELONGATION COMPLEX OF RNA POLYMERASE II WITH ARTIFICIAL RDRP SCAFFOLD</td><td>E.LEHMANN,F.BRUECKNER,P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R4H" title="Link to PDB">1R4H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R4H/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1R4H</a></td><td align="center">A</td><td align="center">10</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050710000000">10-19-04</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE IIIC DOMAIN OF GB VIRUS B IRES ELEMENT</td><td>K.KALUARACHCHI,V.THIVIYANATHAN,R.RIJINBRAND,S.M.LEMON, D.G.GORENSTEIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NJ7" title="Link to PDB">3NJ7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NJ7/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">3NJ7</a></td><td align="center">AB</td><td align="center">60</td><td align="center">24</td><td align="center">1.9</td><td sorttable_customkey="20120108000000">8-25-10</td><td>synthetic</td><td>.9 A RESOLUTION X-RAY STRUCTURE OF (GGCAGCAGCC)2</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BJ6" title="Link to PDB">2BJ6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BJ6/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2BJ6</a></td><td align="center">G</td><td align="center">40</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20050903000000">3-09-05</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF A DECAMERIC HNA-RNA HYBRID</td><td>T.MAIER,I.PRZYLAS,N.STRAETER,P.HERDEWIJN,W.SAENGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LPW" title="Link to PDB">1LPW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LPW/index.html" title="Link to FR3D analysis. 0.4691 pairs per nucleotide">1LPW</a></td><td align="center">AB</td><td align="center">162</td><td align="center">76</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040311000000">11-27-02</td><td>SACCHAROMYCES CEREVISIAE,SACCHAROMYCES CEREVISIAE</td><td>SOLUTION STRUCTURE OF THE YEAST SPLICEOSOMAL U2 SNRNA- INTRON BRANCH SITE HELIX FEATURING A CONSERVED PSEUDOURIDINE</td><td>M.I.NEWBY,N.L.GREENBAUM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DRR" title="Link to PDB">1DRR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DRR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1DRR</a></td><td align="center">B</td><td align="center">100</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19991004000000">4-22-98</td><td>synthetic</td><td>DNA/RNA HYBRID DUPLEX CONTAINING A PURINE-RICH DNA STRAND, NMR, 10 STRUCTURES</td><td>J.I.GYI,A.N.LANE,G.L.CONN,T.BROWN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FNX" title="Link to PDB">1FNX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FNX/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1FNX</a></td><td align="center">R</td><td align="center">210</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20041206000000">6-24-03</td><td>MUS MUSCULUS</td><td>SOLUTION STRUCTURE OF THE HUC RBD1-RBD2 COMPLEXED WITH THE AU-RICH ELEMENT</td><td>M.INOUE,M.HIRAO,K.KASASHIMA,I.-S.KIM,G.KAWAI,T.KIGAWA, H.SAKAMOTO,Y.MUTO,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K5Z" title="Link to PDB">3K5Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K5Z/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K5Z</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/GLD-1 FBEA G4A MUTANT COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BSO" title="Link to PDB">3BSO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSO/index.html" title="Link to FR3D analysis. 0.4375 pairs per nucleotide">3BSO</a></td><td align="center">PT</td><td align="center">16</td><td align="center">7</td><td align="center">1.7</td><td sorttable_customkey="20080801000000">1-08-08</td><td>synthetic,synthetic</td><td>NORWALK VIRUS POLYMERASE BOUND TO CYTIDINE 5'-TRIPHOSPHATE AND PRIMER-TEMPLATE RNA</td><td>D.F.ZAMYATKIN,K.K.S.NG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3H5X/index.html" title="Resolution 1.8A, 16 nucleotides, 7 basepairs, 0.4375 pairs per nucleotide">3H5X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3H5Y/index.html" title="Resolution 1.8A, 16 nucleotides, 7 basepairs, 0.4375 pairs per nucleotide">3H5Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSO/3BSO_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=479D" title="Link to PDB">479D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/479D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">479D</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20010305000000">5-15-00</td><td>synthetic</td><td>CRYSTAL STRUCTURE A DNA:RNA HYBRID DUPLEX, A DYNAMIC MODEL FOR RNASE H RECOGNITION</td><td>Y.XIONG,M.SUNDARALINGAM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DQP/index.html" title="Resolution 2.1A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2DQP</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DQQ/index.html" title="Resolution 2.0A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2DQQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DQO/index.html" title="Resolution 2.3A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2DQO</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/479D/479D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L8U" title="Link to PDB">2L8U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L8U/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2L8U</a></td><td align="center">AB</td><td align="center">18</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20121102000000">2-23-11</td><td>synthetic</td><td>NMR SPECTROSCOPY AND MOLECULAR DYNAMICS SIMULATION OF R(CCGCUGCGG)2 REVEAL A DYNAMIC UU INTERNAL LOOP FOUND IN MYOTONIC DYSTROPHY TYPE 1 - UU PAIR WITH ONE HYDROGEN BOND PAIR</td><td>R.PARKESH,M.A.FOUNTAIN,M.D.DISNEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VNU" title="Link to PDB">2VNU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VNU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VNU</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20080804000000">4-08-08</td><td>SACCHAROMYCES CEREVISIAE</td><td>CRYSTAL STRUCTURE OF SC RRP44</td><td>E.LORENTZEN,J.BASQUIN,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IRN" title="Link to PDB">2IRN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IRN/index.html" title="Link to FR3D analysis. 0.4296 pairs per nucleotide">2IRN</a></td><td align="center">AB</td><td align="center">270</td><td align="center">116</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080802000000">2-20-07</td><td>synthetic</td><td>THE NMR STRUCTURES OF (RGCUGAGGCU)2 AND (RGCGGAUGCU)2</td><td>B.S.TOLBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K5Q" title="Link to PDB">3K5Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K5Q/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K5Q</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/FBE COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JRQ" title="Link to PDB">2JRQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JRQ/index.html" title="Link to FR3D analysis. 0.3182 pairs per nucleotide">2JRQ</a></td><td align="center">A</td><td align="center">176</td><td align="center">56</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20081207000000">7-24-07</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF E. COLI TRNA- VAL3 WITH 1 MODIFICATION (CMO5U34)</td><td>F.A.P.VENDEIX,A.DZIERGOWSKA,E.M.GUSTILO,W.D.GRAHAM,B.SPROAT, A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NTT" title="Link to PDB">1NTT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTT/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1NTT</a></td><td align="center">A</td><td align="center">9</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20031006000000">6-10-03</td><td>synthetic</td><td>'(DCPCPUPCPCPUPUP)3':(RAGGAGGAAA)5', WHERE P=PROPYNYL</td><td>B.M.ZNOSKO,T.W.BARNES III,T.R.KRUGH,D.H.TURNER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTQ/index.html" title="SOLUTION NMR, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1NTQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTS/index.html" title="SOLUTION NMR, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1NTS</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NTT/1NTT_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OKF" title="Link to PDB">1OKF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OKF/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1OKF</a></td><td align="center">B</td><td align="center">180</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030910000000">10-09-03</td><td>synthetic</td><td>NMR STRUCTURE OF AN ALPHA-L-LNA:RNA HYBRID</td><td>J.T.NIELSEN,P.C.STEIN,M.PETERSEN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HHW/index.html" title="SOLUTION NMR, 360 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1HHW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HG9/index.html" title="SOLUTION NMR, 360 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1HG9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1HHX/index.html" title="SOLUTION NMR, 306 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1HHX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H0Q/index.html" title="SOLUTION NMR, 180 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1H0Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OKF/1OKF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1S77" title="Link to PDB">1S77</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S77/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1S77</a></td><td align="center">R</td><td align="center">9</td><td align="center">0</td><td align="center">2.7</td><td sorttable_customkey="20051103000000">3-23-04</td><td>synthetic</td><td>T7 RNAP PRODUCT PYROPHOSPHATE ELONGATION COMPLEX</td><td>Y.W.YIN,T.A.STEITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S76/index.html" title="Resolution 2.9A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1S76</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S77/1S77_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RGO" title="Link to PDB">1RGO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RGO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1RGO</a></td><td align="center">D</td><td align="center">180</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040203000000">3-02-04</td><td>HOMO SAPIENS</td><td>STRUCTURAL BASIS FOR RECOGNITION OF THE MRNA CLASS II AU- RICH ELEMENT BY THE TANDEM ZINC FINGER DOMAIN OF TIS11D</td><td>B.P.HUDSON,M.A.MARTINEZ-YAMOUT,H.J.DYSON,P.E.WRIGHT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NAO" title="Link to PDB">1NAO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NAO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1NAO</a></td><td align="center">A</td><td align="center">9</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990301000000">1-27-97</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN RNA 2'-O-METHYLATED RNA DUPLEX CONTAINING AN RNA/DNA HYBRID SEGMENT AT THE CENTER, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>T.NISHIZAKI,S.IWAI,E.OHTSUKA,H.NAKAMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1C2Q" title="Link to PDB">1C2Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C2Q/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1C2Q</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010511000000">11-29-99</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A DNA.RNA HYBRID CONTAINING AN ALPHAT- ANOMERIC THYMIDINE AND POLARITY REVERSALS</td><td>M.W.GERMANN,J.M.ARAMINI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZH5" title="Link to PDB">1ZH5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZH5/index.html" title="Link to FR3D analysis. 0.2222 pairs per nucleotide">1ZH5</a></td><td align="center">CD</td><td align="center">18</td><td align="center">4</td><td align="center">1.9</td><td sorttable_customkey="20070501000000">1-17-06</td><td>synthetic</td><td>STRUCTURAL BASIS FOR RECOGNITION OF UUUOH 3'-TERMINII OF NASCENT RNA POL III TRANSCRIPTS BY LA AUTOANTIGEN</td><td>M.TEPLOVA,Y.R.YUAN,S.ILIN,L.MALININA,A.T.PHAN,A.TEPLOV, D.J.PATEL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YTY/index.html" title="Resolution 2.3A, 18 nucleotides, 4 basepairs, 0.2222 pairs per nucleotide">1YTY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZH5/1ZH5_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I6H" title="Link to PDB">1I6H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I6H/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1I6H</a></td><td align="center">R</td><td align="center">9</td><td align="center">0</td><td align="center">3.3</td><td sorttable_customkey="20021104000000">4-23-01</td><td>synthetic</td><td>RNA POLYMERASE II ELONGATION COMPLEX</td><td>A.L.GNATT,P.CRAMER,R.D.KORNBERG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1G2E" title="Link to PDB">1G2E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G2E/index.html" title="Link to FR3D analysis. 0.1111 pairs per nucleotide">1G2E</a></td><td align="center">B</td><td align="center">9</td><td align="center">1</td><td align="center">2.3</td><td sorttable_customkey="20010502000000">2-05-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUD AND AU-RICH ELEMENT OF THE TUMOR NECROSIS FACTOR ALPHA RNA</td><td>X.WANG,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KY2" title="Link to PDB">2KY2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KY2/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2KY2</a></td><td align="center">AB</td><td align="center">180</td><td align="center">80</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100206000000">6-02-10</td><td>synthetic</td><td>THE STRUCTURE OF RNA INTERNAL LOOPS WITH TANDEM AG PAIRS: 5'UAGG/3'GGAU</td><td>N.B.HAMMOND,S.D.KENNEDY,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K5Y" title="Link to PDB">3K5Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K5Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K5Y</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/GLD-1 FBEA COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QGB/index.html" title="Resolution 2.4A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3QGB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K5Y/3K5Y_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K64" title="Link to PDB">3K64</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K64/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K64</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/FEM-3 PME COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YVP" title="Link to PDB">1YVP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YVP/index.html" title="Link to FR3D analysis. 0.2381 pairs per nucleotide">1YVP</a></td><td align="center">CDG</td><td align="center">21</td><td align="center">5</td><td align="center">2.2</td><td sorttable_customkey="20050706000000">6-07-05</td><td>synthetic,synthetic</td><td>RO AUTOANTIGEN COMPLEXED WITH RNAS</td><td>A.J.STEIN,G.FUCHS,C.FU,S.L.WOLIN,K.M.REINISCH</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K61" title="Link to PDB">3K61</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K61/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K61</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/FOG-1 FBEA COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRY" title="Link to PDB">2KRY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRY/index.html" title="Link to FR3D analysis. 0.2667 pairs per nucleotide">2KRY</a></td><td align="center">A</td><td align="center">150</td><td align="center">40</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110608000000">8-18-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE HUMAN MITOCHONDRIAL TRNAMET ASL CONTAINING THE 5-FORMYLCYTIDINE MODIFICATION IN POSITION 34</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3I5X" title="Link to PDB">3I5X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I5X/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3I5X</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20090809000000">9-08-09</td><td>synthetic</td><td>STRUCTURE OF MSS116P BOUND TO SSRNA AND AMP-PNP</td><td>M.DEL CAMPO,A.M.LAMBOWITZ</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I62/index.html" title="Resolution 1.9A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3I62</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I61/index.html" title="Resolution 2.1A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3I61</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I5X/3I5X_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2H49" title="Link to PDB">2H49</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2H49/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2H49</a></td><td align="center">AB</td><td align="center">360</td><td align="center">160</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070911000000">11-21-06</td><td>synthetic,synthetic</td><td>THE NMR STRUCTURE OF AN INTERNAL LOOP FROM 23S RIBOSOMAL RNA OF DEINOCOCCUS RADIODURANS DIFFERS FROM THE STRUCTURE IN THE CRYSTAL OF THE RIBOSOMAL SUBUNIT</td><td>N.SHANKAR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GIB" title="Link to PDB">3GIB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GIB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3GIB</a></td><td align="center">H</td><td align="center">9</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20100511000000">11-17-09</td><td>synthetic</td><td>CRYSTAL STUCTURE OF THE COMPLEX OF THE E. COLI HFQ WITH POLY(A)</td><td>T.M.LINK,P.VALENTIN-HANSEN,R.G.BRENNAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AO5" title="Link to PDB">2AO5</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AO5/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">2AO5</a></td><td align="center">AB</td><td align="center">36</td><td align="center">16</td><td align="center">2.1</td><td sorttable_customkey="20080403000000">3-28-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DUPLEX R(GGCGBRUGCGCU)2 WITH TERMINAL AND INTERNAL TANDEM G-U BASE PAIRS</td><td>R.UTSUNOMIYA,K.SUTO,D.BALASUNDARESAN,A.FUKAMIZU,P.K.KUMAR, H.MIZUNO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=421D" title="Link to PDB">421D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/421D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">421D</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">1.8</td><td sorttable_customkey="19990501000000">1-05-99</td><td>synthetic</td><td>'-D(*TP*TP*CP*TP*TP*(BRO)CP*TP*TP*C)-3', 5'- R(*GP*AP*AP*GP*AP*AP*GP*AP*A)-3'</td><td>Y.XIONG,M.SUNDARALINGAM</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GUN/index.html" title="Resolution 2.8A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2GUN</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/421D/421D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2C4R" title="Link to PDB">2C4R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C4R/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2C4R</a></td><td align="center">R</td><td align="center">9</td><td align="center">0</td><td align="center">3.6</td><td sorttable_customkey="20070110000000">10-25-05</td><td>ESCHERICHIA COLI</td><td>CATALYTIC DOMAIN OF E. COLI RNASE E</td><td>M.J.MARCAIDA,A.J.CALLAGHAN,B.F.LUISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MUV" title="Link to PDB">1MUV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MUV/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1MUV</a></td><td align="center">AB</td><td align="center">18</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030612000000">12-18-02</td><td>synthetic</td><td>SHEARED A(ANTI)-A(ANTI) BASE PAIRS IN A DESTABILIZING 2X2 INTERNAL LOOP: THE NMR STRUCTURE OF 5'(RGGCAAGCCU)2</td><td>B.M.ZNOSKO,M.E.BURKARD,S.J.SCHROEDER,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3K62" title="Link to PDB">3K62</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3K62/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3K62</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20090311000000">11-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2/GLD-1 FBEB COMPLEX</td><td>Y.WANG,L.OPPERMAN,M.WICKENS,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IRO" title="Link to PDB">2IRO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IRO/index.html" title="Link to FR3D analysis. 0.3889 pairs per nucleotide">2IRO</a></td><td align="center">AB</td><td align="center">270</td><td align="center">105</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080802000000">2-20-07</td><td>synthetic</td><td>THE NMR STRUCTURES OF (RGCUGAGGCU)2 AND (RGCGGAUGCU)2</td><td>B.S.TOLBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3QGC" title="Link to PDB">3QGC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QGC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3QGC</a></td><td align="center">B</td><td align="center">9</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20121103000000">3-23-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF FBF-2 R288Y MUTANT IN COMPLEX WITH GLD-1 FBEA A7U MUTANT</td><td>Y.WANG,C.QIU,Y.Y.KOH,L.OPPERMAN,L.GROSS,T.M.T.HALL,M.WICKENS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QG9/index.html" title="Resolution 2.2A, 9 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3QG9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QGC/3QGC_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PKM" title="Link to PDB">3PKM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PKM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PKM</a></td><td align="center">G</td><td align="center">17</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20110903000000">3-09-11</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF CAS6 WITH ITS SUBSTRATE RNA</td><td>R.WANG,G.PREAMPLUME,H.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K41" title="Link to PDB">2K41</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K41/index.html" title="Link to FR3D analysis. 0.4706 pairs per nucleotide">2K41</a></td><td align="center">AB</td><td align="center">170</td><td align="center">80</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080411000000">11-04-08</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF URIDINE BULGED RNA DUPLEX</td><td>L.POPENDA,L.BIELECKI,Z.GDANIEC,R.W.ADAMIAK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KRV" title="Link to PDB">2KRV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KRV/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">2KRV</a></td><td align="center">A</td><td align="center">150</td><td align="center">50</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100112000000">12-01-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE E. COLI TRNA-ARG1 (ICG) ASL CONTAINING THE 2-THIOCYTIDINE MODIFICATION</td><td>Y.BILBILLE,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1CVJ" title="Link to PDB">1CVJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CVJ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1CVJ</a></td><td align="center">M</td><td align="center">59</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="19990410000000">10-04-99</td><td>synthetic</td><td>X-RAY CRYSTAL STRUCTURE OF THE POLY(A)-BINDING PROTEIN IN COMPLEX WITH POLYADENYLATE RNA</td><td>R.C.DEO,J.B.BONANNO,N.SONENBERG,S.K.BURLEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L8W" title="Link to PDB">2L8W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L8W/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2L8W</a></td><td align="center">AB</td><td align="center">18</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20121102000000">2-23-11</td><td>synthetic</td><td>R(CCGCUGCGG)2 UU INTERNAL LOOP FOUND IN MYOTONIC DYSTROPHY TYPE 1 - UU PAIR WITH TWO HYDROGEN BOND PAIRS</td><td>R.PARKESH,M.FOUNTAIN,M.DISNEY</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L8C/index.html" title="SOLUTION NMR, 18 nucleotides, 8 basepairs, 0.4444 pairs per nucleotide">2L8C</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L8W/2L8W_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1C4L" title="Link to PDB">1C4L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1C4L/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1C4L</a></td><td align="center">AB</td><td align="center">18</td><td align="center">9</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000908000000">8-09-00</td><td>synthetic,synthetic</td><td>SOLUTION STRUCTURE OF AN RNA DUPLEX INCLUDING A C-U BASE- PAIR</td><td>Y.TANAKA,C.KOJIMA,T.YAMAZAKI,T.S.KODAMA,K.YASUNO, S.MIYASHITA,A.M.ONO,A.S.ONO,M.KAINOSHO,Y.KYOGOKU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2B2D" title="Link to PDB">2B2D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2B2D/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">2B2D</a></td><td align="center">R</td><td align="center">12</td><td align="center">3</td><td align="center">2.9</td><td sorttable_customkey="20060905000000">5-09-06</td><td>ENTEROBACTERIO PHAGE MS2</td><td>RNA STEMLOOP OPERATOR FROM BACTERIOPHAGE QBETA COMPLEXED WITH AN N87S,E89K MUTANT MS2 CAPSID</td><td>W.T.HORN,K.TARS,E.GRAHN,C.HELGSTRAND,A.J.BARON,H.LAGO, C.J.ADAMS,D.S.PEABODY,S.E.V.PHILLIPS,N.J.STONEHOUSE, L.LILJAS,P.G.STOCKLEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2E9Z" title="Link to PDB">2E9Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E9Z/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">2E9Z</a></td><td align="center">BC</td><td align="center">16</td><td align="center">4</td><td align="center">3.0</td><td sorttable_customkey="20090206000000">6-26-07</td><td>synthetic,synthetic</td><td>FOOT-AND-MOUTH DISEASE VIRUS RNA-POLYMERASE IN COMPLEX WITH A TEMPLATE- PRIMER RNA, ATP AND UTP</td><td>C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,C.ESCARMIS,E.DOMINGO, N.VERDAGUER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2E9R" title="Link to PDB">2E9R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E9R/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2E9R</a></td><td align="center">AB</td><td align="center">15</td><td align="center">6</td><td align="center">2.8</td><td sorttable_customkey="20090206000000">6-26-07</td><td>synthetic,synthetic</td><td>FOOT-AND-MOUTH DISEASE VIRUS RNA-DEPENDENT RNA POLYMERASE IN COMPLEX WITH A TEMPLATE-PRIMER RNA AND WITH RIBAVIRIN</td><td>C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,C.ESCARMIS,E.DOMINGO, N.VERDAGUER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K3Z" title="Link to PDB">2K3Z</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K3Z/index.html" title="Link to FR3D analysis. 0.5118 pairs per nucleotide">2K3Z</a></td><td align="center">AB</td><td align="center">170</td><td align="center">87</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080411000000">11-04-08</td><td>synthetic,synthetic</td><td>NMR STRUCTURE OF ADENOSINE BULGED RNA DUPLEX WITH C:G-A TRIPLE</td><td>L.POPENDA,L.BIELECKI,Z.GDANIEC,R.W.ADAMIAK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BSB" title="Link to PDB">3BSB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BSB</a></td><td align="center">C</td><td align="center">9</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20080804000000">4-08-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF HUMAN PUMILIO1 IN COMPLEX WITH CYCLINB REVERSE RNA</td><td>Y.K.GUPTA,D.T.NAIR,R.P.WHARTON,A.K.AGGARWAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BX2" title="Link to PDB">3BX2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BX2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BX2</a></td><td align="center">C</td><td align="center">18</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20081103000000">3-11-08</td><td>SACCHAROMYCES CEREVISIAE</td><td>PUF4 RNA BINDING DOMAIN BOUND TO HO ENDONUCLEASE RNA 3' UTR RECOGNITION SEQUENCE</td><td>M.T.MILLER,J.J.HIGGIN,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2E2J" title="Link to PDB">2E2J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E2J/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2E2J</a></td><td align="center">R</td><td align="center">9</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="20061212000000">12-12-06</td><td>synthetic</td><td>RNA POLYMERASE II ELONGATION COMPLEX IN 5 MM MG+2 WITH GMPCPP</td><td>D.WANG,D.A.BUSHNELL,K.D.WESTOVER,C.D.KAPLAN,R.D.KORNBERG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HXM" title="Link to PDB">3HXM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HXM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HXM</a></td><td align="center">Y</td><td align="center">9</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20090610000000">10-06-09</td><td>synthetic</td><td>STRUCTURE OF AN ARGONAUTE COMPLEXED WITH GUIDE DNA AND TARGET RNA DUPLEX CONTAINING TWO MISMATCHES.</td><td>Y.WANG,H.LI,G.SHENG,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G91" title="Link to PDB">2G91</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G91/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">2G91</a></td><td align="center">AB</td><td align="center">15</td><td align="center">6</td><td align="center">1.5</td><td sorttable_customkey="20070304000000">4-03-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE ANALYSIS OF THE AN RNA NONAMER R(GGUGCGC) D(BRC)R(C)</td><td>K.SHI,B.PAN,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KD4" title="Link to PDB">2KD4</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KD4/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2KD4</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090704000000">4-07-09</td><td>synthetic</td><td>SOLUTION STRUCTURE AND THERMODYNAMICS OF 2',5' RNA INTERCALATION</td><td>E.D.HOROWITZ,S.LILAVIVAT,B.W.HOLLADAY,M.W.GERMANN,N.V.HUD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GQ6" title="Link to PDB">2GQ6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GQ6/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2GQ6</a></td><td align="center">KL</td><td align="center">32</td><td align="center">16</td><td align="center">1.3</td><td sorttable_customkey="20080306000000">6-27-06</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA RACEMATE</td><td>W.RYPNIEWSKI,M.VALLAZZA,M.PERBANDT,S.KLUSSMANN,C.BETZEL, V.A.ERDMANN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GQ5/index.html" title="Resolution 1.4A, 32 nucleotides, 16 basepairs, 0.5000 pairs per nucleotide">2GQ5</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G32/index.html" title="Resolution 1.3A, 32 nucleotides, 16 basepairs, 0.5000 pairs per nucleotide">2G32</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GQ7/index.html" title="Resolution 1.6A, 32 nucleotides, 16 basepairs, 0.5000 pairs per nucleotide">2GQ7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GQ4/index.html" title="Resolution 1.4A, 32 nucleotides, 16 basepairs, 0.5000 pairs per nucleotide">2GQ4</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GPM/index.html" title="Resolution 1.4A, 32 nucleotides, 16 basepairs, 0.5000 pairs per nucleotide">2GPM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GQ6/2GQ6_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q0S" title="Link to PDB">3Q0S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0S/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q0S</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO2 IN COMPLEX WITH ERK2 NRE</td><td>G.LU,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0N/index.html" title="Resolution 2.4A, 15 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3Q0N</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0S/3Q0S_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q0R" title="Link to PDB">3Q0R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0R/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q0R</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO2 IN COMPLEX WITH P38ALPHA NREB</td><td>G.LU,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0M/index.html" title="Resolution 2.7A, 16 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3Q0M</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0R/3Q0R_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MV2" title="Link to PDB">1MV2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MV2/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1MV2</a></td><td align="center">AB</td><td align="center">16</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030612000000">12-18-02</td><td>synthetic</td><td>THE TANDEM, FACE-TO-FACE AP PAIRS IN 5'(RGGCAPGCCU)2</td><td>B.M.ZNOSKO,M.E.BURKARD,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NXR" title="Link to PDB">1NXR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NXR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1NXR</a></td><td align="center">B</td><td align="center">144</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>HIV-1 POLYPURINE HYBRID, R(GAGGACUG):D(CAGTCCTC), NMR, 18 STRUCTURES</td><td>O.Y.FEDOROFF,Y.GE,B.R.REID</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2BGG" title="Link to PDB">2BGG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2BGG/index.html" title="Link to FR3D analysis. 0.3448 pairs per nucleotide">2BGG</a></td><td align="center">PQ</td><td align="center">29</td><td align="center">10</td><td align="center">2.2</td><td sorttable_customkey="20070703000000">3-31-05</td><td>synthetic,synthetic</td><td>THE STRUCTURE OF A PIWI PROTEIN FROM ARCHAEOGLOBUS FULGIDUS COMPLEXED WITH A 16NT SIRNA DUPLEX.</td><td>J.S.PARKER,S.M.ROE,D.BARFORD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KNC" title="Link to PDB">3KNC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3KNC</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20100812000000">12-08-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE CENA-RNA HYBRID OCTAMER CE(GCGTAGCG) :R(CGCUACGC)</td><td>M.OVAERE,L.VAN MEERVELT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EJZ/index.html" title="SOLUTION NMR, 8 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1EJZ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNC/3KNC_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AOI" title="Link to PDB">3AOI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AOI/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3AOI</a></td><td align="center">QSU</td><td align="center">22</td><td align="center">0</td><td align="center">4.3</td><td sorttable_customkey="20100812000000">12-08-10</td><td>synthetic</td><td>RNA POLYMERASE-GFH1 COMPLEX (CRYSTAL TYPE 2)</td><td>S.TAGAMI,S.SEKINE,T.KUMAREVEL,M.YAMAMOTO,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XB2" title="Link to PDB">2XB2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XB2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XB2</a></td><td align="center">E</td><td align="center">15</td><td align="center">0</td><td align="center">3.4</td><td sorttable_customkey="20101205000000">5-12-10</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF THE CORE MAGO-Y14-EIF4AIII-BARENTSZ- UPF3B ASSEMBLY SHOWS HOW THE EJC IS BRIDGED TO THE NMD MACHINERY</td><td>G.BUCHWALD,J.EBERT,C.BASQUIN,J.SAULIERE,U.JAYACHANDRAN, F.BONO,H.LE HIR,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q0Q" title="Link to PDB">3Q0Q</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0Q/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q0Q</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO2 IN COMPLEX WITH P38ALPHA NREA</td><td>G.LU,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0L/index.html" title="Resolution 2.5A, 16 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3Q0L</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0Q/3Q0Q_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QET" title="Link to PDB">1QET</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QET/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1QET</a></td><td align="center">AB</td><td align="center">480</td><td align="center">240</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970506000000">6-05-97</td><td>synthetic</td><td>TANDEM GU MISMATCHES IN RNA, NMR, 30 STRUCTURES</td><td>J.A.MCDOWELL,L.HE,X.CHEN,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3E3J" title="Link to PDB">3E3J</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E3J/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3E3J</a></td><td align="center">R</td><td align="center">16</td><td align="center">0</td><td align="center">6.7</td><td sorttable_customkey="20080411000000">11-04-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN INTERMEDIATE COMPLEX OF T7 RNAP AND 8NT OF RNA</td><td>K.J.DURNIAK,S.BAILEY,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MIS" title="Link to PDB">1MIS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MIS/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1MIS</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>STRUCTURE OF RNA (5'-R(GP*CP*GP*GP*AP*CP*GP*C)-3') ANTI- PARALLEL RNA DUPLEX WITH TANDEM G:A MISMATCHES, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>M.WU,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2E9T" title="Link to PDB">2E9T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2E9T/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2E9T</a></td><td align="center">BC</td><td align="center">28</td><td align="center">12</td><td align="center">2.6</td><td sorttable_customkey="20090206000000">6-26-07</td><td>synthetic,synthetic</td><td>FOOT-AND-MOUTH DISEASE VIRUS RNA-POLYMERASE RNA DEPENDENT IN COMPLEX WITH A TEMPLATE-PRIMER RNA AND 5F-UTP</td><td>C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,C.ESCARMIS,E.DOMINGO, N.VERDAGUER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=8DRH" title="Link to PDB">8DRH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/8DRH/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">8DRH</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000305000000">5-27-98</td><td>synthetic</td><td>HIGH RESOLUTION NMR STRUCTURE OF THE D(GCGTCAGG)R(CCUGACGC) HYBRID, MINIMIZED AVERAGE STRUCTURE</td><td>M.BACHELIN,G.HESSLER,G.KURZ,J.G.HACIA,P.B.DERVAN,H.KESSLER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/8PSH/index.html" title="SOLUTION NMR, 8 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">8PSH</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/8DRH/8DRH_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1JZV" title="Link to PDB">1JZV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1JZV/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1JZV</a></td><td align="center">ABCD</td><td align="center">30</td><td align="center">15</td><td align="center">1.7</td><td sorttable_customkey="20031111000000">11-11-03</td><td>synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF A BULGED RNA FROM THE SL2 STEM-LOOP OF THE HIV-1 PSI-RNA</td><td>Y.XIONG,C.SUDARSANAKUMAR,J.DENG,B.PAN,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=402D" title="Link to PDB">402D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/402D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">402D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">2.3</td><td sorttable_customkey="19990406000000">6-16-98</td><td>synthetic</td><td>'-R(*CP*GP*CP*CP*AP*GP*CP*G)-3'</td><td>S.B.JANG,L.-W.HUNG,Y.-I.CHI,E.L.HOLBROOK,R.CARTER, S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3H3V" title="Link to PDB">3H3V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3H3V/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3H3V</a></td><td align="center">P</td><td align="center">8</td><td align="center">0</td><td align="center">4.0</td><td sorttable_customkey="20100406000000">6-16-09</td><td>synthetic</td><td>YEAST RNAP II CONTAINING POLY(A)-SIGNAL SEQUENCE IN THE ACTIVE SITE</td><td>S.DENGL,P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1R3X" title="Link to PDB">1R3X</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1R3X/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1R3X</a></td><td align="center">C</td><td align="center">80</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19990805000000">5-20-98</td><td>synthetic</td><td>INTRAMOLECULAR DNA TRIPLEX WITH RNA THIRD STRAND, NMR, 10 STRUCTURES</td><td>C.H.GOTFREDSEN,P.SCHULTZE,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q0P" title="Link to PDB">3Q0P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0P/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q0P</a></td><td align="center">C</td><td align="center">16</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO1 IN COMPLEX WITH HUNCHBACK NRE</td><td>G.LU,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OJ8" title="Link to PDB">2OJ8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OJ8/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">2OJ8</a></td><td align="center">A</td><td align="center">80</td><td align="center">30</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20081005000000">5-22-07</td><td>synthetic</td><td>NMR STRUCTURE OF THE UGUU TETRALOOP OF DUCK EPSILON APICAL STEM LOOP OF THE HEPATITIS B VIRUS</td><td>F.C GIRARD,O.M.OTTINK,K.A.M.AMPT,M.TESSARI,S.S.WIJMENGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1GUC" title="Link to PDB">1GUC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1GUC/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1GUC</a></td><td align="center">AB</td><td align="center">480</td><td align="center">240</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971202000000">2-12-97</td><td>synthetic</td><td>SOLUTION NMR STRUCTURE OF AN RNA WITH TANDEM, SYMMETRIC GU MISMATCHES, 30 STRUCTURES</td><td>J.A.MCDOWELL,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2IXZ" title="Link to PDB">2IXZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2IXZ/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">2IXZ</a></td><td align="center">A</td><td align="center">96</td><td align="center">24</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060609000000">9-06-06</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE APICAL STEM-LOOP OF THE HUMAN HEPATITIS B VIRUS ENCAPSIDATION SIGNAL</td><td>S.FLODELL,M.PETERSEN,F.GIRARD,J.ZDUNEK,K.KIDD-LJUNGGREN, J.SCHLEUCHER,S.S.WIJMENGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MV1" title="Link to PDB">1MV1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MV1/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1MV1</a></td><td align="center">AB</td><td align="center">16</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030612000000">12-18-02</td><td>synthetic</td><td>THE TANDEM, SHEARED PA PAIRS IN 5'(RGGCPAGCCU)2</td><td>B.M.ZNOSKO,M.E.BURKARD,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=398D" title="Link to PDB">398D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/398D/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">398D</a></td><td align="center">AB</td><td align="center">24</td><td align="center">8</td><td align="center">1.9</td><td sorttable_customkey="19980610000000">10-06-98</td><td>synthetic</td><td>'-DNA-RNA-5' JUNCTION FORMED DURING INITIATION OF MINUS- STRAND SYNTHESIS OF HIV REPLICATION</td><td>U.MUELLER,G.MEIER,A.MOCHI-ONORI,L.CELLAI,H.HEUMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MWG" title="Link to PDB">1MWG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MWG/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1MWG</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19971203000000">3-12-97</td><td>synthetic</td><td>STRUCTURE OF RIBONUCLEIC ACID, NMR, MINIMIZED AVERAGE STRUCTURE</td><td>M.WU,J.SANTALUCIA,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=472D" title="Link to PDB">472D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/472D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">472D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">1.9</td><td sorttable_customkey="20020311000000">11-27-00</td><td>synthetic,synthetic</td><td>STRUCTURE OF AN OCTAMER RNA WITH TANDEM GG/UU MISPAIRS</td><td>J.DENG,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=124D" title="Link to PDB">124D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/124D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">124D</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19950710000000">10-31-93</td><td>synthetic</td><td>STRUCTURE OF A DNA:RNA HYBRID DUPLEX: WHY RNASE H DOES NOT CLEAVE PURE RNA</td><td>O.Y.FEDOROFF,M.SALAZAR,B.R.REID</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3QRP" title="Link to PDB">3QRP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3QRP/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3QRP</a></td><td align="center">BC</td><td align="center">14</td><td align="center">7</td><td align="center">2.4</td><td sorttable_customkey="20130105000000">5-25-11</td><td>synthetic,synthetic</td><td>STRUCTURE OF THERMUS THERMOPHILUS CSE3 BOUND TO AN RNA REPRESENTING A PRODUCT MIMIC COMPLEX</td><td>M.J.SCHELLENBERG,E.M.GESNER,E.L.GARSIDE,A.M.MACMILLAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GLP" title="Link to PDB">3GLP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GLP/index.html" title="Link to FR3D analysis. 0.3000 pairs per nucleotide">3GLP</a></td><td align="center">AB</td><td align="center">40</td><td align="center">12</td><td align="center">1.2</td><td sorttable_customkey="20110205000000">5-26-09</td><td>synthetic</td><td>.23 A RESOLUTION X-RAY STRUCTURE OF (GCUGCUGC)2</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=259D" title="Link to PDB">259D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/259D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">259D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">1.5</td><td sorttable_customkey="19980208000000">8-26-96</td><td>synthetic</td><td>RNA HYDRATION: A DETAILED LOOK</td><td>M.EGLI,S.PORTMANN,N.USMAN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RXB/index.html" title="Resolution 1.8A, 16 nucleotides, 8 basepairs, 0.5000 pairs per nucleotide">1RXB</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/259D/259D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2OJ7" title="Link to PDB">2OJ7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OJ7/index.html" title="Link to FR3D analysis. 0.2625 pairs per nucleotide">2OJ7</a></td><td align="center">A</td><td align="center">80</td><td align="center">21</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20081005000000">5-22-07</td><td>synthetic</td><td>NMR STRUCTURE OF THE UGUU TETRALOOP OF DUCK EPSILON APICAL STEM LOOP</td><td>F.C.GIRARD,O.M.OTTINK,K.A.M.AMPT,M.TESSARI,S.S.WIJMENGA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GBH" title="Link to PDB">2GBH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GBH/index.html" title="Link to FR3D analysis. 0.4667 pairs per nucleotide">2GBH</a></td><td align="center">A</td><td align="center">75</td><td align="center">35</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061104000000">4-11-06</td><td>synthetic</td><td>NMR STRUCTURE OF STEM REGION OF HELIX-35 OF 23S E.COLI RIBOSOMAL RNA (RESIDUES 736-760)</td><td>A.BAX,J.BOISBOUVIER,D.BRYCE,A.GRISHAEV,C.JARONIEC,E.MICLET, E.NIKONOVICZ,E.O'NEIL-CABELLO,J.YING</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G3S" title="Link to PDB">2G3S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G3S/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2G3S</a></td><td align="center">AB</td><td align="center">80</td><td align="center">40</td><td align="center">1.5</td><td sorttable_customkey="20080401000000">1-16-07</td><td>synthetic</td><td>RNA STRUCTURE CONTAINING GU BASE PAIRS</td><td>S.B.JANG,L.W.HUNG,M.S.JEONG,E.L.HOLBROOK,X.CHEN,D.H.TURNER, S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3I5Y" title="Link to PDB">3I5Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3I5Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3I5Y</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20090809000000">9-08-09</td><td>synthetic</td><td>STRUCTURE OF MSS116P BOUND TO SSRNA CONTAINING A SINGLE 5- BRU AND AMP-PNP</td><td>M.DEL CAMPO,A.M.LAMBOWITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MJ0" title="Link to PDB">3MJ0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MJ0/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3MJ0</a></td><td align="center">B</td><td align="center">8</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20130105000000">5-25-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF DROSOPHIA AGO-PAZ DOMAIN IN COMPLEX WITH 3'-END 2'-O-METHYLATED RNA</td><td>Y.HUANG,L.DING,J.MA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EKD" title="Link to PDB">1EKD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EKD/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1EKD</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010111000000">11-13-00</td><td>synthetic</td><td>NMR AND MOLECULAR MODELING REVEAL THAT DIFFERENT HYDROGEN BONDING PATTERNS ARE POSSIBLE FOR GU PAIRS: ONE HYDROGEN BOND FOR EACH GU PAIR IN R(GGCGUGCC)2 AND TWO FOR EACH GU PAIR IN R(GAGUGCUC)2</td><td>X.CHEN,J.A.MCDOWELL,R.KIERZEK,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BSN" title="Link to PDB">3BSN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BSN/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">3BSN</a></td><td align="center">PT</td><td align="center">16</td><td align="center">6</td><td align="center">1.8</td><td sorttable_customkey="20080801000000">1-08-08</td><td>synthetic,synthetic</td><td>NORWALK VIRUS POLYMERASE BOUND TO 5-NITROCYTIDINE TRIPHOSPHATE AND PRIMER-TEMPLATE RNA</td><td>D.F.ZAMYATKIN,K.K.S.NG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1N35" title="Link to PDB">1N35</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N35/index.html" title="Link to FR3D analysis. 0.3846 pairs per nucleotide">1N35</a></td><td align="center">BC</td><td align="center">13</td><td align="center">5</td><td align="center">2.5</td><td sorttable_customkey="20040112000000">12-25-02</td><td>synthetic,synthetic</td><td>LAMBDA3 ELONGATION COMPLEX WITH FOUR PHOSPHODIESTER BOND FORMED</td><td>Y.TAO,D.L.FARSETTA,M.L.NIBERT,S.C.HARRISON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UON/index.html" title="Resolution 7.6A, 13 nucleotides, 5 basepairs, 0.3846 pairs per nucleotide">1UON</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N35/1N35_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3F73" title="Link to PDB">3F73</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3F73/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3F73</a></td><td align="center">H</td><td align="center">16</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20090412000000">12-16-08</td><td>synthetic</td><td>ALIGNMENT OF GUIDE-TARGET SEED DUPLEX WITHIN AN ARGONAUTE SILENCING COMPLEX</td><td>Y.WANG,H.LI,G.SHENG,S.JURANEK,T.TUSCHL,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YFV" title="Link to PDB">1YFV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YFV/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1YFV</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19980310000000">10-15-97</td><td>synthetic</td><td>STRUCTURE OF (5'-R(GP*GP*CP*GP*AP*GP*CP*C)-3')2 BY 2-D NMR, 1 STRUCTURE</td><td>J.SANTALUCIA JUNIOR,D.H.TURNER</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SA9/index.html" title="Resolution 2.9A, 40 nucleotides, 16 basepairs, 0.4000 pairs per nucleotide">1SA9</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YFV/1YFV_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3BX3" title="Link to PDB">3BX3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3BX3/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3BX3</a></td><td align="center">C</td><td align="center">16</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20081103000000">3-11-08</td><td>synthetic</td><td>PUF4 T650C/C724R MUTANT BOUND TO COX17 RNA 3' UTR RECOGNITION SEQUENCE</td><td>M.T.MILLER,J.J.HIGGIN,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ZDJ" title="Link to PDB">1ZDJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ZDJ/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1ZDJ</a></td><td align="center">R</td><td align="center">16</td><td align="center">4</td><td align="center">2.9</td><td sorttable_customkey="19980807000000">7-08-98</td><td>ENTEROBACTERIO PHAGE MS2</td><td>STRUCTURE OF BACTERIOPHAGE COAT PROTEIN-LOOP RNA COMPLEX</td><td>E.GRAHN,N.STONEHOUSE,K.VALEGARD,S.VANDENWORM,L.LILJAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3R1C" title="Link to PDB">3R1C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R1C/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3R1C</a></td><td align="center">CD</td><td align="center">288</td><td align="center">144</td><td align="center">2.0</td><td sorttable_customkey="20110106000000">6-01-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GCGGCGGC DUPLEX</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=439D" title="Link to PDB">439D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/439D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">439D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">1.6</td><td sorttable_customkey="20020209000000">9-14-01</td><td>synthetic,synthetic</td><td>'-R(*CP*UP*GP*GP*GP*CP*GP*G)-3', 5'- R(*CP*CP*GP*CP*CP*UP*GP*G)-3'</td><td>M.PERBANDT,S.LORENZ,M.VALLAZZA,V.A.ERDMANN,C.BETZEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J9H" title="Link to PDB">1J9H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J9H/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1J9H</a></td><td align="center">AB</td><td align="center">32</td><td align="center">16</td><td align="center">1.4</td><td sorttable_customkey="20030210000000">10-26-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA DUPLEX WITH URIDINE BULGES</td><td>Y.XIONG,J.DENG,C.SUDARSANAKUMAR,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=438D" title="Link to PDB">438D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/438D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">438D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">2.5</td><td sorttable_customkey="19991205000000">5-12-99</td><td>synthetic</td><td>STRUCTURE OF AN RNA DUPLEX R(GGGCGCUCC)2 WITH NON-ADJACENT G.U BASE PAIRS</td><td>K.SHI,M.C.WAHL,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1BYX" title="Link to PDB">1BYX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1BYX/index.html" title="Link to FR3D analysis. 0.2542 pairs per nucleotide">1BYX</a></td><td align="center">AB</td><td align="center">240</td><td align="center">61</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20000810000000">10-20-99</td><td>synthetic</td><td>CHIMERIC HYBRID DUPLEX R(GCAGUGGC).R(GCCA)D(CTGC) COMPRISING THE TRNA-DNA JUNCTION FORMED DURING INITIATION OF HIV-1 REVERSE TRANSCRIPTION</td><td>T.SZYPERSKI,M.GOETTE,M.BILLETER,E.PEROLA,L.CELLAI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WNE" title="Link to PDB">1WNE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WNE/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">1WNE</a></td><td align="center">BC</td><td align="center">13</td><td align="center">4</td><td align="center">3.0</td><td sorttable_customkey="20060708000000">8-31-04</td><td>synthetic,synthetic</td><td>FOOT AND MOUTH DISEASE VIRUS RNA-DEPENDENT RNA POLYMERASE IN COMPLEX WITH A TEMPLATE-PRIMER RNA</td><td>C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,C.ESCARMIS,E.DOMINGO, N.VERDAGUER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M8W" title="Link to PDB">1M8W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8W/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1M8W</a></td><td align="center">C</td><td align="center">15</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20040109000000">9-25-02</td><td>synthetic,synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO1 IN COMPLEX WITH NRE1-19 RNA</td><td>X.WANG,J.MCLACHLAN,P.D.ZAMORE,T.M.T.HALL</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8X/index.html" title="Resolution 2.2A, 15 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1M8X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8W/1M8W_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1H38" title="Link to PDB">1H38</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H38/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1H38</a></td><td align="center">F</td><td align="center">32</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20030811000000">11-20-02</td><td>BACTERIOPHAGE T7</td><td>STRUCTURE OF A T7 RNA POLYMERASE ELONGATION COMPLEX AT 2.9A RESOLUTION</td><td>T.H.TAHIROV,D.TEMYAKOV,M.ANIKIN,V.PATLAN,W.T.MCALLISTER, D.G.VASSYLYEV,S.YOKOYAMA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1S0V/index.html" title="Resolution 3.2A, 32 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1S0V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H38/1H38_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1EKA" title="Link to PDB">1EKA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1EKA/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1EKA</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010111000000">11-13-00</td><td>synthetic</td><td>NMR AND MOLECULAR MODELING REVEAL THAT DIFFERENT HYDROGEN BONDING PATTERNS ARE POSSIBLE FOR GU PAIRS: ONE HYDROGEN BOND FOR EACH GU PAIR IN R(GGCGUGCC)2 AND TWO FOR EACH GU PAIR IN R(GAGUGCUC)2</td><td>X.CHEN,J.A.MCDOWELL,R.KIERZEK,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FXL" title="Link to PDB">1FXL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FXL/index.html" title="Link to FR3D analysis. 0.1250 pairs per nucleotide">1FXL</a></td><td align="center">B</td><td align="center">8</td><td align="center">1</td><td align="center">1.8</td><td sorttable_customkey="20010502000000">2-05-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUD AND AU-RICH ELEMENT OF THE C-FOS RNA</td><td>X.WANG,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2EC0" title="Link to PDB">2EC0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2EC0/index.html" title="Link to FR3D analysis. 0.4667 pairs per nucleotide">2EC0</a></td><td align="center">BC</td><td align="center">30</td><td align="center">14</td><td align="center">2.8</td><td sorttable_customkey="20090206000000">6-26-07</td><td>synthetic,synthetic</td><td>RNA-DEPENDENT RNA POLYMERASE OF FOOT-AND-MOUTH DISEASE VIRUS IN COMPLEX WITH A TEMPLATE-PRIMER RNA AND ATP</td><td>C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,C.ESCARMIS,E.DOMINGO, N.VERDAGUER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3EQT" title="Link to PDB">3EQT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EQT/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3EQT</a></td><td align="center">CD</td><td align="center">16</td><td align="center">8</td><td align="center">2.0</td><td sorttable_customkey="20091003000000">3-10-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN LGP2 C-TERMINAL DOMAIN IN COMPLEX WITH DSRNA</td><td>P.LI,X.LI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QES" title="Link to PDB">1QES</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QES/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1QES</a></td><td align="center">AB</td><td align="center">480</td><td align="center">240</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970506000000">6-05-97</td><td>synthetic</td><td>TANDEM GU MISMATCHES IN RNA, NMR, 30 STRUCTURES</td><td>J.A.MCDOWELL,L.HE,X.CHEN,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=165D" title="Link to PDB">165D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/165D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">165D</a></td><td align="center">AB</td><td align="center">16</td><td align="center">8</td><td align="center">1.6</td><td sorttable_customkey="19960708000000">8-31-94</td><td>synthetic</td><td>THE STRUCTURE OF A MISPAIRED RNA DOUBLE HELIX AT 1.6 ANGSTROMS RESOLUTION AND IMPLICATIONS FOR THE PREDICTION OF RNA SECONDARY STRUCTURE</td><td>W.CRUSE,P.SALUDJIAN,E.BIALA,P.STRAZEWSKI,T.PRANGE,O.KENNARD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3L25" title="Link to PDB">3L25</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3L25/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3L25</a></td><td align="center">CF</td><td align="center">16</td><td align="center">8</td><td align="center">2.0</td><td sorttable_customkey="20120201000000">1-26-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF ZAIRE EBOLA VP35 INTERFERON INHIBITORY DOMAIN BOUND TO 8 BP DSRNA</td><td>D.W.LEUNG,K.C.PRINS,D.M.BOREK,M.FARAHBAKHSH,J.M.TUFARIELLO, P.RAMANAN,J.C.NIX,L.A.HELGESON,Z.OTWINOWSKI,R.B.HONZATKO, C.F.BASLER,G.K.AMARASINGHE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2CJK" title="Link to PDB">2CJK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2CJK/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2CJK</a></td><td align="center">B</td><td align="center">200</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20080406000000">6-28-06</td><td>SACCHAROMYCES CEREVISIAE</td><td>STRUCTURE OF THE RNA BINDING DOMAIN OF HRP1 IN COMPLEX WITH RNA</td><td>J.M.PEREZ-CANADILLAS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R7W" title="Link to PDB">2R7W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7W/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2R7W</a></td><td align="center">X</td><td align="center">7</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20100507000000">7-29-08</td><td>SIMIAN ROTAVIRUS</td><td>CRYSTAL STRUCTURE OF ROTAVIRUS SA11 VP1/RNA (UGUGACC)/MRNA 5'-CAP (M7GPPPG) COMPLEX</td><td>X.LU,S.C.HARRISON,Y.J.TAO,J.T.PATTON,M.L.NIBERT</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7X/index.html" title="Resolution 2.8A, 14 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2R7X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7R/index.html" title="Resolution 2.6A, 7 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2R7R</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7W/2R7W_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=435D" title="Link to PDB">435D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/435D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">435D</a></td><td align="center">AB</td><td align="center">28</td><td align="center">14</td><td align="center">1.4</td><td sorttable_customkey="20000206000000">6-14-99</td><td>synthetic,synthetic</td><td>'-R(*UP*AP*GP*CP*CP*CP*C)-3', 5'-R(*GP*GP*GP*GP*CP*UP*A)-3'</td><td>U.MUELLER,H.SCHUEBEL,M.SPRINZL,U.HEINEMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1DDL" title="Link to PDB">1DDL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1DDL/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1DDL</a></td><td align="center">DE</td><td align="center">9</td><td align="center">0</td><td align="center">2.7</td><td sorttable_customkey="20020610000000">10-30-00</td><td>synthetic,synthetic</td><td>DESMODIUM YELLOW MOTTLE TYMOVIRUS</td><td>S.B.LARSON,J.DAY,M.A.CANADY,A.GREENWOOD,A.MCPHERSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1FL8" title="Link to PDB">1FL8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1FL8/index.html" title="Link to FR3D analysis. 0.2929 pairs per nucleotide">1FL8</a></td><td align="center">A</td><td align="center">280</td><td align="center">82</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010410000000">10-16-00</td><td>synthetic</td><td>HYPERMODIFIED NUCLEOSIDES IN THE ANTICODON OF TRNALYS STABILIZE A CANONICAL U-TURN STRUCTURE</td><td>M.SUNDARAM,P.C.DURANT,D.R.DAVIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VOD" title="Link to PDB">2VOD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VOD/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VOD</a></td><td align="center">C</td><td align="center">14</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20080605000000">5-06-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF N-TERMINAL DOMAINS OF HUMAN LA PROTEIN COMPLEXED WITH RNA OLIGOMER AUAUUUU</td><td>O.KOTIK-KOGAN,E.R.VALENTINE,D.SANFELICE,M.R.CONTE,S.CURRY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VAL" title="Link to PDB">2VAL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VAL/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2VAL</a></td><td align="center">AC</td><td align="center">28</td><td align="center">14</td><td align="center">2.0</td><td sorttable_customkey="20080410000000">10-16-07</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF AN ESCHERICHIA COLI TRNAGLY MICROHELIX AT 2.0 ANGSTROM RESOLUTION</td><td>C.FORSTER,A.B.E.BRAUER,M.PERBANDT,D.LEHMANN,J.P.FURSTE, C.BETZEL,V.A.ERDMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V6W" title="Link to PDB">2V6W</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V6W/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2V6W</a></td><td align="center">AB</td><td align="center">14</td><td align="center">7</td><td align="center">1.8</td><td sorttable_customkey="20070611000000">11-06-07</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>TRNASER ACCEPTOR STEM: CONFORMATION AND HYDRATION OF A MICROHELIX IN A CRYSTAL STRUCTURE AT 1.8 ANGSTROM RESOLUTION</td><td>C.FOERSTER,A.B.E.BRAUER,S.BRODE,J.P.FUERSTE,C.BETZEL, V.A.ERDMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=409D" title="Link to PDB">409D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/409D/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">409D</a></td><td align="center">AB</td><td align="center">42</td><td align="center">18</td><td align="center">2.5</td><td sorttable_customkey="20000101000000">1-13-99</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA R(CCCIUGGG) WITH THREE INDEPENDENT DUPLEXES INCORPORATING TANDEM I.U WOBBLES</td><td>B.PAN,S.N.MITRA,L.SUN,D.HART,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KMS" title="Link to PDB">3KMS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KMS/index.html" title="Link to FR3D analysis. 0.4167 pairs per nucleotide">3KMS</a></td><td align="center">BC</td><td align="center">12</td><td align="center">5</td><td align="center">2.2</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic,synthetic</td><td>G62S MUTANT OF FOOT-AND-MOUTH DISEASE VIRUS RNA-POLYMERASE IN COMPLEX WITH A TEMPLATE- PRIMER RNA TRIGONAL STRUCTURE</td><td>C.FERRER-ORTA,N.VERDAGUER,R.PEREZ-LUQUE</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KLV/index.html" title="Resolution 2.6A, 12 nucleotides, 4 basepairs, 0.3333 pairs per nucleotide">3KLV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KNA/index.html" title="Resolution 2.8A, 12 nucleotides, 3 basepairs, 0.2500 pairs per nucleotide">3KNA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KMS/3KMS_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1SAQ" title="Link to PDB">1SAQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1SAQ/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">1SAQ</a></td><td align="center">AB</td><td align="center">14</td><td align="center">6</td><td align="center">2.7</td><td sorttable_customkey="20050605000000">5-18-04</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE RNA OCTAMER GIC(GA)GCC</td><td>S.B.JANG,K.BAEYENS,M.S.JEONG,J.SANTALUCIA JR,D.TURNER, S.R.HOLBROOK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JLX" title="Link to PDB">2JLX</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLX/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2JLX</a></td><td align="center">D</td><td align="center">14</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20100111000000">11-25-08</td><td>synthetic</td><td>DENGUE VIRUS 4 NS3 HELICASE IN COMPLEX WITH SSRNA AND ADP- VANADATE</td><td>D.H.LUO,T.XU,R.P.WATSON,D.S.BECKER,A.SAMPATH,W.JAHNKE, S.S.YEONG,C.H.WANG,S.P.LIM,S.G.VASUDEVAN,J.LESCAR</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLY/index.html" title="Resolution 2.4A, 14 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JLY</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLX/2JLX_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2K65" title="Link to PDB">2K65</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2K65/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2K65</a></td><td align="center">AB</td><td align="center">280</td><td align="center">140</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100207000000">7-14-09</td><td>synthetic,synthetic</td><td>NMR SOLUTION STRUCTURE OF THE EXON/INTRON BINDING SITE 1 (EBS1/IBS1) OF THE GROUP II INTRON SC.AI5(GAMMA)</td><td>D.KRUSCHEL,R.K.O.SIGEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GVN" title="Link to PDB">3GVN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GVN/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3GVN</a></td><td align="center">AB</td><td align="center">14</td><td align="center">7</td><td align="center">1.2</td><td sorttable_customkey="20110407000000">7-28-09</td><td>synthetic,synthetic</td><td>THE 1.2 ANGSTROEM CRYSTAL STRUCTURE OF AN E.COLI TRNASER ACCEPTOR STEM MICROHELIX REVEALS TWO MAGNESIUM BINDING SITES</td><td>A.EICHERT,J.P.FURSTE,A.SCHREIBER,M.PERBANDT,C.BETZEL, V.A.ERDMANN,C.FORSTER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WMQ" title="Link to PDB">1WMQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WMQ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WMQ</a></td><td align="center">C</td><td align="center">14</td><td align="center">0</td><td align="center">1.6</td><td sorttable_customkey="20060303000000">3-15-05</td><td>synthetic</td><td>STRUCTURE OF THE HUTP ANTITERMINATION COMPLEX BOUND TO A SINGLE STRANDED REGION OF HUT MRNA</td><td>T.S.KUMAREVEL,H.MIZUNO,P.K.R.KUMAR</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WRQ/index.html" title="Resolution 2.2A, 14 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1WRQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WMQ/1WMQ_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3E2E" title="Link to PDB">3E2E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3E2E/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3E2E</a></td><td align="center">R</td><td align="center">7</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20080411000000">11-04-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN INTERMEDIATE COMPLEX OF T7 RNAP AND 7NT OF RNA</td><td>K.J.DURNIAK,S.BAILEY,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2WJ8" title="Link to PDB">2WJ8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2WJ8/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2WJ8</a></td><td align="center">a</td><td align="center">140</td><td align="center">0</td><td align="center">3.3</td><td sorttable_customkey="20090812000000">12-08-09</td><td>ESCHERICHIA COLI</td><td>RESPIRATORY SYNCITIAL VIRUS RIBONUCLEOPROTEIN</td><td>R.G.TAWAR,S.DUQUERROY,C.VONRHEIN,P.F.VARELA,L.DAMIER-PIOLLE, N.CASTAGNE,K.MACLELLAN,H.BEDOUELLE,G.BRICOGNE,D.BHELLA, J.ELEOUET,F.A.REY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KQ2" title="Link to PDB">1KQ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KQ2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1KQ2</a></td><td align="center">R</td><td align="center">7</td><td align="center">0</td><td align="center">2.7</td><td sorttable_customkey="20020507000000">7-05-02</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN HFQ-RNA COMPLEX</td><td>M.A.SCHUMACHER,R.F.PEARSON,T.MOLLER,P.VALENTIN-HANSEN, R.G.BRENNAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ERR" title="Link to PDB">2ERR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ERR/index.html" title="Link to FR3D analysis. 0.1333 pairs per nucleotide">2ERR</a></td><td align="center">B</td><td align="center">210</td><td align="center">28</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061001000000">1-10-06</td><td>HOMO SAPIENS</td><td>NMR STRUCTURE OF THE RNA BINDING DOMAIN OF HUMAN FOX-1 IN COMPLEX WITH UGCAUGU</td><td>F.H.ALLAIN,S.D.AUWETER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=466D" title="Link to PDB">466D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/466D/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">466D</a></td><td align="center">AB</td><td align="center">28</td><td align="center">14</td><td align="center">1.2</td><td sorttable_customkey="20000408000000">8-16-99</td><td>synthetic,synthetic</td><td>DISORDER AND TWIN REFINEMENT OF RNA HEPTAMER DOUBLE HELIX</td><td>U.MUELLER,Y.A.MULLER,R.HERBST-IRMER,M.SPRINZL,U.HEINEMANN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/434D/index.html" title="Resolution 1.2A, 28 nucleotides, 14 basepairs, 0.5000 pairs per nucleotide">434D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/464D/index.html" title="Resolution 1.2A, 26 nucleotides, 12 basepairs, 0.4615 pairs per nucleotide">464D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/466D/466D_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G4B" title="Link to PDB">2G4B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G4B/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2G4B</a></td><td align="center">B</td><td align="center">7</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20080107000000">7-25-06</td><td>synthetic</td><td>STRUCTURE OF U2AF65 VARIANT WITH POLYURIDINE TRACT</td><td>E.A.SICKMIER,C.L.KIELKOPF</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XFM" title="Link to PDB">2XFM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XFM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XFM</a></td><td align="center">B</td><td align="center">70</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20130201000000">1-26-11</td><td>synthetic</td><td>COMPLEX STRUCTURE OF THE MIWI PAZ DOMAIN BOUND TO METHYLATED SINGLE STRANDED RNA</td><td>B.SIMON,J.P.KIRKPATRICK,S.ECKHARDT,P.SEHR,M.A.ANDRADE-NAVARRO, R.S.PILLAI,T.CARLOMAGNO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WWF" title="Link to PDB">1WWF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WWF/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WWF</a></td><td align="center">B</td><td align="center">140</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050504000000">4-05-05</td><td>MOLONEY MURINE LEUKEMIA VIRUS</td><td>NMR STRUCTURE DETERMINED FOR MLV NC COMPLEX WITH RNA SEQUENCE CCUCCGU</td><td>A.DEY,D.YORK,A.SMALLS-MANTEY,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3HSB" title="Link to PDB">3HSB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3HSB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3HSB</a></td><td align="center">X</td><td align="center">7</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20111106000000">6-23-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF YMAH (HFQ) FROM BACILLUS SUBTILIS IN COMPLEX WITH AN RNA APTAMER</td><td>S.BABA,T.SOMEYA,T.KUMASAKA,G.KAWAI,K.NAKAMURA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WPU" title="Link to PDB">1WPU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WPU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WPU</a></td><td align="center">C</td><td align="center">14</td><td align="center">0</td><td align="center">1.5</td><td sorttable_customkey="20070608000000">8-30-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HUTP ANTITERMINATION COMPLEX BOUND TO A SINGLE STRANDED REGION OF HUT MRNA</td><td>T.S.KUMAREVEL,H.MIZUNO,P.K.R.KUMAR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2V7R" title="Link to PDB">2V7R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2V7R/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2V7R</a></td><td align="center">AB</td><td align="center">14</td><td align="center">7</td><td align="center">1.2</td><td sorttable_customkey="20090603000000">3-18-08</td><td>HOMO SAPIENS,HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF A HUMAN TRNAGLY MICROHELIX AT 1.2 ANGSTROM RESOLUTION</td><td>C.FOERSTER,M.MANKOWSKA,J.P.FUERSTE,M.PERBANDT,C.BETZEL, V.A.ERDMANN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3R1E" title="Link to PDB">3R1E</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3R1E/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">3R1E</a></td><td align="center">AB</td><td align="center">14</td><td align="center">6</td><td align="center">1.0</td><td sorttable_customkey="20110106000000">6-01-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF GC(8BRG)GCGGC DUPLEX</td><td>A.KILISZEK,R.KIERZEK,W.J.KRZYZOSIAK,W.RYPNIEWSKI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q0O" title="Link to PDB">3Q0O</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q0O/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q0O</a></td><td align="center">C</td><td align="center">14</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE PUMILIO-HOMOLOGY DOMAIN FROM HUMAN PUMILIO1 IN COMPLEX WITH ERK2 NRE</td><td>G.LU,T.M.T.HALL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2PO1" title="Link to PDB">2PO1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2PO1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2PO1</a></td><td align="center">C</td><td align="center">7</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20090603000000">3-18-08</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE P. ABYSSI EXOSOME RNASE PH RING COMPLEXED WITH A SINGLE STRANDED 10-MER POLY(A) RNA</td><td>M.V.A.S.NAVARRO,B.G.GUIMARAES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WWE" title="Link to PDB">1WWE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WWE/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WWE</a></td><td align="center">B</td><td align="center">140</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061003000000">3-22-05</td><td>MOLONEY MURINE LEUKEMIA VIRUS</td><td>NMR STRUCTURE DETERMINED FOR MLV NC COMPLEX WITH RNA SEQUENCE UUUUGCU</td><td>A.DEY,D.YORK,A.SMALLS-MANTEY,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R7T" title="Link to PDB">2R7T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7T/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2R7T</a></td><td align="center">X</td><td align="center">7</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20100507000000">7-29-08</td><td>SIMIAN ROTAVIRUS</td><td>CRYSTAL STRUCTURE OF ROTAVIRUS SA11 VP1/RNA (UGUGAACC) COMPLEX</td><td>X.LU,S.C.HARRISON,Y.J.TAO,J.T.PATTON,M.L.NIBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2DB3" title="Link to PDB">2DB3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2DB3/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2DB3</a></td><td align="center">G</td><td align="center">24</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20060205000000">5-02-06</td><td>synthetic</td><td>STRUCTURAL BASIS FOR RNA UNWINDING BY THE DEAD-BOX PROTEIN DROSOPHILA VASA</td><td>T.SENGOKU,O.NUREKI,A.NAKAMURA,S.KOBAYASHI,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VUQ" title="Link to PDB">2VUQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VUQ/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2VUQ</a></td><td align="center">AB</td><td align="center">14</td><td align="center">7</td><td align="center">1.2</td><td sorttable_customkey="20091003000000">3-10-09</td><td>HOMO SAPIENS,HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF A HUMAN TRNAGLY ACCEPTOR STEM MICROHELIX (DERIVED FROM THE GENE SEQUENCE DG9990) AT 1.18 ANGSTROEM RESOLUTION</td><td>A.EICHERT,M.PERBANDT,A.SCHREIBER,J.P.FUERSTE,C.BETZEL, V.A.ERDMANN,C.FOERSTER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VON" title="Link to PDB">2VON</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VON/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VON</a></td><td align="center">C</td><td align="center">14</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20080605000000">5-06-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF N-TERMINAL DOMAINS OF HUMAN LA PROTEIN COMPLEXED WITH RNA OLIGOMER AUAAUUU</td><td>O.KOTIK-KOGAN,E.R.VALENTINE,D.SANFELICE,M.R.CONTE,S.CURRY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A0P" title="Link to PDB">2A0P</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A0P/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2A0P</a></td><td align="center">AB</td><td align="center">14</td><td align="center">6</td><td align="center">1.9</td><td sorttable_customkey="20060408000000">8-16-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF RNA OLIGOMER CONTAINING 4'-THIORIBOSE</td><td>P.HAEBERLI,I.BERGER,P.S.PALLAN,M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3AOH" title="Link to PDB">3AOH</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3AOH/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3AOH</a></td><td align="center">Q</td><td align="center">7</td><td align="center">0</td><td align="center">4.1</td><td sorttable_customkey="20100812000000">12-08-10</td><td>synthetic</td><td>RNA POLYMERASE-GFH1 COMPLEX (CRYSTAL TYPE 1)</td><td>S.TAGAMI,S.SEKINE,T.KUMAREVEL,M.YAMAMOTO,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3LOB" title="Link to PDB">3LOB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3LOB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3LOB</a></td><td align="center">R</td><td align="center">7</td><td align="center">0</td><td align="center">3.6</td><td sorttable_customkey="20110904000000">4-21-10</td><td>SPODOPTERA FRUGIPERDA</td><td>CRYSTAL STRUCTURE OF FLOCK HOUSE VIRUS CALCIUM MUTANT</td><td>J.E.JOHNSON,M.BANERJEE,J.A.SPEIR,R.HUANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2W89" title="Link to PDB">2W89</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2W89/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2W89</a></td><td align="center">AB</td><td align="center">28</td><td align="center">14</td><td align="center">2.0</td><td sorttable_customkey="20100511000000">11-17-09</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF THE E.COLI TRNAARG AMINOACYL STEM ISSOACCEPTOR RR-1660 AT 2.0 ANGSTROEM RESOLUTION</td><td>A.EICHERT,A.SCHREIBER,J.P.FUERSTE,M.PERBANDT,C.BETZEL, V.A.ERDMANN,C.FOERSTER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JRG" title="Link to PDB">2JRG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JRG/index.html" title="Link to FR3D analysis. 0.3515 pairs per nucleotide">2JRG</a></td><td align="center">A</td><td align="center">165</td><td align="center">58</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20081207000000">7-24-07</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF E. COLI TRNA- VAL3 WITH 2 MODIFICATIONS (CMO5U34 M6A37)</td><td>F.A.VENDEIX,A.DZIERGOWSKA,E.M.GUSTILO,W.D.GRAHAM,B.SPROAT, A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RQC" title="Link to PDB">2RQC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RQC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2RQC</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20090408000000">8-04-09</td><td>synthetic</td><td>SOLUTION STRUCTURE OF RNA-BINDING DOMAIN 3 OF CUGBP1 IN COMPLEX WITH RNA (UG)3</td><td>K.TSUDA,K.KUWASAKO,M.TAKAHASHI,T.SOMEYA,Y.MUTO,M.INOUE, T.KIGAWA,T.TERADA,M.SHIROUZU,S.YOKOYAMA,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KFY" title="Link to PDB">2KFY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KFY/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2KFY</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100906000000">6-09-10</td><td>synthetic</td><td>NMR STRUCTURE OF THE FIRST QRRM OF HNRNP F IN COMPLEX WITH AGGGAU G- TRACT RNA</td><td>F.H.T.ALLAIN,C.DOMINGUEZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ADC" title="Link to PDB">2ADC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ADC/index.html" title="Link to FR3D analysis. 0.0042 pairs per nucleotide">2ADC</a></td><td align="center">B</td><td align="center">240</td><td align="center">1</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050410000000">10-04-05</td><td>HOMO SAPIENS</td><td>SOLUTION STRUCTURE OF POLYPYRIMIDINE TRACT BINDING PROTEIN RBD34 COMPLEXED WITH CUCUCU RNA</td><td>F.C.OBERSTRASS,S.D.AUWETER,M.ERAT,Y.HARGOUS,A.HENNING, P.WENTER,L.REYMOND,S.PITSCH,D.L.BLACK,F.H.T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R7S" title="Link to PDB">2R7S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7S/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2R7S</a></td><td align="center">X</td><td align="center">6</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20100507000000">7-29-08</td><td>SIMIAN ROTAVIRUS</td><td>CRYSTAL STRUCTURE OF ROTAVIRUS SA11 VP1 / RNA (UGUGCC) COMPLEX</td><td>X.LU,S.C.HARRISON,Y.J.TAO,J.T.PATTON,M.L.NIBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2J0S" title="Link to PDB">2J0S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J0S/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2J0S</a></td><td align="center">E</td><td align="center">6</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20060609000000">9-06-06</td><td>HOMO SAPIENS</td><td>THE CRYSTAL STRUCTURE OF THE EXON JUNCTION COMPLEX AT 2.2 A RESOLUTION</td><td>F.BONO,J.EBERT,E.LORENTZEN,E.CONTI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HYI/index.html" title="Resolution 2.3A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2HYI</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3EX7/index.html" title="Resolution 2.3A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3EX7</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J0Q/index.html" title="Resolution 3.2A, 12 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2J0Q</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2J0S/2J0S_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RAU" title="Link to PDB">1RAU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RAU/index.html" title="Link to FR3D analysis. 0.6667 pairs per nucleotide">1RAU</a></td><td align="center">ABCD</td><td align="center">24</td><td align="center">16</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19960701000000">1-31-94</td><td>synthetic</td><td>SOLUTION STRUCTURE OF AN UNUSUALLY STABLE RNA TETRAPLEX CONTAINING G-AND U-QUARTET STRUCTURES</td><td>C.CHEONG,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2G8F" title="Link to PDB">2G8F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8F/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2G8F</a></td><td align="center">B</td><td align="center">6</td><td align="center">0</td><td align="center">1.6</td><td sorttable_customkey="20080104000000">4-25-06</td><td>synthetic</td><td>B. HALODURANS RNASE H CATALYTIC DOMAIN E188A MUTANT IN COMPLEX WITH MG2+ AND RNA/DNA HYBRID (NON-P NICK AT THE ACTIVE SITE)</td><td>M.NOWOTNY,W.YANG</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8U/index.html" title="Resolution 2.7A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8U</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8W/index.html" title="Resolution 2.0A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8W</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8I/index.html" title="Resolution 1.6A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8I</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8K/index.html" title="Resolution 1.6A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8K</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7Y/index.html" title="Resolution 1.8A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2R7Y</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8H/index.html" title="Resolution 1.9A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8V/index.html" title="Resolution 1.9A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2G8V</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2G8F/2G8F_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AD9" title="Link to PDB">2AD9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AD9/index.html" title="Link to FR3D analysis. 0.1417 pairs per nucleotide">2AD9</a></td><td align="center">B</td><td align="center">120</td><td align="center">17</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050410000000">10-04-05</td><td>HOMO SAPIENS</td><td>SOLUTION STRUCTURE OF POLYPYRIMIDINE TRACT BINDING PROTEIN RBD1 COMPLEXED WITH CUCUCU RNA</td><td>F.C.OBERSTRASS,S.D.AUWETER,M.ERAT,Y.HARGOUS,A.HENNING, P.WENTER,L.REYMOND,S.PITSCH,D.L.BLACK,F.H.T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1OSU" title="Link to PDB">1OSU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1OSU/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1OSU</a></td><td align="center">A</td><td align="center">18</td><td align="center">6</td><td align="center">1.4</td><td sorttable_customkey="19960404000000">4-04-96</td><td>synthetic</td><td>STRUCTURE OF THE RNA HEXAMER, R(UUCGCG), WITH A 5'-UU- OVERHANG EXHIBITING HOOGSTEEN-LIKE TRANS U-U BASE PAIRS</td><td>M.C.WAHL,S.T.RAO,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PBM" title="Link to PDB">1PBM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PBM/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1PBM</a></td><td align="center">AB</td><td align="center">12</td><td align="center">6</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19970707000000">7-07-97</td><td>synthetic</td><td>STRUCTURE OF RIBONUCLEIC ACID, NMR, 1 STRUCTURE</td><td>M.POPENDA,E.BIALA,J.MILECKI,R.W.ADAMIAK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GXB" title="Link to PDB">2GXB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GXB/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">2GXB</a></td><td align="center">EF</td><td align="center">12</td><td align="center">6</td><td align="center">2.2</td><td sorttable_customkey="20070105000000">5-01-07</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE ZA DOMAIN BOUND TO Z-RNA</td><td>A.ATHANASIADIS,D.PLACIDO,A.RICH</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T4X/index.html" title="SOLUTION NMR, 144 nucleotides, 72 basepairs, 0.5000 pairs per nucleotide">1T4X</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GXB/2GXB_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ICE" title="Link to PDB">3ICE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ICE/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3ICE</a></td><td align="center">G</td><td align="center">6</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20091011000000">11-10-09</td><td>synthetic</td><td>RHO TRANSCRIPTION TERMINATION FACTOR BOUND TO RNA AND ADP- BEF3</td><td>N.D.THOMSEN,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J8G" title="Link to PDB">1J8G</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J8G/index.html" title="Link to FR3D analysis. 0.8333 pairs per nucleotide">1J8G</a></td><td align="center">A</td><td align="center">24</td><td align="center">20</td><td align="center">0.6</td><td sorttable_customkey="20021111000000">11-23-01</td><td>synthetic</td><td>X-RAY ANALYSIS OF A RNA TETRAPLEX R(UGGGGU)4 AT ULTRA-HIGH RESOLUTION</td><td>J.DENG,Y.XIONG,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WWG" title="Link to PDB">1WWG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WWG/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WWG</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050504000000">4-05-05</td><td>MOLONEY MURINE LEUKEMIA VIRUS</td><td>NMR STRUCTURE DETERMINED FOR MLV NC COMPLEX WITH RNA SEQUENCE UAUCUG</td><td>A.DEY,D.YORK,A.SMALLS-MANTEY,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MDI" title="Link to PDB">3MDI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MDI/index.html" title="Link to FR3D analysis. 0.1667 pairs per nucleotide">3MDI</a></td><td align="center">C</td><td align="center">6</td><td align="center">1</td><td align="center">2.1</td><td sorttable_customkey="20100206000000">6-02-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE 25KDA SUBUNIT OF HUMAN CLEAVAGE FACTOR IM IN COMPLEX WITH RNA UGUAAA</td><td>Q.YANG,G.M.GILMARTIN,S.DOUBLIE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3FHT" title="Link to PDB">3FHT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FHT/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3FHT</a></td><td align="center">C</td><td align="center">12</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20100502000000">2-17-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF HUMAN DBP5 IN COMPLEX WITH AMPPNP AND RNA</td><td>H.VON MOELLER,E.CONTI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G0H/index.html" title="Resolution 2.7A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3G0H</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3FHT/3FHT_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JLW" title="Link to PDB">2JLW</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLW/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2JLW</a></td><td align="center">C</td><td align="center">12</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20100111000000">11-25-08</td><td>synthetic</td><td>DENGUE VIRUS 4 NS3 HELICASE IN COMPLEX WITH SSRNA2</td><td>D.H.LUO,T.XU,R.P.WATSON,D.S.BECKER,A.SAMPATH,W.JAHNKE, S.S.YEONG,C.H.WANG,S.P.LIM,S.G.VASUDEVAN,J.LESCAR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3G9Y" title="Link to PDB">3G9Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3G9Y/index.html" title="Link to FR3D analysis. 0.1667 pairs per nucleotide">3G9Y</a></td><td align="center">C</td><td align="center">6</td><td align="center">1</td><td align="center">1.4</td><td sorttable_customkey="20090303000000">3-03-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE SECOND ZINC FINGER FROM ZRANB2/ZNF265 BOUND TO 6 NT SSRNA SEQUENCE AGGUAA</td><td>F.E.LOUGHLIN,A.P.MCGRATH,M.LEE,J.M.GUSS,J.P.MACKAY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2RRA" title="Link to PDB">2RRA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2RRA/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2RRA</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20130304000000">4-27-11</td><td>synthetic</td><td>SOLUTION STRUCTURE OF RNA BINDING DOMAIN IN HUMAN TRA2 BETA PROTEIN IN COMPLEX WITH RNA (GAAGAA)</td><td>K.TSUDA,K.KUWASAKO,M.TAKAHASHI,T.SOMEYA,M.INOUE,T.KIGAWA,T.TERADA, M.SHIROUZU,S.SUGANO,Y.MUTO,S.YOKOYAMA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XZO" title="Link to PDB">2XZO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XZO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XZO</a></td><td align="center">D</td><td align="center">6</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20130603000000">3-30-11</td><td>SYNTHETIC CONSTRUCT</td><td>UPF1 HELICASE - RNA COMPLEX</td><td>S.CHAKRABARTI,U.JAYACHANDRAN,F.BONNEAU,F.FIORINI,C.BASQUIN,S.DOMCKE, H.LE HIR,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1M8V" title="Link to PDB">1M8V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1M8V/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1M8V</a></td><td align="center">P</td><td align="center">41</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20031102000000">2-11-03</td><td>synthetic</td><td>STRUCTURE OF PYROCOCCUS ABYSSII SM PROTEIN IN COMPLEX WITH A URIDINE HEPTAMER</td><td>S.THORE,C.MAYER,C.SAUTER,S.WEEKS,D.SUCK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UTD" title="Link to PDB">1UTD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UTD/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UTD</a></td><td align="center">8</td><td align="center">35</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20050301000000">1-15-04</td><td>synthetic,BACILLUS STEAROTHERMOPHILUS</td><td>THE STRUCTURE OF THE TRP RAN-BINDING ATTENUATION PROTEIN (TRAP) BOUND TO A 63-NUCLEOTIDE RNA MOLECULE CONTAINING GAGUUU REPEATS</td><td>N.H.HOPCROFT,A.MANFREDO,A.L.WENDT,A.M.BRZOZOWSKI,P.GOLLNICK, A.A.ANTSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=176D" title="Link to PDB">176D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/176D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">176D</a></td><td align="center">B</td><td align="center">60</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="19940111000000">11-01-94</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF A PEPTIDE NUCLEIC ACID COMPLEXED WITH RNA</td><td>S.C.BROWN,J.M.VEAL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NNC" title="Link to PDB">3NNC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NNC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3NNC</a></td><td align="center">B</td><td align="center">6</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20120310000000">10-27-10</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF CUGBP1 RRM1/2-RNA COMPLEX</td><td>M.TEPLOVA,J.SONG,H.GAW,A TEPLOV,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1AV6" title="Link to PDB">1AV6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1AV6/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1AV6</a></td><td align="center">B</td><td align="center">6</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20000102000000">2-25-98</td><td>engineered</td><td>VACCINIA METHYLTRANSFERASE VP39 COMPLEXED WITH M7G CAPPED RNA HEXAMER AND S-ADENOSYLHOMOCYSTEINE</td><td>A.E.HODEL,P.D.GERSHON,F.A.QUIOCHO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A8V" title="Link to PDB">2A8V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A8V/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2A8V</a></td><td align="center">DE</td><td align="center">9</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20010204000000">4-26-99</td><td>synthetic,ESCHERICHIA COLI</td><td>RHO TRANSCRIPTION TERMINATION FACTOR/RNA COMPLEX</td><td>C.E.BOGDEN,D.FASS,N.BERGMAN,M.D.NICHOLS,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KG0" title="Link to PDB">2KG0</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KG0/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2KG0</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100906000000">6-09-10</td><td>synthetic</td><td>STRUCTURE OF THE SECOND QRRM DOMAIN OF HNRNP F IN COMPLEX WITH A AGGGAU G-TRACT RNA</td><td>F.H.T.ALLAIN,C.DOMINGUEZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I9K" title="Link to PDB">1I9K</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I9K/index.html" title="Link to FR3D analysis. 0.1250 pairs per nucleotide">1I9K</a></td><td align="center">BD</td><td align="center">24</td><td align="center">3</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030605000000">5-30-01</td><td>synthetic</td><td>THE RNA I-MOTIF</td><td>K.SNOUSSI,S.NONIN-LECOMTE,J.L.LEROU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KH9" title="Link to PDB">2KH9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KH9/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2KH9</a></td><td align="center">B</td><td align="center">60</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20100903000000">3-09-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF YEAST PRP24-RRM2 BOUND TO A FRAGMENT OF U6 RNA</td><td>S.A.MARTIN-TUMASZ,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3MDG" title="Link to PDB">3MDG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3MDG/index.html" title="Link to FR3D analysis. 0.1667 pairs per nucleotide">3MDG</a></td><td align="center">C</td><td align="center">6</td><td align="center">1</td><td align="center">2.2</td><td sorttable_customkey="20100206000000">6-02-10</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE 25KDA SUBUNIT OF HUMAN CLEAVAGE FACTOR IM IN COMPLEX WITH RNA UUGUAU</td><td>Q.YANG,G.M.GILMARTIN,S.DOUBLIE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2ADB" title="Link to PDB">2ADB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2ADB/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2ADB</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20050410000000">10-04-05</td><td>HOMO SAPIENS</td><td>SOLUTION STRUCTURE OF POLYPYRIMIDINE TRACT BINDING PROTEIN RBD2 COMPLEXED WITH CUCUCU RNA</td><td>F.C.OBERSTRASS,S.D.AUWETER,M.ERAT,Y.HARGOUS,A.HENNING, P.WENTER,L.REYMOND,S.PITSCH,D.L.BLACK,F.H.T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KXN" title="Link to PDB">2KXN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KXN/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2KXN</a></td><td align="center">A</td><td align="center">72</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120403000000">3-16-11</td><td>synthetic</td><td>NMR STRUCTURE OF HUMAN TRA2BETA1 RRM IN COMPLEX WITH AAGAAC RNA</td><td>A.CLERY,S.JAYNE,N.BENDERSKA,C.DOMINGUEZ,S.STAMM,F.H.-T.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WWD" title="Link to PDB">1WWD</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WWD/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1WWD</a></td><td align="center">B</td><td align="center">120</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061003000000">3-22-05</td><td>MOLONEY MURINE LEUKEMIA VIRUS</td><td>NMR STRUCTURE DETERMINED FOR MLV NC COMPLEX WITH RNA SEQUENCE AACAGU</td><td>A.DEY,D.YORK,A.SMALLS-MANTEY,M.F.SUMMERS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PGL" title="Link to PDB">1PGL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PGL/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1PGL</a></td><td align="center">3</td><td align="center">6</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20040611000000">11-18-03</td><td>BEAN-POD MOTTLE VIRUS (STRAIN</td><td>BEAN POD MOTTLE VIRUS (BPMV), MIDDLE COMPONENT</td><td>T.LIN,J.CAVARELLI,J.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JLZ" title="Link to PDB">2JLZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLZ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2JLZ</a></td><td align="center">C</td><td align="center">12</td><td align="center">0</td><td align="center">2.2</td><td sorttable_customkey="20100111000000">11-25-08</td><td>synthetic</td><td>DENGUE VIRUS 4 NS3 HELICASE IN COMPLEX WITH SSRNA AND ADP</td><td>D.H.LUO,T.XU,R.P.WATSON,D.S.BECKER,A.SAMPATH,W.JAHNKE, S.S.YEONG,C.H.WANG,S.P.LIM,S.G.VASUDEVAN,J.LESCAR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NNA" title="Link to PDB">3NNA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NNA/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3NNA</a></td><td align="center">B</td><td align="center">5</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20120310000000">10-27-10</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF CUGBP1 RRM1/2-RNA COMPLEX</td><td>M.TEPLOVA,J.SONG,H.GAW,A TEPLOV,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2Q66" title="Link to PDB">2Q66</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2Q66/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2Q66</a></td><td align="center">X</td><td align="center">5</td><td align="center">0</td><td align="center">1.8</td><td sorttable_customkey="20090408000000">8-28-07</td><td>synthetic</td><td>STRUCTURE OF YEAST POLY(A) POLYMERASE WITH ATP AND OLIGO(A)</td><td>A.BOHM,P.BALBO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KMQ" title="Link to PDB">3KMQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KMQ/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">3KMQ</a></td><td align="center">BC</td><td align="center">10</td><td align="center">5</td><td align="center">2.1</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic,synthetic</td><td>G62S MUTANT OF FOOT-AND-MOUTH DISEASE VIRUS RNA-POLYMERASE IN COMPLEX WITH A TEMPLATE- PRIMER RNA, TETRAGONAL STRUCTURE</td><td>C.FERRER-ORTA,N.VERDAGUER,R.PEREZ-LUQUE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3O8R" title="Link to PDB">3O8R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O8R/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3O8R</a></td><td align="center">C</td><td align="center">5</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20110501000000">1-05-11</td><td>synthetic</td><td>VISUALIZING ATP-DEPENDENT RNA TRANSLOCATION BY THE NS3 HELICASE FROM HCV</td><td>T.C.APPLEBY,J.R.SOMOZA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JLU" title="Link to PDB">2JLU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2JLU</a></td><td align="center">C</td><td align="center">10</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20100111000000">11-25-08</td><td>synthetic</td><td>DENGUE VIRUS 4 NS3 HELICASE IN COMPLEX WITH SSRNA</td><td>D.H.LUO,T.XU,R.P.WATSON,D.S.BECKER,A.SAMPATH,W.JAHNKE, S.S.YEONG,C.H.WANG,S.P.LIM,S.G.VASUDEVAN,J.LESCAR</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLV/index.html" title="Resolution 2.3A, 10 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2JLV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JLU/2JLU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1N38" title="Link to PDB">1N38</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N38/index.html" title="Link to FR3D analysis. 0.2857 pairs per nucleotide">1N38</a></td><td align="center">BC</td><td align="center">7</td><td align="center">2</td><td align="center">2.8</td><td sorttable_customkey="20040112000000">12-25-02</td><td>synthetic,synthetic</td><td>REOVIRUS POLYMERASE LAMBDA3 ELONGATION COMPLEX WITH ONE PHOSPHODIESTER BOND FORMED</td><td>Y.TAO,D.L.FARSETTA,M.L.NIBERT,S.C.HARRISON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1WTT" title="Link to PDB">1WTT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1WTT/index.html" title="Link to FR3D analysis. 0.4545 pairs per nucleotide">1WTT</a></td><td align="center">A</td><td align="center">121</td><td align="center">55</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010301000000">1-27-99</td><td>synthetic</td><td>HELIX 45 (16S RRNA) FROM B. STEAROTHERMOPHILUS, NMR, 11 STRUCTURES</td><td>J.P.RIFE,P.B.MOORE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3M7N" title="Link to PDB">3M7N</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3M7N/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3M7N</a></td><td align="center">X</td><td align="center">14</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20120404000000">4-28-10</td><td>synthetic</td><td>ARCHAEOGLOBUS FULGIDUS EXOSOME WITH RNA BOUND TO THE ACTIVE SITE</td><td>S.HARTUNG,K.-P.HOPFNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3O8C" title="Link to PDB">3O8C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3O8C/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3O8C</a></td><td align="center">C</td><td align="center">5</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20110501000000">1-05-11</td><td>synthetic</td><td>VISUALIZING ATP-DEPENDENT RNA TRANSLOCATION BY THE NS3 HELICASE FROM HCV</td><td>T.C.APPLEBY,J.R.SOMOZA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3JXR" title="Link to PDB">3JXR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JXR/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">3JXR</a></td><td align="center">AB</td><td align="center">10</td><td align="center">4</td><td align="center">1.2</td><td sorttable_customkey="20100902000000">2-09-10</td><td>synthetic</td><td>X-RAY STRUCTURE OF R[CGCG(5-FLUORO)CG]2</td><td>D.A.ADAMIAK,J.MILECKI,R.W.ADAMIAK,W.RYPNIEWSKI</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JXQ/index.html" title="Resolution 1.4A, 20 nucleotides, 8 basepairs, 0.4000 pairs per nucleotide">3JXQ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3JXR/3JXR_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P6Y" title="Link to PDB">3P6Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P6Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3P6Y</a></td><td align="center">R</td><td align="center">23</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20100311000000">11-03-10</td><td>synthetic</td><td>CF IM25-CF IM68-UGUAA COMPLEX</td><td>H.LI,S.TONG,X.LI,H.SHI,Y.GAO,H.GE,L.NIU,M.TENG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LUU" title="Link to PDB">1LUU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LUU/index.html" title="Link to FR3D analysis. 0.3077 pairs per nucleotide">1LUU</a></td><td align="center">A</td><td align="center">143</td><td align="center">44</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030909000000">9-09-03</td><td>synthetic</td><td>NMR SOLUTION STRUCTURE OF THE ANTICODON OF YEAST TRNA-PHE WITH 4 MODIFICATIONS (OMC32 OMG34 1MG37 5MC40)</td><td>J.W.STUART,K.M.KOSHLAP,R.H.GUENTHER,P.F.AGRIS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LUX/index.html" title="SOLUTION NMR, 140 nucleotides, 40 basepairs, 0.2857 pairs per nucleotide">1LUX</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LUU/1LUU_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1J6S" title="Link to PDB">1J6S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1J6S/index.html" title="Link to FR3D analysis. 0.6000 pairs per nucleotide">1J6S</a></td><td align="center">A</td><td align="center">20</td><td align="center">12</td><td align="center">1.4</td><td sorttable_customkey="20030508000000">8-05-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA TETRAPLEX (UGAGGU)4 WITH A- TETRADS, G-TETRADS, U-TETRADS AND G-U OCTADS</td><td>B.PAN,Y.XIONG,K.SHI,J.DENG,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVM" title="Link to PDB">1UVM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVM</a></td><td align="center">D</td><td align="center">15</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20050702000000">2-19-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 WITH 5NT RNA CONFORMATION A</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XGJ" title="Link to PDB">2XGJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XGJ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XGJ</a></td><td align="center">CD</td><td align="center">10</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20111106000000">6-23-10</td><td>synthetic</td><td>STRUCTURE OF MTR4, A DEXH HELICASE INVOLVED IN NUCLEAR RNA PROCESSING AND SURVEILLANCE</td><td>J.R.WEIR,F.BONNEAU,J.HENTSCHEL,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PEY" title="Link to PDB">3PEY</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PEY/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PEY</a></td><td align="center">B</td><td align="center">5</td><td align="center">0</td><td align="center">1.4</td><td sorttable_customkey="20121103000000">3-23-11</td><td>synthetic</td><td>S. CEREVISIAE DBP5 BOUND TO RNA AND ADP BEF3</td><td>B.MONTPETIT,N.D.THOMSEN,K.J.HELMKE,M.A.SEELIGER,J.M.BERGER,K.WEIS</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PEW/index.html" title="Resolution 1.5A, 5 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">3PEW</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PEY/3PEY_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3RC8" title="Link to PDB">3RC8</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3RC8/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3RC8</a></td><td align="center">E</td><td align="center">5</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20111105000000">5-11-11</td><td>synthetic</td><td>HUMAN MITOCHONDRIAL HELICASE SUV3 IN COMPLEX WITH SHORT RNA FRAGMENT</td><td>Z.DAUTER,R.JEDRZEJCZAK,M.DAUTER,J.WANG,R.SZCZESNY,P.STEPIEN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NMR" title="Link to PDB">3NMR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NMR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3NMR</a></td><td align="center">B</td><td align="center">5</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20120310000000">10-27-10</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF CUGBP1 RRM1/2-RNA COMPLEX</td><td>M.TEPLOVA,J.SONG,H.GAW,A TEPLOV,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3D2S" title="Link to PDB">3D2S</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3D2S/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3D2S</a></td><td align="center">E</td><td align="center">18</td><td align="center">0</td><td align="center">1.7</td><td sorttable_customkey="20080212000000">12-02-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF MBNL1 TANDEM ZINC FINGER 3 AND 4 DOMAIN IN COMPLEX WITH CGCUGU RNA</td><td>M.TEPLOVA,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2JEA" title="Link to PDB">2JEA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2JEA/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2JEA</a></td><td align="center">C</td><td align="center">5</td><td align="center">0</td><td align="center">2.3</td><td sorttable_customkey="20070304000000">4-03-07</td><td>synthetic</td><td>STRUCTURE OF A 9-SUBUNIT ARCHAEAL EXOSOME BOUND TO RNA</td><td>E.LORENTZEN,E.CONTI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R7V" title="Link to PDB">2R7V</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7V/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2R7V</a></td><td align="center">X</td><td align="center">5</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20100507000000">7-29-08</td><td>SIMIAN ROTAVIRUS</td><td>CRYSTAL STRUCTURE OF ROTAVIRUS SA11 VP1/RNA (GGCUUU) COMPLEX</td><td>X.LU,S.C.HARRISON,Y.J.TAO,J.T.PATTON,M.L.NIBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVL" title="Link to PDB">1UVL</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVL/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVL</a></td><td align="center">B</td><td align="center">15</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20060202000000">2-26-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 WITH 5NT RNA CONFORMATION B</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O32" title="Link to PDB">2O32</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O32/index.html" title="Link to FR3D analysis. 0.1786 pairs per nucleotide">2O32</a></td><td align="center">A</td><td align="center">280</td><td align="center">50</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20070602000000">2-06-07</td><td>synthetic</td><td>SOLUTION STRUCTURE OF U2 SNRNA STEM I FROM HUMAN, CONTAINING MODIFIED NUCLEOTIDES</td><td>D.G.SASHITAL,V.VENDITTI,C.G.ANGERS,G.CORNILESCU,S.E.BUTCHER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=377D" title="Link to PDB">377D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/377D/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">377D</a></td><td align="center">AB</td><td align="center">20</td><td align="center">8</td><td align="center">1.8</td><td sorttable_customkey="19981008000000">8-10-98</td><td>synthetic</td><td>'-R(*CP*GP*UP*AP*CP*DG)-3'</td><td>R.BISWAS,S.N.MITRA,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KOS" title="Link to PDB">1KOS</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KOS/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">1KOS</a></td><td align="center">A</td><td align="center">120</td><td align="center">40</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20001010000000">10-22-99</td><td>synthetic</td><td>SOLUTION NMR STRUCTURE OF AN ANALOG OF THE YEAST TRNA PHE T STEM LOOP CONTAINING RIBOTHYMIDINE AT ITS NATURALLY OCCURRING POSITION</td><td>K.M.KOSHLAP,R.GUENTHER,E.SOCHACKA,A.MALKIEWICZ,P.F.AGRIS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VOP" title="Link to PDB">2VOP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VOP/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VOP</a></td><td align="center">B</td><td align="center">5</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20080605000000">5-06-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF N-TERMINAL DOMAINS OF HUMAN LA PROTEIN COMPLEXED WITH RNA OLIGOMER AUUUU</td><td>O.KOTIK-KOGAN,E.R.VALENTINE,D.SANFELICE,M.R.CONTE,S.CURRY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1KNZ" title="Link to PDB">1KNZ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1KNZ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1KNZ</a></td><td align="center">W</td><td align="center">20</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20030501000000">1-17-02</td><td>synthetic</td><td>RECOGNITION OF THE ROTAVIRUS MRNA 3' CONSENSUS BY AN ASYMMETRIC NSP3 HOMODIMER</td><td>R.C.DEO,C.M.GROFT,K.R.RAJASHANKAR,S.K.BURLEY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2R7U" title="Link to PDB">2R7U</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2R7U/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2R7U</a></td><td align="center">X</td><td align="center">5</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20100507000000">7-29-08</td><td>SIMIAN ROTAVIRUS</td><td>CRYSTAL STRUCTURE OF ROTAVIRUS SA11 VP1/RNA (AAAAGCC) COMPLEX</td><td>X.LU,S.C.HARRISON,Y.J.TAO,J.T.PATTON,M.L.NIBERT</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3KOA" title="Link to PDB">3KOA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3KOA/index.html" title="Link to FR3D analysis. 0.3333 pairs per nucleotide">3KOA</a></td><td align="center">BC</td><td align="center">9</td><td align="center">3</td><td align="center">2.4</td><td sorttable_customkey="20100707000000">7-07-10</td><td>synthetic,synthetic</td><td>M296I MUTANT OF FOOT-AND-MOUTH DISEASE VIRUS RNA-POLYMERASE IN COMPLEX WITH A TEMPLATE- PRIMER RNA AND GTP</td><td>C.FERRER-ORTA,N.VERDAGUER,R.PEREZ-LUQUE</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1P79" title="Link to PDB">1P79</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1P79/index.html" title="Link to FR3D analysis. 0.4000 pairs per nucleotide">1P79</a></td><td align="center">A</td><td align="center">20</td><td align="center">8</td><td align="center">1.1</td><td sorttable_customkey="20030411000000">11-04-03</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF A BULGED RNA TETRAPLEX: IMPLICATIONS FOR A NOVEL BINDING SITE IN RNA TETRAPLEX</td><td>B.PAN,Y.XIONG,K.SHI,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2KYE" title="Link to PDB">2KYE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2KYE/index.html" title="Link to FR3D analysis. 0.2727 pairs per nucleotide">2KYE</a></td><td align="center">A</td><td align="center">220</td><td align="center">60</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120606000000">6-30-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF THE PSEUDOURIDINE MODIFIED P6.1 HAIRPIN OF HUMAN TELOMERASE RNA</td><td>N.-K.KIM,C.A.THEIMER,J.R.MITCHELL,K.COLLINS,J.FEIGON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2HT1" title="Link to PDB">2HT1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2HT1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2HT1</a></td><td align="center">JKM</td><td align="center">9</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="20070211000000">11-14-06</td><td>synthetic,synthetic</td><td>THE CLOSED RING STRUCTURE OF THE RHO TRANSCRIPTION TERMINATION FACTOR IN COMPLEX WITH NUCLEIC ACID IN THE MOTOR DOMAINS</td><td>E.SKORDALAKES,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3Q2T" title="Link to PDB">3Q2T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3Q2T/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3Q2T</a></td><td align="center">E</td><td align="center">10</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20120402000000">2-16-11</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF CFIM68 RRM/CFIM25/RNA COMPLEX</td><td>Q.YANG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1YTU" title="Link to PDB">1YTU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1YTU/index.html" title="Link to FR3D analysis. 0.4444 pairs per nucleotide">1YTU</a></td><td align="center">CD</td><td align="center">9</td><td align="center">4</td><td align="center">2.5</td><td sorttable_customkey="20050504000000">4-05-05</td><td>synthetic,synthetic</td><td>STRUCTURAL BASIS FOR 5'-END-SPECIFIC RECOGNITION OF THE GUIDE RNA STRAND BY THE A. FULGIDUS PIWI PROTEIN</td><td>J.B.MA,Y.R.YUAN,G.MEISTER,Y.PEI,T.TUSCHL,D.J.PATEL</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L5D" title="Link to PDB">2L5D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L5D/index.html" title="Link to FR3D analysis. 0.0100 pairs per nucleotide">2L5D</a></td><td align="center">B</td><td align="center">100</td><td align="center">1</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120804000000">4-20-11</td><td>HOMO SAPIENS</td><td>SOLUTION STRUCTURES OF HUMAN PIWI-LIKE 1 PAZ DOMAIN WITH SSRNA (5'- PUGACA)</td><td>L.ZENG,Q.ZHANG,K.YAN,M.ZHOU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2AWE" title="Link to PDB">2AWE</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2AWE/index.html" title="Link to FR3D analysis. 0.6154 pairs per nucleotide">2AWE</a></td><td align="center">ABCD</td><td align="center">39</td><td align="center">24</td><td align="center">2.1</td><td sorttable_customkey="20060702000000">2-07-06</td><td>engineered</td><td>BASE-TETRAD SWAPPING RESULTS IN DIMERIZATION OF RNA QUADRUPLEXES: IMPLICATIONS FOR FORMATION OF I-MOTIF RNA OCTAPLEX</td><td>B.PAN,K.SHI,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PO3" title="Link to PDB">3PO3</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PO3/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PO3</a></td><td align="center">P</td><td align="center">5</td><td align="center">0</td><td align="center">3.3</td><td sorttable_customkey="20110203000000">3-02-11</td><td>synthetic</td><td>ARRESTED RNA POLYMERASE II REACTIVATION INTERMEDIATE</td><td>A.C.M.CHEUNG,P.CRAMER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2C06" title="Link to PDB">2C06</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2C06/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2C06</a></td><td align="center">C</td><td align="center">50</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20060802000000">2-08-06</td><td>synthetic</td><td>NMR-BASED MODEL OF THE COMPLEX OF THE TOXIN KID AND A 5- NUCLEOTIDE SUBSTRATE RNA FRAGMENT (AUACA)</td><td>M.B.KAMPHUIS,A.M.J.J.BONVIN,M.C.MONTI,M.LEMONNIER, A.MUNOZ-GOMEZ,R.H.H.VAN DEN HEUVEL,R.DIAZ-OREJAS,R.BOELENS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3M85" title="Link to PDB">3M85</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3M85/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3M85</a></td><td align="center">X</td><td align="center">12</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20120404000000">4-28-10</td><td>synthetic</td><td>ARCHAEOGLOBUS FULGIDUS EXOSOME Y70A WITH RNA BOUND TO THE ACTIVE SITE</td><td>S.HARTUNG,K.-P.HOPFNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2QQP" title="Link to PDB">2QQP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2QQP/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2QQP</a></td><td align="center">R</td><td align="center">4</td><td align="center">0</td><td align="center">3.8</td><td sorttable_customkey="20100801000000">1-20-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AUTHENTIC PROVIDENCE VIRUS</td><td>J.A.SPEIR,D.J.TAYLOR,J.E.JOHNSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2GRB" title="Link to PDB">2GRB</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2GRB/index.html" title="Link to FR3D analysis. 0.9688 pairs per nucleotide">2GRB</a></td><td align="center">EFGH</td><td align="center">32</td><td align="center">31</td><td align="center">1.4</td><td sorttable_customkey="20070911000000">11-21-06</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN RNA QUADRUPLEX CONTAINING INOSINE- TETRAD</td><td>B.PAN,K.SHI,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVN" title="Link to PDB">1UVN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVN/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVN</a></td><td align="center">B</td><td align="center">12</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20060202000000">2-26-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 CA2+ INHIBITION COMPLEX</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1N1H" title="Link to PDB">1N1H</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1N1H/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1N1H</a></td><td align="center">B</td><td align="center">4</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20040112000000">12-25-02</td><td>synthetic</td><td>INITIATION COMPLEX OF POLYMERASE LAMBDA3 FROM REOVIRUS</td><td>Y.TAO,D.L.FARSETTA,M.L.NIBERT,S.C.HARRISON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P4B" title="Link to PDB">3P4B</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P4B/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3P4B</a></td><td align="center">A</td><td align="center">24</td><td align="center">0</td><td align="center">1.4</td><td sorttable_customkey="20110501000000">1-05-11</td><td>synthetic</td><td>ALTERNATINGLY MODIFIED 2'FLUORO RNA OCTAMER F/RA2U2-P3</td><td>P.S.PALLAN,E.M.GREENE,P.A.JICMAN,R.K.PANDEY,M.MANOHARAN,E.ROZNERS, M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1D0T" title="Link to PDB">1D0T</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1D0T/index.html" title="Link to FR3D analysis. 0.3750 pairs per nucleotide">1D0T</a></td><td align="center">A</td><td align="center">160</td><td align="center">60</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20011205000000">5-24-00</td><td>synthetic</td><td>SOLUTION STRUCTURE OF A PHOSPHOROTHIOATE MODIFIED RNA BINDING SITE FOR PHAGE MS2 COAT PROTEIN</td><td>J.S.SMITH,E.P.NIKONOWICZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3IE1" title="Link to PDB">3IE1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3IE1/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3IE1</a></td><td align="center">EFGH</td><td align="center">14</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20090408000000">8-04-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF H380A MUTANT TTHA0252 FROM THERMUS THERMOPHILUS HB8 COMPLEXED WITH RNA</td><td>H.ISHIKAWA,N.NAKAGAWA,S.KURAMITSU,S.YOKOYAMA,R.MASUI,RIKEN STRUCTURAL GENOMICS/PROTEOMICS INITIATIVE (RSGI)</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1CWP" title="Link to PDB">1CWP</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CWP/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1CWP</a></td><td align="center">DEF</td><td align="center">10</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="19961005000000">5-22-95</td><td>COWPEA CHLOROTIC MOTTLE VIRUS,synthetic</td><td>STRUCTURES OF THE NATIVE AND SWOLLEN FORMS OF COWPEA CHLOROTIC MOTTLE VIRUS DETERMINED BY X-RAY CRYSTALLOGRAPHY AND CRYO-ELECTRON MICROSCOPY</td><td>J.A.SPEIR,J.E.JOHNSON,S.MUNSHI,G.WANG,S.TIMOTHY,T.S.BAKER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1ME1" title="Link to PDB">1ME1</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1ME1/index.html" title="Link to FR3D analysis. 0.5000 pairs per nucleotide">1ME1</a></td><td align="center">A</td><td align="center">80</td><td align="center">40</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040708000000">8-19-03</td><td>synthetic</td><td>CHIMERIC HAIRPIN WITH 2',5'-LINKED RNA LOOP AND RNA STEM</td><td>A.Y.DENISOV,R.N.HANNOUSH,K.GEHRING,M.J.DAMHA</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MV6" title="Link to PDB">1MV6</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MV6/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1MV6</a></td><td align="center">A</td><td align="center">42</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20030612000000">12-18-02</td><td>synthetic</td><td>THE TANDEM, SHEARED PP PAIRS IN 5'(RGGCPPGCCU)2</td><td>B.M.ZNOSKO,M.E.BURKARD,T.R.KRUGH,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1MDG" title="Link to PDB">1MDG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1MDG/index.html" title="Link to FR3D analysis. 0.8000 pairs per nucleotide">1MDG</a></td><td align="center">A</td><td align="center">20</td><td align="center">16</td><td align="center">1.5</td><td sorttable_customkey="20031208000000">8-12-03</td><td>synthetic</td><td>AN ALTERNATING ANTIPARALLEL OCTAPLEX IN AN RNA CRYSTAL STRUCTURE</td><td>B.C.PAN,Y.XIONG,K.SHI,M.SUNDARALINGAM</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVI" title="Link to PDB">1UVI</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVI/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVI</a></td><td align="center">D</td><td align="center">12</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20050702000000">2-19-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 WITH 6NT RNA</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2I2Y" title="Link to PDB">2I2Y</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2I2Y/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2I2Y</a></td><td align="center">B</td><td align="center">112</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20061212000000">12-12-06</td><td>STREPTOCOCCUS SP. 'GROUP G', HOMO</td><td>SOLUTION STRUCTURE OF THE RRM OF SRP20 BOUND TO THE RNA CAUC</td><td>Y.F.HARGOUS,F.H.ALLAIN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2L41" title="Link to PDB">2L41</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2L41/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2L41</a></td><td align="center">B</td><td align="center">80</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20110511000000">11-17-10</td><td>synthetic</td><td>NAB3 RRM - UCUU COMPLEX</td><td>R.STEFL,R.PERGOLI,F.HOBOR,K.KUBICEK,M.ZIMMERMANN,J.PASULKA,C.HOFR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1NB7" title="Link to PDB">1NB7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1NB7/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1NB7</a></td><td align="center">EF</td><td align="center">8</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20050103000000">3-25-03</td><td>synthetic</td><td>HC-J4 RNA POLYMERASE COMPLEXED WITH SHORT RNA TEMPLATE STRAND</td><td>D.J.O'FARRELL,R.TROWBRIDGE,D.J.ROWLANDS,J.JAEGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XC7" title="Link to PDB">2XC7</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XC7/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XC7</a></td><td align="center">P</td><td align="center">40</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20120404000000">4-28-10</td><td>synthetic</td><td>SOLUTION STRUCTURE OF PHAX-RBD IN COMPLEX WITH SSRNA</td><td>A.MOURAO,C.D.MACKERETH,A.VARROT,S.CUSACK,M.SATTLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P4C" title="Link to PDB">3P4C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P4C/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3P4C</a></td><td align="center">A</td><td align="center">8</td><td align="center">0</td><td align="center">1.1</td><td sorttable_customkey="20110501000000">1-05-11</td><td>synthetic</td><td>ALTERNATINGLY MODIFIED 2'FLUORO RNA OCTAMER F/RA2U2-R32</td><td>P.S.PALLAN,E.M.GREENE,P.A.JICMAN,R.K.PANDEY,M.MANOHARAN,E.ROZNERS, M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVJ" title="Link to PDB">1UVJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVJ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVJ</a></td><td align="center">D</td><td align="center">12</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20050702000000">2-19-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 WITH 7NT RNA</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1G3A" title="Link to PDB">1G3A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1G3A/index.html" title="Link to FR3D analysis. 0.2500 pairs per nucleotide">1G3A</a></td><td align="center">AB</td><td align="center">8</td><td align="center">2</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20010108000000">8-01-01</td><td>synthetic</td><td>STRUCTURE OF RNA DUPLEXES (CIGCGICG)2</td><td>X.CHEN,R.KIERZEK,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3P4D" title="Link to PDB">3P4D</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3P4D/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3P4D</a></td><td align="center">A</td><td align="center">8</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20110501000000">1-05-11</td><td>synthetic</td><td>ALTERNATINGLY MODIFIED 2'FLUORO RNA OCTAMER F/RC4G4</td><td>P.S.PALLAN,E.M.GREENE,P.A.JICMAN,R.K.PANDEY,M.MANOHARAN,E.ROZNERS, M.EGLI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VRT" title="Link to PDB">2VRT</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VRT/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VRT</a></td><td align="center">EFGH</td><td align="center">10</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="20091007000000">7-22-08</td><td>ESCHERICHIA COLI,ESCHERICHIA COLI</td><td>CRYSTAL STRUCTURE OF E. COLI RNASE E POSSESSING M1 RNA FRAGMENTS - CATALYTIC DOMAIN</td><td>D.J.KOSLOVER,A.J.CALLAGHAN,M.J.MARCAIDA,E.F.GARMAN, M.MARTICK,W.G.SCOTT,B.F.LUISI</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2TMV" title="Link to PDB">2TMV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TMV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2TMV</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="19890901000000">1-09-89</td><td>synthetic</td><td>VISUALIZATION OF PROTEIN-NUCLEIC ACID INTERACTIONS IN A VIRUS. REFINED STRUCTURE OF INTACT TOBACCO MOSAIC VIRUS AT 2.9 ANGSTROMS RESOLUTION BY X-RAY FIBER DIFFRACTION</td><td>G.STUBBS,R.PATTANAYEK,K.NAMBA</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1CGM/index.html" title="Resolution 3.4A, 3 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1CGM</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2OM3/index.html" title="Resolution 4.4A, 3 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2OM3</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XEA/index.html" title="Resolution 4.6A, 3 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2XEA</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2TMV/2TMV_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UTF" title="Link to PDB">1UTF</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UTF/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UTF</a></td><td align="center">0</td><td align="center">25</td><td align="center">0</td><td align="center">1.9</td><td sorttable_customkey="20050301000000">1-15-04</td><td>synthetic,BACILLUS STEAROTHERMOPHILUS</td><td>THE STRUCTURE OF THE TRP RNA-BINDING ATTENUATION PROTEIN (TRAP) BOUND TO A RNA MOLECULE CONTAINING UAGAU REPEATS (PART I)</td><td>N.H.HOPCROFT,A.MANFREDO,A.L.WENDT,A.M.BRZOZOWSKI,P.GOLLNICK, A.A.ANTSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UTV/index.html" title="Resolution 1.9A, 24 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1UTV</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UTF/1UTF_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2FZ2" title="Link to PDB">2FZ2</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FZ2/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2FZ2</a></td><td align="center">D</td><td align="center">3</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="20070902000000">2-21-06</td><td>TURNIP YELLOW MOSAIC VIRUS</td><td>STRUCTURE OF TURNIP YELLOW MOSAIC VIRUS AT 100 K</td><td>S.B.LARSON,R.W.LUCAS,A.MCPHERSON</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XPJ/index.html" title="Resolution 3.4A, 3 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">2XPJ</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2FZ2/2FZ2_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1RMV" title="Link to PDB">1RMV</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1RMV/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1RMV</a></td><td align="center">B</td><td align="center">3</td><td align="center">0</td><td align="center">2.9</td><td sorttable_customkey="19980305000000">5-15-97</td><td>RIBGRASS MOSAIC VIRUS</td><td>RIBGRASS MOSAIC VIRUS, FIBER DIFFRACTION</td><td>H.WANG,G.STUBBS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2A1R" title="Link to PDB">2A1R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2A1R/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2A1R</a></td><td align="center">C</td><td align="center">6</td><td align="center">0</td><td align="center">2.6</td><td sorttable_customkey="20060812000000">12-20-05</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF PARN NUCLEASE DOMAIN</td><td>M.WU,H.SONG</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O83" title="Link to PDB">2O83</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O83/index.html" title="Link to FR3D analysis. 0.4286 pairs per nucleotide">2O83</a></td><td align="center">AB</td><td align="center">252</td><td align="center">108</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071007000000">7-10-07</td><td>synthetic</td><td>THE STRUCTURE OF TANDEM GA RNA PAIRS WHEN FLANKING PAIRS ARE ISOG-ISOC PAIRS</td><td>S.D.KENNEDY,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2XNR" title="Link to PDB">2XNR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2XNR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2XNR</a></td><td align="center">C</td><td align="center">3</td><td align="center">0</td><td align="center">1.6</td><td sorttable_customkey="20100809000000">9-08-10</td><td>SACCHAROMYCES CEREVISIAE</td><td>STRUCTURAL INSIGHTS INTO CIS ELEMENT RECOGNITION OF NON- POLYADENYLATED RNAS BY THE NAB3-RRM</td><td>B.M.LUNDE,M.HORNER,A.MEINHART</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1H2C" title="Link to PDB">1H2C</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H2C/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1H2C</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">1.6</td><td sorttable_customkey="20031004000000">4-10-03</td><td>ESCHERICHIA COLI</td><td>EBOLA VIRUS MATRIX PROTEIN VP40 N-TERMINAL DOMAIN IN COMPLEX WITH RNA (HIGH-RESOLUTION VP40[55-194] VARIANT).</td><td>F.X.GOMIS-RUTH,A.DESSEN,A.BRACHER,H.D.KLENK,W.WEISSENHORN</td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H2D/index.html" title="Resolution 2.6A, 6 nucleotides, 0 basepairs, 0.0000 pairs per nucleotide">1H2D</a> <a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1H2C/1H2C_compare_circular.html">Compare</a></td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1I5L" title="Link to PDB">1I5L</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1I5L/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1I5L</a></td><td align="center">U</td><td align="center">6</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20030408000000">8-28-01</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF AN SM-LIKE PROTEIN (AF-SM1) FROM ARCHAEOGLOBUS FULGIDUS COMPLEXED WITH SHORT POLY-U RNA</td><td>I.TORO,S.THORE,C.MAYER,J.BASQUIN,B.SERAPHIN,D.SUCK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2VOO" title="Link to PDB">2VOO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2VOO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2VOO</a></td><td align="center">C</td><td align="center">6</td><td align="center">0</td><td align="center">1.8</td><td sorttable_customkey="20080605000000">5-06-08</td><td>HOMO SAPIENS</td><td>CRYSTAL STRUCTURE OF N-TERMINAL DOMAINS OF HUMAN LA PROTEIN COMPLEXED WITH RNA OLIGOMER UUUUUUUU</td><td>O.KOTIK-KOGAN,E.R.VALENTINE,D.SANFELICE,M.R.CONTE,S.CURRY</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2O81" title="Link to PDB">2O81</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2O81/index.html" title="Link to FR3D analysis. 0.3254 pairs per nucleotide">2O81</a></td><td align="center">AB</td><td align="center">252</td><td align="center">82</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20071007000000">7-10-07</td><td>synthetic</td><td>THE STRUCTURE OF TANDEM GA RNA PAIRS WHEN FLANKING PAIRS ARE ISOG-ISOC PAIRS</td><td>G.CHEN,D.H.TURNER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3PDM" title="Link to PDB">3PDM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3PDM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3PDM</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="20111201000000">1-12-11</td><td>synthetic</td><td>HIBISCUS LATENT SINGAPORE</td><td>S.K.TEWARY,S.M.WONG,K.SWAMINATHAN</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3NMA" title="Link to PDB">3NMA</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3NMA/index.html" title="Link to FR3D analysis. 0.2000 pairs per nucleotide">3NMA</a></td><td align="center">BC</td><td align="center">5</td><td align="center">1</td><td align="center">2.6</td><td sorttable_customkey="20130105000000">5-25-11</td><td>synthetic,synthetic</td><td>MUTANT P169S OF FOOT-AND-MOUTH DISEASE VIRUS RNA DEPENDENT RNA- POLYMERASE</td><td>R.AGUDO,C.FERRER-ORTA,A.ARIAS,R.PEREZ-LUQUE,N.VERDAGUER,E.DOMINGO</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1VTM" title="Link to PDB">1VTM</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1VTM/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1VTM</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">3.5</td><td sorttable_customkey="19960707000000">7-31-94</td><td>synthetic</td><td>STRUCTURE OF THE U2 STRAIN OF TOBACCO MOSAIC VIRUS REFINED AT 3.5 ANGSTROMS RESOLUTION USING X-RAY FIBER DIFFRACTION</td><td>G.STUBBS,R.PATTANAYEK</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1LAJ" title="Link to PDB">1LAJ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1LAJ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1LAJ</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">3.4</td><td sorttable_customkey="20040311000000">11-27-02</td><td>TOMATO ASPERMY VIRUS</td><td>THE STRUCTURE OF TOMATO ASPERMY VIRUS BY X-RAY CRYSTALLOGRAPHY</td><td>R.W.LUCAS,S.B.LARSON,M.A.CANADY,A.MCPHERSON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3J06" title="Link to PDB">3J06</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3J06/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3J06</a></td><td align="center">R</td><td align="center">3</td><td align="center">0</td><td align="center">2.8</td><td sorttable_customkey="20110106000000">6-01-11</td><td>TOBACCO MOSAIC VIRUS</td><td>CRYOEM HELICAL RECONSTRUCTION OF TMV</td><td>P.GE,Z.H.ZHOU</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1APG" title="Link to PDB">1APG</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1APG/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1APG</a></td><td align="center">D</td><td align="center">2</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="19960701000000">1-31-94</td><td>synthetic</td><td>X-RAY ANALYSIS OF SUBSTRATE ANALOGS IN THE RICIN A-CHAIN ACTIVE SITE</td><td>A.F.MONZINGO,J.D.ROBERTUS</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1PVO" title="Link to PDB">1PVO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1PVO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1PVO</a></td><td align="center">GHJKL</td><td align="center">10</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20041007000000">7-22-03</td><td>synthetic</td><td>X-RAY CRYSTAL STRUCTURE OF RHO TRANSCRIPTION TERMINATION FACTOR IN COMPLEX WITH SSRNA SUBSTRATE AND ANPPNP</td><td>E.SKORDALAKES,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XPR" title="Link to PDB">1XPR</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPR/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1XPR</a></td><td align="center">GHJKLM</td><td align="center">12</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20040211000000">11-02-04</td><td>synthetic</td><td>STRUCTURAL MECHANISM OF INHIBITION OF THE RHO TRANSCRIPTION TERMINATION FACTOR BY THE ANTIBIOTIC 5A-FORMYLBICYCLOMYCIN (FB)</td><td>E.SKORDALAKES,A.P.BROGAN,B.S.PARK,H.KOHN,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XPU" title="Link to PDB">1XPU</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPU/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1XPU</a></td><td align="center">GHJKLM</td><td align="center">12</td><td align="center">0</td><td align="center">3.0</td><td sorttable_customkey="20040211000000">11-02-04</td><td>synthetic</td><td>STRUCTURAL MECHANISM OF INHIBITION OF THE RHO TRANSCRIPTION TERMINATION FACTOR BY THE ANTIBIOTIC 5A-(3- FORMYLPHENYLSULFANYL)-DIHYDROBICYCLOMYCIN (FPDB)</td><td>E.SKORDALAKES,A.P.BROGAN,B.S.PARK,H.KOHN,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2X1F" title="Link to PDB">2X1F</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X1F/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2X1F</a></td><td align="center">B</td><td align="center">2</td><td align="center">0</td><td align="center">1.6</td><td sorttable_customkey="20100202000000">2-02-10</td><td>synthetic</td><td>STRUCTURE OF RNA15 RRM WITH BOUND RNA (GU)</td><td>C.PANCEVAC,D.C.GOLDSTONE,A.RAMOS,I.A.TAYLOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1XPO" title="Link to PDB">1XPO</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1XPO/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1XPO</a></td><td align="center">GHJKLM</td><td align="center">12</td><td align="center">0</td><td align="center">3.1</td><td sorttable_customkey="20050802000000">2-08-05</td><td>synthetic</td><td>STRUCTURAL MECHANISM OF INHIBITION OF THE RHO TRANSCRIPTION TERMINATION FACTOR BY THE ANTIBIOTIC BICYCLOMYCIN</td><td>E.SKORDALAKES,A.P.BROGAN,B.S.PARK,H.KOHN,J.M.BERGER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ERC" title="Link to PDB">3ERC</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ERC/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3ERC</a></td><td align="center">EFGI</td><td align="center">8</td><td align="center">0</td><td align="center">3.2</td><td sorttable_customkey="20100406000000">6-16-09</td><td>synthetic,synthetic,synthetic</td><td>CRYSTAL STRUCTURE OF THE HETERODIMERIC VACCINIA VIRUS MRNA POLYADENYLATE POLYMERASE WITH THREE FRAGMENTS OF RNA AND 3'-DEOXY ATP</td><td>C.LI,H LI,S.ZHOU,T.L.POULOS,P.D.GERSHON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3ER9" title="Link to PDB">3ER9</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3ER9/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3ER9</a></td><td align="center">D</td><td align="center">2</td><td align="center">0</td><td align="center">2.1</td><td sorttable_customkey="20100406000000">6-16-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF THE HETERODIMERIC VACCINIA VIRUS MRNA POLYADENYLATE POLYMERASE COMPLEX WITH UU AND 3'-DEOXY ATP</td><td>C.LI,H LI,S.ZHOU,T.L.POULOS,P.D.GERSHON</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=3GPQ" title="Link to PDB">3GPQ</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/3GPQ/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">3GPQ</a></td><td align="center">EF</td><td align="center">3</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20100907000000">7-21-09</td><td>synthetic</td><td>CRYSTAL STRUCTURE OF MACRO DOMAIN OF CHIKUNGUNYA VIRUS IN COMPLEX WITH RNA</td><td>H.MALET,S.JAMAL,B.COUTARD,B.CANARD</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1T2R" title="Link to PDB">1T2R</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1T2R/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1T2R</a></td><td align="center">B</td><td align="center">20</td><td align="center">0</td><td align="center">SOLUTION NMR</td><td sorttable_customkey="20040106000000">6-01-04</td><td>DROSOPHILA MELANOGASTER</td><td>STRUCTURAL BASIS FOR 3' END RECOGNITION OF NUCLEIC ACIDS BY THE DROSOPHILA ARGONAUTE 2 PAZ DOMAIN</td><td>A.LINGEL,B.SIMON,E.IZAURRALDE,M.SATTLER</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1QLN" title="Link to PDB">1QLN</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1QLN/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1QLN</a></td><td align="center">R</td><td align="center">2</td><td align="center">0</td><td align="center">2.4</td><td sorttable_customkey="20000402000000">2-04-00</td><td>BACTERIOPHAGE T7</td><td>STRUCTURE OF A TRANSCRIBING T7 RNA POLYMERASE INITIATION COMPLEX</td><td>G.M.T.CHEETHAM,T.A.STEITZ</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=2X1A" title="Link to PDB">2X1A</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/2X1A/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">2X1A</a></td><td align="center">B</td><td align="center">1</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20100202000000">2-02-10</td><td>synthetic</td><td>STRUCTURE OF RNA15 RRM WITH RNA BOUND (G)</td><td>C.PANCEVAC,D.C.GOLDSTONE,A.RAMOS,I.A.TAYLOR</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1B2M" title="Link to PDB">1B2M</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1B2M/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1B2M</a></td><td align="center">CDE</td><td align="center">3</td><td align="center">0</td><td align="center">2.0</td><td sorttable_customkey="20010103000000">3-25-99</td><td>synthetic</td><td>THREE-DIMENSIONAL STRUCTURE OF RIBONULCEASE T1 COMPLEXED WITH AN ISOSTERIC PHOSPHONATE ANALOGUE OF GPU: ALTERNATE SUBSTRATE BINDING MODES AND CATALYSIS.</td><td>R.K.ARNI,L.WATANABE,R.J.WARD,R.J.KREITMAN,K.KUMAR,F.G.WALZ JR.</td><td>&nbsp;</td></tr>
<tr><td><a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1UVK" title="Link to PDB">1UVK</a></td><td><a href="http://rna.bgsu.edu/FR3D/AnalyzedStructures/1UVK/index.html" title="Link to FR3D analysis. 0.0000 pairs per nucleotide">1UVK</a></td><td align="center">BDF</td><td align="center">3</td><td align="center">0</td><td align="center">2.5</td><td sorttable_customkey="20050702000000">2-19-04</td><td>synthetic</td><td>THE STRUCTURAL BASIS FOR RNA SPECIFICITY AND CA2 INHIBITION OF AN RNA-DEPENDENT RNA POLYMERASE PHI6P2 DEAD-END COMPLEX</td><td>P.S.SALGADO,E.V.MAKEYEV,S.BUTCHER,D.BAMFORD,D.I.STUART, J.M.GRIMES</td><td>&nbsp;</td></tr>
</table>
</html>